---

title: Endoplasmic reticulum localization signals
abstract: The invention relates to cellular localization signals. In particular, the invention relates to endoplasmic reticulum localization signals in monomeric or multimeric form. The localization signals are utilized as research tools or are linked to therapeutics. Disclosed are methods of making and using polypeptides and modified polypeptides as signals to localize therapeutics, experimental compounds, peptides, proteins and/or other macromolecules to the endoplasmic reticulum of eukaryotic cells. The polypeptides of the invention optionally include linkage to reporters, epitopes and/or other experimental or therapeutic molecules. The invention also encompasses polynucleotides encoding the localization signals and vectors comprising these polynucleotides.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08703905&OS=08703905&RS=08703905
owner: Intrexon Corporation
number: 08703905
owner_city: Blacksburg
owner_country: US
publication_date: 20120209
---
This application claims benefit of priority to provisional application 60 826 517 filed 21 Sep. 2006.

This application includes a Sequence Listing.txt 143 710 bytes created on Nov. 12 2013 and submitted electronically via EFS Web which is hereby incorporated by reference in its entirety.

The invention relates to subcellular localization signals. In particular the invention relates to endoplasmic reticulum localization signals in monomeric or multimeric form. The multimers may be homomultimers or heteromultimers. The monomers and multimers are utilized as research tools or are linked to therapeutics.

This application has subject matter related to application Ser. Nos. 10 724 532 U.S. Pat. No. 7 071 295 10 682 764 US2004 0185556 PCT US2004 013517 WO2005 040336 11 233 246 and US2004057201 IP WO2005116231 . Each of these patents and applications is hereby incorporated by reference.

Drugs that act intracellularly generally enter cells by diffusion. Most drugs are small molecules because they have the ability to diffuse across plasma membranes or organelle membranes to reach their site of action. To increase the bioavailability of a drug often small molecules must be modified and or formulated for greater solubility and or permeability depending on route of administration. Even small diffusible drugs may not be efficacious at their site of action. For example multidrug resistance MDR may be present which results in active efflux of drugs that enter cells with MDR. MDR often occurs in cancer cells.

In contrast to small molecules high molecular weight compounds and polymer drugs such as polynucleotides polypeptides and other macromolecules have little to no ability to diffuse across membranes. High molecular weight material is generally internalized by endocytosis. The addition of affinity binding partners to high molecular weight material can direct the high molecular weight compound to specific cells and thereby result in increased selective uptake. However once endocytosed the material still remains separated from the cellular cytoplasm by a biological membrane.

Endocytosed material is often delivered to the lysosome where material sensitive to lysosomal enzymes is quickly degraded if steps are not taken to protect its breakdown or to facilitate escape from the lysosome. Delivery of high molecular weight compounds to their site of action at effective levels is a problem. It is therefore desirable to improve delivery to a desired subcellular compartment.

One of the first cellular trafficking signals identified was the endoplasmic reticulum ER retention signal KDEL SEQ ID NO 75 which prevents secretion of proteins routed to the endoplasmic reticulum. When this signal is expressed toward the carboxy terminus in proteins that are normally secreted these proteins are retained in the endoplasmic reticulum and not secreted Munro and Pelham Cell 1987 48 899 907 .

Endogenous and exogenous proteins have varying targeting domains within their primary sequence. Such proteins include those described in Andersson et al. 1999 J Biol Chem 274 15080 4 Cocquerel et al. 1999 J Virol 73 2641 9 Fons et al. 2003 J Cell Biol 160 529 39 Gabathuler et al. 1990 J Cell Biol 111 1803 10 Honsho et al. 1998 J Biol Chem 273 20860 6 Ma et al. 2002 J Biol Chem 277 27328 36 Mitoma et al. 1992 Embo J 11 4197 203 Mziaut et al. 1999 J Biol Chem 274 14122 9 Parker et al. 2004 J Biol Chem 279 23797 805 Pottekat et al. 2004 J Biol Chem 279 15743 51 Ren et al. 2003 J Biol Chem 278 52700 9 Szczesna Skorupa et al. 2001 J Biol Chem 276 45009 14 Vainauskas et al. 2005 J Biol Chem 280 16402 9 Watanabe et al. 1996 J Biol Chem 271 26868 75 Zarei et al. 2004 Proc Natl Acad Sci USA 101 10072 7 and Zarei et al. 2001 J Biol Chem 276 16232 9.

An aspect of the invention is to provide novel monomeric and novel multimeric endoplasmic reticulum localization signals by modifying one or more proteins that naturally locate to the endoplasmic reticulum by truncation or by amino acid substitution. Truncations amino acid substitutions and other modifications of known ER locating proteins are made to minimize endogenous biological activities other than localization. In general the invention relates to cellular localization signals. More specifically the invention relates to endoplasmic reticulum localization signals in monomeric or multimeric form. The multimers may be homomultimers or heteromultimers. Multimers are made to exploit cooperation and synergism among individual signals in order to create a chimeric localization signal with a strength and or performance greater than the constituent individual parts. The monomers and multimers are utilized as research tools or are linked to therapeutics. Disclosed are methods of making and using polypeptides and modified polypeptides as signals to localize therapeutics experimental compounds peptides proteins and or other macromolecules to the endoplasmic reticulum and contiguous structures of eukaryotic cells. The polypeptides of the invention optionally include linkage to reporters epitopes and or other experimental or therapeutic molecules. The invention also encompasses polynucleotides encoding the localization signals and vectors comprising these polynucleotides.

DETAILED DESCRIPTION OF POLYPEPTIDE AND POLYNUCLEOTIDE SEQUENCES SEQ ID NOS 1 16 are example endoplasmic reticulum localization signals and polynucleotides encoding them.

Specifically the polypeptide of SEQ ID NO 1 is encoded by SEQ ID NOS 2 6 wherein the codons of SEQ ID NOS 3 6 have been optimized for vector insertion. SEQ ID NO 4 and SEQ ID NO 6 include flanking restriction sites. SEQ ID NO 5 and SEQ ID NO 6 differ from SEQ ID NO 3 and SEQ ID NO 4 respectively in that an internal EcoRI restriction has been removed. SEQ ID NO 1 is an embodiment of a multimeric ER localization signal of the structure A S1 B 52 B 53 C wherein A is SEQ ID NO 42 B is SEQ ID NO 72 and C is SEQ ID NO 75 and wherein 51 is a two amino acid spacer with the sequence EF S2 is a four amino acid spacer with the sequence PGAG SEQ ID NO 78 and S3 is a three amino acid spacer with the sequence AAA. A multimeric localization signal of structure A S1 B 52 B 53 C is also called herein a heteromultimer see .

SEQ ID NO 7 is an embodiment of a multimer of the structure X S1 Y S2 Y S3 wherein X is SEQ ID NO 60 Y is SEQ ID NO 72 51 is a seven amino acid spacer with the sequence EFGGGGG SEQ ID NO 79 S2 is a four amino acid spacer with the sequence PGAG SEQ ID NO 78 and S3 is a five amino acid spacer with the sequence AAPAA SEQ ID NO 80 . The polypeptide of SEQ ID NO 7 is encoded by SEQ ID NOS 8 12 wherein the codons of SEQ ID NOS 9 12 have been optimized for vector insertion. SEQ ID NO 10 and SEQ ID NO 12 include flanking restriction sites. SEQ ID NO 9 and SEQ ID NO 10 differ from SEQ ID NO 11 and SEQ ID NO 12 respectively in that an internal EcoRI restriction has been removed. A multimer of structure X S1 Y S2 Y S3 is also called herein a heteromultimer see . A vector map of a vector containing SEQ ID NO 7 is shown in labeled Localization Signal . SEQ ID NO 7 was expressed in Cos7 cells as shown in .

SEQ ID NO 13 is an embodiment of a multimer of the structure X S1 Y S2 Y wherein X is SEQ ID NO 60 Y is SEQ ID NO 72 S1 is a seven amino acid spacer with the sequence EFGGGGG SEQ ID NO 79 and S2 is a four amino acid spacer with the sequence PGAG SEQ ID NO 78 . The polypeptide of SEQ ID NO 13 is encoded by SEQ ID NO 14 SEQ ID NO 15 and by SEQ ID NO 16 wherein the codons of SEQ ID NO 15 and SEQ ID NO 16 have been optimized for vector insertion. SEQ ID NO 16 includes flanking restriction sites. A multimer of structure X S1 Y S2 Y is also called herein a heteromultimer see .

SEQ ID NOS 17 38 are full length sequences of proteins that localize to the endoplasmic reticulum. These sequences have the following public database accession numbers NP001007236 Q9Y2B2 CAA77776 AAQ19305 AAF81759 P00180 Q969N2 NP071581 NP003479 CAI20063 Q7M370 CAA23446 AAS89356 BAA19247 B34759 AAB97308 AAP35497 NP999425 NP999113 XP343784.

SEQ ID NOS 39 69 represent examples of monomeric endoplasmic reticulum localization signals. SEQ ID NOS 39 69 are subsequences of SEQ ID NOS 17 38 which represent examples of peptide sequences that confer endoplasmic reticulum routing and or retention.

The invention relates to monomeric or multimeric endoplasmic reticulum localization signals. Various embodiments of the endoplasmic reticulum localization signals are represented in SEQ ID NOS 1 77. More specifically the invention relates to monomeric or multimeric localization signals that comprise any one or more of SEQ ID NOS 39 77. Additionally the invention relates to monomeric or multimeric polypeptide localization signals comprising one or more subsequences of SEQ ID NOS 17 38 or any portion thereof. Furthermore the invention relates to monomeric or multimeric polypeptide localization signals with at least about 80 85 90 95 96 97 98 and 99 sequence identity to a polypeptide comprising one or more of SEQ ID NOS 39 77 or any portion thereof. Furthermore the invention relates to monomeric or multimeric polypeptide localization signals with at least about 80 85 90 95 96 97 98 and 99 sequence identity to a polypeptide comprising one or more subsequences of SEQ ID NOS 17 38.

Multimeric endoplasmic reticulum localization signals which can be homomultimers or heteromultimers are chimeric polypeptides composed of two or more monomers. An example of a monomeric localization signal is the polypeptide represented by SEQ ID NO 39. SEQ ID NO 39 is a selected subsequence of wild type full length SEQ ID NO 17. An example of a homomultimer is a polypeptide comprising a dimer or multimer of SEQ ID NO 39. An example of a heteromultimer is a polypeptide comprising SEQ ID NO 39 and one or more of SEQ ID NOS 40 77. There are numerous ways to combine SEQ ID NOS 39 77 into homomultimeric or heteromultimeric localization signals. Furthermore there are numerous ways to combine additional subsequences of SEQ ID NOS 17 38 with each other and with SEQ ID NOS 39 77 to make multimeric localization signals.

The localization signals of the invention optionally comprise spacer amino acids before after or between monomers. SEQ ID NO 13 is an example of a heteromultimer with the structure X S1 Y S2 Y where X and Y are selected from SEQ ID NOS 39 77 and S1 and S2 are amino acid spacers. This invention intends to capture all combinations of homomultimers and heteromultimers without limitation to the examples given above or below. In this description use of the term localization signal encompasses monomeric homomultimeric and or heteromultimeric polypeptide localization signals.

A monomeric ER localization signal is a polypeptide where at least a portion of the polypeptide is capable of functioning as an endoplasmic reticulum ER routing signal and or as an endoplasmic reticulum retention signal. An ER routing signal functions to direct a polypeptide to the ER while a retention signal functions to retain the polypeptide in the ER or to prevent secretion of ER localized polypeptides.

A homomultimeric localization signal is a multimer where each of the monomers is identical in amino acid sequence.

A heteromultimeric localization signal is a multimer where some of the monomers are not identical in amino acid sequence.

One embodiment of the invention is a monomeric localization signal containing a polypeptide at least 80 identical to one of SEQ ID NOS 39 69.

Another embodiment of the invention is a heteromultimeric localization signal containing polypeptides at least 80 identical to two or more of SEQ ID NOS 39 69.

Another embodiment of the invention is a heteromultimeric localization signal containing two or more of SEQ ID NOS 70 77.

Another embodiment of the invention is a heteromultimeric localization signal containing polypeptides at least 80 identical to two or more of SEQ ID NOS 39 77.

Another embodiment of the invention is a heteromultimeric localization signal containing a polypeptide at least 80 identical to one or more of SEQ ID NOS 39 69 adjacent to one or more of SEQ ID NOS 70 77.

Another embodiment of the invention is a heteromultimeric localization signal containing a polypeptide at least 80 identical to one or more subsequences of SEQ ID NOS 17 38 adjacent to one or more of SEQ ID NOS 70 77.

Another embodiment of the invention is a heteromultimeric localization signal containing polypeptides at least 80 identical to two or more subsequences of SEQ ID NOS 17 38.

The localization signals of the invention are optionally linked to additional molecules or amino acids that provide an epitope a reporter and or an experimental or therapeutic molecule. The epitope and or reporter and or experimental molecule and or therapeutic molecule may be the same molecule. The epitope and or reporter and or experimental molecule and or therapeutic molecule may also be different molecules. Experimental or therapeutic molecules include but are not limited to proteins and polypeptides. In one embodiment a localization signal for tethering a protein or macromolecule of interest to the cyoplasmic face of the ER is made where the localization signal is placed toward the C terminus of the resultant fusion protein C E H . In another embodiment a localization signal for tethering a protein or macromolecule of interest to the cyoplasmic face of the ER is made where the localization signal is placed toward the N terminus of the resultant fusion protein D F G .

The invention also encompasses polynucleotides comprising nucleotide sequences encoding endoplasmic reticulum localization signals. The nucleic acids of the invention are optionally linked to additional nucleotide sequences encoding polypeptides with additional features such as an epitope a reporter an experimental and or therapeutic molecule. The polynucleotides are optionally flanked by nucleotide sequences comprising restriction endonuclease sites and other nucleotides needed for restriction endonuclease activity. The flanking sequences optionally provide unique cloning sites within a vector and optionally provide directionality of subsequence cloning. Further the nucleic acids of the invention are optionally incorporated into vector polynucleotides. The localization signals of this invention have utility in compositions for research tools and or therapeutics.

The present invention relates to endoplasmic reticulum localization signals. Various embodiments of the localization signals are represented by SEQ ID NOS 1 77. Multimeric localization signals are chimeric polypeptides comprising two or more monomeric localization signals. An example of a monomeric localization signal is the polypeptide represented by SEQ ID NO 39. SEQ ID NO 39 is a selected subsequence of wild type full length SEQ ID NO 17. Another example of a monomeric localization signal is the polypeptide represented by SEQ ID NO 68. Each of SEQ ID NOS 39 77 represents an individual localization signal in monomeric form. SEQ ID NOS 39 69 are selected examples of subsequences of SEQ ID NOS 17 38 however other subsequences of SEQ ID NOS 17 38 may also be utilized as monomeric localization signals. Monomeric subsequences of SEQ ID NOS 17 38 may be wild type subsequences. Additionally monomeric subsequences of SEQ ID NOS 17 38 may have some amino acids different than the wild type parent. Furthermore monomeric localization signals may have 80 85 90 95 96 97 98 or 99 sequence identity to a polypeptide comprising one or more of SEQ ID NOS 39 77. Furthermore monomeric localization signals may have 80 85 90 95 96 97 98 and 99 sequence identity to a subsequence of SEQ ID NOS 17 38.

An example of a homomultimeric localization signal is a polypeptide comprising a dimer or multimer of SEQ ID NO 49. An example of a heteromultimeric localization signal is a polypeptide comprising SEQ ID NO 39 and one or more of SEQ ID NOS 40 77. There are numerous ways to combine SEQ ID NOS 39 77 into homomultimeric or heteromultimeric localization signals. Furthermore there are numerous ways to combine additional subsequences of SEQ ID NOS 17 38 with each other and with SEQ ID NOS 39 77 to make multimeric localization signals.

Multimeric localization signals may comprise any two or more of SEQ ID NOS 39 77. A dimer or multimer of SEQ ID NO 66 is an example of a homomultimer. An example of a heteromultimer is a polypeptide comprising SEQ ID NO 77 and one or more of SEQ ID NOS 39 76. Another example of a heteromultimer is a polypeptide comprising SEQ ID NO 70 and one or more of SEQ ID NOS 39 69. Another example of a heteromultimer is a polypeptide comprising SEQ ID NO 72 and one or more of SEQ ID NOS 39 71. There are numerous ways to combine SEQ ID NOS 39 77 into homomultimeric or heteromultimeric localization signals. SEQ ID NOS 39 69 are selected examples of subsequences of SEQ ID NOS 17 38 however additional subsequences wild type or mutated may be utilized to form multimeric localization signals. The instant invention is directed to all possible combinations of homomultimeric and heteromultimeric localization signals without limitation.

SEQ ID NOS 17 38 represent full length sequences of proteins that have endoplasmic reticulum localization activity. SEQ ID NOS 39 69 are subsequences of SEQ ID NOS 17 38 that are capable of conferring endoplasmic reticulum localization. SEQ ID NOS 70 77 are amino acid sequences that confer endoplasmic reticulum retention. Polypeptide subsequences that are identical to their wild type parent may be used as part of a localization signal however in one embodiment some amino acids are mutated to another amino acid such as one of the naturally occurring amino acids including alanine aspartate asparagine cysteine glutamate glutamine phenylalanine glycine histidine isoleucine leucine lysine methionine proline arginine valine tryptophan serine threonine or tyrosine. Mutation of amino acids may be performed for various reasons including but not limited to minimization of undesired biological activity introduction or removal of secondary structure in the polypeptide disruption of protein protein interaction modification of charge hydrophobicity or stability of the polypeptide and introduction or removal of restriction sites in the nucleic acid encoding the polypeptide. As shown by SEQ ID NO 7 and Example 4 below the localization signals of the invention are capable of directing polypeptides of interest to the endoplasmic reticulum of eukaryotic cells.

In general endoplasmic reticulum localization signals are built by identifying proteins that localize to the endoplasmic reticulum. Sometimes it is desirable to utilize wild type truncations as building blocks. However it is sometimes desirable to modify one or more amino acids to enhance the localization. Other reasons for modifying the wild type sequences are to remove undesired characteristics such as enzymatic activity or modulation of an endogenous cellular function. Monomeric building blocks may include an endoplasmic reticulum localization sequence as well as amino acids adjacent and contiguous on either side. Monomeric building blocks may therefore be any length provided the monomer confers endoplasmic localization routing and or retention. For example the monomer may comprise at least 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 100 or more amino acids adjacent to the endoplasmic reticulum localization routing or retention conferring sequence.

For example in one embodiment the invention comprises an endoplasmic reticulum localization signal comprising at least one copy of a peptide selected from the group consisting of 

a a peptide at least 80 85 90 95 96 97 98 or 99 identical to a peptide comprising amino acid residues corresponding to amino acid residues 2338 2428 of the amino acid sequence of SEQ ID NO 17 

b a peptide at least 80 85 90 95 96 97 98 or 99 identical to a peptide comprising amino acid residues corresponding to amino acid residues 2341 2425 of the amino acid sequence of SEQ ID NO 17 

c a peptide at least 80 85 90 95 96 97 98 or 99 identical to a peptide comprising amino acid residues corresponding to amino acid residues 2349 2417 of the amino acid sequence of SEQ ID NO 17 and

d a peptide at least 80 85 90 95 96 97 98 or 99 identical to a peptide comprising amino acid residues corresponding to amino acid residues 2359 2407 of the amino acid sequence of SEQ ID NO 17.

In another embodiment the invention comprises an endoplasmic reticulum localization signal comprising at least one copy of a peptide selected from SEQ ID NOS 70 77 and comprising at least one copy of a peptide selected from the group consisting of 

a a peptide at least 80 85 90 95 96 97 98 or 99 identical to a peptide comprising amino acid residues corresponding to amino acid residues 2338 2428 of the amino acid sequence of SEQ ID NO 17 

b a peptide at least 80 85 90 95 96 97 98 or 99 identical to a peptide comprising amino acid residues corresponding to amino acid residues 2341 2425 of the amino acid sequence of SEQ ID NO 17 

c a peptide at least 80 85 90 95 96 97 98 or 99 identical to a peptide comprising amino acid residues corresponding to amino acid residues 2349 2417 of the amino acid sequence of SEQ ID NO 17 and

d a peptide at least 80 85 90 95 96 97 98 or 99 identical to a peptide comprising amino acid residues corresponding to amino acid residues 2359 2407 of the amino acid sequence of SEQ ID NO 17.

As used herein the terms correspond s to and corresponding to as they relate to sequence alignment are intended to mean enumerated positions within a reference protein e.g. IP3 Receptor SEQ ID NO 17 and those positions that align with the positions on the reference protein. Thus when the amino acid sequence of a subject peptide is aligned with the amino acid sequence of a reference peptide e.g. SEQ ID NO 17 the amino acids in the subject peptide sequence that correspond to certain enumerated positions of the reference peptide sequence are those that align with these positions of the reference peptide sequence but are not necessarily in these exact numerical positions of the reference sequence. Methods for aligning sequences for determining corresponding amino acids between sequences are described below.

Additional embodiments of the invention include monomers based on any putative or real polypeptide or protein that has endoplasmic reticulum localization routing or retention activity such as those identified by SEQ ID NOS 39 77. Furthermore if the protein has more than one localization subsequence then more than one monomer may be identified therein.

Another embodiment of the invention is a nucleic acid molecule comprising a polynucleotide sequence encoding at least one copy of a localization signal polypeptide.

Another embodiment of the invention is a nucleic acid molecule wherein the polynucleotide sequence encodes one or more copies of one or more localization signal polypeptides.

Another embodiment of the invention is a nucleic acid molecule wherein the polynucleotide sequence encodes at least a number of copies of the peptide selected from the group consisting of 2 3 4 5 6 7 8 9 or 10.

Another embodiment of the invention is a vector comprising a nucleic acid molecule encoding at least one copy of an endoplasmic reticulum localization signal.

Another embodiment of the invention is a recombinant host cell comprising a vector comprising a nucleic acid molecule encoding at least one copy of an endoplasmic reticulum localization signal.

Another embodiment of the invention is a method of localizing a polypeptide to an endoplasmic reticulum subcellular compartment in a cell comprising linking a polypeptide open reading frame to a localization signal open reading frame to create a fusion protein coding sequence and transfecting the fusion protein coding sequence into a host cell and culturing the transfected host cell under conditions suitable to produce at least one copy of the fusion protein.

Another embodiment of the invention is a method of delivering a therapeutic molecule to a subcellular location in a cell comprising transfecting a vector comprising a nucleic acid molecule encoding at least one copy of a localization signal linked to a therapeutic molecule into a host cell and culturing the transfected host cell under conditions suitable to produce at least one copy of the localization signal containing therapeutic molecule.

The invention also relates to modified localization signals that are at least about 80 85 90 95 96 97 98 or 99 identical to a reference polypeptide. A modified localization signal is used to mean a peptide that can be created by addition deletion or substitution of one or more amino acids in the primary structure amino acid sequence of a localization signal protein or polypeptide. The terms protein and polypeptide and peptide are used interchangeably herein. The reference polypeptide is considered to be the wild type protein or a portion thereof. Thus the reference polypeptide may be a protein whose sequence was previously modified over a wild type protein. The reference polypeptide may or may not be the wild type protein from a particular organism.

A polypeptide having an amino acid sequence at least for example about 95 identical to a reference an amino acid sequence is understood to mean that the amino acid sequence of the polypeptide is identical to the reference sequence except that the amino acid sequence may include up to about five modifications per each 100 amino acids of the reference amino acid sequence encoding the reference peptide. In other words to obtain a peptide having an amino acid sequence at least about 95 identical to a reference amino acid sequence up to about 5 of the amino acid residues of the reference sequence may be deleted or substituted with another amino acid or a number of amino acids up to about 5 of the total amino acids in the reference sequence may be inserted into the reference sequence. These modifications of the reference sequence may occur at the N terminus or C terminus positions of the reference amino acid sequence or anywhere between those terminal positions interspersed either individually among amino acids in the reference sequence or in one or more contiguous groups within the reference sequence.

As used herein identity is a measure of the identity of nucleotide sequences or amino acid sequences compared to a reference nucleotide or amino acid sequence. In general the sequences are aligned so that the highest order match is obtained. Identity per se has an art recognized meaning and can be calculated using published techniques. See e.g. Computational Molecular Biology Lesk A. M. ed. Oxford University Press New York 1988 Biocomputing Informatics And Genome Projects Smith D. W. ed. Academic Press New York 1993 Computer Analysis of Sequence Data Part I Griffin A. M. and Griffin H. G. eds. Humana Press New Jersey 1994 von Heinje G. Sequence Analysis In Molecular Biology Academic Press 1987 and Sequence Analysis Primer Gribskov M. and Devereux J. eds. M Stockton Press New York 1991 . While there exist several methods to measure identity between two polynucleotide or polypeptide sequences the term identity is well known to skilled artisans Carillo H. Lipton D. Siam J Applied Math 48 1073 1988 . Methods commonly employed to determine identity or similarity between two sequences include but are not limited to those disclosed in Guide to Huge Computers Martin J. Bishop ed. Academic Press San Diego 1994 and Carillo H. Lipton D. Siam J Applied Math 48 1073 1988 . Computer programs may also contain methods and algorithms that calculate identity and similarity. Examples of computer program methods to determine identity and similarity between two sequences include but are not limited to GCG program package Devereux J. et al. Nucleic Acids Research 12 i 387 1984 BLASTP ExPASy BLASTN FASTA Atschul S. F. et al. J Molec Biol 215 403 1990 and FASTDB. Examples of methods to determine identity and similarity are discussed in Michaels G. and Garian R. Vol 1 John Wiley Sons Inc. 2000 which is incorporated by reference. In one embodiment of the present invention the algorithm used to determine identity between two or more polypeptides is BLASTP.

In another embodiment of the present invention the algorithm used to determine identity between two or more polypeptides is FASTDB which is based upon the algorithm of Brutlag et al. Comp. App. Biosci. 6 237 245 1990 incorporated by reference . In a FASTDB sequence alignment the query and subject sequences are amino sequences. The result of sequence alignment is in percent identity. Parameters that may be used in a FASTDB alignment of amino acid sequences to calculate percent identity include but are not limited to Matrix PAM k tuple 2 Mismatch Penalty 1 Joining Penalty 20 Randomization Group Length 0 Cutoff Score 1 Gap Penalty 5 Gap Size Penalty 0.05 Window Size 500 or the length of the subject amino sequence whichever is shorter.

If the subject sequence is shorter or longer than the query sequence because of N terminus or C terminus additions or deletions not because of internal additions or deletions a manual correction can be made because the FASTDB program does not account for N terminus and C terminus truncations or additions of the subject sequence when calculating percent identity. For subject sequences truncated at the N and C terminal ends relative to the query sequence the percent identity is corrected by calculating the number of bases of the query sequence that are N and C terminus to the reference sequence that are not matched aligned as a percent of the total bases of the query sequence. The results of the FASTDB sequence alignment determine matching alignment. The alignment percentage is then subtracted from the percent identity calculated by the above FASTDB program using the specified parameters to arrive at a final percent identity score. This corrected score can be used for the purposes of determining how alignments correspond to each other as well as percentage identity. Residues of the query subject sequences or the reference sequence that extend past the N or C termini of the reference or subject sequence respectively may be considered for the purposes of manually adjusting the percent identity score. That is residues that are not matched aligned with the N or C termini of the comparison sequence may be counted when manually adjusting the percent identity score or alignment numbering.

For example a 90 amino acid residue subject sequence is aligned with a 100 residue reference sequence to determine percent identity. The deletion occurs at the N terminus of the subject sequence and therefore the FASTDB alignment does not show a match alignment of the first 10 residues at the N terminus. The 10 unpaired residues represent 10 of the sequence number of residues at the N and C termini not matched total number of residues in the query sequence so 10 is subtracted from the percent identity score calculated by the FASTDB program. If the remaining 90 residues were perfectly matched the final percent identity would be 90 . In another example a 90 residue subject sequence is compared with a 100 reference sequence. This time the deletions are internal deletions so there are no residues at the N or C termini of the subject sequence which are not matched aligned with the query. In this case the percent identity calculated by FASTDB is not manually corrected.

The multimeric localization signals of the invention optionally comprise spacer amino acids before after or between monomers for example A E . Additionally the localization signals of the invention optionally comprise spacer amino acids before or after the localization signal for example E A B E F A B E F C D E G C D E G and H . The length and composition of the spacer may vary. An example of a spacer is glycine alanine polyglycine or polyalanine. In addition to providing space between monomers spacers can be used for the purpose of engineering restriction sites in the encoding nucleic acid and can be used for modifying secondary structure of the polypeptide encoded. Specific examples of spacers used between monomers in SEQ ID NO 7 are the peptides EFGGGGG SEQ ID NO 79 and PGAG SEQ ID NO 78 . In the instance of SEQ ID NO 7 the proline containing spacer is intended to break an alpha helical secondary structure. At the C terminal end of SEQ ID NO 7 is a five amino acid spacer with the sequence AAPAA SEQ ID NO 80 . This particular spacer provides a linker to another module coding region such as a reporter epitope or experimental or therapeutic polypeptide. The spacer amino acids may be any amino acid and are not limited to alanine glycine and proline. The instant invention is directed to all combinations of homomultimers and heteromultimers with or without spacers and without limitation to the examples given above or below.

The localization signals of the invention are optionally linked to additional molecules or amino acids that provide an epitope a reporter and or an experimental or therapeutic molecule A H A H A H . Non limiting examples of epitope are FLAG Kodak Rochester N.Y. HA hemagluttinin c Myc and His6. Non limiting examples of reporters are alkaline phosphatase galactosidase peroxidase luciferase and fluorescent proteins. Non limiting examples of experimental proteins are enzymes enzyme binding partners signalling factors structural factors and peptide ligands metabolic binding factors nucleic acid binding factors and cellular binding factors. The epitopes reporters and experimental or therapeutic molecules are given by way of example and without limitation. The epitope reporter experimental molecule and or therapeutic molecule may be the same molecule. The epitope reporter experimental molecule and or therapeutic molecule may also be different molecules.

Localization signals and optional amino acids linked thereto can be synthesized chemically or recombinantly using techniques known in the art. Chemical synthesis techniques include but are not limited to peptide synthesis which is often performed using an automated peptide synthesizer. Peptides can also be synthesized utilizing non automated peptide synthesis methods known in the art. Recombinant techniques include insertion of localization signal encoding nucleic acids into expression vectors wherein nucleic acid expression products are synthesized using cellular factors and processes.

Linkage of an epitope reporter experimental or therapeutic molecule to a localization signal can include covalent or enzymatic linkage. When the localization signal comprises material other than a polypeptide such as a lipid or carbohydrate a chemical reaction to link molecules may be utilized. Additionally non standard amino acids and amino acids modified with lipids carbohydrates phosphate or other molecules may be used as precursors to peptide synthesis. The localization signals of the invention have utility as therapeutic targeting molecules. Pure peptides represent embodiments of conventional peptide therapeutics. However polypeptides or proteins linked to localization signals have utility as subcellular tools or therapeutics. For example polypeptides depicted generically in represent localization signals with utility as subcellular tools or therapeutics. Localization signal containing gene constructs are also delivered via gene therapy. depict embodiments of gene therapy vectors for delivering and controlling polypeptide expression in vivo. Polynucleotide sequences linked to the gene construct in include genome integration domains to facilitate integration of the transgene into a viral genome and or host genome.

The invention also encompasses polynucleotides comprising nucleotide sequences encoding monomeric localization signals and multimeric localization signals. The polynucleotides of the invention are optionally linked to additional nucleotide sequences encoding epitopes reporters and or experimental or therapeutic molecules. Further the nucleic acids of the invention are optionally incorporated into vector polynucleotides. The polynucleotides are optionally flanked by nucleotide sequences comprising restriction endonuclease sites and other nucleotides needed for restriction endonuclese activity. The flanking sequences optionally provide cloning sites within a vector. The restriction sites can include but are not limited to any of the commonly used sites in most commercially available cloning vectors. Non limiting examples of such sites are those recognized by NsiI ApaL1 MfeI KpnI BamHI ClaI EcoRI EcoRV SpeI AfIII NdeI NheI XbaI XhoI SphI NaeI SexAI HindIII HpaI and PstI restriction endonucleases. Sites for cleavage by other restriction enzymes including homing endonucleases are also used for this purpose. The polynucleotide flanking sequences also optionally provide directionality of subsequence cloning. It is preferred that 5 and 3 restriction endonuclease sites differ from each other so that double stranded DNA can be directionally cloned into corresponding complementary sites of a cloning vector.

Localization signals with or without epitopes reporters or experimental or therapeutic proteins are alternatively synthesized by recombinant techniques. Polynucleotide expression constructs are made containing desired components and inserted into an expression vector. The expression vector is then transfected into cells and the polypeptide products are expressed and isolated. Localization signals made according to recombinant DNA techniques have utility as research tools and or subcellular therapeutic delivery agents.

The following is an example of how polynucleotides encoding localization signals are produced. Complimentary oligonucleotides encoding the localization signals and flanking sequences are synthesized and annealed. The resulting double stranded DNA molecule is inserted into a cloning vector using techniques known in the art. When the localization signals are placed in frame adjacent to sequences within a transgenic gene construct that is translated into a protein product they form part of a fusion protein when expressed in cells or transgenic animals.

Another embodiment of the invention relates to selective control of transgene expression in a desired cell or organism. The promoter portion of the recombinant gene can be a constitutive promoter a non constitutive promoter a tissue specific promoter constitutive or non constitutive or a selectively controlled promoter. Different selectively controlled promoters are controlled by different mechanisms. For example a tetracycline inducible promoter is activated to express a downstream coding sequence when the cell containing the promoter and other necessary cellular factors is treated with tetracycline. When tetracycline is removed gene expression is subsequently reduced. Other inducible promoters are activated by other drugs or factors. RheoSwitch is an inducible promoter system available from New England Biolabs Ipswich Mass. . Temperature sensitive promoters can also be used to increase or decrease gene expression. An embodiment of the invention comprises a localization signal containing gene construct whose expression is controlled by an inducible promoter. In one embodiment the inducible promoter is tetracycline inducible.

Monomeric and multimeric ER localization signals and methods of making these localization signals are disclosed. Below are examples of methods of using ER localization signals. In general localization signals linked to epitopes reporters and other desired proteins or molecules are delivered via adenovirus lentivirus adeno associated virus or other viral constructs that express protein product in a cell.

Fluorescence microscopy is employed to determine spatial cellular localization. Fluorescence microscopy involves autofluorescence of fluorescent proteins fused to localization signals of the invention. Alternatively fluorescence microscopy involves immunofluorescence of antibodies directed against epitopes fused to localization signals. Anti epitope antibodies are either directly linked to a fluorochrome or are used in combination with a fluorescent secondary antibody.

Known cellular structures and locations are comparatively illustrated with well known and or commercially available stains dyes antibodies and or other reagents that identify cellular locations. Such reagents include but are not limited to DAPI Hoechst stains acridine orange Lysotracker Invitrogen Carlsbad Calif. ERtracker Invitrogen Carlsbad Calif. Golgitracker Invitrogen Carlsbad Calif. Mitotracker Invitrogen Carlsbad Calif. anti CD25 anti myc anti OSBP anti NSF anti transferrin receptor anti T cell transferrin receptor anti AP2 alpha subunit anti clathrin heavy chain anti lamin anti histone anti histone deacetylase anti p53 phalloidin coumarin phalloidin FITC phalloidin phycoerythrin anti oxysterol binding protein anti nem sensitive factor anti gm130 anti lamp 1 anti lamp2 acridine orange nonyl bromide anti tac antigen anti Na K ATPase and anti EGF receptor antibody producing hybridomas available from ATCC .

Electron microscopy is employed to determine location at higher magnifications. Slides of cells expressing localization signals fused to epitopes are prepared using techniques known in the art. Anti epitope antibodies are either directly linked to a gold label or in combination with a gold labeled secondary antibody.

Immunoblotting is employed to determine quantitative expression levels and or to biochemically corroborate microscopic observations. Immunoblotting or western blotting is performed on whole cell lysates and or on cells that have been fractionated by density gradient centrifugation. Antibodies useful for fraction identification by western blot include but are not limited to anti lamin anti histone anti histone deacetylase anti p53 anti oxysterol binding protein anti nem sensitive factor anti gm130 anti lamp1 anti Iamp2 anti tac antigen anti caveolin 1 and anti EGF receptor.

Epitopes for use in localization signal fusion proteins include hemagglutinin HA FLAG and Myc among others. Specifically localization signals fused to an epitope are expressed in Hela HCTZ 16 HT1080 HCN1a HCN2 SHSY5Y ARPE19 HPV16 p5 U87 MG C2Bbe1 HEK293 COS1 COS7 MDCK C2C12 Sol8 P19 10T1 2 and NIH3T3 available from the ATCC . Anti hemagluttinin antibodies and fluorescent secondary antibody are then employed to visualize location using standard methods such as those described in Giepmans et al. 2006 Science 312 217 24 incorporated by reference herein. For electron microscopy methods such as those described in Ukimura et al. 1997 Am J. Pathol. 150 2061 2074 incorporated by reference herein are employed.

Alternatively localization signals fused to a fluorescent protein are expressed in Hela HCT116 HT 1080 HCN1a HCN2 SHSY5Y ARPE19 HPV16 p5 U87 MG C2Bbe1 HEK293 COS 1 COS7 MDCK C2C12 Sol8 P19 10T1 2 and NIH3T3 available from the ATCC . Location is visualized using standard methods such as those described in Giepmans et al. 2006 Science 312 217 24 incorporated by reference herein.

For immunoblot analysis cellular fractions are obtained by taking cells expressing localization signals fused to a hemagluttinin epitope and lightly homogenizing them for example in a Dounce homogenizer. Homogenized cells are then subjected to density gradient centrifugation as is known in the art and described in Current Methods in Cell Biology Volume 1 Chapter 3 pages 3.0.1 3.11.22 Bonafacino et al. editors incorporated by reference herein . Fractions from the density gradient centrifugation are then electrophoresed on an acrylamide gel and subsequently transfered to a membrane electrophoretically. The membrane is then probed with appropriate anti hemagluttinin antibodies and or antibodies to known proteins. By comparing the gel lanes showing an anti hemagglutinnin signal to gel lanes showing antibody signals of known proteins cellular location of a localization signal of the invention is determined biochemically.

A polypeptide comprising a multimeric endoplasmic reticulum localization signal and an epitope is synthesized. The structure of such a polypeptide is generically represented by . The polypeptide is synthesized on an automated peptide synthesizer or is recombinantly expressed and purified. Purified polypeptide is solubilized in media and added to cells. Verification is performed by visualization of antibody binding to the epitope.

A transgene is constructed using a human cytomegalovirus CMV promoter to direct expression of a fusion protein comprising SEQ ID NO 64 SEQ ID NO 69 and SEQ ID NO 72 LOCALIZATION SIGNAL and green fluorescent protein REPORTER . Such a transgene is generically represented by . The transgene is transfected into cells for transient expression. Verification of expression and location is performed by visualization of the fluorescent protein by confocal microscopy.

A transgene construct is built to produce a protein product with expression driven by a tissue specific promoter. The transgene comprises a synthetic gene expression unit engineered to encode three domains. Each of these three domains is synthesized as a pair of complimentary polynucleotides that are annealed in solution ligated and inserted into a vector. Starting at the amino terminus the three domains in the expression unit are nucleotide sequences that encode a kinase inhibitor a FLAG epitope and an endoplasmic reticulum localization signal. The localization signal is a monomeric homomultimeric or heteromultimeric localization signal as described herein. Nucleotide sequences encoding a FLAG epitope are placed downstream of nucleotide sequences encoding the kinase inhibitor. Finally nucleotide sequences encoding the localization signal are placed downstream of those encoding the FLAG epitope. The assembled gene expression unit is subsequently subcloned into an expression vector such as that shown in and used to transiently transfect cells. Verification is performed by microscopic visualization of the epitope immunoreactivity at the endoplasmic reticulum.

Subcellularly localized chloramphenicol acetyltransferase fragment was demonstrated in the endoplasmic reticulum of Cos7 cells using a transgene construct containing an endoplasmic reticulum localization signal a c Myc eptitope and a chloramphenicol acetyltransferase fragment non enzymatic was made. The expression unit contains nucleotides that encode an endoplasmic reticulum localization signal SEQ ID NO 7 LOCALIZATION SIGNAL a c Myc epitope EPITOPE and a fragment of chloramphenicol acetyltransferase POLYPEPTIDE OF INTEREST . This expression unit is subsequently subcloned into a vector between a CMV promoter and an SV40 polyadenylation signal . The completed transgene containing expression vector was then used to transfect Cos7 cells. illustrates the subcellular colocation yellow of the c Myc epitope green with calreticulin red . In the presence of the localization signal chloramphenicol acetyltransferase fragment is located at the endoplasmic reticulum.

Additionally subcellularly localized polypeptide of interest was demonstrated in the endoplasmic reticulum of Cos7 cells using a transgene construct containing an endoplasmic reticulum localization signal a FLAG or modified FLAG eptitope and an ERK decoy polypeptide of interest. The expression unit of the transgene contains nucleotides that encode an ERK decoy POLYPEPTIDE OF INTEREST a FLAG or modified FLAG tag EPITOPE and endoplasmic reticulum localization signal SEQ ID NO 1 LOCALIZATION SIGNAL . This expression unit was subsequently subcloned into a vector between an EF1alpha promoter and an SV40 polyadenylation signal . The completed transgene containing expression vector was then used to transfect Cos7 cells. illustrate the subcellular location red of the FLAG or modified FLAG epitope.

Fluorescent protein localization is demonstrated in vivo by making a transgene construct used to generate mice expressing a fusion protein targeted to the endoplasmic reticulum. The transgene construct is shown generically in . The expression unit contains nucleotides that encode a dimer of SEQ ID NO 49 LOCALIZATION SIGNAL and green fluorescent protein POLYPEPTIDE . This expression unit is subsequently subcloned into a vector between nucleotide sequences including a mammalian promoter and an SV40 polyadenylation signal. The completed transgene is then injected into pronuclei of fertilized mouse oocytes. The resultant pups are screened for the presence of the transgene by PCR. Transgenic founder mice are bred with wild type mice. Heterozygous transgenic animals from at least the third generation are used for the following tests with their non transgenic littermates serving as controls.

Test 1 Southern blotting analysis is performed to determine the copy number. Southern blots are hybridized with a radio labeled probe generated from a fragment of the transgene. The probe detects bands containing DNA from transgenic mice but does not detect bands containing DNA from non transgenic mice. Intensities of the transgenic mice bands are measured and compared with the transgene plasmid control bands to estimate copy number. This demonstrates that mice in Example 4 harbor the transgene in their genomes.

Test 2 Tissues are prepared for microscopic analysis. This experiment demonstrates the transgene is expressed in tissues of transgenic mice because green fluorescent protein is visualized in transgenic tissues but not in non transgenic tissues.

These examples demonstrate delivery of molecules to a localized region of a cell for therapeutic or experimental purposes. The purified polypeptide localization signals linked to therapeutics can be formulated for oral or parenteral administration topical administration or in tablet capsule or liquid form intranasal or inhaled aerosol subcutaneous intramuscular intraperitoneal or other injection intravenous instillation or any other routes of administration. Furthermore the nucleotide sequences encoding the localization signals permit incorporation into a vector designed to deliver and express a gene product in a subcellular compartment. Such vectors include plasmids cosmids artificial chromosomes and modified viruses. Delivery to eukaryotic cells can be accomplished in vivo or ex vivo. Ex vivo delivery methods include isolation of the intended recipient s cells or donor cells and delivery of the vector to those cells followed by treatment of the recipient with the cells. The invention encompasses transgenes comprising localization signals and non human transgenic organisms harboring these transgenes. The transgenes may be under the control of inducible promoters or tissue specific promoters.

Disclosed are endoplasmic localization signals and methods of making and using these localization signals. The localization signals are synthesized chemically or recombinantly and are utilized as research tools or as therapeutic delivery agents. The invention includes linking molecules to cellular localization signals for subcellular therapeutics.

