---

title: Anti-polyubiquitin antibodies
abstract: The invention provides anti-polyubiquitin antibodies and methods of using the same.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09321844&OS=09321844&RS=09321844
owner: Genentech, Inc.
number: 09321844
owner_city: South San Francisco
owner_country: US
publication_date: 20120806
---
This application claims priority under 35 U.S.C. 119 e to U.S. Provisional No. 61 515 729 filed Aug. 5 2011 which is incorporated by reference herein in its entirety.

The instant application contains a Sequence Listing which has been submitted in ASCII format via EFS Web and is hereby incorporated by reference in its entirety. Said ASCII copy created on Aug. 6 2012 is named P4716R1.txt and is 128 464 bytes in size.

The present invention relates to the field of anti polyubiquitin antibodies and more particularly to anti polyubiquitin antibodies that do not specifically bind to monoubiquitin and that are specific for linear polyubiquitin and methods of using the same.

Ubiquitin is a small protein that has important regulatory roles in a wide variety of cellular pathways. The best known of these is ubiquitin s role in protein degradation where covalent attachment of ubiquitin to a target protein enables that targeted protein to be recognized and destroyed by the 26S proteasome see Wilkinson 11 3 141 148 2000 . The covalent attachment of ubiquitin a 76 amino acid protein to a target protein is a three step enzymatic process Pickart 70 503 533 2001 . First ubiquitin activating enzyme E1 forms an ubiquitin E1 thioester in an ATP dependent reaction. The ubiquitin is transferred from the ubiquitin E1 thioester to a member of the ubiquitin conjugating enzyme E2 family in the second step. In the third step with the assistance of a ubiquitin protein ligase E3 an isopeptide bond is formed between the carboxyl terminus of ubiquitin and the 8 amino group of a lysine residue on the target protein. Enzymes termed deubiquitinases remove ubiquitin moieties from target proteins Guterman and Glickman Curr. Prot. Pep. Sci. 5 201 210 2004 .

Ubiquitin contains seven lysine residues Lys6 Lys11 Lys27 Lys33 Lys29 Lys48 and Lys63 and thus ubiquitin itself may serve as a target protein for ubiquitination Peng et al. 21 921 926 2003 Pickart and Fushman 8 610 616 2004 . The molecule produced upon ubiquitination of a ubiquitin protein is termed a polyubiquitin molecule and may comprise two or more ubiquitin moieties. Ubiquitination of ubiquitin may theoretically occur at any of the seven lysine residues Peng et al. 21 921 926 2003 so that different species of polyubiquitins exist having isopeptide bonds to different lysine residues within ubiquitin. Polyubiquitin chains with internal isopeptide linkages at all seven lysine resides have been reported. Iwai and Tokunaga 10 706 713 2009 .

Recently it was discovered that linear polyubiquitin chains also form in which the C terminal glycine of ubiquitin is conjugated to the amino group of the N terminal methionine of another ubiquitin molecule. Iwai and Tokunaga 10 706 713 2009 . Linear polyubiquitin is formed via the linear ubiquitin chain assembly complex LUBAC which is composed of two ring finger proteins HOIL 1L and HOW. Tokunaga et al. 11 123 132 2009 . It is believed that genetically encoded unanchored linear polyubiquitin does not exist in cells as its C terminus is vulnerable to cleavage by isopeptidase T. Iwai and Tokunaga 10 706 713 2009 . This observation suggests that linear polyubiquitin is assembled onto a substrate protein post translationally and that conjugated linear polyubiquitin molecules are potential modulators of protein activity and function. Id. For example linear polyubiquitination of the NF kB essential modulator NEMO has been shown to play a role in NF B activation. Id.

Antibodies which distinguish linear polyubiquitin over polyubiquitin of different lysine linkages would be useful to further examine the role of linear polyubiquitin chains in protein degradation and regulation and to target and modulate linear polyubiquitin in linear polyubiquitin mediated pathways.

The invention provides anti linear polyubiquitin antibodies and methods of using the same. In one embodiment the invention provides an isolated antibody that specifically binds a first polyubiquitin comprising a C terminal to N terminal linkage wherein the antibody does not specifically bind a second polyubiquitin comprising a lysine linkage. In another embodiment the invention provides an isolated antibody that specifically binds both a first polyubiquitin comprising a C terminal to N terminal linkage and a second polyubiquitin comprising a lysine linkage wherein the antibody does not specifically bind monoubiquitin and wherein the antibody binds the second polyubiquitin with a substantially reduced binding affinity as compared to the binding affinity of the antibody for the first polyubiquitin.

In another embodiment the invention provides an isolated antibody that specifically binds C to N terminal linked polyubiquitin wherein the antibody does not specifically bind monoubiquitin. In one aspect the antibody comprises at least one hypervariable HVR sequence selected from HVR L1 HVR L2 HVR L3 HVR H1 HVR H2 and HVR H3 of any of SEQ ID NOs 1 4 19 and 50 57 SEQ ID NOs 2 and 58 63 SEQ ID NOs 3 5 6 20 21 and 64 72 SEQ ID NOs 7 10 13 16 22 and 73 81 SEQ ID NOs 8 11 14 17 23 24 and 82 86 and SEQ ID NOs 9 12 15 18 and 87 93 respectively.

In another aspect the antibody comprises at least one sequence selected from HVR L1 HVR L2 and HVR L3 wherein HVR L1 comprises the amino acid sequence RASQXVXXXVA SEQ ID NO 39 wherein amino acid Xis selected from amino acid D S and G amino acid Xis selected from S and D amino acid Xis selected from S T and N and amino acid Xis selected from A and S wherein HVR L2 comprises the amino acid sequence of SEQ ID NO 2 and wherein HVR L3 comprises the amino acid sequence QQXXXXXPXT SEQ ID NO 40 wherein amino acid Xis selected from S Y and H amino acid Xis selected from Y and F amino acid Xis selected from T Y and A amino acid Xis selected from T Y and S amino acid Xis optional and if present is serine and amino acid Xis selected from P and L.

In another aspect the antibody comprises at least one hypervariable HVR sequence selected from HVR H1 HVR H2 and HVR H3 wherein HVR H1 comprises the amino acid sequence XXXXXXXX SEQ ID NO 41 wherein amino acid Xis selected from T and N amino acid Xis selected from F and I amino acid Xis selected from S T and Y amino acid Xis selected from N D S and Y amino acid Xis selected from T Y S and D amino acid Xis selected from Y D and S amino acid Xis selected from 1 and M and amino acid Xis selected from S and H and wherein HVR H2 comprises the amino acid sequence AXIXXXXXXTX SEQ ID NO 42 wherein amino acid Xis selected from S G W and E amino acid Xis selected from T S and Y amino acid Xis selected from P and S amino acid Xis selected from S and Y amino acid Xis selected from G S and Y amino acid Xis selected from G and S amino acid Xis selected from S and Y and amino acid Xis selected from D and S and HVR H3 comprises the amino acid sequence RXXXXXXXXXXXD SEQ ID NO 43 wherein amino acid Xis selected from T E and G amino acid Xis selected from W A and Y amino acid Xis selected from L G V and S amino acid Xis selected from L S and W amino acid Xis selected from R K and Y amino acid Xis selected from W L G and Y amino acid Xis selected from V L A and G amino acid Xis selected from M and F and wherein amino acids X X and Xare optionally present and if present amino acid Xis S amino acid Xis selected from V and P and amino acid Xis A.

In another aspect the antibody comprises at least one hypervariable HVR sequence selected from HVR L1 HVR L2 and HVR L3 wherein HVR L1 comprises the amino acid sequence RASQ XXXXXXA SEQ ID NO 44 wherein amino acid Xis selected from D A E G L N S T and V amino acid Xis selected from V A L and S amino acid Xis selected from S F G L R and V amino acid Xis selected from T G I N S and V amino acid Xis selected from A H Q R S and Y and amino acid Xis selected from V and L and wherein HVR L2 comprises the amino acid sequence SXXXXYX SEQ ID NO 45 wherein amino acid Xis selected from A and R amino acid Xis selected from S K Q and R amino acid Xis selected from F and Y amino acid Xis selected from L A F G H I K M N P R S V and Y and amino acid Xis selected from S A D F G H V W and Y and wherein HVR L3 comprises the sequence QQ XXXXPPT SEQ ID NO 46 wherein amino acid Xis selected from H and S amino acid Xis selected from Y K N Q R S V and W amino acid Xis selected from T I Q R S and V and amino acid Xis selected from T A D F G K N P Q R S and V.

In another aspect the antibody comprises at least one hypervariable HVR sequence selected from HVR H1 HVR H2 and HVR H3 wherein HVR H1 comprises the amino acid sequence XXXYXS SEQ ID NO 47 wherein amino acid Xis selected from A F K M Q R and S amino acid Xis selected from N and W amino acid Xis selected from T A I L M and V and amino acid Xis selected from I M and V and wherein HVR L2 comprises the amino acid sequence AXXTPXSGXTX SEQ ID NO 48 wherein amino acid Xis selected from T and S amino acid Xis selected from I S and V amino acid Xis selected from S and A amino acid Xis selected from S H I L M and Q amino acid Xis selected from D and N and wherein HVR H3 comprises the amino acid sequence XWXXRWVXD SEQ ID NO 49 wherein amino acid Xis selected from S and T amino acid Xis selected from L and Y amino acid Xis selected from L I and V amino acid Xis selected from M and F.

In another aspect the antibody comprises an HVR L1 sequence of SEQ ID NO 1 or 4 an HVR L2 sequence of SEQ ID NO 2 and an HVR L3 sequence selected from SEQ ID NO 3 5 and 6 respectively. In another aspect the antibody comprises an HVR H1 sequence selected from SEQ ID NO 7 10 13 and 16 an HVR H2 sequence selected from SEQ ID NO 11 23 and 24 and an HVR H3 sequence of SEQ ID NO 12 respectively. In another aspect the antibody comprises an HVR L1 sequence selected from SEQ ID NO 1 and 50 56 an HVR L2 sequence selected from SEQ ID NO 2 and 57 62 and an HVR L3 sequence selected from SEQ ID NO 3 and 63 71 respectively. In another aspect the antibody comprises an HVR H1 sequence selected from SEQ ID NO 7 and 72 80 an HVR H2 sequence selected from SEQ ID NO 8 and 81 85 and an HVR H3 sequence selected from SEQ ID NO 9 and 86 92 respectively.

In another aspect the antibody comprises HVR L1 HVR L2 and HVR L3 sequences corresponding to those set forth for clones 1E2 1D8 1F4 or 1A10 in . In another aspect the antibody comprises HVR H1 HVR H2 and HVR H3 sequences corresponding to those set forth for clones 1E2 1D8 1F4 or 1A10 in . In another aspect the antibody comprises HVR L1 HVR L2 and HVR L3 sequences corresponding to those set forth for clones 1D8.3C2 1D8.3F8 or 1D8.4F5 in . In another aspect the antibody comprises HVR H1 HVR H2 and HVR H3 sequences corresponding to those set forth for clones 1D8.3C2 1D8.3F8 or 1D8.4F5 in .

In another aspect the antibody comprises the HVR L1 sequence of SEQ ID NO 1 the HVR L2 sequence of SEQ ID NO 2 the HVR L3 sequence of SEQ ID NO 3 the HVR H1 sequence of SEQ ID NO 10 the HVR H2 sequence of SEQ ID NO 11 and the HVR H3 sequence of SEQ ID NO 12. In another aspect the antibody comprises the HVR L1 sequence of SEQ ID NO 19 the HVR L2 sequence of SEQ ID NO 2 the HVR L3 sequence of SEQ ID NO 3 the HVR H1 sequence of SEQ ID NO 10 the HVR H2 sequence of SEQ ID NO 23 and the HVR H3 sequence of SEQ ID NO 12. In another aspect the antibody comprises the HVR L1 sequence of SEQ ID NO 1 the HVR L2 sequence of SEQ ID NO 2 the HVR L3 sequence of SEQ ID NO 20 the HVR H1 sequence of SEQ ID NO 10 the HVR H2 sequence of SEQ ID NO 11 and the HVR H3 sequence of SEQ ID NO 12. In another aspect the antibody comprises the HVR L1 sequence of SEQ ID NO 1 the HVR L2 sequence of SEQ ID NO 2 the HVR L3 sequence of SEQ ID NO 20 the HVR H1 sequence of SEQ ID NO 10 the HVR H2 sequence of SEQ ID NO 11 and the HVR H3 sequence of SEQ ID NO 12. In another aspect the antibody comprises the HVR L1 sequence of SEQ ID NO 1 the HVR L2 sequence of SEQ ID NO 2 the HVR L3 sequence of SEQ ID NO 3 the HVR H1 sequence of SEQ ID NO 7 the HVR H2 sequence of SEQ ID NO 8 and the HVR H3 sequence of SEQ ID NO 9. In another aspect the antibody comprises the HVR L1 sequence of SEQ ID NO 1 the HVR L2 sequence selected from SEQ ID NO 2 57 and 59 the HVR L3 sequence selected from SEQ ID NO 3 64 and 71 the HVR H1 sequence of SEQ ID NO 7 or 79 the HVR H2 sequence of SEQ ID NO 8 or 81 and the HVR L3 sequence selected from SEQ ID NO 9 86 88 or 89. In another aspect the antibody comprises the HVR L1 sequence of SEQ ID NO 1 the HVR L2 sequence of SEQ ID NO 57 the HVR L3 sequence of SEQ ID NO 3 the HVR H1 sequence of SEQ ID NO 7 the HVR H2 sequence of SEQ ID NO 8 and the HVR H3 sequence of SEQ ID NO 9. In another aspect the antibody comprises the HVR L1 sequence of SEQ ID NO 1 the HVR L2 sequence of SEQ ID NO 2 the HVR L3 sequence of SEQ ID NO 3 the HVR H1 sequence of SEQ ID NO 7 the HVR H2 sequence of SEQ ID NO 81 and the HVR H3 sequence of SEQ ID NO 9. In another aspect the antibody comprises the HVR L1 sequence of SEQ ID NO 1 the HVR L2 sequence of SEQ ID NO 57 the HVR L3 sequence of SEQ ID NO 3 the HVR H1 sequence of SEQ ID NO 7 the HVR H2 sequence of SEQ ID NO 81 and the HVR H3 sequence of SEQ ID NO 9. In certain aspects any of the antibodies described herein may include a leucine as the first amino acid after the C terminal end of HVR H3 e.g. the amino acid immediately adjacent to the C terminal end of a HVR H3 such as 1E3 TWLLRVMD SEQ ID NO 96 .

In another aspect the antibody comprises a light chain amino acid sequence selected from SEQ ID NOs 25 28 33 35 94 and 193 195. In another aspect the antibody comprises a heavy chain amino acid sequence selected from SEQ ID NOs 29 31 36 38 95 and 196 198.

In another aspect the antibody comprises light chain and heavy chain amino acid sequences with at least 95 sequence identity to the amino acid sequences of one of the following combinations of sequences SEQ ID NOs 25 and 29 SEQ ID NOs 26 and 30 SEQ ID NOs 27 and 31 SEQ ID NOs 28 and 32 SEQ ID NOs 33 and 36 SEQ ID NOs 34 and 37 SEQ ID NOs 35 and 38 SEQ ID NOs 95 and 95 SEQ ID NOs 193 and 196 SEQ ID NOs 194 and 197 and SEQ ID NOs 195 and 198.

In another embodiment the invention provides an isolated antibody wherein the antibody binds to the same antigenic determinant on C to N terminal linked polyubiquitin as any one of the foregoing antibodies and wherein the antibody does not specifically bind to monoubiquitin. In another embodiment the invention provides an isolated antibody that competes with any one of the foregoing antibodies for binding to C to N terminal linked polyubiquitin wherein the antibody does not specifically bind to monoubiquitin. In another embodiment the invention provides any of the foregoing isolated antibodies wherein the antibody specifically binds to a C to N terminal linked polyubiquitinated protein. In another embodiment the invention provides any of the foregoing isolated antibodies wherein the antibody modulates at least one polyubiquitin mediated signaling pathway.

In one general aspect any of the foregoing antibodies is a monoclonal antibody. In another general aspect any of the foregoing antibodies is a human antibody. In another general aspect any of the foregoing antibodies is a humanized antibody. In another general aspect any of the foregoing antibodies is a chimeric antibody. In another general aspect any of the foregoing antibodies is an antibody fragment that binds C to N terminal linked polyubiquitin.

In another embodiment the invention provides an isolated nucleic acid encoding any of the foregoing antibodies. In another embodiment the invention provides a vector comprising an isolated nucleic acid encoding any of the foregoing antibodies. In another embodiment the invention provides a host cell comprising an isolated nucleic acid encoding any of the foregoing antibodies. In another embodiment the invention provides a host cell comprising a vector comprising an isolated nucleic acid encoding any of the foregoing antibodies.

In another embodiment the invention provides a method of producing any of the foregoing antibodies comprising culturing the above recited host cell under conditions wherein the antibody is produced. In one aspect the method further comprises recovering the antibody from the host cell. In another aspect the method further comprises purification of the antibody.

In another embodiment the invention provides an immunoconjugate comprising any of the foregoing antibodies and a cytotoxic agent. In another embodiment the invention provides a pharmaceutical formulation comprising any of the foregoing antibodies and a pharmaceutically acceptable carrier. In one aspect the pharmaceutical formulation further comprises an additional therapeutic agent. In one such aspect the additional therapeutic agent is a chemotherapeutic agent.

In another embodiment the invention provides any of the foregoing antibodies for use as a medicament. In another embodiment the invention provides any of the foregoing antibodies for use in treating a cell cycle related disease or disorder. In one aspect the cell cycle related disease or disorder is selected from a disease or disorder associated with aberrantly increased cell cycle progression and a disease or disorder associated with aberrantly decreased cell cycle progression. In one such aspect the disease or disorder associated with aberrantly increased cell cycle progression is cancer. In another such aspect the disease or disorder associated with aberrantly decreased cell cycle progression is selected from a degenerative muscle disorder and a degenerative nerve disorder.

In another embodiment the invention provides the use of any of the foregoing antibodies in the manufacture of a medicament. In one aspect the medicament is for a disease or disorder selected from cancer a degenerative muscle disorder and a degenerative nerve disorder. In another embodiment the invention provides a method of treating an individual having a disease or disorder selected from cancer a degenerative muscle disorder and a degenerative nerve disorder comprising administering to the individual an effective amount of any of the foregoing antibodies.

In another embodiment the invention provides a method of determining the presence of a polyubiquitin or polyubiquitinated protein in a sample suspected of containing a polyubiquitin or polyubiquitinated protein comprising exposing the same to at least one of the foregoing antibodies and determining the binding of the at least one antibody to a polyubiquitin or polyubiquitinated protein in the sample. In another embodiment the invention provides a method of separating C to N terminal linked polyubiquitinated protein from non C to N terminal linked polyubiquitinated protein in a sample comprising contacting the sample with at least one of the foregoing antibodies. In another embodiment the invention provides a method of determining the function and or activity of C to N terminal linked polyubiquitin in a cell or sample comprising contacting the cell or sample with at least one of the foregoing antibodies and assessing the effect of said contacting step on the cell or sample.

An acceptor human framework for the purposes herein is a framework comprising the amino acid sequence of a light chain variable domain VL framework or a heavy chain variable domain VH framework derived from a human immunoglobulin framework or a human consensus framework as defined below. An acceptor human framework derived from a human immunoglobulin framework or a human consensus framework may comprise the same amino acid sequence thereof or it may contain amino acid sequence changes. In some embodiments the number of amino acid changes are 10 or less 9 or less 8 or less 7 or less 6 or less 5 or less 4 or less 3 or less or 2 or less. In some embodiments the VL acceptor human framework is identical in sequence to the VL human immunoglobulin framework sequence or human consensus framework sequence.

 Affinity refers to the strength of the sum total of noncovalent interactions between a single binding site of a molecule e.g. an antibody and its binding partner e.g. an antigen . Unless indicated otherwise as used herein binding affinity refers to intrinsic binding affinity which reflects a 1 1 interaction between members of a binding pair e.g. antibody and antigen . The affinity of a molecule X for its partner Y can generally be represented by the dissociation constant Kd . Affinity can be measured by common methods known in the art including those described herein. Specific illustrative and exemplary embodiments for measuring binding affinity are described in the following.

An affinity matured antibody refers to an antibody with one or more alterations in one or more hypervariable regions HVRs compared to a parent antibody which does not possess such alterations such alterations resulting in an improvement in the affinity of the antibody for antigen.

An agonist antibody as used herein is an antibody which mimics at least one of the functional activities of a polypeptide of interest.

An antagonist antibody or a blocking antibody is an antibody which inhibits or reduces biological activity of the antigen to which it specifically binds. Certain blocking antibodies or antagonist antibodies substantially or completely inhibit the biological activity of the antigen.

The term antibody herein is used in the broadest sense and encompasses various antibody structures including but not limited to monoclonal antibodies polyclonal antibodies multispecific antibodies e.g. bispecific antibodies and antibody fragments so long as they exhibit the desired antigen binding activity.

An antibody fragment refers to a molecule other than an intact antibody that comprises a portion of an intact antibody that binds the antigen to which the intact antibody binds. Examples of antibody fragments include but are not limited to Fv Fab Fab Fab SH F ab diabodies linear antibodies single chain antibody molecules e.g. scFv and multispecific antibodies formed from antibody fragments.

An antibody that binds to the same epitope as a reference antibody refers to an antibody that blocks binding of the reference antibody to its antigen in a competition assay by 50 or more and conversely the reference antibody blocks binding of the antibody to its antigen in a competition assay by 50 or more. An exemplary competition assay is provided herein.

The terms anti linear linked polyubiquitin antibody and an antibody that binds to linear linked polyubiquitin refer to an antibody that is capable of binding linear linked polyubiquitin with sufficient affinity such that the antibody is useful as a diagnostic and or therapeutic agent in targeting linear linked polyubiquitin. In one embodiment the extent of binding of an anti linear linked polyubiquitin antibody to an unrelated non linear linked polyubiquitin protein is less than about 10 of the binding of the antibody to linear linked polyubiquitin as measured e.g. by a radioimmunoassay RIA . In certain embodiments an antibody that binds to linear linked polyubiquitin has a dissociation constant Kd of 1 M 100 nM 10 nM 1 nM 0.1 nM 0.01 nM or 0.001 nM e.g. 10M or less e.g. from 10M to 10M e.g. from 10M to 10M . In certain embodiments an anti linear linked polyubiquitin antibody binds to an epitope of linear linked polyubiquitin that is conserved among linear linked polyubiquitin from different species.

As used herein the term anti polyubiquitin antibody refers to an antibody that is capable of specifically binding to a polyubiquitin molecule.

As used herein the terms anti ubiquitin antibody and anti monoubiquitin antibody are used interchangeably and refer to an antibody that is capable of specifically binding to a ubiquitin molecule.

The term chimeric antibody refers to an antibody in which a portion of the heavy and or light chain is derived from a particular source or species while the remainder of the heavy and or light chain is derived from a different source or species.

The class of an antibody refers to the type of constant domain or constant region possessed by its heavy chain. There are five major classes of antibodies IgA IgD IgE IgG and IgM and several of these may be further divided into subclasses isotypes e.g. IgG IgG IgG IgG IgA and IgA. The heavy chain constant domains that correspond to the different classes of immunoglobulins are called and respectively.

The term cytotoxic agent as used herein refers to a substance that inhibits or prevents a cellular function and or causes cell death or destruction. Cytotoxic agents include but are not limited to radioactive isotopes e.g. At I I Y Re Re Sm Bi P Pband radioactive isotopes of Lu chemotherapeutic agents or drugs e.g. methotrexate adriamicin vinca alkaloids vincristine vinblastine etoposide doxorubicin melphalan mitomycin C chlorambucil daunorubicin or other intercalating agents growth inhibitory agents enzymes and fragments thereof such as nucleolytic enzymes antibiotics toxins such as small molecule toxins or enzymatically active toxins of bacterial fungal plant or animal origin including fragments and or variants thereof and the various antitumor or anticancer agents disclosed below.

A disorder is any condition that would benefit from treatment with an antibody of the invention. This includes chronic and acute disorders or diseases including those pathological conditions which predispose the mammal to the disorder in question. Non limiting examples of disorders to be treated herein include cancer and hypotrophy disorders including but not limited to degenerative muscle disorders and degenerative nerve disorders.

 Effector functions refer to those biological activities attributable to the Fc region of an antibody which vary with the antibody isotype. Examples of antibody effector functions include C1q binding and complement dependent cytotoxicity CDC Fc receptor binding antibody dependent cell mediated cytotoxicity ADCC phagocytosis down regulation of cell surface receptors e.g. B cell receptor and B cell activation.

An effective amount of an agent e.g. a pharmaceutical formulation refers to an amount effective at dosages and for periods of time necessary to achieve the desired therapeutic or prophylactic result.

The term Fc region herein is used to define a C terminal region of an immunoglobulin heavy chain that contains at least a portion of the constant region. The term includes native sequence Fc regions and variant Fc regions. In one embodiment a human IgG heavy chain Fc region extends from Cys226 or from Pro230 to the carboxyl terminus of the heavy chain. However the C terminal lysine Lys447 of the Fc region may or may not be present. Unless otherwise specified herein numbering of amino acid residues in the Fc region or constant region is according to the EU numbering system also called the EU index as described in Kabat et al. 5th Ed. Public Health Service National Institutes of Health Bethesda Md. 1991.

 Framework or FR refers to variable domain residues other than hypervariable region HVR residues. The FR of a variable domain generally consists of four FR domains FR1 FR2 FR3 and FR4. Accordingly the HVR and FR sequences generally appear in the following sequence in VH or VL FR1 H1 L1 FR2 H2 L2 FR3 H3 L3 FR4.

The terms full length antibody intact antibody and whole antibody are used herein interchangeably to refer to an antibody having a structure substantially similar to a native antibody structure or having heavy chains that contain an Fc region as defined herein.

The terms host cell host cell line and host cell culture are used interchangeably and refer to cells into which exogenous nucleic acid has been introduced including the progeny of such cells. Host cells include transformants and transformed cells which include the primary transformed cell and progeny derived therefrom without regard to the number of passages. Progeny may not be completely identical in nucleic acid content to a parent cell but may contain mutations. Mutant progeny that have the same function or biological activity as screened or selected for in the originally transformed cell are included herein.

A human antibody is one which possesses an amino acid sequence which corresponds to that of an antibody produced by a human or a human cell or derived from a non human source that utilizes human antibody repertoires or other human antibody encoding sequences. This definition of a human antibody specifically excludes a humanized antibody comprising non human antigen binding residues.

A human consensus framework is a framework which represents the most commonly occurring amino acid residues in a selection of human immunoglobulin VL or VH framework sequences. Generally the selection of human immunoglobulin VL or VH sequences is from a subgroup of variable domain sequences. Generally the subgroup of sequences is a subgroup as in Kabat et al. Fifth Edition NIH Publication 91 3242 Bethesda Md. 1991 vols. 1 3. In one embodiment for the VL the subgroup is subgroup kappa I as in Kabat et al. supra. In one embodiment for the VH the subgroup is subgroup III as in Kabat et al. supra.

A humanized antibody refers to a chimeric antibody comprising amino acid residues from non human HVRs and amino acid residues from human FRs. In certain embodiments a humanized antibody will comprise substantially all of at least one and typically two variable domains in which all or substantially all of the HVRs e.g. CDRs correspond to those of a non human antibody and all or substantially all of the FRs correspond to those of a human antibody. A humanized antibody optionally may comprise at least a portion of an antibody constant region derived from a human antibody. A humanized form of an antibody e.g. a non human antibody refers to an antibody that has undergone humanization.

The term hypervariable region or HVR as used herein refers to each of the regions of an antibody variable domain which are hypervariable in sequence and or form structurally defined loops hypervariable loops . Generally native four chain antibodies comprise six HVRs three in the VH H1 H2 H3 and three in the VL L1 L2 L3 . HVRs generally comprise amino acid residues from the hypervariable loops and or from the complementarity determining regions CDRs the latter being of highest sequence variability and or involved in antigen recognition. Exemplary hypervariable loops occur at amino acid residues 26 32 L1 50 52 L2 91 96 L3 26 32 H1 53 55 H2 and 96 101 H3 . Chothia and Lesk 196 901 917 1987 . Exemplary CDRs CDR L1 CDR L2 CDR L3 CDR H1 CDR H2 and CDR H3 occur at amino acid residues 24 34 of L1 50 56 of L2 89 97 of L3 31 35B of H1 50 65 of H2 and 95 102 of H3. Kabat et al. 5th Ed. Public Health Service National Institutes of Health Bethesda Md. 1991 . With the exception of CDR1 in VH CDRs generally comprise the amino acid residues that form the hypervariable loops. CDRs also comprise specificity determining residues or SDRs which are residues that contact antigen. SDRs are contained within regions of the CDRs called abbreviated CDRs or a CDRs. Exemplary a CDRs a CDR L1 a CDR L2 a CDR L3 a CDR H1 a CDR H2 and a CDR H3 occur at amino acid residues 31 34 of L1 50 55 of L2 89 96 of L3 31 35B of H1 50 58 of H2 and 95 102 of H3. See Almagro and Fransson 13 1619 1633 2008 . Unless otherwise indicated HVR residues and other residues in the variable domain e.g. FR residues are numbered herein according to Kabat et al. supra.

An immunoconjugate is an antibody conjugated to one or more heterologous molecule s including but not limited to a cytotoxic agent.

An individual or subject is a mammal. Mammals include but are not limited to domesticated animals e.g. cows sheep cats dogs and horses primates e.g. humans and non human primates such as monkeys rabbits and rodents e.g. mice and rats . In certain embodiments the individual or subject is a human.

An isolated antibody is one which has been separated from a component of its natural environment. In some embodiments an antibody is purified to greater than 95 or 99 purity as determined by for example electrophoretic e.g. SDS PAGE isoelectric focusing IEF capillary electrophoresis or chromatographic e.g. ion exchange or reverse phase HPLC . For review of methods for assessment of antibody purity see e.g. Flatman et al. 848 79 87 2007 .

An isolated nucleic acid refers to a nucleic acid molecule that has been separated from a component of its natural environment. An isolated nucleic acid includes a nucleic acid molecule contained in cells that ordinarily contain the nucleic acid molecule but the nucleic acid molecule is present extrachromosomally or at a chromosomal location that is different from its natural chromosomal location.

 Isolated nucleic acid encoding an anti linear linked polyubiquitin antibody refers to one or more nucleic acid molecules encoding antibody heavy and light chains or fragments thereof including such nucleic acid molecule s in a single vector or separate vectors and such nucleic acid molecule s present at one or more locations in a host cell.

As used herein the terms linear linked polyubiquitin and C terminal to N terminal linked polyubiquitin are interchangeable and refer to a polyubiquitin molecule comprising at least one isopeptide bond between the C terminus e.g. the C terminal glycine of one ubiquitin moiety and the N terminal amino group e.g. the N terminal methionine of another ubiquitin moiety.

As used herein lysine linkage indicates a linkage between one ubiquitin moiety and another ubiquitin moiety which involves a lysine residue e.g. K6 K11 K27 K29 K33 K48 and or K63 .

The term monoclonal antibody as used herein refers to an antibody obtained from a population of substantially homogeneous antibodies i.e. the individual antibodies comprising the population are identical and or bind the same epitope except for possible variant antibodies e.g. containing naturally occurring mutations or arising during production of a monoclonal antibody preparation such variants generally being present in minor amounts. In contrast to polyclonal antibody preparations which typically include different antibodies directed against different determinants epitopes each monoclonal antibody of a monoclonal antibody preparation is directed against a single determinant on an antigen. Thus the modifier monoclonal indicates the character of the antibody as being obtained from a substantially homogeneous population of antibodies and is not to be construed as requiring production of the antibody by any particular method. For example the monoclonal antibodies to be used in accordance with the present invention may be made by a variety of techniques including but not limited to the hybridoma method recombinant DNA methods phage display methods and methods utilizing transgenic animals containing all or part of the human immunoglobulin loci such methods and other exemplary methods for making monoclonal antibodies being described herein.

A naked antibody refers to an antibody that is not conjugated to a heterologous moiety e.g. a cytotoxic moiety or radiolabel. The naked antibody may be present in a pharmaceutical formulation.

 Native antibodies refer to naturally occurring immunoglobulin molecules with varying structures. For example native IgG antibodies are heterotetrameric glycoproteins of about 150 000 daltons composed of two identical light chains and two identical heavy chains that are disulfide bonded. From N to C terminus each heavy chain has a variable region VH also called a variable heavy domain or a heavy chain variable domain followed by three constant domains CH1 CH2 and CH3 . Similarly from N to C terminus each light chain has a variable region VL also called a variable light domain or a light chain variable domain followed by a constant light CL domain. The light chain of an antibody may be assigned to one of two types called kappa and lambda based on the amino acid sequence of its constant domain.

The term package insert is used to refer to instructions customarily included in commercial packages of therapeutic products that contain information about the indications usage dosage administration combination therapy contraindications and or warnings concerning the use of such therapeutic products.

 Percent amino acid sequence identity with respect to a reference polypeptide sequence is defined as the percentage of amino acid residues in a candidate sequence that are identical with the amino acid residues in the reference polypeptide sequence after aligning the sequences and introducing gaps if necessary to achieve the maximum percent sequence identity and not considering any conservative substitutions as part of the sequence identity. Alignment for purposes of determining percent amino acid sequence identity can be achieved in various ways that are within the skill in the art for instance using publicly available computer software such as BLAST BLAST 2 ALIGN or Megalign DNASTAR software. Those skilled in the art can determine appropriate parameters for aligning sequences including any algorithms needed to achieve maximal alignment over the full length of the sequences being compared. For purposes herein however amino acid sequence identity values are generated using the sequence comparison computer program ALIGN 2. The ALIGN 2 sequence comparison computer program was authored by Genentech Inc. and the source code has been filed with user documentation in the U.S. Copyright Office Washington D.C. 20559 where it is registered under U.S. Copyright Registration No. TXU510087. The ALIGN 2 program is publicly available from Genentech Inc. South San Francisco Calif. or may be compiled from the source code. The ALIGN 2 program should be compiled for use on a UNIX operating system including digital UNIX V4.0D. All sequence comparison parameters are set by the ALIGN 2 program and do not vary.

In situations where ALIGN 2 is employed for amino acid sequence comparisons the amino acid sequence identity of a given amino acid sequence A to with or against a given amino acid sequence B which can alternatively be phrased as a given amino acid sequence A that has or comprises a certain amino acid sequence identity to with or against a given amino acid sequence B is calculated as follows 100 times the fraction X Y where X is the number of amino acid residues scored as identical matches by the sequence alignment program ALIGN 2 in that program s alignment of A and B and where Y is the total number of amino acid residues in B. It will be appreciated that where the length of amino acid sequence A is not equal to the length of amino acid sequence B the amino acid sequence identity of A to B will not equal the amino acid sequence identity of B to A. Unless specifically stated otherwise all amino acid sequence identity values used herein are obtained as described in the immediately preceding paragraph using the ALIGN 2 computer program.

The term pharmaceutical formulation refers to a preparation which is in such form as to permit the biological activity of an active ingredient contained therein to be effective and which contains no additional components which are unacceptably toxic to a subject to which the formulation would be administered.

A pharmaceutically acceptable carrier refers to an ingredient in a pharmaceutical formulation other than an active ingredient which is nontoxic to a subject. A pharmaceutically acceptable carrier includes but is not limited to a buffer excipient stabilizer or preservative.

As used herein the term polyubiquitin is defined as all species of native human and synthetic polymeric chains of ubiquitin which fall within human and synthetic classes of different polymeric linkages of ubiquitin including but not limited to linear polyubiquitin K6 linked polyubiquitin K11 linked polyubiquitin K27 linked polyubiquitin K29 linked polyubiquitin K33 linked polyubiquitin K48 linked polyubiquitin and K63 linked polyubiquitin. Polyubiquitin may be of any length and includes at least two ubiquitin moieties.

As used herein treatment and grammatical variations thereof such as treat or treating refers to clinical intervention in an attempt to alter the natural course of the individual being treated and can be performed either for prophylaxis or during the course of clinical pathology. Desirable effects of treatment include but are not limited to preventing occurrence or recurrence of disease alleviation of symptoms diminishment of any direct or indirect pathological consequences of the disease preventing metastasis decreasing the rate of disease progression amelioration or palliation of the disease state and remission or improved prognosis. In some embodiments antibodies of the invention are used to delay development of a disease or to slow the progression of a disease.

As used herein the terms ubiquitin and monoubiquitin are used interchangeably and refer to any native ubiquitin from any vertebrate source including mammals such as primates e.g. humans and rodents e.g. mice and rats unless otherwise indicated. The term encompasses full length unprocessed ubiquitin as well as any shortened or posttranslationally modified form of ubiquitin that results from processing in the cell excepting molecules comprised of multiple ubiquitin moieties. The term also encompasses naturally occurring variants of ubiquitin e.g. splice variants or allelic variants. The amino acid sequence of an exemplary human ubiquitin is shown in SEQ ID NO 97 MQIFVKTLTGKTITLEVEPSDTIENVKAKIQDKEGIPPDQQRLIFA GKQLEDGRTLSDYNIQKESTLHLVLRLRGG SEQ ID NO 97 . Ubiquitin has at least one lysine residue at amino acid 6 amino acid 11 amino acid 27 amino acid 29 amino acid 33 amino acid 48 and or amino acid 63 marked in bold in SEQ ID NO 97 above .

As used herein the term ubiquitin pathway refers to a biochemical pathway in a cell or reconstituted in vitro that includes ubiquitin and or polyubiquitin.

The term variable region or variable domain refers to the domain of an antibody heavy or light chain that is involved in binding the antibody to antigen. The variable domains of the heavy chain and light chain VH and VL respectively of a native antibody generally have similar structures with each domain comprising four conserved framework regions FRs and three hypervariable regions HVRs . See e.g. Kindt et al. 6ed. W.H. Freeman and Co. page 91 2007 . A single VH or VL domain may be sufficient to confer antigen binding specificity. Furthermore antibodies that bind a particular antigen may be isolated using a VH or VL domain from an antibody that binds the antigen to screen a library of complementary VL or VH domains respectively. See e.g. Portolano et al. 150 880 887 1993 Clarkson et al. 352 624 628 1991 .

The term vector as used herein refers to a nucleic acid molecule capable of propagating another nucleic acid to which it is linked. The term includes the vector as a self replicating nucleic acid structure as well as the vector incorporated into the genome of a host cell into which it has been introduced. Certain vectors are capable of directing the expression of nucleic acids to which they are operatively linked. Such vectors are referred to herein as expression vectors. 

In one aspect the invention is based in part on the creation of antibodies that are capable of specifically recognizing a first polyubiquitin molecule containing a first polyubiquitin linkage but not specifically binding to a second polyubiquitin molecule containing a second polyubiquitin linkage. In certain embodiments antibodies that specifically bind to linear polyubiquitin or C terminal to N terminal linked polyubiquitin are provided. Antibodies of the invention are useful both in research and e.g. for the diagnosis or treatment e.g. of diseases and disorders relating to aberrant cell cycle progression.

The unique properties of the anti linear polyubiquitin antibodies of the invention make them particularly useful for distinguishing between different linked forms of polyubiquitin in a cellular system without resorting to cumbersome and expensive genetic manipulation or biophysical methods such as mass spectrometry. The anti linear linked polyubiquitin antibodies of the invention can be used to characterize the function s and activities of specific linear polyubiquitins both in vitro and in vivo. The anti linear linked polyubiquitin antibodies of the invention can also be used to determine the role of specific linear polyubiquitins in the development and pathogenesis of disease. The anti linear linked polyubiquitin antibodies of the invention can further be used to treat diseases in which one or more specific linear polyubiquitins are aberrantly regulated or aberrantly functioning without interfering with the normal activity of polyubiquitins for which the anti polyubiquitin antibodies are not specific.

The involvement of the ubiquitin system in the NF B pathway has been described. Karin et al. 5 749 759 2005 and Chen Z. J. 7 758 765 2005 . Stimulation of cells with proinflammatory cytokines has been shown to result in K63 linked polyubiquitination of RIP1 and NEMO proteins which result in activation of I B kinase IKK . Tokunaga et al. 11 123 132 2009 . IKK is then polyubiquitinated with K48 linked polyubiquitin and subsequently degraded leading to activation of NF B. Id. Recently it was also shown that LUBAC activates the NF B pathway but not the JNK pathway via linear polyubiquitination of NF B. Id. NF B also plays a role in cell proliferation and the cell cycle. Many cancers display aberrant activation of NF B and suppression of NF B suppresses cancer cell proliferation. Garg and Aggarwal 16 1053 68 2002 . The presence of linear linked polyubiquitin on proteins such as NF B plays an important role in modulation of NF B activity and cell cycle progression. Thus the antibodies and Fabs of the invention provide a useful therapeutic means for modulation of disorders and disease states in which cell cycle regulation is aberrant. In one embodiment the anti linear polyubiquitin antibodies of the invention are used to treat diseases and disorders where cell cycle progression is aberrantly upregulated resulting in too much cell division such as cancer. In another embodiment the anti linear linked polyubiquitin antibodies of the invention are used to treat diseases and disorders where cell cycle progression is aberrantly downregulated resulting in too little cell division and concomitant wasting or destruction of tissue. Examples of such diseases include but are not limited to degenerative muscle disorders and degenerative nerve disorders including but not limited to Charcot Marie Tooth syndrome poliomyelitis amyotrophic lateral sclerosis and Guillain Barre syndrome .

As used herein the terms cell proliferative disorder and proliferative disorder refer to disorders that are associated with some degree of abnormal cell proliferation. In one embodiment the cell proliferative disorder is cancer.

The terms cancer and cancerous refer to or describe the physiological condition in mammals that is typically characterized by unregulated cell growth proliferation. Examples of cancer include but are not limited to carcinoma lymphoma e.g. Hodgkin s and non Hodgkin s lymphoma blastoma sarcoma and leukemia. More particular examples of such cancers include squamous cell cancer small cell lung cancer non small cell lung cancer adenocarcinoma of the lung squamous carcinoma of the lung cancer of the peritoneum hepatocellular cancer gastrointestinal cancer appendiceal cancer pancreatic cancer glioblastoma cervical cancer ovarian cancer liver cancer bladder cancer hepatoma breast cancer colon cancer colorectal cancer endometrial or uterine carcinoma salivary gland carcinoma kidney cancer liver cancer prostate cancer vulval cancer thyroid cancer hepatic carcinoma leukemia and other lymphoproliferative disorders and various types of head and neck cancer.

The term tumor as used herein refers to all neoplastic cell growth and proliferation whether malignant or benign and all pre cancerous and cancerous cells and tissues. The terms cancer cancerous cell proliferative disorder proliferative disorder and tumor are not mutually exclusive as referred to herein.

The term degenerative muscle disorder refers to or describes the physiological condition in muscle containing animals that is typically characterized by deterioration or weakening of skeletal and or smooth muscle such that normal muscular function is reduced. Examples of degenerative muscular disorders include but are not limited to muscular dystrophy myotonic dystrophy myotonia congenita cachexia sarcopenia multiple sclerosis amyotrophic lateral sclerosis Isaac s syndrome stiff person syndrome familiar periodic paralyses myopathy myotonia rhabdomyolyses muscle atrophy and various types of muscle weakness and muscle rigidity.

The term degenerative nerve disorder refers to or describes the physiological condition in nerve containing animals that is typically characterized by deterioration of nervous tissue or deterioration of communication between cells in nervous tissue. Examples of degenerative nerve disorders include but are not limited to neurodegenerative diseases including but not limited to Lewy body disease postpoliomyelitis syndrome Shy Draeger syndrome olivopontocerebellar atrophy Parkinson s disease multiple system atrophy amyotrophic lateral sclerosis Guillian Barre syndrome Carcot Marie Tooth syndrome striatonigral degeneration and nervous cell tissue destruction caused by or associated with tauopathies prion diseases bulbar palsy motor neuron disease dementia and nervous system heterodegenerative disorders including but not limited to Canavan disease Huntington s disease neuronal ceroidlipofuscinosis Alexander s disease Tourette s syndrome Menkes kinky hair syndrome Cockayne syndrome Halervorden Spatz syndrome lafora disease Rett syndrome hepatolenticular degeneration Lesch Nyhan syndrome and Unverricht Lundborg syndrome .

In another aspect the anti linear linked polyubiquitin antibodies of the invention find utility as reagents for detection and isolation of linear linked polyubiquitin such as detection of polyubiquitin in various cell types and tissues including the determination of polyubiquitin density and distribution in cell populations and within a given cell and cell sorting based on the presence or amount of polyubiquitin. In yet another aspect the present anti linear linked polyubiquitin antibodies are useful for the development of polyubiquitin antagonists with blocking activity patterns similar to those of the subject antibodies of the invention. As a further example anti linear linked polyubiquitin antibodies of the invention can be used to identify other anti polyubiquitin antibodies that bind substantially the same antigenic determinant s of polyubiquitin as the antibodies exemplified herein including linear and conformational epitopes.

The anti linear linked polyubiquitin antibodies of the invention can be used in assays based on the physiological pathways in which polyubiquitin is involved to screen for small molecule antagonists of linear linked polyubiquitin function. For example since linear linked polyubiquitin chains are known to be necessary for NF B activation Tokunaga et al. 11 123 132 2009 the activity of anti linear linked polyubiquitin antibodies to modulate up or down regulate NF 13 activation in treated cells or tissues may be compared to the activity of one or more potential small molecule antagonists of linear linked polyubiquitin in modulating NF 13 activation.

In one aspect the invention provides isolated antibodies that bind to linear or C terminal to N terminal linked polyubiquitin. In certain embodiments an anti linear linked polyubiquitin antibody specifically binds to C terminal to N terminal linked polyubiquitin but does not specifically bind to monoubiquitin. In certain embodiments an anti linear linked polyubiquitin antibody specifically binds to C terminal to N terminal linked polyubiquitin but does not specifically bind to polyubiquitin having a lysine linkage i.e. K6 K11 K27 K29 K33 K48 and or K63 linkages .

In one aspect the invention provides an anti linear linked polyubiquitin antibody comprising an HVR H1 region comprising the sequence of at least one of SEQ ID NOs 7 10 13 16 22 and 73 81. In one aspect the invention provides an antibody comprising an HVR H2 region comprising the sequence of at least one of SEQ ID NOs 8 11 14 17 23 24 and 82 86. In one aspect the invention provides an antibody comprising an HVR H3 region comprising the sequence of at least one of SEQ ID NOs 9 12 15 18 and 87 93.

In one aspect the invention provides an antibody comprising an HVR H1 region comprising the sequence of at least one of SEQ ID NOs 7 10 13 16 22 and 73 81 and an HVR H2 region comprising the sequence of at least one of SEQ ID NOs 8 11 14 17 23 24 and 82 86. In one aspect the invention provides an antibody comprising an HVR H1 region comprising the sequence of at least one of SEQ ID NOs 7 10 13 16 22 and 73 81 and an HVR H3 region comprising the sequence of at least one of SEQ ID NOs 9 12 15 18 and 87 93. In one aspect the invention provides an antibody comprising an HVR H2 region comprising the sequence of at least one of SEQ ID NOs 8 11 14 17 23 24 and 81 85 and an HVR H3 region comprising the sequence of at least one of SEQ ID NOs 9 12 15 18 and 86 92.

In one aspect the invention provides an antibody comprising an HVR L1 region comprising the sequence of at least one of SEQ ID NOs 1 4 19 and 50 57. In one aspect the invention provides an antibody comprising an HVR L2 region comprising the sequence of at least one of SEQ ID NOs 2 and 58 62. In one aspect the invention provides an antibody comprising an HVR L3 region comprising the sequence of at least one of SEQ ID NOs 3 5 6 20 21 and 64 72.

In one aspect the invention provides an antibody comprising an HVR L1 region comprising the sequence of at least one of SEQ ID NOs 1 4 19 and 50 57 and an HVR L2 region comprising the sequence of at least one of SEQ ID NOs 2 and 58 63. In one aspect the invention provides an antibody comprising an HVR L1 region comprising the sequence of at least one of SEQ ID NOs 1 4 19 and 50 57 and an HVR L3 region comprising the sequence of at least one of SEQ ID NOs 3 5 6 20 21 and 64 72. In one aspect the invention provides an antibody comprising an HVR L2 region comprising the sequence of at least one of SEQ ID NOs 2 and 58 63 and an HVR L3 region comprising the sequence of at least one of SEQ ID NOs 3 5 6 20 21 and 64 72.

In one aspect the invention provides an antibody comprising at least one at least two at least three at least four at least five or all six of the following 

In one aspect the invention provides an antibody that specifically binds linear linked polyubiquitin with high affinity but binds polyubiquitin some other lysine linkage with substantially reduced affinity comprising at least one at least two at least three at least four at least five or all six of the following 

In one aspect the invention provides antibodies comprising heavy chain HVR sequences as depicted in or B. In one embodiment the antibodies comprise light chain HVR sequences as depicted in or A. In one embodiment the antibodies comprise heavy chain HVR sequences as depicted in or B and light chain HVR sequences as depicted in or A. In one embodiment the antibodies comprise light chain HVR sequences as depicted in Tables 5 and 7. In one embodiment the invention provides antibodies comprising heavy chain HVR sequences as depicted in Tables 5 and 7. In one embodiment the antibodies comprise heavy chain HVR sequences as depicted in Tables 5 and 7 and light chain HVR sequences as depicted in Tables 5 and 7.

Some embodiments of antibodies of the invention comprise a light chain variable domain of humanized 4D5 antibody huMAb4D5 8 HERCEPTIN Genentech Inc. South San Francisco Calif. USA also referred to in U.S. Pat. No. 6 407 213 and Lee et al. J. Mol. Biol. 2004 340 5 1073 93 as depicted in SEQ ID NO 98 below.

Antibodies of the invention can comprise any suitable framework variable domain sequence provided binding activity to linear linked polyubiquitin is substantially retained. For example in some embodiments antibodies of the invention comprise a human subgroup III heavy chain framework consensus sequence. In one embodiment of these antibodies the framework consensus sequence comprises substitution at position 71 73 78 and or 102. In some embodiments of these antibodies position 71 is A 73 is T 78 is A and or 102 is L. In one embodiment these antibodies comprise heavy chain variable domain framework sequences of huMAb4D5 8 HERCEPTIN Genentech Inc. South San Francisco Calif. USA also referred to in U.S. Pat. Nos. 6 407 213 5 821 337 and Lee et al. . 2004 340 5 1073 93 . In one embodiment these antibodies further comprise a human id light chain framework consensus sequence. In one embodiment these antibodies comprise at least one two or all of the light chain HVR sequences of SEQ ID NOs 1 6 19 21 and 50 72. In one embodiment these antibodies comprise light chain HVR sequences of huMAb4D5 8 as described in U.S. Pat. Nos. 6 407 213 5 821 337. In one embodiment these antibodies comprise light chain variable domain sequences of huMAb4D5 8 SEQ ID NO 98 and 99 HERCEPTIN Genentech Inc. South San Francisco Calif. USA also referred to in U.S. Pat. Nos. 6 407 213 5 821 337 and Lee et al. . 2004 340 5 1073 93 .

In one embodiment an antibody of the invention is affinity matured to obtain the target binding affinity desired. In one example an affinity matured antibody of the invention which specifically binds to linear linked polyubiquitin with high affinity but binds to polyubiquitin having a lysine linkage with substantially reduced affinity comprises substitution at HVR H1 amino acid position 32. In another example an affinity matured antibody of the invention which specifically binds to linear linked polyubiquitin having a lysine linkage with substantially reduced affinity comprises substitution at HVR H2 amino acid positions 50 54 and or 56. In another example an affinity matured antibody of the invention which specifically binds to linear linked polyubiquitin having a lysine linkage with substantially reduced affinity comprises substitution at HVR H3 amino acid position 103. In another example an affinity matured antibody of the invention which specifically binds to linear linked polyubiquitin with high affinity but binds to polyubiquitin having other lysine linkages with substantially reduced affinity comprises substitution at HVR L1 amino acid positions 28 30 31 and or 32. In another example an affinity matured antibody of the invention which specifically binds to linear linked polyubiquitin with high affinity but binds to polyubiquitin having other lysine linkages with substantially reduced affinity comprises substitution at HVR L3 amino acid positions 92 93 and or 94. In another example an affinity matured antibody of the invention which specifically binds to linear linked polyubiquitin with high affinity but binds to polyubiquitin having other lysine linkages with substantially reduced affinity comprises substitution at HVR L2 amino acid position 52.

In one embodiment an antibody of the invention comprises at least one heavy chain variable domain sequence of SEQ ID NOs 29 32 36 38 95 and 196 198. In one embodiment an antibody of the invention comprises at least one light chain variable domain of SEQ ID NOs 25 28 33 35 94 and 193 195. In one embodiment an antibody of the invention comprises a heavy chain variable domain comprising at least one sequence of SEQ ID NOs 29 32 36 38 95 and 196 198 and also comprises a light chain variable domain comprising at least one sequence of SEQ ID NOs 25 28 33 35 94 and 193 195. In other embodiments an antibody of the invention corresponding to a particular clone number comprises a heavy chain variable domain comprising an HVR H1 HVR H2 and HVR H3 sequence as set forth in for that clone number. In other embodiments an antibody of the invention corresponding to a particular clone number comprises a light chain variable domain comprising an HVR L1 HVR L2 and HVR L3 sequence as set forth in for that clone number. In other embodiments an antibody of the invention corresponding to a particular clone number comprises a heavy chain variable domain comprising an HVR H1 HVR H2 and HVR H3 sequence as set forth in for that clone number and also comprises a light chain variable domain comprising an HVR L1 HVR L2 and HVR L3 sequence as set forth in for that clone number.

In one aspect the invention provides an antibody that competes with any of the above mentioned antibodies for binding to linear linked polyubiquitin. In one aspect the invention provides an antibody that binds to the same antigenic determinant on linear linked polyubiquitin as any of the above mentioned antibodies.

As shown herein the antibodies of the invention specifically bind to an isolated polyubiquitin having a C terminal to N terminal linkage. As shown herein the antibodies of the invention also specifically bind to polyubiquitin having a linear C terminal to N terminal linkage when that polyubiquitin is attached to a heterologous protein.

In any of the above embodiments an anti linear linked polyubiquitin antibody is humanized. In one embodiment an anti linear linked polyubiquitin antibody comprises HVRs as in any of the above embodiments and further comprises an acceptor human framework e.g. a human immunoglobulin framework or a human consensus framework. In another embodiment an anti linear linked polyubiquitin antibody comprises HVRs as in any of the above embodiments and further comprises a VH comprising an FR1 FR2 FR3 or FR4 sequence of any of SEQ ID NOs 29 32 36 38 and 95. In another embodiment an anti linear linked polyubiquitin antibody comprises HVRs as in any of the above embodiments and further comprises a VL comprising an FR1 FR2 FR3 or FR4 sequence of any of SEQ ID NOs 25 28 33 35 94 and 193 195.

In another aspect an anti linear linked polyubiquitin antibody comprises a heavy chain variable domain VH sequence having at least 90 91 92 93 94 95 96 97 98 99 or 100 sequence identity to the amino acid sequence of any of SEQ ID NOs 29 32 26 38 95 and 196 198. In certain embodiments a VH sequence having at least 90 91 92 93 94 95 96 97 98 or 99 identity contains substitutions e.g. conservative substitutions insertions or deletions relative to the reference sequence but an anti linear linked polyubiquitin antibody comprising that sequence retains the ability to bind to linear linked polyubiquitin. In certain embodiments a total of 1 to 10 amino acids have been substituted inserted and or deleted in any one of SEQ ID NOs 29 32 36 38 95 and 196 198. In certain embodiments substitutions insertions or deletions occur in regions outside the HVRs i.e. in the FRs . Optionally the anti linear linked polyubiquitin antibody comprises the VH sequence of any of SEQ ID NOs 29 32 36 38 95 and 196 198 including post translational modifications of that sequence.

In another aspect an anti linear linked polyubiquitin antibody is provided wherein the antibody comprises a light chain variable domain VL having at least 90 91 92 93 94 95 96 97 98 99 or 100 sequence identity to the amino acid sequence of any of SEQ ID NOs 25 28 33 35 94 and 193 195. In certain embodiments a VL sequence having at least 90 91 92 93 94 95 96 97 98 or 99 identity contains substitutions e.g. conservative substitutions insertions or deletions relative to the reference sequence but an anti linear linked polyubiquitin antibody comprising that sequence retains the ability to bind to linear linked polyubiquitin. In certain embodiments a total of 1 to 10 amino acids have been substituted inserted and or deleted in any of SEQ ID NOs 25 28 33 35 94 and 193 195. In certain embodiments the substitutions insertions or deletions occur in regions outside the HVRs i.e. in the FRs . Optionally the anti linear linked polyubiquitin antibody comprises the VL sequence in any of SEQ ID NOs 25 28 33 35 94 and 193 195 including post translational modifications of that sequence.

In another aspect an anti linear linked polyubiquitin antibody is provided wherein the antibody comprises a VH as in any of the embodiments provided above and a VL as in any of the embodiments provided above. In one embodiment the antibody comprises the VH and VL sequences in any of SEQ ID NOs 29 32 36 38 95 and 196 198 and SEQ ID NOs 25 28 33 35 94 and 193 195 respectively including post translational modifications of those sequences.

In a further aspect of the invention an anti linear linked polyubiquitin antibody according to any of the above embodiments is a monoclonal antibody including a chimeric humanized or human antibody. In one embodiment an anti linear linked polyubiquitin antibody is an antibody fragment e.g. a Fv Fab Fab scFv diabody or F ab fragment. In another embodiment the antibody is a full length antibody e.g. an intact IgG1 antibody or other antibody class or isotype as defined herein.

Compositions comprising at least one anti linear linked polyubiquitin antibody or at least one polynucleotide comprising sequences encoding an anti linear linked polyubiquitin antibody are provided. In certain embodiments a composition may be a pharmaceutical composition. As used herein compositions comprise one or more antibodies that bind to one or more polyubiquitin and or one or more polynucleotides comprising sequences encoding one or more antibodies that bind to one or more polyubiquitin. These compositions may further comprise suitable carriers such as pharmaceutically acceptable excipients including buffers which are well known in the art.

In a further aspect an anti linear linked polyubiquitin antibody according to any of the above embodiments may incorporate any of the features singly or in combination as described in Sections 1 7 below 

In certain embodiments an antibody provided herein has a dissociation constant Kd of 100 nM 10 nM 1 nM 0.1 nM 0.01 nM or 0.001 nM e.g. 10M or less e.g. from 10M to 10M e.g. from 10M to 10M .

In one embodiment Kd is measured by a radiolabeled antigen binding assay RIA performed with the Fab version of an antibody of interest and its antigen as described by the following assay. Solution binding affinity of Fabs for antigen is measured by equilibrating Fab with a minimal concentration of I labeled antigen in the presence of a titration series of unlabeled antigen then capturing bound antigen with an anti Fab antibody coated plate see e.g. Chen et al. 293 865 881 1999 . To establish conditions for the assay MICROTITER multi well plates Thermo Scientific are coated overnight with 5 g ml of a capturing anti Fab antibody Cappel Labs in 50 mM sodium carbonate pH 9.6 and subsequently blocked with 2 w v bovine serum albumin in PBS for two to five hours at room temperature approximately 23 C. . In a non adsorbent plate Nunc 269620 100 M or 26 M I antigen are mixed with serial dilutions of a Fab of interest e.g. consistent with assessment of the anti VEGF antibody Fab 12 in Presta et al. 57 4593 4599 1997 . The Fab of interest is then incubated overnight however the incubation may continue for a longer period e.g. about 65 hours to ensure that equilibrium is reached. Thereafter the mixtures are transferred to the capture plate for incubation at room temperature e.g. for one hour . The solution is then removed and the plate washed eight times with 0.1 polysorbate 20 TWEEN 20 in PBS. When the plates have dried 150 l well of scintillant MICROSCINT 20 Packard is added and the plates are counted on a TOPCOUNT gamma counter Packard for ten minutes. Concentrations of each Fab that give less than or equal to 20 of maximal binding are chosen for use in competitive binding assays.

According to another embodiment Kd is measured using surface plasmon resonance assays using a BIACORE 2000 or a BIACORE 3000 BIAcore Inc. Piscataway N.J. at 25 C. with e.g. immobilized antigen CM5 chips at 10 response units RU . Briefly carboxymethylated dextran biosensor chips CM5 BIACORE Inc. are activated with N ethyl N 3 dimethylaminopropyl carbodiimide hydrochloride EDC and N hydroxysuccinimide NHS according to the supplier s instructions. Antigen is diluted with 10 mM sodium acetate pH 4.8 to 5 g ml 0.2 M before injection at a flow rate of 5 l minute to achieve approximately 10 response units RU of coupled protein. Following the injection of antigen 1 M ethanolamine is injected to block unreacted groups. For kinetics measurements two fold serial dilutions of Fab 0.78 nM to 500 nM are injected in PBS with 0.05 polysorbate 20 TWEEN 20 surfactant PBST at 25 C. at a flow rate of approximately 25 l min. Association rates k and dissociation rates k are calculated using a simple one to one Langmuir binding model BIACORE Evaluation Software version 3.2 by simultaneously fitting the association and dissociation sensorgrams. The equilibrium dissociation constant Kd is calculated as the ratio k kSee e.g. Chen et al. 293 865 881 1999 . If the on rate exceeds 10Msby the surface plasmon resonance assay above then the on rate can be determined by using a fluorescent quenching technique that measures the increase or decrease in fluorescence emission intensity excitation 295 nm emission 340 nm 16 nm band pass at 25 C. of a 20 nM anti antigen antibody Fab form in PBS pH 7.2 in the presence of increasing concentrations of antigen as measured in a spectrometer such as a stop flow equipped spectrophometer Aviv Instruments or a 8000 series SLM AMINCO spectrophotometer ThermoSpectronic with a stirred cuvette. Other coupling chemistries for the target antigen to the chip surface e.g. streptavidin biotin hydrophobic interaction or disulfide chemistry are also readily available instead of the amine coupling methodology CM5 chip described above as will be understood by one of ordinary skill in the art.

In certain embodiments an antibody provided herein is an antibody fragment. Antibody fragments include but are not limited to Fab Fab Fab SH F ab Fv and scFv fragments and other fragments described below. For a review of certain antibody fragments see Hudson et al. 9 129 134 2003 . For a review of scFv fragments see e.g. Pluckth n in vol. 113 Rosenburg and Moore eds. Springer Verlag New York pp. 269 315 1994 see also WO 93 16185 and U.S. Pat. Nos. 5 571 894 and 5 587 458. For discussion of Fab and F ab fragments comprising salvage receptor binding epitope residues and having increased in vivo half life see U.S. Pat. No. 5 869 046.

Diabodies are antibody fragments with two antigen binding sites that may be bivalent or bispecific. See for example EP 404 097 WO 1993 01161 Hudson et al. 9 129 134 2003 and Hollinger et al. 90 6444 6448 1993 . Triabodies and tetrabodies are also described in Hudson et al. 9 129 134 2003 .

Single domain antibodies are antibody fragments comprising all or a portion of the heavy chain variable domain or all or a portion of the light chain variable domain of an antibody. In certain embodiments a single domain antibody is a human single domain antibody Domantis Inc. Waltham Mass. see e.g. U.S. Pat. No. 6 248 516 B1 .

Antibody fragments can be made by various techniques including but not limited to proteolytic digestion of an intact antibody as well as production by recombinant host cells e.g. or phage as described herein.

In certain embodiments an antibody provided herein is a chimeric antibody. Certain chimeric antibodies are described e.g. in U.S. Pat. No. 4 816 567 and Morrison et al. 81 6851 6855 1984 . In one example a chimeric antibody comprises a non human variable region e.g. a variable region derived from a mouse rat hamster rabbit or non human primate such as a monkey and a human constant region. In a further example a chimeric antibody is a class switched antibody in which the class or subclass has been changed from that of the parent antibody. Chimeric antibodies include antigen binding fragments thereof.

In certain embodiments a chimeric antibody is a humanized antibody. Typically a non human antibody is humanized to reduce immunogenicity to humans while retaining the specificity and affinity of the parental non human antibody. Generally a humanized antibody comprises one or more variable domains in which HVRs e.g. CDRs or portions thereof are derived from a non human antibody and FRs or portions thereof are derived from human antibody sequences. A humanized antibody optionally will also comprise at least a portion of a human constant region. In some embodiments some FR residues in a humanized antibody are substituted with corresponding residues from a non human antibody e.g. the antibody from which the HVR residues are derived e.g. to restore or improve antibody specificity or affinity.

Humanized antibodies and methods of making them are reviewed e.g. in Almagro and Fransson 13 1619 1633 2008 and are further described e.g. in Riechmann et al. 332 323 329 1988 Queen et al. 86 10029 10033 1989 U.S. Pat. Nos. 5 821 337 7 527 791 6 982 321 and 7 087 409 Kashmiri et al. 36 25 34 2005 describing SDR a CDR grafting Padlan 28 489 498 1991 describing resurfacing Dall Acqua et al. 36 43 60 2005 describing FR shuffling and Osbourn et al. 36 61 68 2005 and Klimka et al. 83 252 260 2000 describing the guided selection approach to FR shuffling .

Human framework regions that may be used for humanization include but are not limited to framework regions selected using the best fit method see e.g. Sims et al. 151 2296 1993 framework regions derived from the consensus sequence of human antibodies of a particular subgroup of light or heavy chain variable regions see e.g. Carter et al. 89 4285 1992 and Presta et al. 151 2623 1993 human mature somatically mutated framework regions or human germline framework regions see e.g. Almagro and Fransson 13 1619 1633 2008 and framework regions derived from screening FR libraries see e.g. Baca et al. 272 10678 10684 1997 and Rosok et al. 271 22611 22618 1996 .

In certain embodiments an antibody provided herein is a human antibody. Human antibodies can be produced using various techniques known in the art. Human antibodies are described generally in van Dijk and van de Winkel 5 368 74 2001 and Lonberg 20 450 459 2008 .

Human antibodies may be prepared by administering an immunogen to a transgenic animal that has been modified to produce intact human antibodies or intact antibodies with human variable regions in response to antigenic challenge. Such animals typically contain all or a portion of the human immunoglobulin loci which replace the endogenous immunoglobulin loci or which are present extrachromosomally or integrated randomly into the animal s chromosomes. In such transgenic mice the endogenous immunoglobulin loci have generally been inactivated. For review of methods for obtaining human antibodies from transgenic animals see Lonberg 23 1117 1125 2005 . See also e.g. U.S. Pat. Nos. 6 075 181 and 6 150 584 describing XENOMOUSE technology U.S. Pat. No. 5 770 429 describing HM technology U.S. Pat. No. 7 041 870 describing K M MOUSE technology and U.S. Patent Application Publication No. US 2007 0061900 describing VM technology . Human variable regions from intact antibodies generated by such animals may be further modified e.g. by combining with a different human constant region.

Human antibodies can also be made by hybridoma based methods. Human myeloma and mouse human heteromyeloma cell lines for the production of human monoclonal antibodies have been described. See e.g. Kozbor 133 3001 1984 Brodeur et al. pp. 51 63 Marcel Dekker Inc. New York 1987 and Boerner et al. 147 86 1991 . Human antibodies generated via human B cell hybridoma technology are also described in Li et al. 103 3557 3562 2006 . Additional methods include those described for example in U.S. Pat. No. 7 189 826 describing production of monoclonal human IgM antibodies from hybridoma cell lines and Ni 26 4 265 268 2006 describing human human hybridomas . Human hybridoma technology Trioma technology is also described in Vollmers and Brandlein 20 3 927 937 2005 and Vollmers and Brandlein 27 3 185 91 2005 .

Human antibodies may also be generated by isolating Fv clone variable domain sequences selected from human derived phage display libraries. Such variable domain sequences may then be combined with a desired human constant domain. Techniques for selecting human antibodies from antibody libraries are described below.

Antibodies of the invention may be isolated by screening combinatorial libraries for antibodies with the desired activity or activities. For example a variety of methods are known in the art for generating phage display libraries and screening such libraries for antibodies possessing the desired binding characteristics. Such methods are reviewed e.g. in Hoogenboom et al. in 178 1 37 O Brien et al. ed. Human Press Totowa N.J. 2001 and further described e.g. in the McCafferty et al. 348 552 554 Clackson et al. 352 624 628 1991 Marks et al. 222 581 597 1992 Marks and Bradbury in 248 161 175 Lo ed. Human Press Totowa N.J. 2003 Sidhu et al. 338 2 299 310 2004 Lee et al. 340 5 1073 1093 2004 Fellouse 101 34 12467 12472 2004 and Lee et al. 284 1 2 119 132 2004 .

In certain phage display methods repertoires of VH and VL genes are separately cloned by polymerase chain reaction PCR and recombined randomly in phage libraries which can then be screened for antigen binding phage as described in Winter et al. 12 433 455 1994 . Phage typically display antibody fragments either as single chain Fv scFv fragments or as Fab fragments. Libraries from immunized sources provide high affinity antibodies to the immunogen without the requirement of constructing hybridomas. Alternatively the naive repertoire can be cloned e.g. from human to provide a single source of antibodies to a wide range of non self and also self antigens without any immunization as described by Griffiths et al. 12 725 734 1993 . Finally naive libraries can also be made synthetically by cloning unrearranged V gene segments from stem cells and using PCR primers containing random sequence to encode the highly variable CDR3 regions and to accomplish rearrangement in vitro as described by Hoogenboom and Winter 227 381 388 1992 . Patent publications describing human antibody phage libraries include for example U.S. Pat. No. 5 750 373 and US Patent Publication Nos. 2005 0079574 2005 0119455 2005 0266000 2007 0117126 2007 0160598 2007 0237764 2007 0292936 and 2009 0002360.

Antibodies or antibody fragments isolated from human antibody libraries are considered human antibodies or human antibody fragments herein.

In certain embodiments an antibody provided herein is a multispecific antibody e.g. a bispecific antibody. Multispecific antibodies are monoclonal antibodies that have binding specificities for at least two different sites. In certain embodiments one of the binding specificities is for linear linked polyubiquitin and the other is for any other antigen. In certain embodiments bispecific antibodies may bind to two different epitopes of linear linked polyubiquitin. Bispecific antibodies may also be used to localize cytotoxic agents to cells which express linear linked polyubiquitin. Bispecific antibodies can be prepared as full length antibodies or antibody fragments.

Techniques for making multispecific antibodies include but are not limited to recombinant co expression of two immunoglobulin heavy chain light chain pairs having different specificities see Milstein and Cuello 305 537 1983 WO 93 08829 and Traunecker et al. 10 3655 1991 and knob in hole engineering see e.g. U.S. Pat. No. 5 731 168 . Multi specific antibodies may also be made by engineering electrostatic steering effects for making antibody Fc heterodimeric molecules WO 2009 089004A1 cross linking two or more antibodies or fragments see e.g. U.S. Pat. No. 4 676 980 and Brennan et al. 229 81 1985 using leucine zippers to produce bi specific antibodies see e.g. Kostelny et al. 148 5 1547 1553 1992 using diabody technology for making bispecific antibody fragments see e.g. Hollinger et al. 90 6444 6448 1993 and using single chain Fv sFv dimers see e.g. Gruber et al. 152 5368 1994 and preparing trispecific antibodies as described e.g. in Tutt et al. 147 60 1991 .

Engineered antibodies with three or more functional antigen binding sites including Octopus antibodies are also included herein see e.g. US 2006 0025576A1 .

The antibody or fragment herein also includes a Dual Acting FAb or DAF comprising an antigen binding site that binds to linear linked polyubiquitin as well as another different antigen see US 2008 0069820 for example .

In certain embodiments amino acid sequence variants of the antibodies provided herein are contemplated. For example it may be desirable to improve the binding affinity and or other biological properties of the antibody. Amino acid sequence variants of an antibody may be prepared by introducing appropriate modifications into the nucleotide sequence encoding the antibody or by peptide synthesis. Such modifications include for example deletions from and or insertions into and or substitutions of residues within the amino acid sequences of the antibody. Any combination of deletion insertion and substitution can be made to arrive at the final construct provided that the final construct possesses the desired characteristics e.g. antigen binding.

In certain embodiments antibody variants having one or more amino acid substitutions are provided. Sites of interest for substitutional mutagenesis include the HVRs and FRs. Conservative substitutions are shown in Table 1 under the heading of conservative substitutions. More substantial changes are provided in Table 1 under the heading of exemplary substitutions and as further described below in reference to amino acid side chain classes. Amino acid substitutions may be introduced into an antibody of interest and the products screened for a desired activity e.g. retained improved antigen binding decreased immunogenicity or improved ADCC or CDC.

Non conservative substitutions will entail exchanging a member of one of these classes for another class.

One type of substitutional variant involves substituting one or more hypervariable region residues of a parent antibody e.g. a humanized or human antibody . Generally the resulting variant s selected for further study will have modifications e.g. improvements in certain biological properties e.g. increased affinity reduced immunogenicity relative to the parent antibody and or will have substantially retained certain biological properties of the parent antibody. An exemplary substitutional variant is an affinity matured antibody which may be conveniently generated e.g. using phage display based affinity maturation techniques such as those described herein. Briefly one or more HVR residues are mutated and the variant antibodies displayed on phage and screened for a particular biological activity e.g. binding affinity .

Alterations e.g. substitutions may be made in HVRs e.g. to improve antibody affinity. Such alterations may be made in HVR hotspots i.e. residues encoded by codons that undergo mutation at high frequency during the somatic maturation process see e.g. Chowdhury 207 179 196 2008 and or SDRs a CDRs with the resulting variant VH or VL being tested for binding affinity. Affinity maturation by constructing and reselecting from secondary libraries has been described e.g. in Hoogenboom et al. in 178 1 37 O Brien et al. ed. Human Press Totowa N.J. 2001 . In some embodiments of affinity maturation diversity is introduced into the variable genes chosen for maturation by any of a variety of methods e.g. error prone PCR chain shuffling or oligonucleotide directed mutagenesis . A secondary library is then created. The library is then screened to identify any antibody variants with the desired affinity. Another method to introduce diversity involves HVR directed approaches in which several HVR residues e.g. 4 6 residues at a time are randomized. HVR residues involved in antigen binding may be specifically identified e.g. using alanine scanning mutagenesis or modeling. CDR H3 and CDR L3 in particular are often targeted.

In certain embodiments substitutions insertions or deletions may occur within one or more HVRs so long as such alterations do not substantially reduce the ability of the antibody to bind antigen. For example conservative alterations e.g. conservative substitutions as provided herein that do not substantially reduce binding affinity may be made in HVRs. Such alterations may be outside of HVR hotspots or SDRs. In certain embodiments of the variant VH and VL sequences provided above each HVR either is unaltered or contains no more than one two or three amino acid substitutions.

A useful method for identification of residues or regions of an antibody that may be targeted for mutagenesis is called alanine scanning mutagenesis as described by Cunningham and Wells 1989 244 1081 1085. In this method a residue or group of target residues e.g. charged residues such as arg asp his lys and glu are identified and replaced by a neutral or negatively charged amino acid e.g. alanine or polyalanine to determine whether the interaction of the antibody with antigen is affected. Further substitutions may be introduced at the amino acid locations demonstrating functional sensitivity to the initial substitutions. Alternatively or additionally a crystal structure of an antigen antibody complex to identify contact points between the antibody and antigen. Such contact residues and neighboring residues may be targeted or eliminated as candidates for substitution. Variants may be screened to determine whether they contain the desired properties.

Amino acid sequence insertions include amino and or carboxyl terminal fusions ranging in length from one residue to polypeptides containing a hundred or more residues as well as intrasequence insertions of single or multiple amino acid residues. Examples of terminal insertions include an antibody with an N terminal methionyl residue. Other insertional variants of the antibody molecule include the fusion to the N or C terminus of the antibody to an enzyme e.g. for ADEPT or a polypeptide which increases the serum half life of the antibody.

In certain embodiments an antibody provided herein is altered to increase or decrease the extent to which the antibody is glycosylated. Addition or deletion of glycosylation sites to an antibody may be conveniently accomplished by altering the amino acid sequence such that one or more glycosylation sites is created or removed.

Where the antibody comprises an Fc region the carbohydrate attached thereto may be altered. Native antibodies produced by mammalian cells typically comprise a branched biantennary oligosaccharide that is generally attached by an N linkage to Asn297 of the CH2 domain of the Fc region. See e.g. Wright et al. 15 26 32 1997 . The oligosaccharide may include various carbohydrates e.g. mannose N acetyl glucosamine GlcNAc galactose and sialic acid as well as a fucose attached to a GlcNAc in the stem of the biantennary oligosaccharide structure. In some embodiments modifications of the oligosaccharide in an antibody of the invention may be made in order to create antibody variants with certain improved properties.

In one embodiment antibody variants are provided having a carbohydrate structure that lacks fucose attached directly or indirectly to an Fc region. For example the amount of fucose in such antibody may be from 1 to 80 from 1 to 65 from 5 to 65 or from 20 to 40 . The amount of fucose is determined by calculating the average amount of fucose within the sugar chain at Asn297 relative to the sum of all glycostructures attached to Asn 297 e.g. complex hybrid and high mannose structures as measured by MALDI TOF mass spectrometry as described in WO 2008 077546 for example. Asn297 refers to the asparagine residue located at about position 297 in the Fc region Eu numbering of Fc region residues however Asn297 may also be located about 3 amino acids upstream or downstream of position 297 i.e. between positions 294 and 300 due to minor sequence variations in antibodies. Such fucosylation variants may have improved ADCC function. See e.g. US Patent Publication Nos. US 2003 0157108 Presta L. US 2004 0093621 Kyowa Hakko Kogyo Co. Ltd . Examples of publications related to defucosylated or fucose deficient antibody variants include US 2003 0157108 WO 2000 61739 WO 2001 29246 US 2003 0115614 US 2002 0164328 US 2004 0093621 US 2004 0132140 US 2004 0110704 US 2004 0110282 US 2004 0109865 WO 2003 085119 WO 2003 084570 WO 2005 035586 WO 2005 035778 WO2005 053742 WO2002 031140 Okazaki et al. 336 1239 1249 2004 Yamane Ohnuki et al. 87 614 2004 . Examples of cell lines capable of producing defucosylated antibodies include Lec13 CHO cells deficient in protein fucosylation Ripka et al. 249 533 545 1986 US Pat Appl No US 2003 0157108 A1 Presta L and WO 2004 056312 A1 Adams et al. especially at Example 11 and knockout cell lines such as alpha 1 6 fucosyltransferase gene FUT8 knockout CHO cells see e.g. Yamane Ohnuki et al. 87 614 2004 Kanda Y. et al. 94 4 680 688 2006 and WO2003 085107 .

Antibodies variants are further provided with bisected oligosaccharides e.g. in which a biantennary oligosaccharide attached to the Fc region of the antibody is bisected by GlcNAc. Such antibody variants may have reduced fucosylation and or improved ADCC function. Examples of such antibody variants are described e.g. in WO 2003 011878 Jean Mairet et al. U.S. Pat. No. 6 602 684 Umana et al. and US 2005 0123546 Umana et al. . Antibody variants with at least one galactose residue in the oligosaccharide attached to the Fc region are also provided. Such antibody variants may have improved CDC function. Such antibody variants are described e.g. in WO 1997 30087 Patel et al. WO 1998 58964 Raju S. and WO 1999 22764 Raju S. .

In certain embodiments one or more amino acid modifications may be introduced into the Fc region of an antibody provided herein thereby generating an Fc region variant. The Fc region variant may comprise a human Fc region sequence e.g. a human IgG1 IgG2 IgG3 or IgG4 Fc region comprising an amino acid modification e.g. a substitution at one or more amino acid positions.

In certain embodiments the invention contemplates an antibody variant that possesses some but not all effector functions which make it a desirable candidate for applications in which the half life of the antibody in vivo is important yet certain effector functions such as complement and ADCC are unnecessary or deleterious. In vitro and or in vivo cytotoxicity assays can be conducted to confirm the reduction depletion of CDC and or ADCC activities. For example Fc receptor FcR binding assays can be conducted to ensure that the antibody lacks Fc R binding hence likely lacking ADCC activity but retains FcRn binding ability. The primary cells for mediating ADCC NK cells express Fc RIII only whereas monocytes express Fc RI Fc RII and Fc RIII. FcR expression on hematopoietic cells is summarized in Table 3 on page 464 of Ravetch and Kinet 9 457 492 1991 . Non limiting examples of in vitro assays to assess ADCC activity of a molecule of interest is described in U.S. Pat. No. 5 500 362 see e.g. Hellstrom I. et al. 83 7059 7063 1986 and Hellstrom I et al. 82 1499 1502 1985 U.S. Pat. No. 5 821 337 see Bruggemann M. et al. 166 1351 1361 1987 . Alternatively non radioactive assays methods may be employed see for example ACTI non radioactive cytotoxicity assay for flow cytometry CellTechnology Inc. Mountain View Calif. and CytoTox 96 non radioactive cytotoxicity assay Promega Madison Wis. . Useful effector cells for such assays include peripheral blood mononuclear cells PBMC and Natural Killer NK cells. Alternatively or additionally ADCC activity of the molecule of interest may be assessed in vivo e.g. in a animal model such as that disclosed in Clynes et al. 95 652 656 1998 . C1 q binding assays may also be carried out to confirm that the antibody is unable to bind C1q and hence lacks CDC activity. See e.g. C1q and C3c binding ELISA in WO 2006 029879 and WO 2005 100402. To assess complement activation a CDC assay may be performed see for example Gazzano Santoro et al. 202 163 1996 Cragg M. S. et al. 101 1045 1052 2003 and Cragg M. S. and M. J. Glennie 103 2738 2743 2004 . FeRn binding and in vivo clearance half life determinations can also be performed using methods known in the art see e.g. Petkova S. B. et al. 18 12 1759 1769 2006 .

Antibodies with reduced effector function include those with substitution of one or more of Fc region residues 238 265 269 270 297 327 and 329 U.S. Pat. No. 6 737 056 . Such Fc mutants include Fc mutants with substitutions at two or more of amino acid positions 265 269 270 297 and 327 including the so called DANA Fc mutant with substitution of residues 265 and 297 to alanine U.S. Pat. No. 7 332 581 .

Certain antibody variants with improved or diminished binding to FcRs are described. See e.g. U.S. Pat. No. 6 737 056 WO 2004 056312 and Shields et al. 9 2 6591 6604 2001 . 

In certain embodiments an antibody variant comprises an Fc region with one or more amino acid substitutions which improve ADCC e.g. substitutions at positions 298 333 and or 334 of the Fc region EU numbering of residues .

In some embodiments alterations are made in the Fc region that result in altered i.e. either improved or diminished C1q binding and or Complement Dependent Cytotoxicity CDC e.g. as described in U.S. Pat. No. 6 194 551 WO 99 51642 and Idusogie et al. 164 4178 4184 2000 .

Antibodies with increased half lives and improved binding to the neonatal Fc receptor FcRn which is responsible for the transfer of maternal IgGs to the fetus Guyer et al. 117 587 1976 and Kim et al. 24 249 1994 are described in US2005 0014934A1 Hinton et al. . Those antibodies comprise an Fc region with one or more substitutions therein which improve binding of the Fc region to FcRn. Such Fc variants include those with substitutions at one or more of Fc region residues 238 256 265 272 286 303 305 307 311 312 317 340 356 360 362 376 378 380 382 413 424 or 434 e.g. substitution of Fc region residue 434 U.S. Pat. No. 7 371 826 .

See also Duncan Winter 322 738 40 1988 U.S. Pat. No. 5 648 260 U.S. Pat. No. 5 624 821 and WO 94 29351 concerning other examples of Fc region variants.

In certain embodiments it may be desirable to create cysteine engineered antibodies e.g. thioMAbs in which one or more residues of an antibody are substituted with cysteine residues. In particular embodiments the substituted residues occur at accessible sites of the antibody. By substituting those residues with cysteine reactive thiol groups are thereby positioned at accessible sites of the antibody and may be used to conjugate the antibody to other moieties such as drug moieties or linker drug moieties to create an immunoconjugate as described further herein. In certain embodiments any one or more of the following residues may be substituted with cysteine V205 Kabat numbering of the light chain A118 EU numbering of the heavy chain and 5400 EU numbering of the heavy chain Fc region. Cysteine engineered antibodies may be generated as described e.g. in U.S. Pat. No. 7 521 541.

In certain embodiments an antibody provided herein may be further modified to contain additional nonproteinaceous moieties that are known in the art and readily available. The moieties suitable for derivatization of the antibody include but are not limited to water soluble polymers. Non limiting examples of water soluble polymers include but are not limited to polyethylene glycol PEG copolymers of ethylene glycol propylene glycol carboxymethylcellulose dextran polyvinyl alcohol polyvinyl pyrrolidone poly 1 3 dioxolane poly 1 3 6 trioxane ethylene maleic anhydride copolymer polyaminoacids either homopolymers or random copolymers and dextran or poly n vinyl pyrrolidone polyethylene glycol propropylene glycol homopolymers prolypropylene oxide ethylene oxide co polymers polyoxyethylated polyols e.g. glycerol polyvinyl alcohol and mixtures thereof. Polyethylene glycol propionaldehyde may have advantages in manufacturing due to its stability in water. The polymer may be of any molecular weight and may be branched or unbranched. The number of polymers attached to the antibody may vary and if more than one polymer is attached they can be the same or different molecules. In general the number and or type of polymers used for derivatization can be determined based on considerations including but not limited to the particular properties or functions of the antibody to be improved whether the antibody derivative will be used in a therapy under defined conditions etc.

In another embodiment conjugates of an antibody and nonproteinaceous moiety that may be selectively heated by exposure to radiation are provided. In one embodiment the nonproteinaceous moiety is a carbon nanotube Kam et al. 102 11600 11605 2005 . The radiation may be of any wavelength and includes but is not limited to wavelengths that do not harm ordinary cells but which heat the nonproteinaceous moiety to a temperature at which cells proximal to the antibody nonproteinaceous moiety are killed.

Antibodies may be produced using recombinant methods and compositions e.g. as described in U.S. Pat. No. 4 816 567. In one embodiment isolated nucleic acid encoding an anti linear linked polyubiquitin antibody described herein is provided. Such nucleic acid may encode an amino acid sequence comprising the VL and or an amino acid sequence comprising the VH of the antibody e.g. the light and or heavy chains of the antibody . In a further embodiment one or more vectors e.g. expression vectors comprising such nucleic acid are provided. In a further embodiment a host cell comprising such nucleic acid is provided. In one such embodiment a host cell comprises e.g. has been transformed with 1 a vector comprising a nucleic acid that encodes an amino acid sequence comprising the VL of the antibody and an amino acid sequence comprising the VH of the antibody or 2 a first vector comprising a nucleic acid that encodes an amino acid sequence comprising the VL of the antibody and a second vector comprising a nucleic acid that encodes an amino acid sequence comprising the VH of the antibody. In one embodiment the host cell is eukaryotic e.g. a Chinese Hamster Ovary CHO cell or lymphoid cell e.g. Y0 NS0 Sp20 cell . In one embodiment a method of making an anti linear linked polyubiquitin antibody is provided wherein the method comprises culturing a host cell comprising a nucleic acid encoding the antibody as provided above under conditions suitable for expression of the antibody and optionally recovering the antibody from the host cell or host cell culture medium .

For recombinant production of an anti linear linked polyubiquitin antibody nucleic acid encoding an antibody e.g. as described above is isolated and inserted into one or more vectors for further cloning and or expression in a host cell. Such nucleic acid may be readily isolated and sequenced using conventional procedures e.g. by using oligonucleotide probes that are capable of binding specifically to genes encoding the heavy and light chains of the antibody .

Suitable host cells for cloning or expression of antibody encoding vectors include prokaryotic or eukaryotic cells described herein. For example antibodies may be produced in bacteria in particular when glycosylation and Fc effector function are not needed. For expression of antibody fragments and polypeptides in bacteria see e.g. U.S. Pat. Nos. 5 648 237 5 789 199 and 5 840 523. See also Charlton 248 B. K. C. Lo ed. Humana Press Totowa N.J. 2003 pp. 245 254 describing expression of antibody fragments in . After expression the antibody may be isolated from the bacterial cell paste in a soluble fraction and can be further purified.

In addition to prokaryotes eukaryotic microbes such as filamentous fungi or yeast are suitable cloning or expression hosts for antibody encoding vectors including fungi and yeast strains whose glycosylation pathways have been humanized resulting in the production of an antibody with a partially or fully human glycosylation pattern. See Gerngross 22 1409 1414 2004 and Li et al. 24 210 215 2006 .

Suitable host cells for the expression of glycosylated antibody are also derived from multicellular organisms invertebrates and vertebrates . Examples of invertebrate cells include plant and insect cells. Numerous baculoviral strains have been identified which may be used in conjunction with insect cells particularly for transfection of cells.

Plant cell cultures can also be utilized as hosts. See e.g. U.S. Pat. Nos. 5 959 177 6 040 498 6 420 548 7 125 978 and 6 417 429 describing PLANTIBODIES technology for producing antibodies in transgenic plants .

Vertebrate cells may also be used as hosts. For example mammalian cell lines that are adapted to grow in suspension may be useful. Other examples of useful mammalian host cell lines are monkey kidney CV1 line transformed by SV40 COS 7 human embryonic kidney line 293 or 293 cells as described e.g. in Graham et al. 36 59 1977 baby hamster kidney cells BHK mouse sertoli cells TM4 cells as described e.g. in Mather 23 243 251 1980 monkey kidney cells CV1 African green monkey kidney cells VERO 76 human cervical carcinoma cells HELA canine kidney cells MDCK buffalo rat liver cells BRL 3A human lung cells W138 human liver cells Hep G2 mouse mammary tumor MMT 060562 TRI cells as described e.g. in Mather et al. 383 44 68 1982 MRC 5 cells and FS4 cells. Other useful mammalian host cell lines include Chinese hamster ovary CHO cells including DHFR CHO cells Urlaub et al. 77 4216 1980 and myeloma cell lines such as Y0 NS0 and Sp2 0. For a review of certain mammalian host cell lines suitable for antibody production see e.g. Yazaki and Wu 248 B. K. C. Lo ed. Humana Press Totowa N.J. pp. 255 268 2003 .

Anti linear linked polyubiquitin antibodies provided herein may be identified screened for or characterized for their physical chemical properties and or biological activities by various assays known in the art.

In one aspect an antibody of the invention is tested for its antigen binding activity e.g. by known methods such as ELISA Western blot etc.

In another aspect competition assays may be used to identify an antibody that competes with e.g. any of Fabs or antibodies described herein such as 1E3 1D8 1F4 1A10 1D8.3C2 1D8.3F8 1D8.4F5 1F11 2A2 2C11 2H5 3E4 4C9 4E4 4G7 3F5 3A7 and 4C10 or hybrid antibodies as described herein for example 4E4 T110A 4C9 T110A 4G7 Y102G 1F11 3F5 1F11 3E4 1F11 4G7 2C11 3E4 2C11 3F5 2C11 4G7 2H5 3E4 2H5 3F5 2H5 4G7 1F11 3F5 Y102L for binding to linear linked polyubiquitin. In certain embodiments such a competing antibody binds to the same epitope e.g. a linear or a conformational epitope that is bound by any of Fabs or antibodies described herein such as 1E3 1D8 1F4 1A10 1D8.3C2 1D8.3F8 1D8.4F5 1F11 2A2 2C11 2H5 3E4 4C9 4E4 4G7 3F5 3A7 and 4C10 or hybrid antibodies as described herein for example 4E4 T110A 4C9 T110A 4G7 Y102G 1F11 3F5 1F11 3E4 1F11 4G7 2C11 3E4 2C11 3F5 2C11 4G7 2H5 3E4 2H5 3F5 2H5 4G7 1F11 3F5 Y102L for binding to linear linked polyubiquitin. Detailed exemplary methods for mapping an epitope to which an antibody binds are provided in Morris 1996 Epitope Mapping Protocols in vol. 66 Humana Press Totowa N.J. .

In an exemplary competition assay immobilized linear linked polyubiquitin is incubated in a solution comprising a first labeled antibody that binds to linear linked polyubiquitin e.g. antibodies 1E3 1D8 1F4 1A10 1D8.3C2 1D8.3F8 1D8.4F5 1F11 2A2 2C11 2H5 3E4 4C9 4E4 4G7 3F5 3A7 and 4C10 or hybrid antibodies 4E4 T110A 4C9 T110A 4G7 Y102G 1F11 3F5 1F11 3E4 1F11 4G7 2C11 3E4 2C11 3F5 2C11 4G7 2H5 3E4 2H5 3F5 2H5 4G7 1F11 3F5 Y102L for binding to linear linked polyubiquitin and a second unlabeled antibody that is being tested for its ability to compete with the first antibody for binding to linear linked polyubiquitin. The second antibody may be present in a hybridoma supernatant. As a control immobilized linear linked polyubiquitin is incubated in a solution comprising the first labeled antibody but not the second unlabeled antibody. After incubation under conditions permissive for binding of the first antibody to linear linked polyubiquitin excess unbound antibody is removed and the amount of label associated with immobilized linear linked polyubiquitin is measured. If the amount of label associated with immobilized linear linked polyubiquitin is substantially reduced in the test sample relative to the control sample then that indicates that the second antibody is competing with the first antibody for binding to linear linked polyubiquitin. See Harlow and Lane 1988 ch. 14 Cold Spring Harbor Laboratory Cold Spring Harbor N.Y. .

In one aspect assays are provided for identifying anti linear linked polyubiquitin antibodies thereof having biological activity. Biological activity may include e.g. modulating the rate of degradation of linear linked polyubiquitinated proteins in a cell or tissue and modulating the rate of cell cycle progression of a cell. Antibodies having such biological activity in vivo and or in vitro are also provided.

The invention also provides immunoconjugates comprising an anti linear linked polyubiquitin antibody herein conjugated to one or more cytotoxic agents such as chemotherapeutic agents or drugs growth inhibitory agents toxins e.g. protein toxins enzymatically active toxins of bacterial fungal plant or animal origin or fragments thereof or radioactive isotopes.

In one embodiment an immunoconjugate is an antibody drug conjugate ADC in which an antibody is conjugated to one or more drugs including but not limited to a maytansinoid see U.S. Pat. Nos. 5 208 020 5 416 064 and European Patent EP 0 425 235 B1 an auristatin such as monomethylauristatin drug moieties DE and DF MMAE and MMAF see U.S. Pat. Nos. 5 635 483 and 5 780 588 and 7 498 298 a dolastatin a calicheamicin or derivative thereof see U.S. Pat. Nos. 5 712 374 5 714 586 5 739 116 5 767 285 5 770 701 5 770 710 5 773 001 and 5 877 296 Hinman et al. 53 3336 3342 1993 and Lode et al. 58 2925 2928 1998 an anthracycline such as daunomycin or doxorubicin see Kratz et al. 13 477 523 2006 Jeffrey et al. 16 358 362 2006 Torgov et al. 16 717 721 2005 Nagy et al. 97 829 834 2000 Dubowchik et al. . 12 1529 1532 2002 King et al. 45 4336 4343 2002 and U.S. Pat. No. 6 630 579 methotrexate vindesine a taxane such as docetaxel paclitaxel larotaxel tesetaxel and ortataxel a trichothecene and CC1065.

In another embodiment an immunoconjugate comprises an antibody as described herein conjugated to an enzymatically active toxin or fragment thereof including but not limited to diphtheria A chain nonbinding active fragments of diphtheria toxin exotoxin A chain from ricin A chain abrin A chain modeccin A chain alpha sarcin proteins dianthin proteins proteins PAPI PAPII and PAP S inhibitor curcin crotin sapaonaria officinalis inhibitor gelonin mitogellin restrictocin phenomycin enomycin and the tricothecenes.

In another embodiment an immunoconjugate comprises an antibody as described herein conjugated to a radioactive atom to form a radioconjugate. A variety of radioactive isotopes are available for the production of radioconjugates. Examples include At I I Y Re Re sm Bi P Pband radioactive isotopes of Lu. When the radioconjugate is used for detection it may comprise a radioactive atom for scintigraphic studies for example tc99m or 1123 or a spin label for nuclear magnetic resonance NMR imaging also known as magnetic resonance imaging mri such as iodine 123 again iodine 131 indium 111 fluorine 19 carbon 13 nitrogen 15 oxygen 17 gadolinium manganese or iron.

Conjugates of an antibody and cytotoxic agent may be made using a variety of bifunctional protein coupling agents such as N succinimidyl 3 2 pyridyldithio propionate SPDP succinimidyl 4 N maleimidomethyl cyclohexane 1 carboxylate SMCC iminothiolane IT bifunctional derivatives of imidoesters such as dimethyl adipimidate HCl active esters such as disuccinimidyl suberate aldehydes such as glutaraldehyde bis azido compounds such as bis p azidobenzoyl hexanediamine bis diazonium derivatives such as bis p diazoniumbenzoyl ethylenediamine diisocyanates such as toluene 2 6 diisocyanate and bis active fluorine compounds such as 1 5 difluoro 2 4 dinitrobenzene . For example a ricin immunotoxin can be prepared as described in Vitetta et al. 238 1098 1987 . Carbon 14 labeled 1 isothiocyanatobenzyl 3 methyldiethylene triaminepentaacetic acid MX DTPA is an exemplary chelating agent for conjugation of radionucleotide to the antibody. See WO94 11026. The linker may be a cleavable linker facilitating release of a cytotoxic drug in the cell. For example an acid labile linker peptidase sensitive linker photolabile linker dimethyl linker or disulfide containing linker Chari et al. 52 127 131 1992 U.S. Pat. No. 5 208 020 may be used.

The immunoconjugates or ADCs herein expressly contemplate but are not limited to such conjugates prepared with cross linker reagents including but not limited to BMPS EMCS GMBS HBVS LC SMCC MBS MPBH SBAP SIA STAB SMCC SMPB SMPH sulfo EMCS sulfo GMBS sulfo KMUS sulfo MBS sulfo SIAB sulfo SMCC and sulfo SMPB and SVSB succinimidyl 4 vinylsulfone benzoate which are commercially available e.g. from Pierce Biotechnology Inc. Rockford Ill. U.S.A .

In certain embodiments any of the anti linear linked polyubiquitin antibodies provided herein is useful for detecting the presence of linear linked polyubiquitin in a biological sample. The term detecting as used herein encompasses quantitative or qualitative detection. In certain embodiments a biological sample comprises a cell or tissue such as but not limited to a tumor cell a muscle cell or a nerve cell.

In one embodiment an anti linear linked polyubiquitin antibody for use in a method of diagnosis or detection is provided. In a further aspect a method of detecting the presence of linear linked polyubiquitin in a biological sample is provided. In certain embodiments the method comprises contacting the biological sample with an anti linear linked polyubiquitin antibody as described herein under conditions permissive for binding of the anti linear linked polyubiquitin antibody to a polyubiquitin or polyubiquitinated protein and detecting whether a complex is formed between the anti linear linked polyubiquitin antibody and the polyubiquitin or polyubiquitinated protein. Such method may be an in vitro or in vivo method. In one embodiment an anti linear linked polyubiquitin antibody is used to select subjects eligible for therapy with an anti linear linked polyubiquitin antibody e.g. where linear linked polyubiquitin is a biomarker for selection of patients.

Exemplary disorders that may be diagnosed using an antibody of the invention include cell cycle related diseases or disorders which may be a disease or disorder associated with aberrantly increased cell cycle progression or a disease or disorder associated with aberrantly decreased cell cycle progression. In one aspect a disease or disorder associated with aberrantly increased cell cycle progression is cancer. In another aspect a disease or disorder associated with aberrantly decreased cell cycle progression is e.g. a degenerative muscle disorder or a degenerative nerve disorder.

In certain embodiments labeled anti linear linked polyubiquitin antibodies are provided. Labels include but are not limited to labels or moieties that are detected directly such as fluorescent chromophoric electron dense chemiluminescent and radioactive labels as well as moieties such as enzymes or ligands that are detected indirectly e.g. through an enzymatic reaction or molecular interaction. Exemplary labels include but are not limited to the radioisotopes P C I H and I fluorophores such as rare earth chelates or fluorescein and its derivatives rhodamine and its derivatives dansyl umbelliferone luceriferases e.g. firefly luciferase and bacterial luciferase U.S. Pat. No. 4 737 456 luciferin 2 3 dihydrophthalazinediones horseradish peroxidase HRP alkaline phosphatase galactosidase glucoamylase lysozyme saccharide oxidases e.g. glucose oxidase galactose oxidase and glucose 6 phosphate dehydrogenase heterocyclic oxidases such as uricase and xanthine oxidase coupled with an enzyme that employs hydrogen peroxide to oxidize a dye precursor such as HRP lactoperoxidase or microperoxidase biotin avidin spin labels bacteriophage labels stable free radicals and the like.

Pharmaceutical formulations of an anti linear linked polyubiquitin antibody as described herein are prepared by mixing such antibody having the desired degree of purity with one or more optional pharmaceutically acceptable carriers 16th edition Osol A. Ed. 1980 in the form of lyophilized formulations or aqueous solutions. Pharmaceutically acceptable carriers are generally nontoxic to recipients at the dosages and concentrations employed and include but are not limited to buffers such as phosphate citrate and other organic acids antioxidants including ascorbic acid and methionine preservatives such as octadecyldimethylbenzyl ammonium chloride hexamethonium chloride benzalkonium chloride benzethonium chloride phenol butyl or benzyl alcohol alkyl parabens such as methyl or propyl paraben catechol resorcinol cyclohexanol 3 pentanol and m cresol low molecular weight less than about 10 residues polypeptides proteins such as serum albumin gelatin or immunoglobulins hydrophilic polymers such as polyvinylpyrrolidone amino acids such as glycine glutamine asparagine histidine arginine or lysine monosaccharides disaccharides and other carbohydrates including glucose mannose or dextrins chelating agents such as EDTA sugars such as sucrose mannitol trehalose or sorbitol salt forming counter ions such as sodium metal complexes e.g. Zn protein complexes and or non ionic surfactants such as polyethylene glycol PEG . Exemplary pharmaceutically acceptable carriers herein further include interstitial drug dispersion agents such as soluble neutral active hyaluronidase glycoproteins sHASEGP for example human soluble PH 20 hyaluronidase glycoproteins such as rHuPH20 HYLENEX Baxter International Inc. . Certain exemplary sHASEGPs and methods of use including rHuPH20 are described in US Patent Publication Nos. 2005 0260186 and 2006 0104968. In one aspect a sHASEGP is combined with one or more additional glycosaminoglycanases such as chondroitinases.

Exemplary lyophilized antibody formulations are described in U.S. Pat. No. 6 267 958. Aqueous antibody formulations include those described in U.S. Pat. No. 6 171 586 and WO2006 044908 the latter formulations including a histidine acetate buffer.

The formulation herein may also contain more than one active ingredients as necessary for the particular indication being treated preferably those with complementary activities that do not adversely affect each other. For example it may be desirable to further provide one or more chemotherapeutic agents. Such active ingredients are suitably present in combination in amounts that are effective for the purpose intended.

Active ingredients may be entrapped in microcapsules prepared for example by coacervation techniques or by interfacial polymerization for example hydroxymethylcellulose or gelatin microcapsules and poly methylmethacylate microcapsules respectively in colloidal drug delivery systems for example liposomes albumin microspheres microemulsions nano particles and nanocapsules or in macroemulsions. Such techniques are disclosed in 16th edition Osol A. Ed. 1980 .

Sustained release preparations may be prepared. Suitable examples of sustained release preparations include semipermeable matrices of solid hydrophobic polymers containing the antibody which matrices are in the form of shaped articles e.g. films or microcapsules.

The formulations to be used for in vivo administration are generally sterile. Sterility may be readily accomplished e.g. by filtration through sterile filtration membranes.

Any of the anti linear linked polyubiquitin antibodies provided herein may be used in therapeutic methods.

In one aspect an anti linear linked polyubiquitin antibody for use as a medicament is provided. In further aspects an anti linear linked polyubiquitin antibody for use in treating disorders associated with aberrant cell cycle regulation including but not limited to proliferation disorders such as cancer and hypotrophy disorders including but not limited to degenerative muscle disorders and degenerative nerve disorders is provided. In certain embodiments an anti linear linked polyubiquitin antibody for use in a method of treatment is provided. In certain embodiments the invention provides an anti linear linked polyubiquitin antibody for use in a method of treating an individual having a disorder associated with aberrant cell cycle regulation comprising administering to the individual an effective amount of the anti linear linked polyubiquitin antibody. In one such embodiment the method further comprises administering to the individual an effective amount of at least one additional therapeutic agent e.g. as described below. In further embodiments the invention provides an anti linear linked polyubiquitin antibody for use in modulating cell cycle regulation such that the rate of cell cycle progression is adjusted. In certain embodiments the invention provides an anti linear linked polyubiquitin antibody for use in a method of modulating the rate of cell cycle progression in an individual comprising administering to the individual an effective of the anti linear linked polyubiquitin antibody to modulate cell cycle progression and thereby adjust the rate of cell division. An individual according to any of the above embodiments is preferably a human.

In a further aspect the invention provides for the use of an anti linear linked polyubiquitin antibody in the manufacture or preparation of a medicament. In one embodiment the medicament is for treatment of disorders associated with aberrant cell cycle regulation including but not limited to proliferation disorders such as cancer and hypotrophy disorders including but not limited to degenerative muscle disorders and degenerative nerve disorders . In a further embodiment the medicament is for use in a method of treating a disorder associated with aberrant cell cycle regulation comprising administering to an individual having such a disorder an effective amount of the medicament. In one such embodiment the method further comprises administering to the individual an effective amount of at least one additional therapeutic agent e.g. as described below. In a further embodiment the medicament is for modulating the rate of cell cycle progression. In a further embodiment the medicament is for use in a method of modulating the rate of cell cycle progression in an individual comprising administering to the individual an amount effective of the medicament to adjust the rate of cellular division. An individual according to any of the above embodiments may be a human.

In a further aspect the invention provides a method for treating a disorder associated with aberrant cell cycle regulation. In one embodiment the method comprises administering to an individual having such a disorder associated with aberrant cell cycle regulation an effective amount of an anti linear linked polyubiquitin antibody. In one such embodiment the method further comprises administering to the individual an effective amount of at least one additional therapeutic agent as described below. An individual according to any of the above embodiments may be a human.

In a further aspect the invention provides pharmaceutical formulations comprising any of the anti linear linked polyubiquitin antibodies provided herein e.g. for use in any of the above therapeutic methods. In one embodiment a pharmaceutical formulation comprises any of the anti linear linked polyubiquitin antibodies provided herein and a pharmaceutically acceptable carrier. In another embodiment a pharmaceutical formulation comprises any of the anti linear linked polyubiquitin antibodies provided herein and at least one additional therapeutic agent e.g. as described below.

Antibodies of the invention can be used either alone or in combination with other agents in a therapy. For instance an antibody of the invention may be co administered with at least one additional therapeutic agent. In certain embodiments an additional therapeutic agent is a chemotherapeutic agent.

A chemotherapeutic agent is a chemical compound useful in the treatment of cancer. Examples of chemotherapeutic agents include alkylating agents such as thiotepa and CYTOXAN cyclosphosphamide alkyl sulfonates such as busulfan improsulfan and piposulfan aziridines such as benzodopa carboquone meturedopa and uredopa ethylenimines and methylamelamines including altretamine triethylenemelamine trietylenephosphoramide triethiylenethiophosphoramide and trimethylolomelamine acetogenins especially bullatacin and bullatacinone delta 9 tetrahydrocannabinol dronabinol MARINOL beta lapachone lapachol colchicines betulinic acid a camptothecin including the synthetic analogue topotecan HYCAMTIN CPT 11 irinotecan CAMPTOSAR acetylcamptothecin scopolectin and 9 aminocamptothecin bryostatin callystatin CC 1065 including its adozelesin carzelesin and bizelesin synthetic analogues podophyllotoxin podophyllinic acid teniposide cryptophycins particularly cryptophycin 1 and cryptophycin 8 dolastatin duocarmycin including the synthetic analogues KW 2189 and CB1 TM1 eleutherobin pancratistatin a sarcodictyin spongistatin nitrogen mustards such as chlorambucil chlornaphazine cholophosphamide estramustine ifosfamide mechlorethamine mechlorethamine oxide hydrochloride melphalan novembichin phenesterine prednimustine trofosfamide uracil mustard nitrosureas such as carmustine chlorozotocin fotemustine lomustine nimustine and ranimnustine antibiotics such as the enediyne antibiotics e.g. calicheamicin especially calicheamicin gammall and calicheamicin omegaIl see e.g. Agnew Chem. Intl. Ed. Engl. 33 183 186 1994 dynemicin including dynemicin A an esperamicin as well as neocarzinostatin chromophore and related chromoprotein enediyne antiobiotic chromophores aclacinomysins actinomycin authramycin azaserine bleomycins cactinomycin carabicin caminomycin carzinophilin chromomycinis dactinomycin daunorubicin detorubicin 6 diazo 5 oxo L norleucine ADRIAMYCIN doxorubicin including morpholino doxorubicin cyanomorpholino doxorubicin 2 pyrrolino doxorubicin and deoxydoxorubicin epirubicin esorubicin idarubicin marcellomycin mitomycins such as mitomycin C mycophenolic acid nogalamycin olivomycins peplomycin potfiromycin puromycin quelamycin rodorubicin streptonigrin streptozocin tubercidin ubenimex zinostatin zorubicin anti metabolites such as methotrexate and 5 fluorouracil 5 FU folic acid analogues such as denopterin methotrexate pteropterin trimetrexate purine analogs such as fludarabine 6 mercaptopurine thiamiprine thioguanine pyrimidine analogs such as ancitabine azacitidine 6 azauridine carmofur cytarabine dideoxyuridine doxifluridine enocitabine floxuridine androgens such as calusterone dromostanolone propionate epitiostanol mepitiostane testolactone anti adrenals such as aminoglutethimide mitotane trilostane folic acid replenisher such as frolinic acid aceglatone aldophosphamide glycoside aminolevulinic acid eniluracil amsacrine bestrabucil bisantrene edatraxate defofamine demecolcine diaziquone elformithine elliptinium acetate an epothilone etoglucid gallium nitrate hydroxyurea lentinan lonidainine maytansinoids such as maytansine and ansamitocins mitoguazone mitoxantrone mopidanmol nitraerine pentostatin phenamet pirarubicin losoxantrone 2 ethylhydrazide procarbazine PSK polysaccharide complex JHS Natural Products Eugene Oreg. razoxane rhizoxin sizofuran spirogermanium tenuazonic acid triaziquone 2 2 2 trichlorotriethylamine trichothecenes especially T 2 toxin verracurin A roridin A and anguidine urethan vindesine ELDISINE FILDESIN dacarbazine mannomustine mitobronitol mitolactol pipobroman gacytosine arabinoside Ara C thiotepa taxoids e.g. TAXOL paclitaxel Bristol Myers Squibb Oncology Princeton N.J. ABRAXANE Cremophor free albumin engineered nanoparticle formulation of paclitaxel American Pharmaceutical Partners Schaumberg Ill. and TAXOTERE doxetaxel Rh ne Poulenc Rorer Antony France chloranbucil gemcitabine GEMZAR 6 thioguanine mercaptopurine methotrexate platinum analogs such as cisplatin and carboplatin vinblastine VELBAN platinum etoposide VP 16 ifosfamide mitoxantrone vincristine ONCOVIN oxaliplatin leucovovin vinorelbine NAVELBINE novantrone edatrexate daunomycin aminopterin ibandronate topoisomerase inhibitor RFS 2000 difluoromethylomithine DMFO retinoids such as retinoic acid capecitabine XELODA pharmaceutically acceptable salts acids or derivatives of any of the above as well as combinations of two or more of the above such as CHOP an abbreviation for a combined therapy of cyclophosphamide doxorubicin vincristine and prednisolone and FOLFOX an abbreviation for a treatment regimen with oxaliplatin ELOXATIN combined with 5 FU and leucovovin.

Also included in this definition are anti hormonal agents that act to regulate reduce block or inhibit the effects of hormones that can promote the growth of cancer and are often in the form of systemic or whole body treatment. They may be hormones themselves. Examples include anti estrogens and selective estrogen receptor modulators SERMs including for example tamoxifen including NOLVADEX tamoxifen EVISTA raloxifene droloxifene 4 hydroxytamoxifen trioxifene keoxifene LY117018 onapristone and FARESTON toremifene anti progesterones estrogen receptor down regulators ERDs agents that function to suppress or shut down the ovaries for example leutinizing hormone releasing hormone LHRH agonists such as LUPRON and ELIGARD leuprolide acetate goserelin acetate buserelin acetate and tripterelin other anti androgens such as flutamide nilutamide and bicalutamide and aromatase inhibitors that inhibit the enzyme aromatase which regulates estrogen production in the adrenal glands such as for example 4 5 imidazoles aminoglutethimide MEGASE megestrol acetate AROMASIN exemestane formestanie fadrozole RIVISOR vorozole FEMARA letrozole and ARIMIDEX anastrozole. In addition such definition of chemotherapeutic agents includes bisphosphonates such as clodronate for example BONEFOS or OSTAC DIDROCAL etidronate NE 58095 ZOMETA zoledronic acid zoledronate FOSAMAX alendronate AREDIA pamidronate SKELID tiludronate or ACTONEL risedronate as well as troxacitabine a 1 3 dioxolane nucleoside cytosine analog antisense oligonucleotides particularly those that inhibit expression of genes in signaling pathways implicated in abherant cell proliferation such as for example PKC alpha Raf H Ras and epidermal growth factor receptor EGF R vaccines such as THERATOPE vaccine and gene therapy vaccines for example ALLOVECTIN vaccine LEUVECTIN vaccine and VAXID vaccine LURTOTECAN topoisomerase 1 inhibitor ABARELIX rmRH lapatinib ditosylate an ErbB 2 and EGFR dual tyrosine kinase small molecule inhibitor also known as GW572016 and pharmaceutically acceptable salts acids or derivatives of any of the above.

Such combination therapies noted above encompass combined administration where two or more therapeutic agents are included in the same or separate formulations and separate administration in which case administration of the antibody of the invention can occur prior to simultaneously and or following administration of the additional therapeutic agent and or adjuvant. Antibodies of the invention can also be used in combination with radiation therapy.

An antibody of the invention and any additional therapeutic agent can be administered by any suitable means including parenteral intrapulmonary and intranasal and if desired for local treatment intralesional administration. Parenteral infusions include intramuscular intravenous intraarterial intraperitoneal or subcutaneous administration. Dosing can be by any suitable route e.g. by injections such as intravenous or subcutaneous injections depending in part on whether the administration is brief or chronic. Various dosing schedules including but not limited to single or multiple administrations over various time points bolus administration and pulse infusion are contemplated herein.

Antibodies of the invention would be formulated dosed and administered in a fashion consistent with good medical practice. Factors for consideration in this context include the particular disorder being treated the particular mammal being treated the clinical condition of the individual patient the cause of the disorder the site of delivery of the agent the method of administration the scheduling of administration and other factors known to medical practitioners. The antibody need not be but is optionally formulated with one or more agents currently used to prevent or treat the disorder in question. The effective amount of such other agents depends on the amount of antibody present in the formulation the type of disorder or treatment and other factors discussed above. These are generally used in the same dosages and with administration routes as described herein or about from 1 to 99 of the dosages described herein or in any dosage and by any route that is empirically clinically determined to be appropriate.

The location of the binding target of an antibody of the invention may be taken into consideration in preparation and administration of the antibody. When the binding target is an intracellular molecule certain embodiments of the invention provide for the antibody or antigen binding fragment thereof to be introduced into the cell where the binding target is located. In one embodiment an antibody of the invention can be expressed intracellularly as an intrabody. The term intrabody as used herein refers to an antibody or antigen binding portion thereof that is expressed intracellularly and that is capable of selectively binding to a target molecule as described in Marasco Gene Therapy 4 11 15 1997 Kontermann Methods 34 163 170 2004 U.S. Pat. Nos. 6 004 940 and 6 329 173 U.S. Patent Application Publication No. 2003 0104402 and PCT Publication No. WO2003 077945. Intracellular expression of an intrabody is effected by introducing a nucleic acid encoding the desired antibody or antigen binding portion thereof lacking the wild type leader sequence and secretory signals normally associated with the gene encoding that antibody of antigen binding fragment into a target cell. Any standard method of introducing nucleic acids into a cell may be used including but not limited to microinjection ballistic injection electroporation calcium phosphate precipitation liposomes and transfection with retroviral adenoviral adeno associated viral and vaccinia vectors carrying the nucleic acid of interest. One or more nucleic acids encoding all or a portion of an anti polyubiquitin antibody of the invention can be delivered to a target cell such that one or more intrabodies are expressed which are capable of intracellular binding to a polyubiquitin and modulation of one or more polyubiquitin mediated cellular pathways.

In another embodiment internalizing antibodies are provided. Antibodies can possess certain characteristics that enhance delivery of antibodies into cells or can be modified to possess such characteristics. Techniques for achieving this are known in the art. For example cationization of an antibody is known to facilitate its uptake into cells see e.g. U.S. Pat. No. 6 703 019 . Lipofections or liposomes can also be used to deliver the antibody into cells. Where antibody fragments are used the smallest inhibitory fragment that specifically binds to the binding domain of the target protein is generally advantageous. For example based upon the variable region sequences of an antibody peptide molecules can be designed that retain the ability to bind the target protein sequence. Such peptides can be synthesized chemically and or produced by recombinant DNA technology. See e.g. Marasco et al. Proc. Natl. Acad. Sci. USA 90 7889 7893 1993 .

Entry of modulator polypeptides into target cells can be enhanced by methods known in the art. For example certain sequences such as those derived from HIV Tat or the Antennapedia homeodomain protein are able to direct efficient uptake of heterologous proteins across cell membranes. See e.g. Chen et al. Proc. Natl. Acad. Sci. USA 96 4325 4329 1999 .

When the binding target is located in the brain certain embodiments of the invention provide for the antibody or antigen binding fragment thereof to traverse the blood brain barrier. Certain neurodegenerative diseases are associated with an increase in permeability of the blood brain barrier such that the antibody or antigen binding fragment can be readily introduced to the brain. When the blood brain barrier remains intact several art known approaches exist for transporting molecules across it including but not limited to physical methods lipid based methods and receptor and channel based methods.

Physical methods of transporting the antibody or antigen binding fragment across the blood brain barrier include but are not limited to circumventing the blood brain barrier entirely or by creating openings in the blood brain barrier. Circumvention methods include but are not limited to direct injection into the brain see e.g. Papanastassiou et al. Gene Therapy 9 398 406 2002 interstitial infusion convection enhanced delivery see e.g. Bobo et al. Proc. Natl. Acad. Sci. USA 91 2076 2080 1994 and implanting a delivery device in the brain see e.g. Gill et al. Nature Med. 9 589 595 2003 and Gliadel Wafers Guildford Pharmaceutical . Methods of creating openings in the barrier include but are not limited to ultrasound see e.g. U.S. Patent Publication No. 2002 0038086 osmotic pressure e.g. by administration of hypertonic mannitol Neuwelt E. A. Implication of the Blood Brain Barrier and its Manipulation vols. 1 2 Plenum Press N.Y. 1989 permeabilization by e.g. bradykinin or permeabilizer A 7 see e.g. U.S. Pat. Nos. 5 112 596 5 268 164 5 506 206 and 5 686 416 and transfection of neurons that straddle the blood brain barrier with vectors containing genes encoding the antibody or antigen binding fragment see e.g. U.S. Patent Publication No. 2003 0083299 .

Lipid based methods of transporting the antibody or antigen binding fragment across the blood brain barrier include but are not limited to encapsulating the antibody or antigen binding fragment in liposomes that are coupled to antibody binding fragments that bind to receptors on the vascular endothelium of the blood brain barrier see e.g. U.S. Patent Application Publication No. 20020025313 and coating the antibody or antigen binding fragment in low density lipoprotein particles see e.g. U.S. Patent Application Publication No. 20040204354 or apolipoprotein E see e.g. U.S. Patent Application Publication No. 20040131692 .

Receptor and channel based methods of transporting the antibody or antigen binding fragment across the blood brain barrier include but are not limited to using glucocorticoid blockers to increase permeability of the blood brain barrier see e.g. U.S. Patent Application Publication Nos. 2002 0065259 2003 0162695 and 2005 0124533 activating potassium channels see e.g. U.S. Patent Application Publication No. 2005 0089473 inhibiting ABC drug transporters see e.g. U.S. Patent Application Publication No. 2003 0073713 coating antibodies with a transferrin and modulating activity of the one or more transferrin receptors see e.g. U.S. Patent Application Publication No. 2003 0129186 and cationizing the antibodies see e.g. U.S. Pat. No. 5 004 697 .

For the prevention or treatment of disease the appropriate dosage of an antibody of the invention when used alone or in combination with one or more other additional therapeutic agents will depend on the type of disease to be treated the type of antibody the severity and course of the disease whether the antibody is administered for preventive or therapeutic purposes previous therapy the patient s clinical history and response to the antibody and the discretion of the attending physician. The antibody is suitably administered to the patient at one time or over a series of treatments. Depending on the type and severity of the disease about 1 g kg to 15 mg kg e.g. 0.1 mg kg 10 mg kg of antibody can be an initial candidate dosage for administration to the patient whether for example by one or more separate administrations or by continuous infusion. One typical daily dosage might range from about 1 g kg to 100 mg kg or more depending on the factors mentioned above. For repeated administrations over several days or longer depending on the condition the treatment would generally be sustained until a desired suppression of disease symptoms occurs. One exemplary dosage of the antibody would be in the range from about 0.05 mg kg to about 10 mg kg. Thus one or more doses of about 0.5 mg kg 2.0 mg kg 4.0 mg kg or 10 mg kg or any combination thereof may be administered to the patient. Such doses may be administered intermittently e.g. every week or every three weeks e.g. such that the patient receives from about two to about twenty or e.g. about six doses of the antibody . An initial higher loading dose followed by one or more lower doses may be administered. However other dosage regimens may be useful. The progress of this therapy is easily monitored by conventional techniques and assays.

It is understood that any of the above formulations or therapeutic methods may be carried out using an immunoconjugate of the invention in place of or in addition to an anti linear linked polyubiquitin antibody.

In another aspect of the invention an article of manufacture containing materials useful for the treatment prevention and or diagnosis of the disorders described above is provided. The article of manufacture comprises a container and a label or package insert on or associated with the container. Suitable containers include for example bottles vials syringes IV solution bags etc. The containers may be formed from a variety of materials such as glass or plastic. The container holds a composition which is by itself or combined with another composition effective for treating preventing and or diagnosing the condition and may have a sterile access port for example the container may be an intravenous solution bag or a vial having a stopper pierceable by a hypodermic injection needle . At least one active agent in the composition is an antibody of the invention. The label or package insert indicates that the composition is used for treating the condition of choice. Moreover the article of manufacture may comprise a a first container with a composition contained therein wherein the composition comprises an antibody of the invention and b a second container with a composition contained therein wherein the composition comprises a further cytotoxic or otherwise therapeutic agent. The article of manufacture in this embodiment of the invention may further comprise a package insert indicating that the compositions can be used to treat a particular condition. Alternatively or additionally the article of manufacture may further comprise a second or third container comprising a pharmaceutically acceptable buffer such as bacteriostatic water for injection BWFI phosphate buffered saline Ringer s solution and dextrose solution. It may further include other materials desirable from a commercial and user standpoint including other buffers diluents filters needles and syringes.

It is understood that any of the above articles of manufacture may include an immunoconjugate of the invention in place of or in addition to an anti linear linked polyubiquitin antibody.

The following are examples of methods and compositions of the invention. It is understood that various other embodiments may be practiced given the general description provided above.

The antigen used for sorting the phage display libraries was linear diubiquitin Boston Biochem . Linear diubiquitin is a head to tail fusion of two ubiquitins through a peptide bond between the carboxy terminus of the first ubiquitin and the amino terminus of the second ubiquitin.

The na ve YSGX Fab phage display library was subjected to four rounds of sorting against linear diubiquitin. No enrichment was observed after four rounds of sorting see Table 2 . The YSGX Fab phage display library contains randomized amino acids in all three heavy chain CDRs and light chain CDR L3 see U.S. Published Patent Application No. 2005 0106667 and Fellouse F. et al. 373 924 40 2007 and is based on a humanized antibody 4D5.

The na ve common light chain YSGX Fab phage display library was subjected to four rounds of sorting against linear diubiquitin. Twenty five fold enrichment was observed after four rounds of sorting see Table 2 . The common light chain YSGX Fab phage display library contains randomized amino acids in all three heavy chain CDRs as described for the YSGX library see U.S. Published Patent Application No. 2005 0106667 and Fellouse F. et al. 373 924 40 2007 however the light chain sequence is fixed as a modified version of humanized antibody 4D5.

The na ve VH Fab phage display library was subjected to four rounds of sorting against linear diubiquitin. Eight hundred fold enrichment was observed after four rounds of sorting see Table 1 . The VH Fab phage display library contains randomized amino acids in all three heavy chain CDRs see U.S. Published Patent Application No. 2005 0119455 and Lee C. W. et al. 340 1073 93 2004 and is based on a humanized antibody 4D5.

Linear diubiquitin Boston Biochem was immobilized on a 96 well Maxisorb immunoplate NUNC . Plates were coated overnight at 4 C. with 5 g mL linear diubiquitin in 50 mM sodium carbonate buffer pH 9.6. The coated plates were blocked with 200 L well of 2.5 milk in phosphate buffered saline PBS containing 0.05 Tween 20 PBST for one hour at 25 C. with shaking. The na ve phage libraries were precipitated from glycerol stocks with volume of 20 polyethylene glycol PEG 2.5M NaCl resuspended in 2.5 milk PBST and incubated at 25 C. for one hour while the plate was blocking. After one hour the blocking buffer was dumped off of the plate and 100 L well of phage in 2.5 milk PBST was added and incubated at 25 C. for four hours with shaking. After binding the plate was washed ten times with PBST by manually filling the wells and dumping off the buffer between washes. Phage were eluted with 150 L well of 50 mM HCl 500 mM KCl for 30 minutes at 25 C. with shaking. The elution was neutralized with 150 L well of 1 M Tris pH 7.5 and subsequently propagated in XL1 Blue Agilent with the addition of M13K07 helper phage.

Amplified phage were used for additional rounds of selection against linear diubiquitin as described above. In rounds two through four soluble ubiquitin of different forms were added to the phage for counterselection. In the second round 10 g mL of soluble monoubiquitin Boston Biochem was used. In the third and fourth rounds 10 g mL each of soluble monoubiquitin Boston Biochem K11 linked diubiquitin Genentech K48 linked polyubiquitin 2 7 Boston Biochem and K63 linked polyubiquitin 2 7 chains Boston Biochem were used. Enrichment was calculated for rounds two through four by comparing the number of phage recovered with linear diubiquitin compared to an uncoated well. Enrichment was observed in rounds two through four for the common light chain library and the VH library but not the YSGX library see Table 2 .

Ninety six individual clones from the second round of sorting of the VH library 192 clones from the third round of sorting of the VH library 96 clones from the fourth round of sorting of the VH library and 192 clones from the fourth round of sorting of the common light chain library were screened. Since no enrichment was seen for the YSGX library no clones were screened from this library. Individual clones were grown up in a 96 well format in 1 mL of 2YT broth containing 50 g mL carbenicillin and 1 10phage mL M13K07 helper phage at 37 C. overnight with shaking. Cells were pelleted by spinning at 3000 rpm for ten minutes. Supernatants from those cultures were used in high throughput phage spot enzyme linked immunosorbant assays ELISAs for binding to linear diubiquitin Boston Biochem monoubiquitin Boston Biochem K11 linked diubiquitin Genentech K48 linked polyubiquitin 2 7 Boston Biochem K63 linked polyubiquitin 2 7 Boston Biochem an anti gD antibody Genentech or an uncoated well. All of the Fab libraries contain a carboxy terminal gD tag on the light chain which allows for assessment of display level by anti gD antibody binding. The panel of ubiquitin proteins was immobilized on 384 well Maxisorb immunoplates NUNC . Plates were coated at 4 C. overnight with 2 g mL of each protein in 50 mM sodium carbonate buffer pH 9.6. The coated plates were blocked with 60 L well of 2.5 milk in PBST for one hour at 25 C. with shaking. After one hour the blocking buffer was dumped off of the plate and 20 L well of PBST and 10 L well of phage supernatant were added. Plates were incubated at 25 C. for one hour with shaking. The plate was then washed six times with PBST by manually filling the wells and dumping off the wash buffer. A 1 5 000 dilution of an anti M13 horseradish peroxidase HRP conjugated secondary antibody GE Healthcare in PBST was used for detection of phage binding. 30 L well of the secondary dilution was added and the plate was incubated at 25 C. for 30 minutes with shaking. The plate was then washed six times with PBST and twice with PBS both manually. Bound secondary antibody was detected using a TMB substrate KPL followed by quenching with an equal volume of 1 M phosphoric acid. The absorbance was read at 450 nm.

From the common light chain library several weak linear diubiquitin specific binders were identified see . The light and heavy chain variable domains of these clones were sequenced. Two unique sequences 1F4 SEQ ID NOs 27 and 31 and 1A10 SEQ ID NOs 28 and 31 were identified however based on the light chain sequence it was determined that one of these clones 1F4 was actually from the YSGX library that does not contain a fixed light chain see . From the VH library several clones showing strong linear diubiquitin binding with weaker binding to K63 linked polyubiquitin were identified see . The heavy chain variable domains of these VH library clones were sequenced. The CDR H1 CDR H2 and CDR H3 sequences are expected to be clone specific whereas the heavy chain framework sequences should be identical based on the VH library design. The entire light chain sequence both framework and CDRs is expected to be invariant due to the library design. The CDR L1 sequence is RASQDVSTAVA SEQ ID NO 1 the CDR L2 sequence is SASFLYS SEQ ID NO 2 and the CDR L3 sequence is QQSYTTPPT SEQ ID NO 3 . Two unique heavy chain sequences were identified 1D8 SEQ ID NO 30 and 1E3 SEQ ID NO 29 see .

The 1D8 and 1E3 phagemid clones from the VH library were converted from bivalent Fab zip format to monovalent Fab display for affinity maturation purposes. The leucine zipper between the end of the CH1 constant domain and the start of gene III gpIII was removed using Kunkel mutagenesis see Kunkel 82 488 1985 . Mutagenic oligonucleotide F220 delzip TCTTGTGACAAAACTCACAGTGGCGGTGGCTCTGGT SEQ ID NO 100 was combined with 1 g of 1D8 or 1E3 phagemid Kunkel DNA.

The 1A10 and 1F4 clones from the common light chain library and the YSGX library respectively were converted from bivalent Fab C format to monovalent Fab display for affinity maturation purposes. The cysteine between the end of the CH1 constant domain and gpIII was removed using Kunkel mutagenesis. Mutagenic oligonucleotide F1120 delCGRP TGTGACAAAACTCACCTCAGTGGCGGTGGCTCTGGTTCCGGTGATTTTGATTATGAAA AG SEQ ID NO 101 was combined with 1 g of 1A10 or 1F4 phagemid Kunkel DNA. The resulting monovalent Fab phagemids were used to produce phage for ICELISAs.

Phage displaying monovalent Fab for 1D8 1E3 1A10 and 1F4 were tested in an ICELISA to get an estimate of relative affinity for linear diubiquitin. An initial titer ELISA was done to determine the amount of phage at which a signal of OD 0.5 would be achieved. Linear diubiquitin Boston Biochem was immobilized on 96 well Maxisorb immunoplates NUNC . Linear diubiquitin was coated overnight at 4 C. at 1 g mL in 50 mM sodium carbonate buffer pH 9.6. The coated plates were blocked with 200 L well of 2.5 milk in PBST for one hour at 25 C. with shaking. Twelve two fold serial dilutions of the phage were made in 2.5 milk in PBST from OD 4.0 to OD 0.002. After one hour the blocking buffer was dumped off of the plate and 100 L well of each phage dilution was added and incubated at 25 C. for 15 minutes with shaking. The plate was then washed six times with PBST using a plate washer. A 1 5000 dilution of an anti M13 phage HRP conjugated secondary antibody GE Healthcare in PBST was used for detection of phage binding. 100 L well of the secondary dilution was added and the plate was incubated at 25 C. for 30 minutes with shaking. The plate was then washed 12 times with PBST using a plate washer and twice with PBS manually. Bound secondary antibody was detected using a TMB substrate KPL followed by quenching with an equal volume of 1 M phosphoric acid. The absorbance was read at 450 nm. The concentration of phage at which an OD 0.5 was OD 1.0 of phage for clone 1F4 an OD 0.125 for clone 1D8 and an OD 0.5 for clone 1E3. For clone 1A10 even at phage OD 4.0 the ODwas only 0.376. This binding is quite weak and therefore clone 1A10 was not pursued further. Two fold serial dilutions of soluble linear diubiquitin from 10 M to 5 nM for clone 1F4 and three fold serial dilutions of soluble linear diubiquitin from 10 M to 56 M for clones 1D8 and 1E3 plus the selected phage concentrations in 2.5 milk in PBST were incubated at 25 C. for one hour with shaking. The amount of unbound phage at each linear diubiquitin concentration was then measured by incubating the mixtures with a 96 well Maxisorb immunoplate that had been coated with 1 g mL linear diubiquitin and blocked with 2.5 milk in PBST. The phage linear diubiquitin mixture was incubated on the plate for 15 minutes at 25 C. with shaking. The plate was then washed six times with PBST using a plate washer. A 1 5000 dilution of an anti M13 phage HRP conjugated secondary antibody GE Healthcare in PBST was used for detection of phage binding. 100 L well of the secondary dilution was added and the plate was incubated at 25 C. for 30 minutes with shaking. The plate was then washed 12 times with PBST using a plate washer and twice with PBS manually. Bound secondary antibody was detected using a TMB substrate KPL followed by quenching with an equal volume of 1 M phosphoric acid. The absorbance was read at 450 nm. The absorbance was plotted against soluble linear diubiquitin concentration and shows that for 1F4 the ICis greater than 10 M see for 1D8 the ICis near 5 M see and for 1E3 the ICis 80 nM see .

Clones derived from the Fab phage display libraries are expressed under the control of the alkaline phosphatase PhoA promoter. Both the light chain and the heavy chain contain an amino terminal bacterial still signal sequence to allow secretion in and are expressed from a single phagemid vector. The heavy chain carboxyl terminus is fused in frame to the C terminus of gene product III gpIII of the M13 bacteriophage allowing for display of a monovalent Fab fragment on phage. In order to express soluble Fab a stop codon was introduced into the 1D8 and 1E3 monovalent phagemids between the end of the CH1 constant domain of the Fab and the start of gpIII. Mutagenic oligonucleotides 

3 FabdelzipTAA CACCGGAACCAGAGCCACCGCCACTTTATGTGTGAGTT TTGTCACAAGATTTGGG SEQ ID NO 103 were used to insert the stop codon using the QuikChange Lightning Site Directed Mutagenesis kit Agilent . The resulting soluble Fab expression plasmids were transformed into the strain 62A7 Genentech and plated on solid agar containing carbenicillin. Single colonies were used to inoculate 25 mL of 2YT broth containing 50 g mL carbenicillin. The culture was grown overnight at 37 C. and 5 mL were used to inoculate 500 mL of complete C.R.A.P. media 3.57 g NH4 2SO4 0.71 g sodium citrate 2H2O 1.07 g KCl 5.36 g yeast extract certified 5.36 g Hycase SF Sheffield pH adjusted to 7.3 by addition of KOH and volume adjusted to 872 mL with ultrapure water autoclaved cooled to 55 C. to which was added per L 110 mL 1M MOPS pH 7.3 11 mL 50 glucose and 7 mL 1M MgSO4 with 50 g mL carbenicillin. The cultures were grown at 30 C. for 24 hours with shaking. Cells were harvested by centrifugation and pellets were stored at 20 C. The Fab was purified by resuspending the cell pellet in 35 mL of cold wash buffer Phosphate Buffered Saline PBS 150 mM NaCl containing 10 g mL DNaseI Invitrogen 0.2 mg mL lysozyme USB and 1 mM phenylmethylsulphonylfluoride PMSF Calbiochem . The pellet was resuspended by vortexing rapidly. To allow complete lysis the cells were incubated for 15 minutes at 25 C. Cell debris was pelleted by centrifugation and lysate was loaded on 1 mL protein A sepharose GE Healthcare column preequilibrated with cold wash buffer. The column was washed with 50 mL of cold wash buffer eluted with 3 mL of 0.1 M acetic acid and neutralized with 150 L of 1 M Tris pH 11.0. The Fab was concentrated using Amicon Ultra 15 centrifugal filter units 10 kDa cut off Millipore . The resulting Fab concentration was determined spectrophotometrically 1 OD 1.5 mg mL .

The 1D8 and 1E3 Fabs from Example 1E were tested for binding to linear diubiquitin Boston Biochem monoubiquitin Boston Biochem K1 linked diubiquitin Genentech K48 linked diubiquitin Boston Biochem and K63 linked diubiquitin Boston Biochem in a western blot. 1 g of each protein in 1 LDS buffer Invitrogen with reducing agent was heated at 70 C. for ten minutes and run on 4 12 NuPAGE Bis Tris 1.0 mm gels in MES buffer Invitrogen in duplicate. Gels were transferred at 30 V constant for 1.5 hours by wet transfer in 10 methanol and 1 NuPAGE transfer buffer Invitrogen to 0.2 m nitrocellulose Invitrogen . Non specific binding sites on the membranes were blocked by incubation in 5 milk in PBST for 1.5 hours at 25 C. with shaking. The membranes were then incubated in 5 g mL of 1D8 or 1E3 Fab in 5 milk in PBST for one hour at 25 C. with shaking. The membrane was washed three times in PBST with shaking. The Fabs were detected by incubating the membrane in a 1 10000 dilution of a goat anti human Fab fragment specific HRP conjugated secondary antibody Sigma Aldrich in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then washed three times in PBST followed by one wash in PBS. The secondary antibody was detected using Super Signal West Pico chemiluminescent substrate Thermo Scientific followed by exposure of the blots to film. The 1E3 Fab detects only the linear diubiquitin but not monoubiquitin K11 linked diubiquitin K48 linked diubiquitin or K63 linked diubiquitin see . The 1D8 Fab did not detect any forms of ubiquitin by western blot see .

The affinity of the 1D8 and 1E3 Fabs from Example 1E was analyzed by surface plasmon resonance SPR using a BIACORE 3000 GE Healthcare . Approximately 120 resonance units RUs of linear diubiquitin Boston Biochem K48 linked diubiquitin Boston Biochem and K63 linked diubiquitin Boston Biochem were immobilized on flow cell two flow cell three and flow cell four respectively of a CM5 chip using the amine coupling protocol supplied by the manufacturer. Flow cell one was activated and ethanolamine blocked without immobilizing protein to be used for reference subtraction. Two fold serial dilutions 0.5 500 nM of 1E3 Fab in 10 mM Hepes pH 7.2 150 mM NaCl and 0.01 Tween 20 HBST were injected 60 L total at a flow rate of 30 L minute over each flow cell using HBST as the running buffer. The signal for each flow cell was recorded and the reference signal was subtracted. Different regeneration conditions were scouted. Even with 10 mM HCl the chip surface could not be completely regenerated back to baseline. When 10 mM glycine pH 1.7 was tested this altered the chip surface and decreased the binding capacity.

An alternative approach was tested using a Fab capture method on a BIACORE 3000 GE Healthcare . Approximately 11 000 resonance units RUs of an anti human Fab capture antibody GE Healthcare were immobilized on flow cells one and two of a CM5 chip using the amine coupling protocol supplied by the manufacturer. 10 L of 10 g mL Fab in 10 mM Hepes pH 7.2 150 mM NaCl and 0.01 Tween 20 HBST was injected at a flow rate of 10 L minute over flow cell two resulting in capture of approximately 430 RUs of Fab. Flow cell one had only the capture antibody on it to serve as a reference subtraction. Two fold serial dilutions 1 1000 nM of linear diubiquitin Boston Biochem or K63 linked diubiquitin Boston Biochem in HBST were injected 60 L total at a flow rate of 30 L minute over flow cells one and two. The signal for each flow cell was recorded and the reference signal was subtracted. Following a dissociation period of four minutes the chip surface was regenerated with two injections of 30 L of 10 mM glycine pH 2.1 at a flow rate of 30 L minute. Data were difficult to fit to any binding model because the diubiquitin did not fully dissociate from the chip. In addition the association rates were very fast and binding did not reach a plateau even at the highest concentration of diubiquitin used. Therefore it is difficult to estimate a Kfor these Fabs.

A TAA stop codon was inserted separately into either CDR L1 CDR L2 CDR L3 CDR H3 or both CDRs L3 and H3 resulting in L1 L2 L3 H3 and L3 H3 stop templates respectively for library synthesis using Kunkel mutagenesis. Stop codons force diversity within a particular CDR loop by requiring repair of the stop in order to get full length Fab expression and display on phage. The stop codon mutagenic oligonucleotides listed below were combined with 1 g of the corresponding monovalent phagemid Kunkel DNA. The resulting monovalent Fab phagemid stop templates were used for affinity maturation library generation.

For clones 1D8 and 1E3 the CDR L1 stop mutagenic oligonucleotide used to insert a TAA stop codon at position 24 Kabat numbering within CDR L1 of the light chain was 4D5LC1.stop GTCACCATCACCTGCTAAGCCAGTCAGGATGTG SEQ ID NO 104 . For clones 1D8 and 1E3 the CDR L2 stop mutagenic oligonucleotide used to insert a TAA stop codon at position 50 Kabat numbering within CDR L2 of the light chain was 4D5LC2.stop GAAGCTTCTGATTTACTAAGCATCCTTCCTCTAC SEQ ID NO 105 . For clones 1D8 and 1E3 the CDR L3 stop mutagenic oligonucleotide used to insert a TAA stop codon at position 89 Kabat numbering within CDR L3 of the light chain was 4D5LC3.stop GCAACTTATTACTGTTAACAATCTTATACTACTC SEQ ID NO 106 . The CDR H3 stop mutagenic oligonucleotide used to insert a TAA stop codon at position 95 Kabat numbering within CDR H3 of the heavy chain of clone 1D8 was VH3.1D8.H3stop GCCGTCTATTATTGTGCTCGTTAAGCCGGGTCCCGCTTGTTGTCG SEQ ID NO 107 . The CDR H3 stop mutagenic oligonucleotide used to insert a TAA stop codon at position 98 Kabat numbering within CDR H3 of the heavy chain of clone 1E3 was 413Vh5SRo6 GAGGACACTGCCGTCTATTATTGTGCTCGTGAGGCCTCGTAACTGCCCCCCTACGTTA TGGACTACTGGGGTCAAGGAACACTAGTC SEQ ID NO 108 .

For clone 1F4 the CDR L1 stop mutagenic oligonucleotide used to insert a TAA stop codon at position 27 Kabat numbering within CDR L1 of the light chain was CLC.L1stop CATCACCTGCCGTGCCAGTTAATCCGTGTCCAGCGCTGTAG SEQ ID NO 109 . For clone 1F4 the CDR L2 stop mutagenic oligonucleotide used to insert a TAA stop codon at position 52 Kabat numbering within CDR L2 of the light chain was CLC.L2stop CTTCTGATTTACTCGGCATAAAGCCTCTACTCTGGAGTC SEQ ID NO 110 . For clone 1F4 the CDR L3 stop mutagenic oligonucleotide used to insert a TAA stop codon at position 90 Kabat numbering within CDR L3 of the light chain was CLC4.1F4.L3stop GCAACTTATTACTGTCAGTAATATTATTATTATTCTCCG SEQ ID NO 111 . The CDR H3 stop mutagenic oligonucleotide used to insert a TAA stop codon at position 94 Kabat numbering within CDR H3 of the heavy chain of clone 1F4 was CLC4.1F4.H3stop GCCGTCTATTATTGTGCTTAAGGTTACGTTTGGAAAGGTG SEQ ID NO 112 .

A total of ten affinity maturation libraries were generated for each clone 1F4 1D8 and 1E3 . All libraries were generated by Kunkel mutagenesis see Kunkel 82 488 1985 . In the case of soft randomization degenerate oligonucleotides were synthesized such that the wild type residue would be retained 50 of the time and 50 of the time one of the remaining 19 amino acids would be encoded. To achieve soft randomization oligonucleotides were designed such that certain nucleotide positions were occupied 70 of the time with the indicated base and 10 of the time occupied by one of the three other bases Gallop et al. 37 1233 1994 . For those oligonucleotides that follow where such soft randomization was included at a particular base the presence of soft randomization is indicated by the presence of a number at that base position. The number 5 indicates that the base adenine is present 70 of the time at that position while the bases guanine cytosine and thymine are each present 10 of the time. Similarly the number 6 refers to guanine 7 to cytosine and 8 to thymine where in each case each of the other three bases is present only 10 of the time. In the case of hard randomization degenerate oligonucleotides were synthesized such that amino acid diversity found at certain positions within natural human antibodies would be allowed. In this case degenerate codons were used where the letter R encodes for guanine or adenine Y encodes for thymine or cytosine M encodes for adenine or cytosine K encodes for guanine or thymine S encodes for guanine or cytosine W encodes for adenine or thymine H encodes for adenine cytosine or thymine B encodes for guanine thymine or cytosine V encodes for guanine cytosine or adenine D encodes for guanine adenine or thymine and N encodes for guanine adenine cytosine or thymine.

Ten libraries were generated for 1F4 and designated L1 L2 L3 L1 L2 L3 H3 L3 H3 L1 H2 L2 H1 H2 H3 and L3 H1 H2. The 1F4 L1 library had positions 28 33 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. The L1 mutagenic oligonucleotides F111 L1 ACCTGCCGTGCCAGTCAGRDTRKTRVWANWTHTGTAGCCTGGTATCAACAGAAAC SEQ ID NO 113 and F202 L1 ACCTGCCGTGCCAGTCAGRDTRKTRVWANWT HTCTGGCCTGGTATCAACAGAAAC SEQ ID NO 114 were mixed at a 1 2 ratio resulting in the L1 oligo mix and combined with 20 g of Kunkel DNA of the 1F4 L1 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1F4 L2 library had positions 50 53 and 55 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. The L2 mutagenic oligonucleotides F201 L2 CCGAAGCTTCTGATTTA CKBGGCATCCAVCCTCTACTCTGGAGTCCCT SEQ ID NO 115 and F203 L2 CCGAAGCTTCTGATTTACKBGGCATCCAVCCTCGMATCTGGAGTCCCTTCTCGC SEQ ID NO 116 were mixed at a 1 1 ratio resulting in the L2 oligo mix and combined with 20 g of Kunkel DNA of the 1F4 L2 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1F4 L3 library had positions 91 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. The mutagenic oligonucleotides F133a GCAACTTA TTACTGTCAGCAATMTDMCRVTNHTCCTYKGACGTTCGGACAGGGTACC SEQ ID NO 117 F133b GCAACTTATTACTGTCAGCAATMTDMCRVTNHTC CTTWTACGTTCGGACAGGGTACC SEQ ID NO 118 F133c GCAACTTATT ACTGTCAGCAASRTDMCRVTNHTCCTYKGACGTTCGGACAGGGTACC SEQ ID NO 119 F133d GCAACTTATTACTGTCAGCAASRTDMCRVTNHTCCTT WTACGTTCGGACAGGGTACC SEQ ID NO 120 were mixed at a 1 1 1 1 ratio resulting in the L3 hard oligo mix . The mutagenic oligonucleotides F563 L3soft1 ACTTATTACTGTCAGCAA878857577577CCT777ACGTTCGGACAGGGTACC SEQ ID NO 121 F564 L3soft2 ACTTATTACTGTCAGCAA878857577577CCTTWTACGT TCGGACAGGGTACC SEQ ID NO 122 and F565 L3soft3 ACTTATTACTGTCAGCAA 878857577577CCTYKGACGTTCGGACAGGGTACC SEQ ID NO 123 were mixed at a 1 0.5 1 ratio resulting in the L3 soft oligo mix . The L3 hard oligo mix the L3 soft oligo mix and the mutagenic oligonucleotide 1F4.L3 soft GCAACTTATTACTGTCAG CAA857857857857878CCG788ACGTTCGGACAGGGTACCAAG SEQ ID NO 124 were then mixed at a 1 1 1 ratio resulting in the L3 total oligo mix and combined with 20 g of Kunkel DNA of the 1F4 L3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1F4 H3 library had positions 95 97 99 100 and 100a Kabat numbering of the heavy chain soft randomized. Mutagenic oligonucleotide CLC.1F4.H3soft GACACTGCCGTCTATTATTGTGCTCGC668TAC688TGG555668678ATGGACTACTGGGG TCAAGGAACC SEQ ID NO 125 was combined with 20 g of Kunkel DNA of the 1F4 H3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1F4 L3 H3 library had positions 91 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. It also had positions 95 97 99 100 and 100a Kabat numbering of the heavy chain soft randomized. The L3 total oligo mix and the mutagenic oligonucleotide CLC.1F4.H3soft SEQ ID NO 126 described above were mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1F4 L3 H3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1F4 L1 H2 library had positions 28 33 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. It also had positions 50 52 53 54 and 58 Kabat numbering of the heavy chain soft randomized. The L1 oligo mix described above and the mutagenic oligonucleotide CLC4.1F4.H2soft GGTAAGGGCCTGGAATGGGTTGCA878ATT857TCT857857AG CTATACT878TATGCCGATAGCGTCAAGGGCCG SEQ ID NO 126 were mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1F4 L1 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1F4 L2 H1 library had positions 50 53 and 55 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. It also had positions 30 33 Kabat numbering of the heavy chain soft randomized. The L2 oligo mix described above and the mutagenic oligonucleotide CLC4.1F4.H1soft GCAGCTTCTGGCTTCAACTTT857878857857ATGCACTGGGTGCGTCAGGCC SEQ ID NO 127 were mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1F4 L2 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1F4 H2 H3 library had positions 50 52 53 54 58 95 97 99 100 and 100a Kabat numbering of the heavy chain soft randomized. Mutagenic oligonucleotides CLC4.1F4.H2soft SEQ ID NO 126 and CLC4.1F4.H3soft SEQ ID NO 125 described above were mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1F4 H3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1F4 L1 L2 L3 library had positions 28 33 50 53 and 55 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. It also had positions 91 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. The L1 oligo mix the L2 oligo mix and the L3 total oligo mix were mixed at a 1 1 1 ratio and combined with 20 g of Kunkel DNA of the 1F4 L3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1F4 L3 H1 H2 library had positions 91 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. It also had positions 30 33 50 52 53 54 and 58 Kabat numbering of the heavy chain soft randomized. The L3 total oligo mix CLC4.1F4.H1soft SEQ ID NO 127 and CLC4.1F4.H2soft SEQ ID NO 126 described above were mixed at a 1 1 1 ratio and combined with 20 g of Kunkel DNA of the 1F4 L3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

Ten libraries were generated for 1D8 and designated L1 L2 L3 L1 L2 L3 H3 L3 H3 L1 H2 L2 H1 H2 H3 and L3 H1 H2. The 1D8 l library had positions 28 33 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. The L1 mutagenic oligonucleotides F111 L1 SEQ ID NO 113 and F202 L1 SEQ ID NO 114 described above were mixed at a 1 2 ratio resulting in the L1 oligo mix and combined with 20 g of Kunkel DNA of the 1D8 l stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1D8 L2 library had positions 50 53 and 55 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. The L2 mutagenic oligonucleotides F201 L2 SEQ ID NO 115 and F203 L2 SEQ ID NO 116 described above were mixed at a 1 1 ratio resulting in the L2 oligo mix and combined with 20 g of Kunkel DNA of the 1D8 L2 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1D8 L3 library had positions 91 94 and 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. The mutagenic oligonucleotides F133a SEQ ID NO 117 F133b SEQ ID NO 118 F133c SEQ ID NO 119 and F133d SEQ ID NO 120 described above were mixed at a 1 1 1 1 ratio resulting in the L3 hard oligo mix . The mutagenic oligonucleotides F563 L3soft1 SEQ ID NO 121 F564 L3soft2 SEQ ID NO 122 and F565 L3soft3 SEQ ID NO 123 described above were mixed at a 1 0.5 1 ratio resulting in the L3 soft oligo mix . The L3 hard oligo mix and the L3 soft oligo mix were then mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1D8 L3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1D8 H3 library had positions 96 100c Kabat numbering of the heavy chain soft randomized. Mutagenic oligonucleotide VH3.1D8.H3soft GCCGTCTATTATTGTGCTCG TGAG678668878565788788878688ATGGACTACTGGGGTCAAGGAACC SEQ ID NO 128 was combined with 20 g of Kunkel DNA of the 1D8 H3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1D8 L3 H3 library had positions 91 94 and 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. It also had positions 96 100c Kabat numbering of the heavy chain soft randomized. The L3 soft oligo mix the L3 hard oligo mix and the mutagenic oligonucleotide VH3.1D8.H3soft SEQ ID NO 128 described above were mixed at a 0.5 0.5 1 ratio and combined with 20 g of Kunkel DNA of the 1D8 L3 H3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1D8 L1 H2 library had positions 28 33 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. It also had positions 50 52 53 54 and 58 Kabat numbering of the heavy chain soft randomized. The L1 oligo mix described above and the mutagenic oligonucleotide VH3.1D8.H2soft GGTAAGGGCCTGGAATGGGTTGCT668ATT878CCT857668 GGTTATACT657TATGCCGATAGCGTCAAGGGCCG SEQ ID NO 129 were mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1D8 L1 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1D8 L2 H1 library had positions 50 53 and 55 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. It also had positions 30 33 Kabat numbering of the heavy chain soft randomized. The L2 oligo mix described above and the mutagenic oligonucleotide VH3.1D8.H1soft GCAGCTTCTGGCTTCACCTTC577657857657ATTCACTGGGTGCGTCAGGCC SEQ ID NO 130 were mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1D8 L2 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1D8 H2 H3 library had positions 50 52 53 54 58 and 96 100c Kabat numbering of the heavy chain soft randomized. Mutagenic oligonucleotides VH3.1D8.H2soft SEQ ID NO 129 and VH3.1D8.H3soft SEQ ID NO 128 described above were mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1D8 H3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1D8 L1 L2 L3 library had positions 28 33 50 53 and 55 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. It also had positions 91 94 and 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. The L1 oligo mix the L2 oligo mix the L3 hard oligo mix and the L3 soft oligo mix were mixed at a 1 1 0.5 0.5 ratio and combined with 20 g of Kunkel DNA of the 1D8 L3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1D8 L3 H1 H2 library had positions 91 94 and 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. It also had positions 30 33 50 52 53 54 and 58 Kabat numbering of the heavy chain soft randomized. The L3 hard oligo mix the L3 soft oligo mix VH3.1D8.H1soft SEQ ID NO 130 and VH3.1D8.H2soft SEQ ID NO 129 described above were mixed at a 0.5 0.5 1 1 ratio and combined with 20 g of Kunkel DNA of the 1D8 L3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

Ten libraries were generated for 1E3 and designated L1 L2 L3 L1 L2 L3 H3 L3 H3 L1 H2 L2 H1 H2 1 13 and L3 H1 H2. The 1E3 l library had positions 28 33 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. The L1 mutagenic oligonucleotides F111 L1 SEQ ID NO 113 and F202 L1 SEQ ID NO 114 described above were mixed at a 1 2 ratio resulting in the L1 oligo mix and combined with 20 g of Kunkel DNA of the 1E3 L1 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1E3 L2 library had positions 50 53 and 55 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. The L2 mutagenic oligonucleotides F201 L2 SEQ ID NO 115 and F203 L2 SEQ ID NO 116 described above were mixed at a 1 1 ratio resulting in the L2 oligo mix and combined with 20 g of Kunkel DNA of the 1E3 L2 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1E3 L3 library had positions 91 94 and 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. The mutagenic oligonucleotides F133a SEQ ID NO 117 F133b SEQ ID NO 118 F133c SEQ ID NO 119 and F133d SEQ ID NO 120 described above were mixed at a 1 1 1 1 ratio resulting in the L3 hard oligo mix . The mutagenic oligonucleotides F563 L3soft1 SEQ ID NO 121 F564 L3soft2 SEQ ID NO 122 and F565 L3soft3 SEQ ID NO 123 described above were mixed at a 1 0.5 1 ratio resulting in the L3 soft oligo mix . The L3 hard oligo mix and the L3 soft oligo mix were then mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1E3 L3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1E3 H3 library had positions 95 97 98 99 and 100a Kabat numbering of the heavy chain soft randomized. Mutagenic oligonucleotide VH4.1E3.H3soft GACACTGCCGTCTATTATTGTGCTCGT577TGG788788565TGG688ATGGACTACTGGGG TCAAGGAACCCTG SEQ ID NO 131 was combined with 20 g of Kunkel DNA of the 1E3 H3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1E3 L3 H3 library had positions 91 94 and 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. It also had positions 95 97 98 99 and 100a Kabat numbering of the heavy chain soft randomized. The L3 soft oligo mix the L3 hard oligo mix and the mutagenic oligonucleotide VH4.1E3.H3soft SEQ ID NO 131 described above were mixed at a 0.5 0.5 1 ratio and combined with 20 g of Kunkel DNA of the 1E3 L3 H3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1E3 L1 H2 library had positions 28 33 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. It also had positions 50 52 53 54 and 58 Kabat numbering of the heavy chain soft randomized. The L1 oligo mix described above and the mutagenic oligonucleotide VH4.1E3.H2soft GGTAAGGGCCTGGAATGGGTTGCT878ATT577CCT878878GGTTCTA CT657TATGCCGATAGCGTCAAGGGCCG SEQ ID NO 132 were mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1E3 l stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1E3 L2 H1 library had positions 50 53 and 55 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. It also had positions 30 33 Kabat numbering of the heavy chain soft randomized. The L2 oligo mix described above and the mutagenic oligonucleotide VH4.1E3.H1soft GCAGCTTCTGGCTTCACCTTC878558577857ATTAGCTGGGTGCGTCAGGCC SEQ ID NO 133 were mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1E3 L2 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1E3 H2 H3 library had positions 50 52 53 54 58 95 97 98 99 and 100a Kabat numbering of the heavy chain soft randomized. Mutagenic oligonucleotides VH4.1E3.H2soft SEQ ID NO 132 and VH4.1E3.H3soft SEQ ID NO 133 described above were mixed at a 1 1 ratio and combined with 20 g of Kunkel DNA of the 1E3 H3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1E3 L1 L2 L3 library had positions 28 33 50 53 and 55 Kabat numbering of the light chain hard randomized to allow for amino acid diversity found at these positions within natural human antibodies. It also had positions 91 94 and 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. The L1 oligo mix the L2 oligo mix the L3 hard oligo mix and the L3 soft oligo mix were mixed at a 1 1 0.5 0.5 ratio and combined with 20 g of Kunkel DNA of the 1E3 L3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The 1E3 L3 H1 H2 library had positions 91 94 and 96 Kabat numbering of the light chain either hard randomized to allow for amino acid diversity found at these positions within natural human antibodies or soft randomized. It also had positions 30 33 50 52 53 54 and 58 Kabat numbering of the heavy chain soft randomized. The L3 hard oligo mix the L3 soft oligo mix VH4.1E3.H1soft SEQ ID NO 133 and VH4.1E3.H2soft SEQ ID NO 132 described above were mixed at a 0.5 0.5 1 1 ratio and combined with 20 g of Kunkel DNA of the 1E3 L3 stop template described in Example 2A to generate the library by Kunkel mutagenesis.

The mutagenesis reactions were electroporated into electrocompetent XL1 Blue Agilent and recovered in 25 mL of SOC medium for 45 minutes at 37 C. with shaking. Twenty microliters were removed and ten fold serial dilutions were plated onto solid agar plates containing carbenicillin and grown overnight at 37 C. to determine the library size. The remaining culture was transferred to 500 mL of 2YT broth containing 50 g mL carbenicillin and 1 10phage mL M13K07 helper phage. The cells were infected at 37 C. for one hour with shaking. 50 g mL of kanamycin was added and the cultures were grown for another seven hours at 37 C. with shaking. The temperature was then shifted to 30 C. and the cultures were grown for another 22 hours. The libraries each contained at least 9.5 10colony forming units CFUs . The phage were purified from the culture supernatant by two rounds of precipitation with volume of 20 polyethylene glycol PEG 2.5M NaCl.

The 1F4 1D8 and 1E3 affinity maturation libraries underwent four rounds of sorting. Each of the ten sub libraries were sorted in parallel for the first round and then pooled for sorts two through four. The first round was plate based sorting with linear diubiquitin immobilized on a 96 well Maxisorb immunoplate NUNC . Plates were coated overnight at 4 C. with 5 g mL linear diubiquitin Boston Biochem in 50 mM sodium carbonate buffer pH 9.6. The coated plates were blocked with 200 L well of 2.5 milk in PBS containing 0.05 Tween 20 PBST for one hour at 25 C. with shaking. The phage libraries were diluted to an OD 2.0 in 2.5 milk in PBST and 30 g mL of K63 linked polyubiquitin 2 7 Boston Biochem was added for counterselection. After one hour the blocking buffer was dumped off of the plate and 100 L well of the phage was added and incubated at 25 C. for three hours with shaking. After binding the plate was washed 20 times with PBST by manually filling the wells and dumping off the buffer between washes. Phage were eluted with 150 L well of 50 mM HCl 500 mM KCl for 30 minutes at 25 C. with shaking. The elution was neutralized with 150 L well of 1 M Tris pH 7.5 and subsequently propagated in XL1 Blue Agilent with the addition of M13K07 helper phage.

Amplified phage were used for additional rounds of selection against linear diubiquitin in plate based sorting. Solution based sorting was not possible because biotinylation of linear diubiquitin interfered with Fab binding. Stringency of the later sorts was increased in three ways by adding 30 g mL soluble monoubiquitin K11 linked polyubiquitin K48 linked polyubiquitin 2 7 and K63 linked polyubiquitin 2 7 to the phage for counterselection by increasing the number and duration of plate washes and by decreasing the amount of phage used and the duration of phage binding. The second sort was done exactly as the first sort except that the soluble ubiquitins added was expanded to include the above listed chains the amount of phage used was OD 1.0 the duration of phage binding was decreased to 2 hours and the number of plate washes was increased to 30. The third sort was done exactly as the second sort except that the duration of phage binding was reduced to 1.5 hours and the number of plate washes was increased to 40 followed by four additional washes of 15 minutes each with shaking at 25 C. with five quick washes in between each 15 minute wash. The fourth sort was done exactly as the third sort except that the amount of phage used was reduced to OD 0.5 the duration of phage binding was reduced to 1 hour and the washes included 40 quick washes followed by four 15 minute washes with shaking at 37 C. Enrichment was calculated for rounds two through four by comparing the number of phage recovered with linear diubiquitin compared to an uncoated well. Enrichment was observed in rounds two through four for all three libraries see Table 3 .

After four rounds of sorting 96 individual clones were picked from the 1F4 third round sort 1D8 second round sort 1D8 third round sort 1D8 fourth round sort and 1E3 third round sort and grown up in 96 well format in 1 mL of 2YT broth containing 50 g mL carbenicillin and 1 10phage mL M13K07 helper phage. Supernatants from those cultures were used in high throughput phage spot ELISAs for binding to linear diubiquitin Boston Biochem monoubiquitin Boston Biochem K11 linked diubiquitin Genentech K48 linked diubiquitin Boston Biochem K63 linked diubiquitin Boston Biochem an anti gD antibody Genentech or an uncoated well as described in Example 1B . From the 1F4 third round sort all clones were very weak linear diubiquitin binders showing ODof less than 0.4 and therefore were not pursued. From the 1D8 second round sort 93 of the clones were linear diubiquitin specific but sequencing revealed that all were the parental wild type 1D8 sequence. From the 1D8 third round sort 48 of the clones were linear diubiquitin specific but sequencing revealed that all but two were the parental wild type 1D8 sequence. The two non parental clones 1D8.3C2 SEQ ID NOs 33 and 36 and 1D8.3F8 SEQ ID NOs 34 and 37 see were tested by phage ICELISA as in example 1D and demonstrated to have ICs in the low M range for linear diubiquitin only slightly improved over 1D8 see . From the 1D8 fourth round sort 94 of the clones showed strong binding to both linear diubiquitin and K63 linked diubiquitin ODgreater than 1.0 for linear ODgreater than 0.5 for K63 and therefore were not pursued. Only one clone 1D8.4F5 SEQ ID NOs 35 and 38 see showed strong linear diubiquitin binding with little K63 linked diubiquitin binding OD 1.3 OD 0.1 for K63 . The phage ICfor 1D8.4F5 was also measured by ELISA as in example 1D and determined to be 2 M only slightly better than the parental clone 1D8 see . From the 1E3 third round sort 33 of the clones were linear diubiquitin specific however they were all very weak binders ODof less than 0.5 and therefore were not pursued. An additional 27 clones showed stronger binding to linear diubiquitin ODof greater than 0.5 but less than 1.0 however they also showed increased binding to K63 linked diubiquitin ODof greater than 0.1 and therefore were not pursued.

A TAA stop codon was inserted separately into either CDR L1 CDR L2 CDR L3 CDR H1 CDR H2 and CDR H3 resulting in L1 L2 L3 H1 H2 and H3 stop templates respectively for library synthesis using Kunkel mutagenesis. Stop codons force diversity within a particular CDR loop by requiring repair of the stop in order to get full length Fab expression and display on phage. The stop codon mutagenic oligonucleotides listed below were combined with 1 g of 1E3 monovalent phagemid Kunkel DNA. The resulting monovalent Fab phagemid stop templates were used for affinity maturation library generation.

The mutagenic oligonucleotide used to insert a TAA stop codon at position 31 Kabat numbering within CDR L1 of the 1E3 light chain was E3.L1 stop GCCAGTCAGGATGTG TCCTAAGCTGTAGCCTGGTATCAAC SEQ ID NO 134 . The mutagenic oligonucleotide used to insert a TAA stop codon at position 53 Kabat numbering within CDR L2 of the 1E3 light chain was E3.L2stop CTGATTTACTCGGCATCCTAACTCTACTCTGGAGTCCCTTC SEQ ID NO 135 . The mutagenic oligonucleotide used to insert a TAA stop codon at position 93 Kabat numbering within CDR L3 of the 1E3 light chain was E3.L3stop CTTATTACT GTCAGCAATCTTATTAAACTCCTCCCACGTTCGGACAG SEQ ID NO 136 . The mutagenic oligonucleotide used to insert a TAA stop codon at position 32 Kabat numbering within CDR H1 of the 1E3 light chain was E3.H1stop GGCTTCACCTTC AGTAATTAATATATTAGCTGGGTGCGTC SEQ ID NO 137 . The mutagenic oligonucleotide used to insert a TAA stop codon at position 54 Kabat numbering within CDR H2 of the 1E3 light chain was E3.H2stop GTTGCTTCTATTACTCCTTAAAGCGGTTCTACTGACTATG SEQ ID NO 138 . The mutagenic oligonucleotide used to insert a TAA stop codon at position 99 Kabat numbering within CDR H3 of the 1E3 light chain was E3.H3stop GCTCGTACCTGGTTGCTCTAATGGGTTATGGACTACTGG SEQ ID NO 139 .

Since the first attempt at affinity maturation of 1F4 and 1D8 resulted in only modest improvements in affinity with ICs still in the low M range clone 1E3 which had a starting ICof 80 nM was focused on. The first attempt at 1E3 affinity maturation produced many clones that showed strong binding to both linear and K63 linked diubiquitin. Therefore a different approach was taken to minimize K63 linked diubiquitin binding by limiting the number of mutations incorporated into single clones. Single position NNK randomization was used to incorporate a single amino acid change into one CDR at a time. Six single CDR libraries designated L1 L2 L3 H1 H2 and H3 were generated where a single residue in any one clone was allowed to retain the wild type residue or to change to any one of the other 19 amino acids.

The L1 library had positions 28 34 Kabat numbering of the light chain hard randomized individually using the NNK codon to allow for all 20 amino acids. The L1 mutagenic oligonucleotides

E3.L1.7 CATCACCTGCCGTGCCAGTCAGGATGTGTCCACTGCTGTANNKTGGTATCAA CAGAAACCAGG SEQ ID NO 146 were mixed at a 1 1 1 1 1 1 1 ratio and combined with 20 g of Kunkel DNA of the 1E3 l stop template described in Example 3A to generate the library by Kunkel mutagenesis.

The L2 library had positions 50 56 Kabat numbering of the light chain hard randomized individually using the NNK codon to allow for all 20 amino acids. The L2 mutagenic oligonucleotides

E3.L2.7 GCTCCGAAGCTTCTGATTTACTCGGCATCCTTCCTCTACNNKGGAGTCCCTT CTCGCTTCTCTG SEQ ID NO 153 were mixed at a 1 1 1 1 1 1 1 ratio and combined with 20 of Kunkel DNA of the 1E3 L2 stop template described in Example 3A to generate the library by Kunkel mutagenesis.

The L3 library had positions 91 96 Kabat numbering of the light chain hard randomized individually using the NNK codon to allow for all 20 amino acids. The L3 mutagenic oligonucleotides

E3.L3.6 GCAACTTATTACTGTCAGCAATCTTATACTACTCCTNNKACGTTCGGACA GGGTACCAAG SEQ ID NO 159 were mixed at a 1 1 1 1 1 1 ratio and combined with 20 g of Kunkel DNA of the 1E3 L3 stop template described in Example 3A to generate the library by Kunkel mutagenesis.

The H1 library had positions 30 35 Kabat numbering of the heavy chain hard randomized individually using the NNK codon to allow for all 20 amino acids. The H1 mutagenic oligonucleotides

E3.H1.6 GCAGCTTCTGGCTTCACCTTCAGTAATACTTATATTNNKTGGGTGCG TCAGGCCCCG SEQ ID NO 165 were mixed at a 1 1 1 1 1 1 ratio and combined with 20 g of Kunkel DNA of the 1E3 H1 stop template described in Example 3A to generate the library by Kunkel mutagenesis.

The H2 library had positions 49 58 Kabat numbering of the heavy chain hard randomized individually using the NNK codon to allow for all 20 amino acids. The H2 mutagenic oligonucleotides

E3.H2.10 GGTAAGGGCCTGGAATGGGTTGCTTCTATTACTCCTTCTAGCGGTTCTNNKG ACTATGCCGATAGCGTCAAGGGC SEQ ID NO 175 and

E3.H2.11 GGTAAGGGCCTGGAATGGGTTGCTTCTATTACTCCTTCTAGCGGTTCTACT NNKTATGCCGATAGCGTCAAGGGC SEQ ID NO 176 were mixed at a 1 1 1 1 1 1 1 1 1 1 1 ratio and combined with 20 g of Kunkel DNA of the 1E3 H2 stop template described in Example 3A to generate the library by Kunkel mutagenesis.

The H3 library had positions 95 102 Kabat numbering of the heavy chain hard randomized individually using the NNK codon to allow for all 20 amino acids. The H3 mutagenic oligonucleotides

E3.H3.10 GCCGTCTATTATTGTGCTCGTACCTGGTTGCTCCGGTGGGTTATGGACNNKT GGGGTCAAGGAACCCTGGTC SEQ ID NO 186 were mixed at a 1 1 1 1 1 1 1 1 1 1 ratio and combined with 20 g of Kunkel DNA of the 1E3 H3 stop template described in Example 3A to generate the library by Kunkel mutagenesis.

The mutagenesis reactions were electroporated into electrocompetent XL1 Blue Agilent and recovered in 25 mL of SOC medium for 45 minutes at 37 C. with shaking. Twenty microliters were removed and ten fold serial dilutions were plated onto solid agar plates containing carbenicillin and grown overnight at 37 C. to determine the library size. The remaining culture was transferred to 500 mL of 2YT broth containing 50 g mL carbenicillin and 1 10phage mL M13K07 helper phage. The cells were infected at 37 C. for one hour with shaking. 50 g mL of kanamycin was added and the cultures were grown for another seven hours at 37 C. with shaking. The temperature was then shifted to 30 C. and the cultures were grown for another 22 hours. The libraries each contained at least 1.5 10colony forming units CFUs . The phage were purified from the culture supernatant by two rounds of precipitation with volume of 20 polyethylene glycol PEG 2.5M NaCl.

Sixty four individual clones from each of the six libraries were sequenced to make sure the amino acid diversity in the library accurately reflected the design. The libraries all were as designed.

The six CDR NNK affinity maturation libraries underwent three rounds of sorting in parallel against either linear diubiquitin or K63 linked diubiquitin. Plates were coated overnight at 4 C. with 5 g mL linear diubiquitin Boston Biochem or K63 linked diubiquitin Boston Biochem in 50 mM sodium carbonate buffer pH 9.6. The coated plates were blocked with 200 L well of 2.5 milk in PBS containing 0.05 Tween 20 PBST for one hour at 25 C. with shaking. The phage libraries were diluted to an OD 1.0 in 2.5 milk in PBST. After one hour the blocking buffer was dumped off of the plate and 100 L well of the phage was added and incubated at 25 C. for 1.5 hours with shaking. After binding the plate was washed 10 times with PBST by manually filling the wells and dumping off the buffer between washes. Phage were eluted with 100 L well of 50 mM HCl 500 mM KCl for 30 minutes at 25 C. with shaking. The elution was neutralized with 100 L well of 1 M Tris pH 7.5 and subsequently propagated in XL1 Blue Agilent with the addition of M13K07 helper phage.

Amplified phage were used for additional rounds of selection against linear diubiquitin or K63 linked diubiquitin in plate based sorting. Solution based sorting was not possible because biotinylation of linear diubiquitin interfered with 1E3 Fab binding. Stringency of the later sorts was increased in two ways by increasing the number and duration of plate washes and by decreasing the amount of phage used and the duration of phage binding. The second sort was done exactly as the first sort except that the amount of phage used was OD 0.5 and the number of plate washes were increased to 21 with the last wash incubating at 25 C. with shaking for 5 minutes. The third sort was done exactly as the second sort except that the duration of phage binding was reduced to one hour and the number of plate washes was increased to 30. For the linear diubiquitin sort this was followed by four additional washes of 15 minutes each with shaking at 25 C. with five quick washes in between each 15 minute wash. Then a one hour wash with shaking at 25 C. was performed followed by five quick washes. Enrichment was calculated for rounds two and three by comparing the number of phage recovered with linear diubiquitin or K63 linked diubiquitin compared to an uncoated well. Strong enrichment was observed in rounds two and three for all six libraries sorted against linear diubiquitin with only modest enrichment seen for K63 linked diubiquitin see Table 4 .

After three rounds of sorting 64 individual clones were picked for each of the six libraries from the linear diubiquitin second round sort the linear diubiquitin third round sort and the K63 linked diubiquitin third round sort and grown up in 96 well format in 1 mL of 2YT broth containing 50 g mL carbenicillin and 1 10phage mL M13K07 helper phage. Supernatants from those cultures were used in high throughput phage spot ELISAs for binding to 1 g mL coated linear diubiquitin Boston Biochem K63 linked diubiquitin Boston Biochem an anti gD antibody Genentech or an uncoated well as previously described Example 1B . The variable domains of these clones were also sequenced see Table 5 linear diubiquitin and Table 6 K63 linked diubiquitin . Sequencing of H3 clones revealed that a T110A mutation was present in some clones outside of the region targeted for randomization in the library design in addition to the intended mutation. This is likely due to an oligonucleotide synthesis error or a mutagenesis error.

Single spot competition phage ELISAs were done to determine which clones had the biggest improvement in affinity for linear diubiquitin compared to the parental 1E3 clone. The phage supernatants from the phage spot ELISAs Example 3C were used. The competition ELISA was done as described for the ICELISA Example 1D except phage supernatants were used instead of purified phage and only a single concentration 25 nM of soluble linear diubiquitin Boston Biochem was used. The competition was also done for each clone without addition of any soluble linear diubiquitin to determine the phage binding signal in the absence of any competing antigen. The percent inhibition in binding in the presence of 25 nM linear diubiquitin was calculated as 1 ODfor 25 nM linear ODfor no linear 100 . The 1E3 parental clone showed variable percent inhibition of binding ranging from 20 to 75 inhibition in the presence of 25 nM linear diubiquitin see Table 5 . This was due to variability in the ODfor binding linear diubiquitin in the absence of any competing soluble linear diubiquitin. Clones showing 60 percent inhibition or greater or those which were isolated many times in the linear diubiquitin sort were selected for further analysis by phage ICELISA.

Table 5 below shows CDR L1 L2 L3 H1 H2 and H3 sequences from clones isolated from the sorting of the 1E3 L1 L2 L3 H1 H2 and H3 NNK libraries respectively against linear diubiquitin. Also shown are the ODsignals from the competition spot ELISA in the absence and presence of 25 nM soluble linear diubiquitin. The percent inhibition in binding in the presence of 25 nM linear diubiquitin was calculated as 1 ODfor 25 nM linear ODfor no linear 100 . Table 6 below shows CDR L1 L2 L3 H1 H2 and H3 sequences from clones isolated from the sorting of the 1E3 L1 L2 L3 H1 H2 and H3 NNK libraries respectively against K63 linked diubiquitin.

Table 5 discloses the CDR L1 sequences as SEQ ID NOS 1 56 199 200 200 57 201 202 202 203 204 204 207 207 208 50 50 50 50 50 50 209 209 54 54 54 54 210 55 55 53 211 212 212 212 214 52 52 215 215 215 216 216 217 217 217 218 1 1 1 1 1 1 1 1 1 1 1 1 1 1 and 1 respectively in order of appearance. Table 5 discloses the CDR L2 sequences as SEQ ID NOS 2 59 59 58 58 58 219 60 60 60 60 60 60 60 60 60 60 220 220 220 220 221 221 223 223 226 226 226 62 227 228 228 229 61 61 230 233 233 237 237 238 238 238 239 239 239 2 2 2 2 2 2 2 2 2 2 2 and 2 respectively in order of appearance. Table 5 discloses the CDR L3 sequences as SEQ ID NOS 3 6 6 240 240 240 242 66 66 66 66 243 68 244 244 247 247 248 248 248 70 64 64 64 64 249 250 250 71 71 251 72 72 72 72 72 72 72 69 65 65 65 252 252 252 252 252 67 67 67 67 67 67 67 67 253 3 3 3 3 and 254 respectively in order of appearance. Table 5 discloses the CDR H1 sequences as SEQ ID NOS 255 256 73 79 79 257 257 75 75 75 75 77 77 77 77 77 77 77 258 258 259 81 74 74 74 74 74 74 74 74 74 74 74 74 74 74 78 76 76 76 76 76 76 76 76 76 76 80 80 80 260 260 255 255 255 255 255 255 255 255 255 255 255 255 and 255 respectively in order of appearance. Table 5 discloses the CDR H2 sequences as SEQ ID NOS 8 17 84 261 261 261 261 261 263 85 83 83 264 82 82 82 82 82 82 82 82 82 82 82 86 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 and 8 respectively in order of appearance. Table 5 discloses the CDR H3 sequences as SEQ ID NOS 265 269 269 270 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 273 274 274 274 274 274 278 278 278 278 281 281 282 282 282 282 265 265 265 265 265 265 and 265 respectively in order of appearance.

Table 6 discloses the CDR L1 sequences as SEQ ID NOS 1 283 50 50 50 50 50 50 50 50 50 50 50 50 50 50 50 50 50 50 50 50 50 284 54 285 285 288 55 55 55 53 289 289 290 290 290 290 290 290 294 327 295 295 295 295 297 297 1 1 1 1 1 1 and 1 respectively in order of appearance. Table 6 discloses the CDR L2 sequences as SEQ ID NOS 2 298 299 60 60 300 302 305 305 307 307 307 309 309 309 310 310 314 314 314 314 314 314 314 314 314 314 314 315 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 and 2 respectively in order of appearance. Table 6 discloses the CDR L3 sequences as SEQ ID NOS 3 6 6 6 6 6 316 318 318 319 66 66 320 321 321 321 321 324 324 324 325 301 326 301 326 382 328 328 329 64 64 330 330 331 331 331 331 331 71 71 71 71 71 71 332 69 333 333 67 67 67 67 67 334 334 334 335 335 335 3 3 3 and 3 respectively in order of appearance. Table 6 discloses the CDR H1 sequences as SEQ ID NOS 255 336 337 337 337 337 337 337 337 337 337 337 337 337 337 340 340 343 343 343 343 343 343 344 344 344 344 344 344 81 81 81 81 81 81 74 74 74 74 74 74 74 74 74 74 74 74 74 74 76 76 76 76 76 76 76 76 76 76 and 255 respectively in order of appearance. Table 6 discloses the CDR H2 sequences as SEQ ID NOS 8 346 347 345 348 349 349 349 349 349 349 349 349 349 349 349 349 349 351 351 351 351 352 352 352 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 353 354 8 8 8 8 8 and 8 respectively in order of appearance. Table 6 discloses the CDR H3 sequences as SEQ ID NOS 355 358 358 359 359 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 362 367 355 355 355 355 355 355 355 355 355 and 355 respectively in order of appearance. 0 is an ochre stop TAA and q is an amber stop TAG which can replaced by Gln O for example with the use of suppressor tRNA cell lines.

Eight clones from the L1 library six from the L2 library nine from the L3 library nine from the H1 library five from the H2 library and seven from the H3 library were tested in a phage ICELISA as described in example 1D. Eight three fold serial dilutions of linear diubiquitin Boston Biochem from 500 nM to 0.23 nM were used. In each experiment the WT 1E3 clone was included for comparison. The ICvalue for 1E3 varies between experiments however clones which show higher affinity for linear diubiquitin than the parental clone can be identified. In an attempt to identify mutants with improved affinity for linear diubiquitin with minimal K63 diubiquitin binding clones that were tested in the ICELISA were narrowed down by taking into account whether they had improved ICvalues compared to the parental 1E3 whether they were isolated in the K63 linked diubiquitin sort example 3C as well as their signal for binding to K63 linked diubiquitin in the phage spot ELISA example 3C . Clones with improved ICvalues compared to 1E3 that were isolated multiple times in the linear diubiquitin sort but were not isolated in the K63 linked diubiquitin sort and demonstrated no K63 linked diubiquitin binding in the spot ELISA ODof less than 0.1 were selected for further analysis 1F11 2A2 2C11 2H5 3E4 4C9 4E4 and 4G7 see Table 7 . If a clone with an improved ICthat was isolated multiple times in the linear diubiquitin sort was isolated only once in the K63 linked diubiquitin sort and demonstrated no K63 linked diubiquitin binding in the phage spot ELISA then it was considered for further analysis 3F5 see Table 7 . Clones 3A7 and 4C10 which were isolated multiple times in both the linear diubiquitin and the K63 linked diubiquitin sorts were chosen as negative controls. These 11 clones along with 1E3 were tested further in a phage specificity ELISA. Table 7 below shows the CDR L1 L2 L3 H1 H2 and H3 sequences of clones from the sorting of the 1E3 L1 L2 L3 H1 H2 and H3 NNK libraries respectively against linear diubiquitin that were further characterized by phage ICELISA. The ICand the fold improvement over the parental 1E3 ICis given. Binding to K63 linked diubiquitin is defined as having an ODof greater than 0.1 in the phage spot ELISA. Also shown is the percent inhibition in binding in the presence of 25 nM soluble linear diubiquitin in the phage spot competition ELISA. The number of times each clone was isolated in the library sorts against linear diubiquitin or K63 linked diubiquitin is indicated.

Table 7 discloses the CDR L1 sequences as SEQ ID NOS 1 1 and 50 57 respectively in order of appearance. Table 7 discloses the CDR L2 sequences as SEQ ID NOS 2 and 58 63 respectively in order of appearance. Table 7 discloses the CDR L3 sequences as SEQ ID NOS 3 3 64 65 3 and 66 72 respectively in order of appearance. Table 7 discloses the CDR H1 sequences as SEQ ID NOS 255 and 73 81 respectively in order of appearance. Table 7 discloses the CDR H2 sequences as SEQ ID NOS 8 and 82 86 respectively in order of appearance. Table 7 discloses the CDR H3 sequences as SEQ ID NOS 368 372 368 and 373 375 respectively in order of appearance.

1F11 2A2 2C11 2H5 3E4 4C9 4E4 4G7 3F5 3A7 and 4C10 along with the parental clone 1E3 were tested in a phage specificity ELISA against monoubiquitin Boston Biochem linear diubiquitin Boston Biochem K11 linked diubiquitin Genentech K48 linked diubiquitin Boston Biochem K63 linked diubiquitin Boston Biochem an anti gD antibody Genentech or an uncoated well. The panel of ubiquitin proteins was immobilized on 96 well Maxisorb immunoplates NUNC see . Plates were coated at 4 C. overnight with 1 g mL of each protein in 50 mM sodium carbonate buffer pH 9.6. The coated plates were blocked with 200 L well of 2.5 milk in PBST for one hour at 25 C. with shaking. Six two fold serial dilutions of phage from OD 1.0 to OD 0.03 were made in 2.5 milk in PBST. After one hour the blocking buffer was dumped off of the plate and 100 L of the phage serial dilutions was added. Plates were incubated at 25 C. for one hour with shaking. The plate was then washed six times with PBST using a plate washer. A 1 5 000 dilution of an anti M13 horseradish peroxidase HRP conjugated secondary antibody GE Healthcare in PBST was used for detection of phage binding. 100 L well of the secondary dilution was added and the plate was incubated at 25 C. for 1.25 hours with shaking. The plate was then washed six times with PBST using a plate washer and twice manually with PBS. Bound secondary antibody was detected using a TMB substrate KPL followed by quenching with an equal volume of 1 M phosphoric acid. The absorbance was read at 450 nm.

Clones 3A7 and 4C10 which were isolated multiple times in both the linear diubiquitin and the K63 linked diubiquitin sorts were used as negative controls. Both demonstrated significant K63 linked binding at a phage OD 1.0 OD 0.44 and OD 0.5 for 3A7 and 4C10 respectively see . 2A2 which was isolated twice in the K63 linked diubiquitin sort showed intermediate levels of K63 linked binding OD 0.2 at a phage OD 1.0 . All other clones demonstrated negligible K63 linked diubiquitin binding OD

Clones which demonstrated improved ICs over the parental 1E3 example 3E and showed negligible K63 linked diubiquitin binding in the phage specificity ELISA example 3F were chosen for further consideration. These included light chain mutants 1F11 2C11 and 2H5 and heavy chain mutants 3E4 3F5 and 4G7. To see whether there was an additive effect in affinity improvement double mutants were constructed combining different combinations of light and heavy chain mutations. Light chain variable domains were removed by digesting the phagemids with EcoRV and KpnI and were then cloned into the various heavy chain mutant phagemids using the same sites.

Double mutants 1F11 3E4 1F11 3F5 1F11 4G7 2C11 3E4 2C11 3F5 2C11 4G7 2H5 3E4 2H5 3F5 and 2H5 4G7 were compared to their respective single mutants and parental 1E3 in a phage ICELISA as previously described examples 1D and 3E . Double mutants 1F11 3F5 and 2H5 3F5 demonstrated an additive improvement in affinity over their individual single mutants see Table 8 . 2C11 3F5 was border line additive. The single mutant 3F5 in this particular assay gave a lower IC 9 nM than in the other two experiments 12 or 13 nM which could make 2C11 3F5 ICof 10 nM appear not to be additive. Therefore 2C11 3F5 was also chosen for further consideration.

Double mutants 1F11 3F5 2C11 3F5 and 2H5 3F5 were compared to their respective single mutants and parental 1E3 in a phage specificity ELISA as in example 3F. All double mutants showed negligible K63 linked diubiquitin binding OD

The parental clone 1E3 single mutants 1F11 2C11 2H5 3F5 and double mutants 1F11 3F5 2C11 3F5 2H5 3F5 were cloned as Fab expression constructs by inserting a TAA stop codon into the phagemids at the end of the CH1 domain as described in example 1E. These Fabs were expressed in and purified as described in example 1E.

The parental clone 1E3 single mutants 1F11 2C11 2H5 3F5 and double mutants 1F11 3F5 2C11 3F5 2H5 3F5 were also expressed in HEK293 cells as human immunoglobulins IgGs . Expression constructs were generated by cloning the Fab variable domains into pRK mammalian expression constructs encoding the heavy and light chains of human kappa IgG1 Gorman et al. DNA Prot. Eng. Tech. 2 3 10 1990 . IgGs were purified by affinity chromatography on protein A sepharose columns by standard methodologies as described for the Fab purification in Example 1E.

The affinity of the double mutant Fabs from Example 3J was analyzed by surface plasmon resonance SPR using a BIACORE 3000 GE Healthcare and direct binding as described in example 1G. Approximately 150 resonance units RUs of linear diubiquitin Boston Biochem K48 linked diubiquitin Boston Biochem and K63 linked diubiquitin Boston Biochem were immobilized on flow cell two flow cell three and flow cell four respectively of a CM5 chip using the amine coupling protocol supplied by the manufacturer. Flow cell one was activated and ethanolamine blocked without immobilizing protein to be used for reference subtraction. Even with 10 mM glycine pH 1.7 the chip surface could not be regenerated back to baseline and an increase in RUs was seen suggesting that the chip surface was altered.

An alternative approach was tested using an IgG capture method with the IgGs from example 3K on a BIACORE 3000 GE Healthcare . Approximately 8 000 resonance units RUs of an anti human Fc capture antibody GE Healthcare were immobilized on flow cells one and two of a CM5 chip using the amine coupling protocol supplied by the manufacturer. 60 L of 1 g mL IgG in 10 mM Hepes pH 7.2 150 mM NaCl and 0.01 Tween 20 HBST was injected at a flow rate of 30 L minute over flow cell two resulting in capture of approximately 750 RUs of IgG. Flow cell one had only the capture antibody on it to serve as a reference subtraction. Two fold serial dilutions 3.9 500 nM of linear diubiquitin Boston Biochem or K63 linked diubiquitin Boston Biochem in HBST were injected 60 L total at a flow rate of 30 L minute over flow cells one and two. The signal for each flow cell was recorded and the reference signal was subtracted. Following a dissociation period of four minutes the chip surface was regenerated with one injection of 15 L of 3M MgClat a flow rate of 30 L minute. Much like the Fab capture Biacore experiment see example 1G data were difficult to fit to any binding model because the diubiquitin did not fully dissociate from the chip. In addition the association rates were very fast and binding did not reach a plateau even at the highest concentration of diubiquitin used. Therefore it is difficult to estimate a KD for these IgGs.

To rank the affinity and specificity of the double mutants the IgGs described in example 3K were tested in a western blot for binding to linear diubiquitin Boston Biochem and K63 linked diubiquitin Boston Biochem . 1 g of K63 linked diubiquitin and five three fold serial dilutions of linear diubiquitin 1000 333 111 37 12 ng in 1 LDS buffer Invitrogen with reducing agent was heated at 70 C. for ten minutes and run on 4 12 NuPAGE Bis Tris 1.0 mm gels in MES buffer Invitrogen . Gels were transferred at 30 V constant for 1 hour by wet transfer in 10 methanol and 1 NuPAGE transfer buffer Invitrogen to 0.2 m nitrocellulose Invitrogen . Non specific binding sites on the membranes were blocked by incubation in 5 milk in PBST for 1 hour at 25 C. with shaking. The membranes were then incubated in 1 g mL of 1E3 1F11 2C11 2H5 3F5 1F11 3F5 2C11 3F5 or 2H5 3F5 IgG in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were washed three times in PBST with shaking. The IgGs were detected by incubating the membrane in a 1 10 000 dilution of a goat anti human Fc fragment specific IR Dye 800CW conjugated secondary antibody Rockland Immunochemicals in 5 milk in PBST for 30 minutes at 25 C. with shaking. The membranes were then washed three times in PBST followed by one wash in PBS. The secondary antibody was detected and quantified using the LI COR Odyssey infrared imaging system LI COR Biosciences .

Single mutants 1F11 and 3F5 were considerably more sensitive than the parental 1E3 whereas single mutants 2C11 and 2H5 were only slightly improved see . The double mutant 1F11 3F5 had an additive improvement in sensitivity over the respective single mutants whereas 2C11 3F5 and 2H5 3F5 were not any better than 3F5 alone.

To see whether a further improvement in affinity could be achieved by combining three mutations the 1F11 2C11 3F5 and 1F11 2H5 3F5 triple mutants were generated. Mutagenic oligonucleotides 5 1F11 S52K CCGAAGCTTCTGATTTACTCGGCAAAGTTCCTCTA CTCTGGAGTCCC SEQ ID NO 187 and 3 1F11 S52K GGGACTCCAGAGTAG AGGAACTTTGCCGAGTAAATCAGAAGCTTCGG SEQ ID NO 188 were combined with either the 2C11 or 2H5 pRK light chain constructs from example 3K and the QuikChange Lightning Site Directed Mutagenesis kit Agilent was used to generate the double mutant light chains. Triple mutants were then generated by combining the double mutant light chain pRK constructs with the 3F5 heavy chain pRK construct and IgGs were expressed in HEK293 cells and purified as described in example 3K.

To rank the affinity and specificity of the triple mutants 1F11 2C11 3F5 and 1F11 2H5 3F5 the IgGs described in example 3N were tested in a western blot for binding to linear diubiquitin Boston Biochem and K63 linked diubiquitin Boston Biochem as described in example 3M. Neither 1F11 2C11 3F5 nor 1F11 2H5 3F5 was more sensitive than the double mutant 1F11 3F5 see .

One H3 mutant 4E4 that demonstrated an improved ICvalue over the parental 1E3 in example 3E was not originally considered when making double mutants because it had a small amount of K63 linked diubiquitin binding in the specificity ELISA OD 0.16 at a phage OD 1.0 see example 3F . Since none of the IgG mutants tested so far demonstrated any K63 linked diubiquitin binding in the western blots the 4E4 clone was analyzed further. 4E4 was an H3 clone that actually contained two mutations Y102L immediately adjacent to CDR H3 and T110A in framework 4. To determine whether the unintentional T110A mutation had any affect on affinity both the Y102L single mutant and the Y102L T110A double mutant heavy chain was made in the context of the 1E3 parental heavy chain or the mutant 3F5 heavy chain. To insert the Y102L mutation mutagenic oligonucleotides 5 Y102L CGGTGGGTTATGGACCTGTGGGGTCAAGGAACCCTGGTC ACCGTCTCCTCGGCCTCC SEQ ID NO 189 and 3 Y102L GGAGGCCGAGGAGA CGGTGACCAGGGTTCCTTGACCCCACAGGTCCATAACCCACCG SEQ ID NO 190 were combined with either the 1E3 or 3F5 IgG heavy chain pRK expression construct and the mutants were synthesized using the QuikChange Lightning Site Directed Mutagenesis kit Agilent . To insert the Y102L T110A mutations mutagenic oligonucleotides 5 Y102L T110A CGGTGGGTTATGGACCTGTGGGGTCAAGGAACCCTGGTCGCGGTCTCCTCGGCCTCC SEQ ID NO 191 and 3 Y102L T110A GGAGGCCGAGGAGACCGCGACCAGG GTTCCTTGACCCCACAGGTCCATAACCCACCG SEQ ID NO 192 were combined with either the 1E3 or 3F5 IgG heavy chain pRK expression construct and the mutants were synthesized using the QuikChange Lightning Site Directed Mutagenesis kit Agilent . The resulting heavy chain IgG pRK constructs were combined with either the parental 1E3 or the mutant 1F11 light chain IgG pRK constructs and the resulting IgGs were expressed in HEK293 cells and purified as described in example 3K.

These mutants Y012L vs. Y102L T110A 3F5 Y102L vs. 3F5 Y102L T110A 1F11 Y102L vs. 1F11 Y102L T110A and 1F11 3F5 Y102L vs. 1F11 3F5 Y102L T110A were then compared side by side in a western blot for binding linear or K63 linked diubiquitin as described in example 3M. Generally having the T110A mutation in combination with Y102L did not significantly improve sensitivity compared to Y102L alone see . Also from the phage ICs it is known that T110A alone clone 4E1 does not improve affinity compared to the parental 1E3 see Table 7 . Therefore the Y102L mutation alone was considered for further analysis.

The triple mutant 1F11 3F5 Y102L was then compared to the parental 1E3 each of the single mutants 1F11 3F5 Y102L as well as the double mutants 1F11 3F5 1F11 Y102L 3F5 Y102L side by side in a western blot for binding linear or K63 linked diubiquitin as described in example 3M. The triple mutant was more sensitive than the parental clone 1E3 all of the single mutants and all of the double mutants for binding linear diubiquitin see . In addition it showed no binding to K63 linked diubiquitin indicating that the specificity is maintained.

The 1E3 parental and 1F11 3F5 Y102L IgGs were tested for their ability to detect pure diubiquitin chains in a western blot. Seven two fold serial dilutions of linear diubiquitin Boston Biochem from 1 g to 16 ng and 1 g each of monoubiquitin Boston Biochem K11 linked diubiquitin Genentech K48 linked diubiquitin Boston Biochem and K63 linked diubiquitin Boston Biochem in 1 LDS buffer Invitrogen with reducing agent Invitrogen were heated at 70 C. for ten minutes and run on 4 12 Bis Tris NuPAGE 1.0 mm gels Invitrogen in MES buffer Invitrogen in triplicate. One gel was stained by SimplyBlue Coomassie stain Invitrogen to detect all proteins. For comparison to get an idea of affinity a second western blot was done with the anti K63 antibody Apu3.A8 which has a known KD of 8.7 nM for K63 linked diubiquitin see Newton K. et al. 2008 Cell 134 668 678 . For this western seven two fold serial dilutions of K63 linked diubiquitin Boston Biochem from 1 g to 16 ng and 1 g each of monoubiquitin Boston Biochem linear diubiquitin Boston Biochem K11 linked diubiquitin Genentech and K48 linked diubiquitin Boston Biochem were run on the gel as described above. The three gels were transferred individually at 30 V constant for one hour by wet transfer in 10 methanol and 1 NuPAGE transfer buffer Invitrogen to 0.2 M nitrocellulose Invitrogen . Non specific binding sites on the membranes were blocked by incubation in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then incubated in 1 g mL of 1E3 1F11 3F5 Y102L or Apu3.A8 in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were washed three times in PBST with shaking. The IgGs were detected by incubating the membranes in a 1 10000 dilution of a goat anti human Fc specific HRP conjugated F ab 2 secondary antibody Jackson Immunoresearch in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then washed three times in PBST followed by one wash in PBS. The secondary antibody was detected using Super Signal West Pico chemiluminescent substrate Thermo Scientific followed by exposure of the blots to film.

The 1E3 limit of detection was 250 ng of linear diubiquitin see . In contrast 1F11 3F5 Y102L was much more sensitive and could detect as little as 31 ng of linear diubiquitin. In addition 1F11 3F5 Y102L was highly specific demonstrating no binding to any of the other forms of ubiquitin tested. For comparison the limit of detection of Apu3.A8 which has a Kof 8.7 nM was 62 ng. The Kof 1F11 3F5 Y102L for linear diubiquitin is therefore likely in the low nM range.

The linear specific antibodies were generated against a linear diubiquitin antigen so they presumably recognize either the linkage itself or the surrounding surface residues on the proximal and distal ubiquitins that are placed in close proximity due to the conformation of diubiquitin which results from the linear linkage. Since diubiquitin is the smallest recognition unit of antigen and linear polyubiquitin is a polymeric chain with diubiquitin as the repeating monomer unit the antibodies should also bind the polyubiquitin form. To examine this the 1F11 3F5 Y102L IgG was tested for its ability to detect pure polyubiquitin chains in a western blot. 1 g each of monoubiquitin Boston Biochem linear polyubiquitin 2 7 Enzo Lifesciences K11 linked polyubiquitin Genentech K48 linked polyubiquitin 2 7 Boston Biochem and K63 linked polyubiquitin 2 7 Boston Biochem in 1 LDS buffer Invitrogen with reducing agent Invitrogen was heated at 70 C. for ten minutes and run on 4 12 Bis Tris NuPAGE 1.0 mm gels Invitrogen in MES buffer Invitrogen in triplicate. One gel was stained by SimplyBlue Coomassie stain Invitrogen to detect all proteins. The other two gels were transferred separately at 30 V constant for one hour by wet transfer in 10 methanol and 1 NuPAGE transfer buffer Invitrogen to 0.2 m nitrocellulose Invitrogen . Non specific binding sites on the membranes were blocked by incubation in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then incubated in 1 g mL of 1F11 3F5 Y102L IgG or a 1 200 dilution of a mouse pan ubiquitin antibody P4D1 non linkage specific Santa Cruz Biotechnology in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were washed three times in PBST with shaking. The 1F11 3F5 Y102L IgG was detected by incubating the membrane in a 1 10000 dilution of a goat anti human Fey specific HRP conjugated F ab 2 secondary antibody Jackson Immunoresearch in 5 milk in PBST for one hour at 25 C. with shaking. The P4D1 IgG was detected by incubating the membrane in a 1 10 000 dilution of a goat anti mouse Fey specific HRP conjugated F ab 2 secondary antibody Jackson Immunoresearch in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then washed three times in PBST followed by one wash in PBS. The secondary antibodies were detected using Super Signal West Pico chemiluminescent substrate Thermo Scientific followed by exposure of the blots to film. Whereas the control pan ubiquitin antibody P4D1 recognizes monoubiquitin linear polyubiquitin 2 7 albeit poorly K11 linked polyubiquitin K48 linked polyubiquitin 2 7 and K63 linked polyubiquitin 2 7 the 1F11 3F5 Y102L IgG recognizes only linear polyubiquitin see . Thus just as with linear diubiquitin the 1F11 3F5 Y102L antibody can detect polyubiquitin chains containing the linear linkage but does not recognize polyubiquitin chains of other linkages.

HeLa S3 cells were grown in suspension culture in 50 50 F 12 Dulbecco s Modified Eagle Media DMEM supplemented with 10 fetal bovine serum FBS 2 mM L glutamine 1 glycine hypoxanthine thymidine GHT solution and 1 penicillin streptomycin. The day before the experiment the cells were split 1 2. The cells were grown overnight until reaching a density of 0.38 10cells mL 98 viable . The cells were divided into three flasks of 1.5 L of cells each. Flask 1 was pretreated with 5.8 M MG132 Cayman Chemicals for 10 minutes. Flask 2 and 3 received no pretreatment. At time zero flasks 2 and 3 also were treated with 5.8 M MG132. In addition at time zero flasks 1 and 3 were treated with 100 ng mL TNF Shenandoah Biotechnology and flask 2 was treated with 500 ng mL TNF . At time zero five minutes and 20 minutes 444 mL of cells were removed from each flask spun down at 800 rpm for five minutes at 4 C. and the supernatants were aspirated. Cells were immediately washed with 40 mL of cold PBS pelleted at 800 rpm for five minutes at 4 C. and the supernatants were aspirated. Each pellet was lysed in 13 mL of cold lysis buffer 20 mM Tris pH 7.5 150 mM NaCl 1 mM EDTA 1 Triton X 100 10 mM N ethylmaleimide NEM 25 M MG132 50 mM NaF Complete protease inhibitor cocktail tablets Roche and PhosSTOP phosphatase inhibitor cocktail tablets Roche for 10 minutes at 4 C. with rocking. Debris was pelleted by spinning at 10 000 g for five minutes at 4 C. Lysates were precleared with 133 L of Protein A Dynabeads Invitrogen for 1 hour at 4 C. with rocking. Beads were pelleted by spinning at 2000 rpm for five minutes. The supernatant was removed and stored at 80 C.

Linear polyubiquitin chains have been suggested to play a signaling role in the NF B pathway. Therefore the above lysates were probed with 1F11 3F5 Y102L in a western blot. Thirteen L of each lysate in 1 LDS sample buffer Invitrogen with reducing agent Invitrogen was heated at 70 C. for ten minutes and run on 4 12 Bis Tris NuPAGE 1.0 mm gels Invitrogen in MES buffer Invitrogen in duplicate. As specificity controls 250 ng of purified linear Enzo Lifesciences and K63 linked polyubiquitin chains Boston Biochem were run on the gel. The gels were transferred individually at 30 V constant for one hour by wet transfer in 10 methanol and 1 NuPAGE transfer buffer Invitrogen to 0.45 m nitrocellulose Invitrogen . Non specific binding sites on the membranes were blocked by incubation in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then incubated in 1 g mL of 1F11 3F5 Y102L or Apu3.A8 anti K63 in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were washed three times in PBST with shaking. The IgGs were detected by incubating the membranes in a 1 10 000 dilution of a goat anti human Fc specific HRP conjugated F ab 2 secondary antibody Jackson Immunoresearch in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then washed three times in PBST followed by one wash in PBS. The secondary antibody was detected using Super Signal West Pico chemiluminescent substrate Thermo Scientific followed by exposure of the blots to film. An additional western blot was done to assess activation of the NF B pathway by probing for I B levels. Upon TNF signaling this leads to ubiquitination and degradation of the inhibitor of NF B I B . Five L of the above lysates in 1 LDS sample buffer Invitrogen with reducing agent Invitrogen was heated at 70 C. for ten minutes and run on a 4 12 Bis Tris NuPAGE 1.0 mm gel Invitrogen in MES buffer Invitrogen in duplicate. The gel was transferred at 30 V constant for two hours by wet transfer in 20 methanol and 1 NuPAGE transfer buffer Invitrogen to Invitrolon PVDF Invitrogen . The membrane was blocked in 5 milk in PBST for one hour at 25 C. with shaking and then probed with 1 1000 dilutions of an anti I B Cell Signaling and an anti tubulin Cell Signaling antibody as a loading control at 4 C. overnight with shaking. The following day the blots were washed three times in PBST with shaking. The IgGs were detected by incubating the membranes in a 1 10 000 dilution of a goat anti rabbit Fey specific HRP conjugated F ab 2 secondary antibody Jackson Immunoresearch in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then washed three times in PBST followed by one wash in PBS. The secondary antibody was detected using Super Signal West Pico chemiluminescent substrate Thermo Scientific followed by exposure of the blots to film.

In cells which were stimulated with 100 ng mL TNF and no pretreatment with MG132 the amount of linear polyubiquitin chains increases from time zero to five minutes to 20 minutes see . In contrast K63 linked polyubiquitin chains which were much more abundant at all three time points showed an increase from zero to five minutes and then a decrease back down to the starting levels at 20 minutes. In cells which were stimulated with 500 ng mL TNF and no pretreatment with MG132 the linear polyubiquitin chains demonstrated the same pattern of increasing from zero to 20 minutes however the abundance of the linear chains at each time point was increased compared to the cells treated with 100 ng mL TNF . In contrast the K63 linked chains demonstrated the same pattern and abundance as compared to the cells treated with 100 ng mL TNF . In cells which were pretreated for 10 minutes with 5.8 M MG132 and then 100 ng mL TNF the levels of both linear and K63 linked chains remained fairly constant over the three time points. The blot for I B demonstrates that under all three experimental conditions the NF B pathway is activated as evidenced by degradation of I B upon TNF treatment over time however the extent of activation in greater in the absence of MG132 pretreatment see . This demonstrates that 1F11 3F5 Y102L can recognize endogenous linear polyubiquitin chains and suggests that these chains are up regulated in HeLa S3 cells upon TNF stimulation.

The 1F11 3F5 Y102L IgG was tested to see whether it is capable of immunoprecipitating linear polyubiquitin chains. As a positive control the Apu3.A8 anti K63 antibody was also used to monitor immunoprecipitation of K63 linked chains. As a negative control an unrelated human kappa IgG1 antibody was used as an isotype control. Three immunoprecipitation IP conditions were tested. In reaction 1 which contained all chains 2 g each of monoubiquitin Boston Biochem linear polyubiquitin 2 7 Enzo Lifesciences K11 linked polyubiquitin Genentech K48 linked polyubiquitin 2 7 Boston Biochem and K63 linked polyubiquitin 2 7 Boston Biochem were mixed. In reaction 2 which lacked linear chains 2 g each of monoubiquitin K11 linked polyubiquitin K48 linked polyubiquitin 2 7 and K63 linked polyubiquitin 2 7 were mixed. Reaction 3 consisted of 2 g of linear polyubiquitin 2 7 chains alone. Each reaction was diluted in 500 L of 4 M urea IP buffer 4 M urea 20 mM Tris pH 7.5 135 mM NaCl 1 Triton X 100 10 glycerol 1 mM EDTA 1.5 mM MgCl . The reactions were precleared with 50 L of Protein A Dynabeads Invitrogen for three hours at 25 C. with rotation. The beads were then captured on a magnetic stand and the supernatants were transferred to new tubes. Twenty g of 1F11 3F5 Y102L anti linear Apu3.A8 anti K63 or an isotype control IgG was added to each IP reaction and incubated overnight at 25 C. with rotation. The following day 100 L of Protein A Dynabeads were added to each reaction and the IgGs were captured for 15 minutes at 25 C. with rotation. The beads were then washed three times with 1 mL each of 4M urea IP buffer followed by two washes with 1 mL each of PBS. During the final wash the beads were transferred to new tubes to avoid eluting any proteins bound to the tube walls. The beads were resuspended in 30 L of 1 LDS sample buffer Invitrogen with reducing agent Invitrogen and heated at 70 C. for 10 minutes to elute the immunoprecipitated proteins. The beads were then captured on a magnetic stand and the supernatant was split in half and loaded in duplicate onto two 4 12 Bis Tris NuPAGE 1.0 mm gels Invitrogen . As positive and negative controls 1 g each of purified linear polyubiquitin 2 7 Enzo Lifesciences and K63 linked polyubiquitin 2 7 Boston Biochem were also run on the gels. The gels were run in MES buffer Invitrogen and were then transferred individually at 30 V constant for one hour by wet transfer in 10 methanol and 1 NuPAGE transfer buffer Invitrogen to 0.2 m nitrocellulose Invitrogen . Non specific binding sites on the membranes were blocked by incubation in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then incubated in 1 g mL of 1F11 3F5 Y102L or Apu3.A8 anti K63 in 5 milk in PBST for 1.5 hours at 25 C. with shaking. The membranes were washed three times in PBST with shaking. The IgGs were detected by incubating the membranes in a 1 10 000 dilution of a goat anti human Fc specific HRP conjugated F ab 2 secondary antibody Jackson Immunoresearch in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then washed three times in PBST followed by one wash in PBS. The secondary antibody was detected using Super Signal West Pico chemiluminescent substrate Thermo Scientific followed by exposure of the blots to film. 1F11 3F5 Y102L is able to immunoprecipitate linear polyubiquitin chains in 4M urea however it is not specific under these conditions as it is also able to pull down K63 linked chains see .

To determine whether different concentrations of urea could help improve specificity immunoprecipitations were carried out under different buffer conditions. Two g each of linear polyubiquitin 2 7 and K63 linked polyubiquitin 2 7 were mixed and diluted with 500 l of IP buffer 20 mM Tris pH 7.5 135 mM NaCl 1 Triton X 100 10 glycerol 1 mM EDTA 1.5 mM MgCl containing 0 2 4 or 6 M urea. An additional IP was done using 500 L of PBST. The reactions were precleared with 50 L of Protein A Dynabeads Invitrogen for 30 minutes at 25 C. with rotation. The beads were then captured on a magnetic stand and the supernatants were transferred to new tubes. Twenty g of 1F11 3F5 Y102L anti linear or Apu3.A8 anti K63 IgG was added to each IP reaction and incubated at 25 C. with rotation for one hour. Next 100 L of Protein A Dynabeads were added to each reaction and the IgGs were captured for 15 minutes at 25 C. with rotation. The beads were then washed three times with 1 mL each of the corresponding buffer used in the IP 0 2 4 or 6 M urea FP buffer or PBST followed by two washes with 1 mL each of PBS. During the final wash the beads were transferred to new tubes to avoid eluting any proteins bound to the tube walls. The beads were resuspended in 20 L at of 1 LDS sample buffer Invitrogen with reducing agent Invitrogen and heated at 70 C. for 10 minutes to elute the immunoprecipitated proteins. The beads were then captured on a magnetic stand and the supernatant was split in half and loaded in duplicate onto two 4 12 Bis Tris NuPAGE 1.0 mm gels Invitrogen . As positive and negative controls 1 g each of purified linear polyubiquitin 2 7 and K63 linked polyubiquitin 2 7 were also run on the gels. The gels were run in MES buffer Invitrogen and were then transferred individually at 30 V constant for one hour by wet transfer in 10 methanol and 1 NuPAGE transfer buffer Invitrogen to 0.2 m nitrocellulose Invitrogen . Non specific binding sites on the membranes were blocked by incubation in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then incubated in 1 g mL of 1F11 3F5 Y102L or Apu3.A8 anti K63 in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were washed three times in PBST with shaking. The IgGs were detected by incubating the membranes in a 1 10 000 dilution of a goat anti human Fey specific HRP conjugated F ab 2 secondary antibody Jackson Immunoresearch in 5 milk in PBST for one hour at 25 C. with shaking. The membranes were then washed three times in PBST followed by one wash in PBS. The secondary antibody was detected using Super Signal West Pico chemiluminescent substrate Thermo Scientific followed by exposure of the blots to film. As the concentration of urea is increased in the IP buffer the 1F11 3F5 Y102L IP becomes more specific see . At 6 M urea very little K63 linked polyubiquitin is pulled down by the 1F11 3F5 Y102L IgG and yet it is still able to pull down a significant amount of linear polyubiquitin.

To see whether even higher concentrations of urea could further improve specificity the IPs were repeated using 6 7 or 8 M urea IP buffer. Two g each of linear polyubiquitin 2 7 and K63 linked polyubiquitin 2 7 were mixed and diluted with 500 of IP buffer 20 mM Tris pH 7.5 135 mM NaCl 1 Triton X 100 10 glycerol 1 mM EDTA 1.5 mM MgCl containing 6 7 or 8 M urea. The reactions were precleared with 50 L of Protein A Dynabeads Invitrogen for 15 minutes at 25 C. with rotation. The beads were then captured on a magnetic stand and the supernatants were transferred to new tubes. Twenty g of 1F11 3F5 Y102L anti linear Apu3.A8 anti K63 or an isotype control IgG was added to each IP reaction and incubated at 25 C. with rotation for one hour. Next 100 L of Protein A Dynabeads were added to each reaction and the IgGs were captured for 15 minutes at 25 C. with rotation. The beads were then washed three times with 1 mL each of the corresponding buffer used in the IP 6 7 or 8 M urea IP buffer followed by two washes with 1 mL each of PBS. During the final wash the beads were transferred to new tubes to avoid eluting any proteins bound to the tube walls. The beads were resuspended in 30 L of 1 LDS sample buffer Invitrogen with reducing agent Invitrogen and heated at 70 C. for 10 minutes to elute the immunoprecipitated proteins. The beads were then captured on a magnetic stand and the supernatant was split in half and loaded in duplicate onto two 4 12 Bis Tris NuPAGE 1.0 mm gels Invitrogen . As positive and negative controls 1 g each of purified linear polyubiquitin 2 7 and K63 linked polyubiquitin 2 7 were also run on the gels. The gels were run in MES buffer Invitrogen and the western blots were performed as described in the preceding paragraph. One blot was probed with 1F11 3F5 Y102L to detect linear chains and the other was probed with Apu3.A8 anti K63 to detect K63 linked chains. In 6 M urea there is a small amount of K63 linked chains pulled down by 1F11 3F5 Y102L as seen in the previous IPs see . In 7 M urea 1F11 3F5 Y102L behaves just like the isotype control and does not pull down any K63 linked chains however it still retains the ability to IP a significant amount of linear polyubiquitin when compared to what was present in the starting material. In 8 M urea however the amount of linear chains pulled down by 1F11 3F5 Y102L is dramatically reduced. Thus 7 M urea is the most specific condition which strikes a balance between pulling down a significant amount of linear chains without bringing down K63 linked chains.

In the previous IPs the heavy chain of the IgGs used in the pull down is eluted from the beads and detected by the secondary antibody used in the western blots. This band can often obscure bands of the pull down material. To make the blots cleaner and to be absolutely sure no K63 linked chains were being pulled down the IgGs were crosslinked to the beads before the IPs. Twenty g of either 1F11 3F5 Y102L or an isotype control IgG were incubated with 200 L of Protein A Dynabeads in PBST for 30 minutes at 25 C. with rotation. The beads were captured on a magnetic stand and washed twice with 800 L of conjugation buffer 20 mM sodium phosphate pH 7.5 150 mM NaCl . The IgG coupled beads were resuspended in 1 mL of 5 mM Bis sulfosuccinimidyl suberate BS3 in conjugation buffer and incubated at 25 C. with rotation for 30 minutes to crosslink. While crosslinking the IgGs to the beads the IP reactions were set up. Four g each of linear polyubiquitin 2 7 K11 linked polyubiquitin K48 linked polyubiquitin 2 7 and K63 linked polyubiquitin 2 7 were mixed and diluted with 500 L of IP buffer 20 mM Tris pH 7.5 135 mM NaCl 1 Triton X 100 10 glycerol 1 mM EDTA 1.5 mM MgCl containing 0 4 or 7 M urea. The mixtures were precleared with 50 L of Protein A Dynabeads for 45 minutes at 25 C. with rotation. After 30 minutes the crosslinking reaction was quenched by the addition of 48 L of 1 M Tris pH 7.5 and incubation at 25 C. with rotation for 15 minutes. The crosslinked beads were washed three times with 800 L of IP buffer containing the corresponding amount of urea to be used in the IP i.e. 0 4 or 7 M urea . After washing the IgG crosslinked beads were resuspended in the precleared IP reactions and incubated at 25 C. with rotation for one hour. The IgG crosslinked beads were then washed three times with 1 mL of IP buffer containing the corresponding amount of urea followed by two washes with 1 mL of PBS. During the final wash the beads were transferred to new tubes to avoid eluting any proteins bound to the tube walls. The beads were resuspended in 50 L of 1 LDS sample buffer Invitrogen with reducing agent Invitrogen and heated at 70 C. for 10 minutes to elute the immunoprecipitated proteins. The beads were then captured on a magnetic stand and the supernatant was split and loaded in quadruplicate onto 4 12 Bis Tris NuPAGE 1.0 mm gels Invitrogen . As positive and negative controls 1 g each of purified linear polyubiquitin 2 7 K11 linked polyubiquitin K48 linked polyubiquitin and K63 linked polyubiquitin 2 7 were also run on the gels. The gels were run in MES buffer Invitrogen and the western blots were performed as described above in this example. One blot was probed with 1F11 3F5 Y102L to detect linear chains one blot was probed with 2A3 2E6 anti K11 to detect K11 linked chains one blot was probed with Apu2.07 anti K48 to detect K48 linked chains and one blot was probed with Apu3.A8 anti K63 to detect K63 linked chains. When comparing the amount of material in the IPs with the amount of material present in the starting inputs overall much less linear polyubiquitin was pulled down in each IP with 1F11 3F5 Y102L compared to the IPs done with free IgG that was subsequently captured on the beads in the previous paragraphs see . This could be due to the fact that 1F11 3F5 Y102L is a member of the human IgG1 VH3 subgroup which contains a second Protein A binding site in the heavy chain variable domain in addition to the usual binding site in the Fc domain. Therefore precoupling and crosslinking of the IgGs to Protein A beads before antigen binding could diminish the binding capacity of this antibody. Under these precoupling and crosslinking conditions 4 M urea is the most specific condition where 1F11 3F5 Y102L is able to pull down linear chains without bringing down any K11 K48 or K63 linked chains.

To see whether the reduction in linear chains pulled down was due to the additional Protein A binding site in the heavy chain variable domain the IPs were done using IgGs crosslinked to Protein G beads. Protein G also has two binding sites on human IgG1 but both are in the constant domains CH1 and Fc . The IPs were repeated as described above except Protein G Dynabeads Invitrogen were used for precoupling and crosslinking the IgGs. Much more linear polyubiquitin chains were pulled down in each condition by 1F11 3F5 Y102L compared to the experiments with the IgGs crosslinked to Protein A see . Upon over exposure of the anti K63 blot at 4 M urea there is a small amount of K63 linked chains pulled down by 1F11 3F5 Y102L. Thus 7 M urea seems to be the most specific condition when 1F11 3F5 Y102L is precoupled to Protein G beads however this comes at the expense of pulling down less linear polyubiquitin compared to when the IPs are done with free IgG that is subsequently captured on beads compare to . It is possible that precoupling and crosslinking of Protein G beads to the CH1 domain may also diminish binding to linear polyubiquitin through steric hindrance of the neighboring VH domain although less significantly than precoupling and crosslinking of Protein A beads to the VH domain.

The IPs using free IgG followed by subsequent capture with Protein A beads were repeated using a mixture of polyubiquitin chains of different linkages as the substrate and were analyzed more extensively by western blot and mass spectrometry. Two g each of linear polyubiquitin 2 7 Boston Biochem K11 linked polyubiquitin Genentech K48 linked polyubiquitin 2 7 Boston Biochem and K63 linked polyubiquitin 2 7 Boston Biochem were mixed and diluted with 500 L of IP buffer 20 mM Tris pH 7.5 135 mM NaCl 1 Triton X 100 10 glycerol 1 mM EDTA 1.5 mM MgCl containing 0 4 5 6 or 7 M urea. Each IP was done in duplicate one for western blots and the other for mass spectrometry analysis. The reactions were precleared with 50 L of Protein A Dynabeads Invitrogen for 15 minutes at 25 C. with rotation. The beads were then captured on a magnetic stand and the supernatants were transferred to new tubes. Twenty g of 1F11 3F5 Y102L anti linear or an isotype control IgG was added to each IP reaction and incubated at 25 C. with rotation for one hour. Next 100 L of Protein A Dynabeads were added to each reaction and the IgGs were captured for 15 minutes at 25 C. with rotation. The beads were then washed three times with 1 mL each of the corresponding buffer used in the IP 0 4 5 6 or 7 M urea IP buffer followed by two washes with 1 mL each of PBS. During the final wash the beads were transferred to new tubes to avoid eluting any proteins bound to the tube walls. The beads were resuspended in 50 L of 1 LDS sample buffer Invitrogen with reducing agent Invitrogen and heated at 70 C. for 10 minutes to elute the immunoprecipitated proteins. The beads were then captured on a magnetic stand and the supernatant was split and loaded in quadruplicate onto 4 12 Bis Tris NuPAGE 1.0 mm gels Invitrogen for western blots. As positive and negative controls 500 ng each of purified linear polyubiquitin 2 7 K11 linked polyubiquitin K48 linked polyubiquitin 2 7 and K63 linked polyubiquitin 2 7 were also run on the gels. The other set of IPs were run on 4 12 Bis Tris NuPAGE 1.0 mm gels Invitrogen for mass spectrometry AQUA analysis. The gels were run in MES buffer Invitrogen and the western blots were performed as described above in this example. The blots were probed with 1F11 3F5 Y102L anti linear polyubiquitin 2A3 2E6 anti K11 linked polyubiquitin Apu2.07 anti K48 linked polyubiquitin and Apu3.A8 anti K63 polyubiquitin antibodies see . The other gels for mass spectrometry AQUA were stained with SimplyBlue Coomasie Safe stain Invitrogen see . In the absence of urea F11 3F5 Y102L is able to IP chains of all linkages see . As the concentration of urea is increased in the IP buffer the 1F11 3F5 Y102L IP becomes more specific. At 7 M urea no K11 linked K48 linked or K63 linked polyubiquitin is pulled down by the 1F11 3F5 Y102L IgG and yet it is still able to pull down a significant amount of linear polyubiquitin relative to the starting input.

Regions B and C of the Coomassie stained gel were excised subjected to in gel tryptic digestion and analyzed by mass spectrometry AQUA see . Gel pieces were destained using 50 mM ammonium bicarbonate 50 methanol and then desiccated with acetonitrile ACN . To permit effective uptake of trypsin gel pieces were incubated on ice for 2 hr with 20 ng L modified sequencing grade trypsin Promega Madison Wis. diluted in 50 mM ammonium bicarbonate 5 ACN. Digests were performed overnight at 37 C. and stopped by the addition of 50 ACN 5 formic acid FA . Isotope labeled internal standard peptides 1 pmol were added to each sample prior to two rounds of extraction 1st 50 ACN 5 FA 2nd 100 ACN . Extracted peptides were dried completely and resuspended in 10 ACN 5 FA 0.01 H2O2 at least 30 minutes prior to mass spectrometric analysis. Samples were loaded directly onto a Thermo AQUASIL C18 column 2.1 150 mm and separated using an Agilent 1200 capillary LC at a flow rate of 200 l min over a 26 minute gradient of 5 to 90 buffer B 98 ACN 0.1 FA . Mass spectrometric detection was performed on an ABI 4000 QTRAP using a segmented multiple reaction monitoring MRM method for detecting both labeled and unlabeled peptides covering the sequence of ubiquitin. Quantitation was performed by comparing peak areas between labeled and unlabeled versions of each peptide using ABI Multiquant 1.1 software. By measuring the abundances of the GG signature peptides corresponding to modification of the seven lysines of ubiquitin and the amino terminus 7M urea was confirmed to be the most specific condition for immunoprecipitation of linear polyubiquitin chains see . In 7M urea the antibody is able to recover 67 of the linear chains present in the input from region B which contains the longer chains tetraubiquitin through heptaubiquitin however it is less efficient at recovering the short chains found in region C diubiquitin and triubiquitin with only 2 of linear chains recovered see . This is likely due to avidity from the bivalent antibody and the multiple linkages found in longer chains.

The Linear Ubiquitin Assembly Complex LUBAC is an E3 ligase has been demonstrated to assemble linear polyubiquitin chains Kirisako T. et al. 2006 25 4877 4887 . The open reading frames ORFs of two of the members of this complex Hoil 1L and Hoip were synthesized Blue Heron Biotechnology and cloned into the pBI CMV1 mammalian expression vector Clonetech containing a bidirectional CMV promoter. Hod 1L was cloned into multiple cloning site MCS 1 using restriction enzyme sites MluI and EcoRV and Hoip was cloned into the MCS2 using restriction enzyme sites EcoRI and PstI. The construct was verified by sequencing the ORFs. The resulting construct pBI CMV1 Hoil1L Hoip or the empty vector was transfected into 293T cells. 293T cells were split 1 20 into 20 ten cm plates two days before transfection and grown at 37 C. in 5 CO2. On the day of transfection 150 L of Lipofectamine 2000 was diluted into 2.5 mL of Opti MEM media lacking serum for each plasmid to be transfected. Also 50 g of either pBI CMV1 empty vector or pBI CMV1 Hoil1L Hoip was diluted into 2.5 mL of Opti MEM media lacking serum. These dilutions were incubated at 25 C. for five minutes. The diluted Lipofectamine and the diluted DNA were combined mixed gently by inversion and incubated 25 C. for 30 minutes. 500 L of the DNA Lipofectamine mixture was then added to each ten cm plate ten plates per plasmid . Forty eight hours after transfection the media was collected and the cells were scraped off the plates. The cells were spun down at 10 000 rpm for ten minutes at 4 C. The supernatants were removed and the cells were washed in 40 mL of cold PBS. The cells were spun down at 10 000 rpm for ten minutes at 4 C. The supernatants were removed and the cells were resuspended in 4 mL of lysis buffer containing 8M urea 50 mM Tris pH 7.5 25 mM NaCl 10 L mL HALT protease and phosphatase inhibitors Thermo Scientific 5 mM EDTA and 2 mM NEM. Lysates were sonicated briefly to reduce viscosity and then frozen at 80 C. To determine whether Hoil 1L and Hoip were overexpressed and whether this leads to an increase in linear polyubiquitin chain assembly the lysates were analyzed by western blot. One L of each lysate was mixed with LDS sample buffer containing reducing reagent and then loaded onto a 4 12 NuPAGE Bis Tris 1.0 mm gel Invitrogen and run in MES buffer Invitrogen in quadruplicate. The gels were transferred individually at 30V for two hours in 1 NuPAGE transfer buffer containing 10 methanol to 0.45 m nitrocellulose. The membranes were blocked in 5 milk in PBST for one hour at 25 C. with shaking and then incubated in the primary antibody. The first blot was probed with 1 t mL 1F11 3F5 Y102L anti linear polyUb IgG in 5 milk in PBST. The second blot was probed with a 1 500 dilution of an anti Hoil 1L RBCK antibody Abcam ab38540 in 5 milk in PBST. The third blot was probed with 1 g mL anti Hoip RNF31 antibody Abcam ab85294 in 5 milk in PBST. The fourth blot was probed with a 1 1000 dilution of an anti tubulin antibody Cell Signaling 9F3 2128 in 5 milk in PBST. Blots one two and three were incubated in their respective primary antibodies for one hour at 25 C. with shaking. The anti tubulin blot was incubated overnight at 4 C. with rotation. Blots were then washed three times in PBST0.05 and then incubated in a 1 10 000 dilution of secondary antibody in 5 milk in PBST for one hour at 25 C. with shaking. The anti linear polyUb blot was probed with a goat anti human F ab 2 HRP secondary Jackson Immunoresearch and the other three blots were probed with a goat anti rabbit F ab 2 HRP secondary Jackson Immunoresearch . The blots were then washed three times with PBST and then once with PBS. The secondary antibody was detected using Super Signal West Pico chemiluminescent substrate Thermo Scientific followed by exposure of the blots to film. The anti tubulin blot demonstrates that equal amounts of cells were used in the transfections and that equivalent amounts of lysate were loaded on the gel see . The anti Hoil 1L and anti Hoip blots show that when pBI CMV1 Hoil1L Hoip is transfected both Hoil 1L and Hoip are overexpressed relative to endogenous levels of the two proteins. Finally overexpression of Hoil 1L and Hoip leads to a dramatic increase in the levels of linear polyUb chains as expected. Given that these are detected by the 1F11 3F5 Y102L IgG indicates that this antibody can recognize endogenous enzymatically synthesized linear polyUb chains.

In addition to probing these lysates by western blot immunoprecipitations of linear polyUb was also performed. 500 L of the above lysates were diluted to 7M urea with 71 L of 50 mM Tris pH 7.5 25 mM NaCl. The lysates were then precleared with 200 l of Protein A Dynabeads Invitrogen in 7M urea 50 mM Tris pH 7.5 25 mM NaCl for one hour at 25 C. with rotation. The beads were captured with a magnetic stand and the supernatants were transferred to new tubes. The precleared lysates were then spun at 14 000 rpm for five minutes to pellet any precipitation. The supernatants were transferred to new tubes and 40 g of 1F11 3F5 Y102L IgG or an isotype control IgG were added. The immunoprecipitations were then incubated overnight at 25 C. with rotation. The following day the IgGs were captured by the addition of 200 L of Protein A Dynabeads in 7M urea 50 mM Tris pH 7.5 25 mM NaCl for 15 minutes at 25 C. with rotation. The beads were captured on a magnetic stand and washed five times with 1 mL of 7M urea 50 mM Tris pH 7.5 25 mM NaCl and then three times with 1 mL of PBS. After the final wash the beads were transferred to new tubes to avoid eluting any protein sticking to the tube walls. The beads were then resuspended in 30 l of 1 LDS sample buffer with reducing agent Invitrogen and eluted at 70 C. for ten minutes. Two 4 12 NuPAGE Bis Tris 1.0 mm gels were run one for western blot and the other for Coomassie staining for mass spec AQUA analysis. For western blot 10 of each IP was run along with 0.1 of each lysate. For mass spec 90 of each IP was run along with 1 of each lysate. The western blot with 1F11 3F5 Y102L was done as described in the above paragraph except the transfer done for one hour. The gel for mass spec AQUA analysis was stained with Simply Blue Safe Stain Invitrogen . The western blot shows that the 1F11 3F5 Y102L IgG can immunoprecipitate linear polyUb chains from cells lysates see . More chains are pulled down from the LUBAC overexpression lysates than from the vector alone consistent with the much higher levels of linear chains present when LUBAC is overexpressed.

The high molecular weight regions of the gel indicated were excised see and subjected to mass spectrometry AQUA as described above in Example 4D. In the vector control samples no linear polyubiquitin linkages were identified in the input samples see Table 9 . The majority of the polyubiquitin chains identified were of the K48 and K63 linkages.

Consistent with this no linear polyubiquitin was pulled down in the IP with either the anti linear antibody or the isotype control antibody. In LUBAC over expressed cells the levels of linear polyubiquitin linkages reaches 4808 pmols which is similar in abundance to K63 linkages 5075 pmols . When the anti linear antibody is used for IP it significantly enriches for linear linkages with some additional K11 K48 and K63 linkages being pulled down see . This is likely due to the presence of mixed linkage chains or substrates modified with multiple homogenous chains of different linkages since only 7 and 4 pmols of K48 and K63 linkages respectively were pulled down by the anti linear antibody from the vector control cells see Table 9 . Additionally only 1 pmol of K48 linkages and no K68 linkages were identified in the IP with the isotype control antibody from LUBAC over expressed cells indicating that these linkages are simply not sticky. This demonstrates that the anti linear antibody is able to enrich for endogenous linear polyubiquitin chains see .

To determine whether 1F11 3F5 Y102L is functional for immunofluorescence HeLa cells over expressing Hoil 1L and Hoip were stained with the antibody. HeLa cells 5 000 cells 100 l well were seeded in a clear bottomed black walled 96 well plate and grown for 24 h. The cells were transfected with Hoil Hoip plasmids for 18 hours using lipofectamine 2000 according to manufacturer s protocols. The cells were rinsed with PBS fixed with ice cold methanol at 20 C. for 10 minutes permeabilized with PBS 0.1 Triton X 100 at room temperature for 5 min then blocked with PBS 0.3 Triton X 100 5 BSA at room temperature for 1 hour. The linear ubiquitin antibody 1 g ml was incubated with or without poly ubiquitin chains 5 g ml at room temperature for 1 hour and then used to label cells at room temperature for 1 hour. After 6 washes 10 min each with PBS 0.05 Triton X 100 the cells were stained with DyLight488 conjugated donkey anti human antibody 1 500 Jackson ImmunoResearch Laboratories at room temperature for 1 hour. The cells then were washed 6 times 10 minutes each with PBS containing 0.05 Triton X 100 stained with Hoechst 1 10 000 at room temperature for 10 min and washed with PBS. The plate was covered with black seals and imaged using ImageXpress Micro imaging system. Untransfected cells did not show any signal when stained with 1F11 3F5 Y102L but cells over expressing Hoil 1L and Hoip demonstrated a cytoplasmic punctate staining pattern see . This staining was specific for linear polyubiquitin chains as it could be blocked by the addition of recombinant linear polyubiquitin.

To better understand the interaction of the anti linear Fab with linear polyubiquitin the 1F11 3F5 Y102L Fab was co crystallized with linear diubiquitin. The Fab fragment of 1F11 3F5 Y102L was expressed in and purified over a Protein A column as described above in Example 1E. The Fab was further purified over a 5 mL SP HiTrap column GE Healthcare in 20 mM 2 N morpholino ethanesulfonic acid MES pH 5.5 with a 0 100 linear gradient of 20 mM MES pH 5.5 0.5 M NaCl. Fractions containing the Fab fragment were pooled and run over a 320 mL S75 sizing column GE Healthcare in 25 mM Tris pH 7.5 150 mM NaCl. Fractions containing the Fab fragment were pooled and concentrated to 22 mg mL.

A head to tail fusion of two ubiquitin subunits through a canonical peptide bond linear diubiquitin was also expressed in . BL21 Gold Agilent Technologies cells were transformed with a linear diubiquitin expression plasmid constructed in a pET15b vector Novagen . This construct has an N terminal Histag SEQ ID NO 376 followed by a thrombin cleavage site under the control of the T7 promoter and lac operator. An overnight culture of BL21 Gold transformed with the pET15b linear diubiquitin expression plasmid grown in LB media was diluted 50 fold into 1 L of warmed Terrific Broth containing 1 glycerol 0.1 M MOPS pH 7.3 and 50 g mL carbenicilin. The culture was grown at 37 C. with shaking at 250 rpms in a 2.5 L ultra yield flask Thomson to an OD 1.64. Expression was induced by adding 0.5 mM IPTG and the culture was grown overnight at 16 C. with shaking at 250 rpms. Next day the cells were pelleted by spinning at 8K rpm for 10 minutes and the pellets were frozen in liquid nitrogen. Cells were resuspended and lysed in 40 mM Tris pH 8.0 0.3 M NaCl and Complete EDTA free protease inhibitor tablets Roche by microfluidizing three times. Cell debris was pelleted by spinning at 10K rpm for 1 hour and the supernatant was filtered through a 0.45 M low protein binding filter. The His diUb His disclosed as SEQ ID NO 376 was purified over a 5 mL Ni NTA agarose Qiagen column. The column was washed with 12 column volumes of Buffer A 20 mM Tris pH 8.0 1 M NaCl 20 mM imidazole and eluted with four column volumes of Buffer B 20 mM Tris pH 8.0 1 M NaCl 250 mM imidazole . The expression of the diubiquitin was high such that it exceeded the binding capacity of the column and some diubiquitin was eluted in the wash with Buffer A. Thus the wash material was run over a second 5 mL Ni NTA agarose column and purified as described above. The Histag SEQ ID NO 376 was removed by cleaving with 1800 units of Thrombin GE Healthcare at 4 C. for 3 days while dialyzing into 25 mM Tris pH 8.0 150 mM NaCl 2 mM CaClusing a 3500 MWCO dialysis tubing Spectrum Medical . The dialyzed and cleaved material was split in half and the free diubiquitin was separated from the Histag SEQ ID NO 376 using two 5 mL Ni NTA agarose columns. The flow through and all washes were collected. The columns were washed with four column volumes of Buffer C 25 mM Tris pH 8.0 0.5 M NaCl four column volumes of Buffer D 25 mM Tris pH 8.0 0.5 M NaCl 20 mM imidazole and then one column volume of Buffer E 25 mM Tris pH 8.0 0.5 M NaCl 250 mM imidazole . The presence of detagged diubiquitin was monitored by running samples from the flow through and each wash on an 18 Novex tris glycine gel Invitrogen and staining with Simply Blue Safe Stain Invitrogen . The majority of the detagged diubiquitin was present in the flow through wash with Buffer C and wash with Buffer D. These were pooled concentrated and purified further over a 320 mL S75 sizing column GE Healthcare in 25 mM Tris pH 7.5 150 mM NaCl. Fractions containing the diubiquitin were pooled and concentrated to 15.3 mg mL.

The Fab diubiquitin complex was set up using a three fold molar excess of linear diubiquitin to Fab and was incubated at 4 C. overnight. The complex was then purified over a 320 mL S75 sizing column GE Healthcare in 25 mM Tris pH 7.5 150 mM NaCl. Fractions containing the complex were pooled and concentrated to 20 mg mL.

Crystals were grown using the sitting drop vapor diffusion method in drops containing 0.2 L of 1F11 3F5 Y102L Fab linear diubiquitin complex 20 mg mL complex in 25 mM Tris pH 7.5 150 mM NaCl and 0.2 L of mother liquor 20 isopropanol 0.1 M MES pH 6.0 20 polyethylene glycol PEG 2K monomethylether MME . Initial crystals grew under these conditions at 19 C. over 27 days. Crystals were optimized in sitting drops using a microbridge with 2 L of 1F11 3F5 Y102L Fab linear diubiquitin complex 20 mg mL complex in 25 mM Tris pH 7.5 150 mM NaCl and 3 L of mother liquor 18 isopropanol 0.09 M MES pH 6.0 19.8 PEG 2K MME 10 mM sodium bromide . Crystals grew at 19 C. over five days and could be manipulated to obtain single diffracting crystals. Crystals were cryoprotected using 20 isopropanol 0.1 M MES pH 6.0 20 PEG 2K MME with an additional 25 30 PEG 2K MME. Crystallographic data was collected at Berkeley Advanced Light Source beamline 5.0.2 and was processed using HKL2000. Crystals belonged to the P1 space group with unit cell dimensions a 53 b 60 c 96 87 77 and 72 with two complexes in the asymmetric unit. The structure was solved by molecular replacement using the program Phaser and the coordinates of a variant of the humanized 4D5 Fab fragment PDB code for 4D5 1FVE and of human monoubiquitin PDB code 1UBQ . Model building was carried out in Coot and the structure was refined using Phenix. The resolution of the structure is 2.43 and the complex has been refined to an R of 22.8 and Rfree of 25.0 see . Panel A .

Analysis of the structure indicates that the majority of the binding to diubiquitin is mediated through contacts with the heavy chain CDRs. Between the heavy chain and diubiquitin there is 785 of buried surface area in contrast to no buried surface area between the light chain and diubiquitin. The structural epitope and paratope is defined as residues which bury at least 25 of their solvent accessible surface area upon binding see Panels B and C and or have at least one atom within 4.5 of the interacting chain see Tables 10 and 11 .

There are 15 heavy chain residues and only five light chain residues which make up the Fab paratope. The diubiquitin epitope consists of 18 residues from the distal Ub C terminus involved in the linkage and eight residues from the proximal ubiquitin N terminus involved in the linkage . The contact interface also consists of nine hydrogen bonds between the heavy chain and diubiquitin and three hydrogen bonds between the light chain and diubiquitin see Table 12 . Rather than making exclusive contact with the linear linkage itself the specificity appears to be derived from multiple interactions with surface residues from both the proximal and distal ubiquitins.

Despite free linear polyubiquitin and free K63 linked polyubiquitin chains adopting similar structures due to Lys63 and the free amino terminus situated only 6 apart in the monoubiquitin structure PDB code 1UBQ this antibody preferentially binds linear chains. This specificity is achieved through the dual recognition of the relative orientations of both the proximal and distal ubiquitin subunits. Recognition of the proximal ubiquitin is achieved through interaction of CDR H2 residues with the 60 s loop of the proximal subunit. In particular two hydrogen bonds are made with the side chain of Gln62 of the proximal ubiquitin both involving residues of CDR H2. One is between the main chain carbonyl oxygen of Pro52a and the side chain amine of Gln62. The other is between the main chain amide of Gly55 and the side chain carbonyl of Gln62. There is also a third hydrogen bond between the main chain carbonyl of Ser54 and the main chain amide of Lys63. In addition to these three hydrogen bonds numerous Van der Waals interactions also occur between CDR H2 and the proximal ubiquitin. Although the distal ubiquitin of K63 linked diubiquitin could theoretically bind in the same orientation to the antibody the proximal ubiquitin would be rotated slightly due to the 6 difference in position of Met1 and Lys63. This rotation would likely disrupt the hydrogen bonds and Van der Waals interactions between CDR H2 and the 60 s loop. Therefore specificity is encoded by recognition of the relative spatial orientations of the proximal and distal ubiquitins resulting from the linear linkage.

The structure also illustrates why the particular mutations selected in the affinity maturation help increase affinity. In CDR H2 the mutation that made the most significant improvement in sensitivity in western blots was S56Q see clone 3F5 . The structure shows that Gln56 makes two hydrogen bonds one with the carbonyl oxygen of Gly75 of the distal ubiquitin and the second with the carbonyl oxygen of Gly76 of the distal ubiquitin which participates in the linear linkage between the distal and proximal ubiquitins see Panel A . The shorter side chain of Ser likely would not reach these residues to make the hydrogen bonds. The mutation which made the second largest improvement in sensitivity in western blots was S52K in CDR L2 see clone 1F11 . Lys52 is placed in proximity to Asp32 found at the carboxy terminal end of the alpha helix of the distal ubiquitin. Although Lys52 is too far away to make a salt bridge with the side chain of Asp32 it may make a favorable electrostatic interaction given that it is also oriented towards the negative end of the helix dipole see Panel B . The third mutation which improved sensitivity of the antibody was Y102L in CDR H3 see clone Y102L . Although this residue does not contact the diubiquitin it may help stabilize a favorable conformation of CDR H3 for improved binding. Leu102 intercalates between Val2 and Leu4 of framework 1 of the heavy chain see Panel C . The more hydrophobic side chain of Leu could provide a more favorable interaction with Val2 and Leu4 than Tyr and might help position the rest of CDR H3 to make contacts with the diubiquitin.

Although the foregoing invention has been described in some detail by way of illustration and example for purposes of clarity of understanding the descriptions and examples should not be construed as limiting the scope of the invention. The disclosures of all patent and scientific literature cited herein are expressly incorporated in their entirety by reference.

