---

title: System and methods for detecting genetic variation
abstract: The invention provides methods, apparatuses, and compositions for high-throughput amplification sequencing of specific target sequences in one or more samples. In some aspects, barcode-tagged polynucleotides are sequenced simultaneously and sample sources are identified on the basis of barcode sequences. In some aspects, sequencing data are used to determine one or more genotypes at one or more loci comprising a causal genetic variant. In some aspects, systems and methods of detecting genetic variation are provided.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09092401&OS=09092401&RS=09092401
owner: Counsyl, Inc.
number: 09092401
owner_city: South San Francisco
owner_country: US
publication_date: 20121031
---
Next generation sequencing NGS allows small scale inexpensive genome sequencing with a turnaround time measured in days. However as NGS is generally performed and understood all regions of the genome are sequenced with roughly equal probability meaning that a large amount of genomic sequence is collected and discarded to collect sequence information from the relatively low percentage of areas where function is understood well enough to interpret potential mutations. Generally purifying from a full genome sample only those regions one is interested in is conducted as a separate step from sequencing. It is usually a days long low efficiency process in the current state of the art.

Direct Targeted Sequencing DTS is a modification to the standard sequencing protocol employed by Illumina Inc. that allows the sequencing substrate i.e. the flow cell to become a genomic sequence capture substrate as well. Without adding another instrument to the normal flow of a typical next generation sequencing protocol the DTS protocol modifies the sequencing surface to capture gDNA from a specially prepared library. The captured library is then sequenced as a normal gDNA library would be. However modification of the sequencing substrate and accompanying library preparation according to previous suggestions result in inefficiencies reduced reliability and reproducibility and waste valuable sample. Improvements to the DTS process are therefore desirable.

In one aspect the invention provides an apparatus and a method of producing an apparatus for sequencing a plurality of target polynucleotides. In one embodiment the method comprises a providing a solid support having a reactive surface and b attaching to the solid support a plurality of oligonucleotides. In some embodiments the plurality of oligonucleotides comprises i a plurality of different first oligonucleotides comprising sequence A and sequence B wherein sequence A is common among all first oligonucleotides and further wherein sequence B is different for each different first oligonucleotide is at the 3 end of each first oligonucleotide and is complementary to a sequence comprising a causal genetic variant or a sequence within 200 nucleotides of a causal genetic variant ii a plurality of second oligonucleotides comprising sequence A at each 3 end and iii a plurality of third oligonucleotides comprising sequence C at each 3 end wherein sequence C is the same as a sequence shared by a plurality of different target polynucleotides. In some embodiments A B and C are different sequences and comprise 5 or more nucleotides each.

In some embodiments sequences A B and C have less than 90 sequence identity with one another. In some embodiments the plurality of oligonucleotides comprise a reactive moiety such that a reaction between the reactive surface and the reactive moiety attaches the plurality of oligonucleotides to the solid support. In some embodiments the plurality of first oligonucleotides comprises at least about 100 different first oligonucleotides each comprising a different sequence B. In some embodiments sequence B of one or more of the plurality of first oligonucleotides comprises a sequence selected from the group consisting of SEQ ID NOs 22 121 shown in . In some embodiments the solid support is a channel of a flow cell. In some embodiments the reactive surface comprises functionalized polyacrylamide which may be produced from a polymerization mixture comprising acrylamide N 5 bromoacetamidylpentyl acrylamide tetramethylethylenediamine and potassium persulfate. In some embodiments the amount of the plurality of second oligonucleotides is at least about 1000 fold or 10000 fold higher than the amount of the plurality of first oligonucleotides and the amount of the plurality of second oligonucleotides and the amount of the plurality of third oligonucleotides are in a ratio of about 1 to 1. In some embodiments each of the first oligonucleotides is added to the solid support at a concentration of about 50 pM. In some embodiments the concentration of the plurality of second oligonucleotides and of the plurality of third oligonucleotides is about 500 nM. In some embodiments the invention provides a method of sequencing a plurality of target polynucleotides the method comprising exposing an apparatus produced according to a method of the invention to a sample comprising target polynucleotides and non target polynucleotides wherein sequencing data is enriched for target genomic sequences relative to non target genomic sequences. In some embodiments the plurality of different first oligonucleotides further comprises additional first oligonucleotides comprising sequence A and sequence B wherein sequence B is different for each different additional first oligonucleotide is at the 3 end of each additional first oligonucleotide and is complementary to a sequence comprising a non subject sequence or a sequence within 200 nucleotides of a non subject sequence.

In one aspect the invention provides a method for sequencing a plurality of target polynucleotides in a sample. In one embodiment the method comprises a fragmenting target polynucleotides to produce fragmented polynucleotides b joining adapter oligonucleotides to the fragmented polynucleotides each of the adapter oligonucleotides comprising sequence D to produce adapted polynucleotides comprising sequence D hybridized to complementary sequence D at both ends of the adapted polynucleotides optionally wherein sequence D is produced by extension of a target polynucleotide 3 end c amplifying the adapted polynucleotides using amplification primers comprising sequence C sequence D and a barcode associated with the sample wherein sequence D is positioned at the 3 end of the amplification primers d hybridizing amplified target polynucleotides to a plurality of different first oligonucleotides that are attached to a solid surface e performing bridge amplification on a solid surface and f sequencing a plurality of polynucleotides from step e . The solid surface may comprise a plurality of oligonucleotides as described herein including an apparatus as described herein and optionally produced according to the methods described herein. In some embodiments the solid surface comprises i a plurality of different first oligonucleotides comprising sequence A and sequence B wherein sequence A is common among all first oligonucleotides and further wherein sequence B is different for each different first oligonucleotide is at the 3 end of each first oligonucleotide and is complementary to a sequence comprising a causal genetic variant or a sequence within 200 nucleotides of a causal genetic variant ii a plurality of second oligonucleotides comprising sequence A at each 3 end and iii a plurality of third oligonucleotides comprising sequence C at each 3 end. In some embodiments sequences A B and C are different sequences and comprise 5 or more nucleotides each.

In some embodiments the method further comprises a second amplification step before step d wherein amplified polynucleotides are amplified using a second amplification primer having a 3 end comprising sequence complementary to at least a portion of one or more sequences added to the target polynucleotides in step c . In some embodiments sequences A B and C have less than 90 sequence identity with one another. In some embodiments the plurality of first oligonucleotides comprises at least about 100 different first oligonucleotides each comprising a different sequence B. In some embodiments sequence B of one or more of the plurality of first oligonucleotides comprises a sequence selected from the group consisting of SEQ ID NOs 22 121 shown in . In some embodiments each barcode differs from every other barcode in a pool of two or more samples at at least three nucleotide positions. In some embodiments samples are pooled such that all four nucleotide bases A G C and T are approximately evenly represented at every position along each barcode in the pool. In some embodiments one or more barcodes are selected from the group consisting of AGGTCA CAGCAG ACTGCT TAACGG GGATTA AACCTG GCCGTT CGTTGA GTAACC CTTAAC TGCTAA GATCCG CCAGGT TTCAGC ATGATC and TCGGAT. In some embodiments the barcode is located between sequence C and sequence D. In some embodiments the method further comprises the step of identifying the sample from which a target polynucleotide is derived based on the barcode sequence. In some embodiments the fragmented polynucleotides have a median length between about 200 and about 1000 base pairs. In some embodiments step f comprises i sequencing by extension of a first sequencing primer that hybridizes to a position located 3 from the barcode and then ii sequencing by extension of a second sequencing primer that hybridizes to a position located 5 from the barcode. In some embodiments the solid support is a channel of a flow cell. In some embodiments steps b and c are performed by an automated system such as a liquid handler e.g. a Biomek FXP . In some embodiments step d is performed by an automated system such as a system comprising a cBot machine. In some embodiments the automated system that performs step d also performs step e . In some embodiments sequencing data are generated for at least about 100 different target polynucleotides. In some embodiments step d utilizes at least about 10 g of DNA in a single flow cell. In some embodiments the method is performed on a plurality of samples in parallel. In some embodiments step c is performed in quadruplicate for each of a plurality of samples. In some embodiments the amount of DNA is measured at the completion of one or more of steps a b and c . In some embodiments one or more of steps a b and c has a minimum threshold for the amount of DNA remaining at the end of that step to be used in the next step such as 1 g 0.8 g 13 g respectively. In some embodiments sequencing data are generated for at least about 10target sequences in a single reaction. In some embodiments sequencing data are generated for less than about 10target sequences per sample in a single reaction. In some embodiments presence or absence of one or more causal genetic variants is determined with an accuracy of at least about 90 . In some embodiments the plurality of different first oligonucleotides further comprises additional first oligonucleotides comprising sequence A and sequence B wherein sequence B is different for each different additional first oligonucleotide is at the 3 end of each additional first oligonucleotide and is complementary to a sequence comprising a non subject sequence or a sequence within 200 nucleotides of a non subject sequence.

In one aspect the invention provides a method of enriching a plurality of different target polynucleotides in a sample. In some embodiments the method comprises a joining an adapter oligonucleotide to each of the target polynucleotides wherein the adapter oligonucleotide comprises sequence Y b hybridizing a plurality of different oligonucleotide primers to the adapted target polynucleotides wherein each oligonucleotide primer comprises sequence Z and sequence W wherein sequence Z is common among all oligonucleotide primers and further wherein sequence W is different for each different oligonucleotide primer is positioned at the 3 end of each oligonucleotide primer and is complementary to a sequence comprising a causal genetic variant or a sequence within 200 nucleotides of a causal genetic variant c in an extension reaction extending the oligonucleotide primers along the adapted target polynucleotides to produce extended primers comprising sequence Z and sequence Y wherein sequence Y is complementary to sequence Y and d exponentially amplifying the purified extension products using a pair of amplification primers comprising i a first amplification primer comprising sequence V and sequence Z wherein sequence Z is positioned at the 3 end of the first amplification primer and ii a second amplification primer comprising sequence X and sequence Y wherein sequence Y is positioned at the 3 end of the second amplification primer. In some embodiments sequences W Y and Z are different sequences and comprise 5 or more nucleotides each. Each oligonucleotide primer may or may not comprise a first binding partner. In some embodiments the method further comprises before step d exposing the extended primers to a solid surface comprising a second binding partner that binds to the first binding partner thereby purifying the extended primers away from one or more components of the extension reaction. In some embodiments the method does not comprise a purification step.

In some embodiments the plurality of oligonucleotide primers comprises at least about 100 different oligonucleotide primers each comprising a different sequence W. In some embodiments sequence W of one or more of the plurality of oligonucleotide primers comprises a sequence selected from the group consisting of SEQ ID NOs 22 121 shown in . In some embodiments the target polynucleotides comprise fragmented polynucleotides. In some embodiments the fragmented polynucleotides have a median length between about 200 and about 1000 base pairs. In some embodiments the fragmented polynucleotides are treated to produce blunt ends or to have a defined overhang prior to step a such as an overhang consisting of an adenine. In some embodiments the first binding partner and the second binding partner are members of a binding pair such as streptavidin and biotin. In some embodiments the solid surface is a bead such as a bead that is responsive to a magnetic field. In some embodiments the purifying step comprises application of a magnetic field to purify the beads. In some embodiments the extended primers are purified away from the target polynucleotides. In some embodiments the method further comprises sequencing the products of step d . In some embodiments sequencing comprises amplifying the products of step d by bridge amplification with bound oligonucleotides attached to a solid support to produce double stranded bridge polynucleotides cleaving one strand of a bridge polynucleotide at a cleavage site in a bound oligonucleotide denaturing the cleaved bridge polynucleotide to produce a free single stranded polynucleotide comprising a target sequence attached to the solid support and sequencing the target sequence by extending a sequencing primer hybridized to at least a portion of one or more sequences added during one or more of steps a c or d . In some embodiments sequencing comprises amplifying the products of step d by extension of a bound primer on a solid support to produce bound templates hybridizing a sequencing primer to a bound template extending the sequencing primer and identifying nucleotides added by extension of the sequencing primer. In some embodiments the plurality of different oligonucleotide primers further comprises additional oligonucleotide primers comprising sequence Z and sequence W wherein sequence W is different for each different additional oligonucleotide primer is at the 3 end of each additional oligonucleotide primer and is complementary to a sequence comprising a non subject sequence or a sequence within 200 nucleotides of a non subject sequence.

In one aspect the invention provides a method of enriching a plurality of different target polynucleotides in a sample. In some embodiments the method comprises a hybridizing a plurality of different oligonucleotide primers to the target polynucleotides wherein each oligonucleotide primer comprises sequence Z and sequence W wherein sequence Z is common among all oligonucleotide primers and further wherein sequence W is different for each different oligonucleotide primer is positioned at the 3 end of each oligonucleotide primer and is complementary to a sequence comprising a causal genetic variant or a sequence within 200 nucleotides of a causal genetic variant b in an extension reaction extending the oligonucleotide primers along the target polynucleotides to produce extended primers c joining an adapter oligonucleotide to each extended primer wherein the adapter oligonucleotide comprises sequence Y and further wherein sequence Y is the complement of a sequence Y and d exponentially amplifying the purified extension products using a pair of amplification primers comprising i a first amplification primer comprising sequence V and sequence Z wherein sequence Z is positioned at the 3 end of the first amplification primer and ii a second amplification primer comprising sequence X and sequence Y wherein sequence Y is positioned at the 3 end of the second amplification primer. In some embodiments sequences W Y and Z are different sequences and comprise 5 or more nucleotides each. Each oligonucleotide primer may or may not comprise a first binding partner. In some embodiments the method further comprises before step d exposing the extended primers to a solid surface comprising a second binding partner that binds to the first binding partner thereby purifying the extended primers away from one or more components of the extension reaction. In some embodiments the method does not comprise a purification step.

In some embodiments the plurality of oligonucleotide primers comprises at least about 100 different oligonucleotide primers each comprising a different sequence W. In some embodiments sequence W of one or more of the plurality of oligonucleotide primers comprises a sequence selected from the group consisting of SEQ ID NOs 22 121 shown in . In some embodiments the target polynucleotides comprise fragmented polynucleotides. In some embodiments the fragmented polynucleotides have a median length between about 200 and about 1000 base pairs. In some embodiments step b further comprises treating the extended primers and the target polynucleotides to which they are hybridized to produce blunt ends or to have a defined overhang prior to step c such as an overhang consisting of an adenine. In some embodiments the first binding partner and the second binding partner are members of a binding pair such as streptavidin and biotin. In some embodiments the solid surface is a bead such as a bead that is responsive to a magnetic field. In some embodiments the purifying step comprises application of a magnetic field to purify the beads. In some embodiments the extended primers are purified away from the target polynucleotides. In some embodiments the method further comprises sequencing the products of step d . In some embodiments sequencing comprises amplifying the products of step d by bridge amplification with bound oligonucleotides attached to a solid support to produce double stranded bridge polynucleotides cleaving one strand of a bridge polynucleotide at a cleavage site in a bound oligonucleotide denaturing the cleaved bridge polynucleotide to produce a free single stranded polynucleotide comprising a target sequence attached to the solid support and sequencing the target sequence by extending a sequencing primer hybridized to at least a portion of one or more sequences added during one or more of steps b c or d . In some embodiments sequencing comprises amplifying the products of step d by extension of a bound primer on a solid support to produce bound templates hybridizing a sequencing primer to a bound template extending the sequencing primer and identifying nucleotides added by extension of the sequencing primer. In some embodiments the plurality of different oligonucleotide primers further comprises additional oligonucleotide primers comprising sequence Z and sequence W wherein sequence W is different for each different additional oligonucleotide primer is at the 3 end of each additional oligonucleotide primer and is complementary to a sequence comprising a non subject sequence or a sequence within 200 nucleotides of a non subject sequence.

In one aspect the invention provides a method of detecting genetic variation in a subject s genome. In some embodiments the method comprises a providing a plurality of clusters of polynucleotides wherein i each cluster comprises multiple copies of a nucleic acid duplex attached to a support ii each duplex in a cluster comprises a first molecule comprising sequences A B G D C from 5 to 3 and a second molecule comprising sequences C D G B A from 5 to 3 iii sequence A is complementary to sequence A sequence B is complementary to sequence B sequence C is complementary to sequence C sequence D is complementary to sequence D and sequence G is complementary to sequence G iv sequence G is a portion of a target polynucleotide sequence from a subject and is different for each of a plurality of clusters and v sequence B is located 5 with respect to sequence G in the corresponding target polynucleotide sequence b sequencing sequence G by extension of a first primer comprising sequence D to produce an R1 sequence for each cluster c sequencing sequence B by extension of a second primer comprising sequence A to produce R2 sequence for each cluster d performing a first alignment using a first algorithm to align all R1 sequences to a first reference sequence e performing a second alignment using a second algorithm to locally align R1 sequences identified in said first alignment as likely to contain an insertion or deletion with respect to the first reference sequence to produce a single consensus alignment for each insertion or deletion f performing an R2 alignment by aligning all R2 sequences to a second reference sequence and g transmitting a report identifying sequence variation identified by steps d to f to a receiver.

In some embodiments the first reference sequence comprises a reference genome. In some embodiments the second reference sequence consists of every sequence B for every different target polynucleotide. In some embodiments R2 sequences are aligned independently of R1 sequences. In some embodiments the method further comprises discarding an R1 sequence that aligns to a first position in the first reference sequence that is more than 10 000 base pairs away from a second position in the first reference sequence to which the R2 sequence for the same cluster aligns. In some embodiments the method further comprises deleting a portion of an R1 sequence for a cluster when the portion of R1 sequence to be deleted is identical to at least a portion of sequence B for that cluster and sequence G is shorter than the R1 sequence for that cluster. In some embodiments the method further comprises deleting a portion of an R1 sequence for a cluster when the portion of R1 sequence to be deleted is identical to at least a portion of any sequence B the portion includes either the 5 or 3 nucleotide of R1 and either i no R2 sequence was produced for the cluster or ii R2 sequence produced is not identical to any sequence B. In some embodiments performing the first alignment with a system using the first algorithm takes less time and or uses less system memory to align all R1 reads than would be taken and or used if the system used the second algorithm to perform the first alignment. In some embodiments the first algorithm is based on the Burrows Wheeler transform. In some embodiments the second algorithm is based on the Smith Waterman algorithm or a hash function. In some embodiments R1 and R2 sequences are generated for at least 100 different target polynucleotides. In some embodiments sequences A B C and D are at least 5 nucleotides in length. In some embodiments sequence G of every cluster is 1 to 1000 nucleotides in length. In some embodiments each probe sequence B of a plurality of clusters is complementary to a sequence comprising a causal genetic variant or a sequence within 200 nucleotides of a causal genetic variant. In some embodiments sequence B of one or more of the clusters comprises a sequence selected from the group consisting of SEQ ID NOs 22 121. In some embodiments an R1 sequence is produced for at least about 10clusters in a single reaction. In some embodiments presence absence or allele ratio of one or more causal genetic variants is determined with an accuracy of at least about 90 . In some embodiments the consensus sequence identifies an insertion a deletion or an insertion and a deletion in a target polynucleotide with an accuracy of at least about 90 . In some embodiments each probe sequence B of a plurality of clusters is complementary to a sequence comprising a non subject sequence or a sequence within 200 nucleotides of a non subject sequence. In some embodiments the presence or absence of one or more non subject sequences is determined with an accuracy of at least about 90 . In some embodiments the method further comprises calculating a plurality of probabilities based on the R1 sequences for the subject and including the probabilities in the report wherein each probability is a probability of the subject or a subject s offspring having or developing a disease or trait.

In some embodiments each first molecule comprises a barcode sequence. In some embodiments each barcode differs from every other barcode in a plurality of different barcodes analyzed in parallel. In some embodiments the barcode sequence is associated with a single sample in a pool of samples sequenced in a single reaction. In some embodiments each of a plurality of barcode sequences is uniquely associated with a single sample in a pool of samples sequenced in a single reaction. In some embodiments the barcode sequence is located 5 from sequence D . In some embodiments the method further comprises hybridizing a third primer to sequence C and sequencing the barcode sequence by extension of the third primer to produce a barcode sequence for each cluster. In some embodiments the method further comprises grouping sequences from the clusters based on the barcode sequences. In some embodiments the method further comprises discarding all but one of a plurality of R1 sequences having the same sequence and alignment within a barcode sequence grouping.

In one aspect the invention provides a method of detecting genetic variation in a subject s genome. In some embodiments the method comprises a providing sequencing data for a plurality of clusters of polynucleotides wherein i each cluster comprised multiple copies of a nucleic acid duplex attached to a support ii each duplex in a cluster comprised a first molecule comprising sequences A B G D C from 5 to 3 and a second molecule comprising sequences C D G B A from 5 to 3 iii sequence A is complementary to sequence A sequence B is complementary to sequence B sequence C is complementary to sequence C sequence D is complementary to sequence D and sequence G is complementary to sequence G iv sequence G is a portion of a target polynucleotide sequence from a subject and is different for each of a plurality of clusters v sequence B is located 5 with respect to sequence G in the corresponding target polynucleotide sequence viii the sequencing data comprises R1 sequences generated by extension of a first primer comprising sequence D and vi the sequencing data comprises R2 sequences generated by extension of a second primer comprising sequence A b performing a first alignment using a first algorithm to align all R1 sequences to a first reference sequence c performing a second alignment using a second algorithm to locally align R1 sequences identified in said first alignment as likely to contain an insertion or deletion with respect to the first reference sequence to produce a single consensus alignment for each insertion or deletion d performing an R2 alignment by aligning all R2 sequences to a second reference sequence and e transmitting a report identifying sequence variation identified by steps b to d to a receiver.

In some embodiments the first reference sequence comprises a reference genome. In some embodiments the second reference sequence consists of every sequence B for every different target polynucleotide. In some embodiments R2 sequences are aligned independently of R1 sequences. In some embodiments the method further comprises discarding an R1 sequence that aligns to a first position in the first reference sequence that is more than 10 000 base pairs away from a second position in the first reference sequence to which the R2 sequence for the same cluster aligns. In some embodiments the method further comprises deleting a portion of an R1 sequence for a cluster when the portion of R1 sequence to be deleted is identical to at least a portion of sequence B for that cluster and sequence G is shorter than the R1 sequence for that cluster. In some embodiments the method further comprises deleting a portion of an R1 sequence for a cluster when the portion of R1 sequence to be deleted is identical to at least a portion of any sequence B the portion includes either the 5 or 3 nucleotide of R1 and either i no R2 sequence was produced for the cluster or ii R2 sequence produced is not identical to any sequence B. In some embodiments performing the first alignment with a system using the first algorithm takes less time and or uses less system memory to align all R1 reads than would be taken and or used if the system used the second algorithm to perform the first alignment. In some embodiments the first algorithm is based on the Burrows Wheeler transform. In some embodiments the second algorithm is based on the Smith Waterman algorithm or a hash function. In some embodiments the sequencing data comprises R1 and R2 sequences for at least 100 different target polynucleotides. In some embodiments sequences A B C and D are at least 5 nucleotides in length. In some embodiments sequence G of every cluster is 1 to 1000 nucleotides in length. In some embodiments each probe sequence B of a plurality of clusters is complementary to a sequence comprising a causal genetic variant or a sequence within 200 nucleotides of a causal genetic variant. In some embodiments sequence B of one or more of the clusters comprises a sequence selected from the group consisting of SEQ ID NOs 22 121. In some embodiments sequencing data comprises at least about 10R1 sequences from a single reaction. In some embodiments presence absence or allele ratio of one or more causal genetic variants is determined with an accuracy of at least about 90 . In some embodiments the consensus sequence identifies an insertion a deletion or an insertion and a deletion in a target polynucleotide with an accuracy of at least about 90 . In some embodiments each probe sequence B of a plurality of clusters is complementary to a sequence comprising a non subject sequence or a sequence within 200 nucleotides of a non subject sequence. In some embodiments the presence or absence of one or more non subject sequences is determined with an accuracy of at least about 90 . In some embodiments the method further comprises calculating a plurality of probabilities based on the R1 sequences for the subject and including the probabilities in the report wherein each probability is a probability of the subject or a subject s offspring having or developing a disease or trait.

In some embodiments each first molecule comprises a barcode sequence. In some embodiments each barcode differs from every other barcode in a plurality of different barcodes analyzed in parallel. In some embodiments the barcode sequence is associated with a single sample in a pool of samples sequenced in a single reaction and represented in the sequencing data. In some embodiments each of a plurality of barcode sequences is uniquely associated with a single sample in a pool of samples sequenced in a single reaction. In some embodiments the barcode sequence is located 5 from sequence D . In some embodiments the sequencing data further comprises a barcode sequence for each cluster generated by extension of a third primer comprising sequence C. In some embodiments the method further comprises grouping sequences from the clusters based on the barcode sequences. In some embodiments the method further comprises discarding all but one of a plurality of R1 sequences having the same sequence and alignment within a barcode sequence grouping.

All publications patents and patent applications mentioned in this specification are herein incorporated by reference to the same extent as if each individual publication patent or patent application was specifically and individually indicated to be incorporated by reference.

The terms polynucleotide nucleotide nucleotide sequence nucleic acid and oligonucleotide are used interchangeably. They refer to a polymeric form of nucleotides of any length either deoxyribonucleotides or ribonucleotides or analogs thereof. Polynucleotides may have any three dimensional structure and may perform any function known or unknown. The following are non limiting examples of polynucleotides coding or non coding regions of a gene or gene fragment intergenic DNA loci locus defined from linkage analysis exons introns messenger RNA mRNA transfer RNA ribosomal RNA short interfering RNA siRNA short hairpin RNA shRNA micro RNA miRNA small nucleolar RNA ribozymes cDNA recombinant polynucleotides branched polynucleotides plasmids vectors isolated DNA of any sequence isolated RNA of any sequence nucleic acid probes adapters and primers. A polynucleotide may comprise modified nucleotides such as methylated nucleotides and nucleotide analogs. If present modifications to the nucleotide structure may be imparted before or after assembly of the polymer. The sequence of nucleotides may be interrupted by non nucleotide components. A polynucleotide may be further modified after polymerization such as by conjugation with a labeling component tag reactive moiety or binding partner. Polynucleotide sequences when provided are listed in the 5 to 3 direction unless stated otherwise.

As used herein the term target polynucleotide refers to a nucleic acid molecule or polynucleotide in a population of nucleic acid molecules having a target sequence to which one or more oligonucleotides of the invention are designed to hybridize. In some embodiments a target sequence uniquely identifies a sequence derived from a sample such as a particular genomic mitochondrial bacterial viral or RNA e.g. mRNA miRNA primary miRNA or pre miRNA sequence. In some embodiments a target sequence is a common sequence shared by multiple different target polynucleotides such as a common adapter sequence joined to different target polynucleotides. Target polynucleotide may be used to refer to a double stranded nucleic acid molecule comprising a target sequence on one or both strands or a single stranded nucleic acid molecule comprising a target sequence and may be derived from any source of or process for isolating or generating nucleic acid molecules. A target polynucleotide may comprise one or more e.g. 1 2 3 4 5 6 7 8 9 10 or more target sequences which may be the same or different. In general different target polynucleotides comprise different sequences such as one or more different nucleotides or one or more different target sequences.

 Hybridization and annealing refer to a reaction in which one or more polynucleotides react to form a complex that is stabilized via hydrogen bonding between the bases of the nucleotide residues. The hydrogen bonding may occur by Watson Crick base pairing Hoogstein binding or in any other sequence specific manner. The complex may comprise two strands forming a duplex structure three or more strands forming a multi stranded complex a single self hybridizing strand or any combination of these. A hybridization reaction may constitute a step in a more extensive process such as the initiation of a PCR or the enzymatic cleavage of a polynucleotide by a ribozyme. A first sequence that can be stabilized via hydrogen bonding with the bases of the nucleotide residues of a second sequence is said to be hybridizable to the second sequence. In such a case the second sequence can also be said to be hybridizable to the first sequence.

In general a complement of a given sequence is a sequence that is fully complementary to and hybridizable to the given sequence. In general a first sequence that is hybridizable to a second sequence or set of second sequences is specifically or selectively hybridizable to the second sequence or set of second sequences such that hybridization to the second sequence or set of second sequences is preferred e.g. thermodynamically more stable under a given set of conditions such as stringent conditions commonly used in the art to hybridization with non target sequences during a hybridization reaction. Typically hybridizable sequences share a degree of sequence complementarity over all or a portion of their respective lengths such as between 25 100 complementarity including at least about 25 30 35 40 45 50 55 60 65 70 75 80 85 90 91 92 93 94 95 96 97 98 99 and 100 sequence complementarity.

The term hybridized as applied to a polynucleotide refers to a polynucleotide in a complex that is stabilized via hydrogen bonding between the bases of the nucleotide residues. The hydrogen bonding may occur by Watson Crick base pairing Hoogstein binding or in any other sequence specific manner. The complex may comprise two strands forming a duplex structure three or more strands forming a multi stranded complex a single self hybridizing strand or any combination of these. The hybridization reaction may constitute a step in a more extensive process such as the initiation of a PCR reaction ligation reaction sequencing reaction or cleavage reaction.

The practice of the present invention employs unless otherwise indicated conventional techniques of immunology biochemistry chemistry molecular biology microbiology cell biology genomics and recombinant DNA which are within the skill of the art. See e.g. Sambrook Fritsch and Maniatis MOLECULAR CLONING A LABORATORY MANUAL 2nd edition 1989 CURRENT PROTOCOLS IN MOLECULAR BIOLOGY F. M. Ausubel et al. eds. 1987 the series METHODS IN ENZYMOLOGY Academic Press Inc. PCR 2 A PRACTICAL APPROACH M. J. MacPherson B. D. Hames and G. R. Taylor eds. 1995 Harlow and Lane eds. 1988 ANTIBODIES A LABORATORY MANUAL and ANIMAL CELL CULTURE R. I. Freshney ed. 1987 .

In one aspect the invention provides a method of producing an apparatus for sequencing a plurality of target polynucleotides. In one embodiment the method comprises a providing a solid support having a reactive surface and b attaching to the solid support a plurality of oligonucleotides. In some embodiments the plurality of oligonucleotides comprises i a plurality of different first oligonucleotides comprising sequence A and sequence B wherein sequence A is common among all first oligonucleotides and further wherein sequence B is different for each different first oligonucleotide is at the 3 end of each first oligonucleotide and is complementary to a sequence comprising a causal genetic variant or a sequence within 200 nucleotides of a causal genetic variant ii a plurality of second oligonucleotides comprising sequence A at each 3 end and iii a plurality of third oligonucleotides comprising sequence C at each 3 end wherein sequence C is the same as a sequence shared by a plurality of different target polynucleotides. In some embodiments one or more of sequences A B and C are different sequences. In some embodiments one or more of sequences A B and C are about less than about or more than about 5 10 15 20 25 30 40 50 60 70 80 90 or more different from one or more of the other of sequences A B and C e.g. have less than about 10 20 30 40 50 60 70 80 90 or more sequence identity . In some embodiments one or more of sequences A B and C comprise about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 or more nucleotides each.

A variety of suitable solid support materials are known in the art. Non limiting examples of solid support materials include silica based substrates such as glass fused silica and other silica containing materials silicone hydrides or plastic materials such as polyethylene polystyrene poly vinyl chloride polypropylene nylons polyesters polycarbonates poly methyl methacrylate and cyclic olefin polymer substrates and other solid support materials such as gold titanium dioxide or silicon supports. The solid support materials may be provided in any suitable form including but not limited to beads nanoparticles nanocrystals fibers microfibers nanofibers nanowires nanotubes mats planar sheets planar wafers or slides multiwell plates optical slides flow cells and channels. A solid support may further include one or more additional structures such as channels microfluidic channels capillaries and wells. In some embodiments the solid support is a channel of a flow cell.

When referring to immobilization or attachment of molecules e.g. nucleic acids to a solid support the terms immobilized and attached are used interchangeably herein and both terms are intended to encompass direct or indirect covalent or non covalent attachment unless indicated otherwise. In some embodiments of the invention covalent attachment may be preferred but generally all that is required is that the molecules e.g. nucleic acids remain immobilized or attached to the support under the conditions in which it is intended to use the support for example in nucleic acid amplification and or sequencing applications.

In some embodiments a solid support material comprises a material that is reactive such that under specified conditions a molecule such as an oligonucleotide or modified oligonucleotide can be attached directly to the surface of the solid support. In some embodiments a solid support material comprises an inert substrate or matrix e.g. glass slides polymer beads or other solid support material that has been functionalized for example by application of a layer or coating of an intermediate material comprising reactive groups which permit attachment e.g. covalent attachment to biomolecules such as polynucleotides. Examples of such supports include but are not limited to polyacrylamide hydrogels supported on an inert substrate such as glass. In such embodiments the biomolecules e.g. oligonucleotide may be directly covalently attached to the intermediate material e.g. the hydrogel but the intermediate material may itself be non covalently attached to the substrate or matrix e.g. the glass substrate .

A non limiting example of a reactive surface includes the use of biotinylated albumins BSA to form a stable attachment of biotin groups by physisorption of the protein onto surfaces. Covalent modification can be performed using silanes which have been used to attach molecules to a solid support usually a glass slide. By way of example a mixture of tetraethoxysilane and triethoxy bromoacetamidopropyl silane e.g. in a ratio of 1 100 can be used to prepare functionalized glass slides which permit attachment of nucleic acids including a thiophosphate or phosphorothioate functionality. Biotin molecules can be attached to surfaces using appropriately reactive species such as biotin PEG succinimidyl ester which reacts with an amino surface.

In some embodiments oligonucleotides to be attached to the solid support comprise a reactive moiety. In general a reactive moiety includes any moiety that facilitates attachment to the solid support by reacting with the reactive surface. In some embodiments functionalized polyacrylamide hydrogels are used to attach a plurality of oligonucleotides comprising a reactive moiety wherein the reactive moiety is a sulfur containing nucleophilic group. Examples of appropriate sulfur nucleophile containing polynucleotides are disclosed in Zhao et al Nucleic Acids Research 2001 29 4 955 959 and Pirrung et al Langmuir 2000 16 2185 2191 and include for example simple thiols thiophosphates and thiophosphoramidates. Preferred hydrogels are those formed from a mixture of i a first co monomer which is acrylamide methacrylamide hydroxyethyl methacrylate or N vinyl pyrrolidinone and ii a second co monomer which is a functionalized acrylamide or acrylate such as N 5 bromoacetamidylpentyl acrylamide tetramethylethylenediamine In some embodiments a reactive surface comprising a functionalized polyacrylamide is produced from a polymerization mixture comprising acrylamide N 5 bromoacetamidylpentyl acrylamide tetramethylethylenediamine and potassium persulfate. Further non limiting examples of support materials and reactive surfaces are provided by US20120053074 and WO2005065814 which are hereby incorporated by reference in their entireties.

Oligonucleotides to which the solid support is exposed for attachment may be of any suitable length and may comprise one or more sequence elements. Examples of sequence elements include but are not limited to one or more amplification primer annealing sequences or complements thereof one or more sequencing primer annealing sequences or complements thereof one or more common sequences shared among multiple different oligonucleotides or subsets of different oligonucleotides one or more restriction enzyme recognition sites one or more target recognition sequences complementary to one or more target polynucleotide sequences one or more random or near random sequences e.g. one or more nucleotides selected at random from a set of two or more different nucleotides at one or more positions with each of the different nucleotides selected at one or more positions represented in a pool of oligonucleotides comprising the random sequence one or more spacers and combinations thereof. Two or more sequence elements can be non adjacent to one another e.g. separated by one or more nucleotides adjacent to one another partially overlapping or completely overlapping. For example an amplification primer annealing sequence can also serve as a sequencing primer annealing sequence. Sequence elements can be located at or near the 3 end at or near the 5 end or in the interior of the oligonucleotide. In general as used herein a sequence element located at the 3 end includes the 3 most nucleotide of the oligonucleotide and a sequence element located at the 5 end includes the 5 most nucleotide of the oligonucleotide. In some embodiments a sequence element is about less than about or more than about 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 40 50 or more nucleotides in length. In some embodiments an oligonucleotide is about less than about or more than about 5 10 15 20 25 30 35 40 45 50 60 70 80 90 100 or more nucleotides in length.

A spacer may consist of a repeated single nucleotide e.g. 1 2 3 4 5 6 7 8 9 10 or more of the same nucleotide in a row or a sequence of 2 3 4 5 6 7 8 9 10 or more nucleotides repeated 1 2 3 4 5 6 7 8 9 10 or more times. A spacer may comprise or consist of a specific sequence such as a sequence that does not hybridize to any target sequence in a sample. A spacer may comprise or consist of a sequence of randomly selected nucleotides.

In some embodiments a plurality of different first oligonucleotides are attached to the solid support each comprising a sequence A that is common among all first oligonucleotides and a sequence B that is different for each different first oligonucleotide. In some embodiments sequence B of each first oligonucleotide is complementary to a different target sequence. In some embodiments the plurality of first oligonucleotides comprises about less than about or more than about 5 10 25 50 75 100 125 150 175 200 300 400 500 750 1000 2500 5000 7500 10000 20000 50000 or more different first oligonucleotides each comprising a different sequence B. In some embodiments sequence B of one or more of the plurality of first oligonucleotides comprises a sequence selected from the group consisting of SEQ ID NOs 22 121 shown in e.g. 1 5 10 25 50 75 or 100 different oligonucleotides each with a different sequence from . In some embodiments sequence B or the target sequence to which it specifically hybridizes comprises a causal genetic variant. In some embodiments sequence B or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of a causal genetic variant. Causal genetic variants are typically located downstream of a first oligonucleotide such that at least a portion of the causal genetic variant serves as template for extension of a first oligonucleotide. In general causal genetic variants are genetic variants for which there is statistical biological and or functional evidence of association with a disease or trait. A single causal genetic variant can be associated with more than one disease or trait. In some embodiments a causal genetic variant can be associated with a Mendelian trait a non Mendelian trait or both. Causal genetic variants can manifest as variations in a polynucleotide such 1 2 3 4 5 6 7 8 9 10 20 50 or more sequence differences such as between a polynucleotide comprising the causal genetic variant and a polynucleotide lacking the causal genetic variant at the same relative genomic position . Non limiting examples of types of causal genetic variants include single nucleotide polymorphisms SNP deletion insertion polymorphisms DIP copy number variants CNV short tandem repeats STR restriction fragment length polymorphisms RFLP simple sequence repeats SSR variable number of tandem repeats VNTR randomly amplified polymorphic DNA RAPD amplified fragment length polymorphisms AFLP inter retrotransposon amplified polymorphisms IRAP long and short interspersed elements LINE SINE long tandem repeats LTR mobile elements retrotransposon microsatellite amplified polymorphisms retrotransposon based insertion polymorphisms sequence specific amplified polymorphism and heritable epigenetic modification for example DNA methylation . A causal genetic variant may also be a set of closely related causal genetic variants. Some causal genetic variants may exert influence as sequence variations in RNA polynucleotides. At this level some causal genetic variants are also indicated by the presence or absence of a species of RNA polynucleotides. Also some causal genetic variants result in sequence variations in protein polypeptides. A number of causal genetic variants are known in the art. An example of a causal genetic variant that is a SNP is the Hb S variant of hemoglobin that causes sickle cell anemia. An example of a causal genetic variant that is a DIP is the delta508 mutation of the CFTR gene which causes cystic fibrosis. An example of a causal genetic variant that is a CNV is trisomy 21 which causes Down s syndrome. An example of a causal genetic variant that is an STR is tandem repeat that causes Huntington s disease. provides a table of non limiting examples of causal genetic variants and associated diseases. Non limiting examples of causal genetic variants are also described in US20100022406 which is hereby incorporated by reference in its entirety.

Causal genetic variants can be originally discovered by statistical and molecular genetic analyses of the genotypes and phenotypes of individuals families and populations. The causal genetic variants for Mendelian traits are typically identified in a two stage process. In the first stage families in which multiple individuals who possess the trait are examined for genotype and phenotype. Genotype and phenotype data from these families is used to establish the statistical association between the presence of the Mendelian trait and the presence of a number of genetic markers. This association establishes a candidate region in which the causal genetic variant is likely to map. In a second stage the causal genetic variant itself is identified. The second step typically entails sequencing the candidate region. More sophisticated one stage processes are possible with more advanced technologies which permit the direct identification of a causal genetic variant or the identification of smaller candidate regions. After one causal genetic variant for a trait is discovered additional variants for the same trait can be discovered by simple methods. For example the gene associated with the trait can be sequenced in individuals who possess the trait or their relatives. The invention of new methods for discovering causal genetic variants is an active area of research. The application of existing methods and the incorporation of new methods is expected to continue to result in the discovery of additional causal genetic variants which can be used or tested for by the devices systems and methods herein. Many causal genetic variants are cataloged in databases including the Online Mendelian Inheritance in Man OMIM and the Human Gene Mutation Database HGMD . Causal genetic variants are also reported in the scholarly literature at conferences and in personal communications between scholars.

A causal genetic variant may exist at any frequency within a specified populations. In some embodiments at least one of the causal genetic variants causes a trait having an incidence of no more than 1 a reference population. In another embodiment at least one of the causal genetic variants causes a trait having an incidence of no more than 1 10 000 in a reference population. In some embodiments a causal genetic variant is associated with a disease or trait. In some embodiments a causal genetic variant is a genetic variant the presence of which increases the risk of having or developing a disease or trait by about less than about or more than about 1 5 10 15 20 25 30 40 50 60 70 80 90 100 200 300 400 500 or more. In some embodiments a causal genetic variant is a genetic variant the presence of which increases the risk of having or developing a disease or trait by about less than about or more than about 1 fold 2 fold 3 fold 4 fold 5 fold 6 fold 7 fold 8 fold 9 fold 10 fold 25 fold 50 fold 100 fold 500 fold 1000 fold 10000 fold or more. In some embodiments a causal genetic variant is a genetic variant the presence of which increases the risk of having or developing a disease or trait by any statistically significant amount such as an increase having a p value of about or less than about 0.1 0.05 10 10 10 10 10 10 10 10 10 10 10 10 10 or smaller.

In some embodiments a causal genetic variant has a different degree of association with a disease or trait between two or more different populations of individuals such as between two or more human populations. In some embodiments a causal genetic variant has a statistically significant association with a disease or trait only within one or more populations such as one or more human populations. A human population can be a group of people sharing a common genetic inheritance such as an ethnic group for example Caucasian . A human population can be a haplotype population or group of haplotype populations for example haplotype H1 M52 . A human population can be a national group for example Americans English Irish . A human population can be a demographic population such as those delineated by age sex and socioeconomic factors. Human populations can be historical populations. A population can consist of individuals distributed over a large geographic area such that individuals at extremes of the distribution may never meet one another. The individuals of a population can be geographically dispersed into discontinuous areas. Populations can be informative about biogeographical ancestry. Populations can also be defined by ancestry. Genetic studies can define populations. In some embodiments a population may be based on ancestry and genetics with major human populations corresponding to continental scale groupings which include Western Eurasian sub Saharan African East Asian and Native American. Most humans can be assigned to at least one of these populations on the basis of ancestry. A number of smaller populations are also distinguished as continental groups including Indigenous Australian Oceanian and Bushmen.

Very often populations can be further decomposed into sub populations. The relationship between populations and subpopulations can be hierarchical. For example the Oceanian population can be further sub divided into sub populations including Polynesians Melanesians and Micronesians. The Western Eurasian population can be further sub divided into sub populations including European Western Central Asian South Asian and North African. The European population can be further sub divided into sub populations including North Western European Southern European and Ashkenazi Jewish populations. The North Western European population can be further sub divided into national populations including English Irish German Finnish and the like. The East Asian population can be further sub divided into Chinese Japanese and Korean subpopulations. The South Asian population can be further sub divided into Indian and Pakistani populations. The Indian population can be further sub divided into Dravidian people Brahui people Kannadigas Malayalis Tamils Telugus Tuluvas and Gonds. A sub population may serve as a population for the purpose of identifying a causal genetic variant.

In some embodiments a causal genetic variant is associated with a disease such as a rare genetic disease. Examples of diseases with which a causal genetic variant may be associated include but are not limited to 21 Hydroxylase Deficiency ABCC8 Related Hyperinsulinism ARSACS Achondroplasia Achromatopsia Adenosine Monophosphate Deaminase 1 Agenesis of Corpus Callosum with Neuronopathy Alkaptonuria Alpha 1 Antitrypsin Deficiency Alpha Mannosidosis Alpha Sarcoglycanopathy Alpha Thalassemia Alzheimers Angiotensin II Receptor Type I Apolipoprotein E Genotyping Argininosuccinicaciduria Aspartylglycosaminuria Ataxia with Vitamin E Deficiency Ataxia Telangiectasia Autoimmune Polyendocrinopathy Syndrome Type 1 BRCA1 Hereditary Breast Ovarian Cancer BRCA2 Hereditary Breast Ovarian Cancer one or more other types of cancer Bardet Biedl Syndrome Best Vitelliform Macular Dystrophy Beta Sarcoglycanopathy Beta Thalassemia Biotinidase Deficiency Blau Syndrome Bloom Syndrome CFTR Related Disorders CLN3 Related Neuronal Ceroid Lipofuscinosis CLN5 Related Neuronal Ceroid Lipofuscinosis CLN8 Related Neuronal Ceroid Lipofuscinosis Canavan Disease Carnitine Palmitoyltransferase IA Deficiency Carnitine Palmitoyltransferase II Deficiency Cartilage Hair Hypoplasia Cerebral Cavernous Malformation Choroideremia Cohen Syndrome Congenital Cataracts Facial Dysmorphism and Neuropathy Congenital Disorder of Glycosylationla Congenital Disorder of Glycosylation Ib Congenital Finnish Nephrosis Crohn Disease Cystinosis DFNA 9 COCH Diabetes and Hearing Loss Early Onset Primary Dystonia DYTI Epidermolysis Bullosa Junctional Herlitz Pearson Type FANCC Related Fanconi Anemia FGFR1 Related Craniosynostosis FGFR2 Related Craniosynostosis FGFR3 Related Craniosynostosis Factor V Leiden Thrombophilia Factor V R2 Mutation Thrombophilia Factor XI Deficiency Factor XIII Deficiency Familial Adenomatous Polyposis Familial Dysautonomia Familial Hypercholesterolemia Type B Familial Mediterranean Fever Free Sialic Acid Storage Disorders Frontotemporal Dementia with Parkinsonism 17 Fumarase deficiency GJB2 Related DFNA 3 Nonsyndromic Hearing Loss and Deafness GJB2 Related DFNB 1 Nonsyndromic Hearing Loss and Deafness GNE Related Myopathies Galactosemia Gaucher Disease Glucose 6 Phosphate Dehydrogenase Deficiency Glutaricacidemia Type 1 Glycogen Storage Disease Type 1a Glycogen Storage Disease Type 1b Glycogen Storage Disease Type II Glycogen Storage Disease Type III Glycogen Storage Disease Type V Gracile Syndrome HFE Associated Hereditary Hemochromatosis Halder AIMs Hemoglobin S Beta Thalassemia Hereditary Fructose Intolerance Hereditary Pancreatitis Hereditary Thymine Uraciluria Hexosaminidase A Deficiency Hidrotic Ectodermal Dysplasia 2 Homocystinuria Caused by Cystathionine Beta Synthase Deficiency Hyperkalemic Periodic Paralysis Type 1 Hyperornithinemia Hyperammonemia Homocitrullinuria Syndrome Hyperoxaluria Primary Type 1 Hyperoxaluria Primary Type 2 Hypochondroplasia Hypokalemic Periodic Paralysis Type 1 Hypokalemic Periodic Paralysis Type 2 Hypophosphatasia Infantile Myopathy and Lactic Acidosis Fatal and Non Fatal Forms Isovaleric Acidemias Krabbe Disease LGMD2I Leber Hereditary Optic Neuropathy Leigh Syndrome French Canadian Type Long Chain 3 Hydroxyacyl CoA Dehydrogenase Deficiency MELAS MERRF MTHFR Deficiency MTHFR Thermolabile Variant MTRNR1 Related Hearing Loss and Deafness MTTS1 Related Hearing Loss and Deafness MYH Associated Polyposis Maple Syrup Urine Disease Type 1A Maple Syrup Urine Disease Type 1B McCune Albright Syndrome Medium Chain Acyl Coenzyme A Dehydrogenase Deficiency Megalencephalic Leukoencephalopathy with Subcortical Cysts Metachromatic Leukodystrophy Mitochondrial Cardiomyopathy Mitochondrial DNA Associated Leigh Syndrome and NARP Mucolipidosis IV Mucopolysaccharidosis Type I Mucopolysaccharidosis Type IIIA Mucopolysaccharidosis Type VII Multiple Endocrine Neoplasia Type 2 Muscle Eye Brain Disease Nemaline Myopathy Neurological phenotype Niemann Pick Disease Due to Sphingomyelinase Deficiency Niemann Pick Disease Type C1 Nijmegen Breakage Syndrome PPT1 Related Neuronal Ceroid Lipofuscinosis PROP1 related pituitary hormome deficiency Pallister Hall Syndrome Paramyotonia Congenita Pendred Syndrome Peroxisomal Bifunctional Enzyme Deficiency Pervasive Developmental Disorders Phenylalanine Hydroxylase Deficiency Plasminogen Activator Inhibitor I Polycystic Kidney Disease Autosomal Recessive Prothrombin G20210A Thrombophilia Pseudovitamin D Deficiency Rickets Pycnodysostosis Retinitis Pigmentosa Autosomal Recessive Bothnia Type Rett Syndrome Rhizomelic Chondrodysplasia Punctata Type 1 Short Chain Acyl CoA Dehydrogenase Deficiency Shwachman Diamond Syndrome Sjogren Larsson Syndrome Smith Lemli Opitz Syndrome Spastic Paraplegia 13 Sulfate Transporter Related Osteochondrodysplasia TFR2 Related Hereditary Hemochromatosis TPP1 Related Neuronal Ceroid Lipofuscinosis Thanatophoric Dysplasia Transthyretin Amyloidosis Trifunctional Protein Deficiency Tyrosine Hydroxylase Deficient DRD Tyrosinemia Type I Wilson Disease X Linked Juvenile Retinoschisis and Zellweger Syndrome Spectrum.

In some embodiments sequence B of one or more of the plurality of first oligonucleotides or the target sequence to which it specifically hybridizes comprises a non subject sequence. In some embodiments sequence B or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of a non subject sequence. In general a non subject sequence corresponds to a polynucleotide derived from an organism other than the individual being tested such as DNA or RNA from bacteria archaea viruses protists fungi or other organism. A non subject sequence may be indicative of the identity of an organism or class of organisms and may further be indicative of a disease state such as infection. An example of non subject sequences useful in identifying an organism include without limitation rRNA sequences such as 16s rRNA sequences see e.g. WO2010151842 . In some embodiments non subject sequences are analyzed instead of or separately from causal genetic variants. In some embodiments causal genetic variants and non subject sequences are analyzed in parallel such as in the same sample e.g. using a mixture of first oligonucleotides some with a sequence B that specifically hybridizes to a sequence comprising or near a causal genetic variant and some with a sequence B that specifically hybridizes to a sequence comprising or near a non subject sequence and or in the same report.

In some embodiments sequence B of one or more of the plurality of first oligonucleotides or the target sequence to which it specifically hybridizes comprises an ancestry informative marker AIM . In some embodiments sequence B or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of an AIM. In general an AIM is a genetic variant that differs in frequency between two or more populations of individuals such as two or more human populations and may be used to infer the ancestry of a subject either alone or in combination with one or more other AIMs. An AIM may be used to classify a person as belonging to or not belonging to one or more populations such as a population that is at increased risk for one of the causal genetic variants. For example an AIM can be diagnostic for a population in which a trait is at increased prevalence. In certain instances the AIM may distinguish between populations with finer granularity for example between sub continental groups or related ethnic groups. In some embodiments AIMs are analyzed instead of or separately from causal genetic variants and or non subject sequences. In some embodiments AIMs causal genetic variants and or non subject sequences are analyzed in parallel such as in the same sample e.g. using a mixture of first oligonucleotides some with a sequence B that specifically hybridizes to a sequence comprising or near a causal genetic variant and some with a sequence B that specifically hybridizes to a sequence comprising or near an AIM and or in the same report. Non limiting examples of types of AIMs include single nucleotide polymorphisms SNP deletion insertion polymorphisms DIP copy number variants CNV short tandem repeats STR restriction fragment length polymorphisms RFLP simple sequence repeats SSR variable number of tandem repeats VNTR randomly amplified polymorphic DNA RAPD amplified fragment length polymorphisms AFLP inter retrotransposon amplified polymorphisms IRAP long and short interspersed elements LINE SINE long tandem repeats LTR mobile elements retrotransposon microsatellite amplified polymorphisms retrotransposon based insertion polymorphisms sequence specific amplified polymorphism and heritable epigenetic modification for example DNA methylation . AIMs can also be sequence variations in RNA polynucleotides. Some AIMs can also be indicated by the presence or the concentration of a species of RNA polynucleotides. Some AIMs can also be sequence variations in protein polypeptides. Some AIMs can also be indicated by the presence or absence of a species of protein polypeptides. A number of ancestry informative markers are identified in . Other AIMs are described in US 2007 0037182. An AIM may or may not also be a causal genetic variant. For example the Duffy Null FY 0 genetic variant causes an absence of a blood antigen. This variant is at nearly 100 frequency in sub Saharan African populations and at nearly 0 frequency in populations outside of sub Saharan Africa. Many causal genetic variants associated with pigmentation are also AIMs. AIMs that are not causal genetic variants can be indirectly associated with traits caused by other AIMs.

AIMs can be discovered by determining the frequency of genetic variants in a plurality of populations. This may be achieved by determining the frequency of already known variants in individuals from various populations. It may also be achieved intrinsically during the process of variant discovery. Both tasks were undertaken by the International HapMap project which catalogued SNP polymorphisms. Ancestry informative markers can be ranked by a variety of measurements which judge their predictive power. One measurement is Wright s F statistic called Fst or FST. This variable is known by other names including Fixation index. Another metric for ranking AIMs is informativeness. Another method of ranking AIMs is the PCA correlated SNPs method of Paschou et al. Paschou et al. PCA correlated SNPs for structure identification in worldwide human populations. PLoS Genet 2007 vol. 3 9 pp. 1672 86 .

To achieve a pre selected degree of confidence e.g. at least about 80 85 90 95 97.5 99 or more in ancestry inference on the basis of ancestry informative markers and to achieve ancestry inference for a plurality of populations it may be necessary to examine more than one ancestry informative marker. A sufficiently large panel of randomly selected genetic variants can be used to infer ancestry e.g. about or more than about 5 10 15 25 50 100 250 500 1000 2500 5000 or more AIMs . A targeted set of especially appropriate AIMs can be constructed. Many researchers have published lists of suggested ancestry informative markers for example Seldin et al. Application of ancestry informative markers to association studies in European Americans. PLoS Genet 2008 vol. 4 1 pp. e5 Halder et al. A panel of ancestry informative markers for estimating individual biogeographical ancestry and admixture from four continents utility and applications. Hum Mutat 2008 vol. 29 5 pp. 648 58 Tian et al. Analysis and application of European genetic substructure using 300 K SNP information. PLoS Genet 2008 vol. 4 1 pp. e4 Price et al. Discerning the ancestry of European Americans in genetic association studies. PLoS Genet 2008 vol. 4 1 pp. e236 Paschou et al. PCA correlated SNPs for structure identification in worldwide human populations. PLoS Genet 2007 vol. 3 9 pp. 1672 86 and Bauchet et al. Measuring European population stratification with microarray genotype data. Am J Hum Genet 2007 vol. 80 5 pp. 948 56 . These and similar lists can be used to build a panel of AIMs for which a device or method herein can be configured to test for.

In some embodiments a plurality of second nucleotides and a plurality of third nucleotides are attached to the solid support in addition to the plurality of first nucleotides. In some embodiments the second nucleotides all comprise sequence A at the 3 end where sequence A in the plurality of second oligonucleotides is the same as sequence A in all of the first oligonucleotides. In some embodiments the third oligonucleotides comprise sequence C at the 3 end where sequence C is complementary to a sequence shared by a plurality of different target polynucleotides. In some embodiments extension of a first oligonucleotide along a target polynucleotide serving as a template generates an extension product comprising sequence C which is complementary and specifically hybridizable to sequence C. In some embodiments the amount of the plurality of second oligonucleotides exposed to the solid support is about less than about or more than about 10 fold 50 fold 100 fold 1000 fold 5000 fold 7500 fold 10000 fold 12500 fold 15000 fold 20000 fold 50000 fold 100000 fold or more higher than the amount of the plurality of first oligonucleotides exposed to the solid support such as in a reaction for attached the plurality of oligonucleotides to the solid support. In some embodiments the ratio or the inverse ratio of the amount of the plurality of second oligonucleotides to the amount of third oligonucleotides exposed to the solid support is about less than about or more than about 1 1 2 1 3 1 4 1 5 1 6 1 7 1 8 1 9 1 10 1 or more. In some embodiments the plurality of first oligonucleotides is added to the solid support at a concentration of about less than about or more than about 0.5 pM 1 pM 5 pM 10 pM 25 pM 50 pM 75 pM 100 pM 200 pM 500 pM 1 nM 10 nM 100 nM 500 nM or higher. In some embodiments the concentration of the plurality of second oligonucleotides and or the third oligonucleotides is about less than about or more than about 0.5 nM 1 nM 5 nM 10 nM 25 nM 50 nM 75 nM 100 nM 200 nM 500 nM 1 M 5 M 10 M 25 M 50 M 100 M 500 M or higher.

In some embodiments one or more the plurality of oligonucleotides comprise one or more blocking groups. In general a blocking group is any modification that prevents extension of a 3 end of an oligonucleotide such as by a polymerase a ligase and or other enzymes. A blocking group may be added before or after an oligonucleotide is attached to the solid support. In some embodiments a blocking group is added during an amplification or sequencing process. Examples of blocking groups include but are not limited to alkyl groups non nucleotide linkers phosphorothioate alkane diol residues peptide nucleic acid and nucleotide derivatives lacking a 3 OH including for example cordycepin.

In some embodiments one or more of the oligonucleotides attached to the substrate comprise a cleavage site such that cleavage at that site releases all or a portion of the cleaved polynucleotide from attachment to the solid support. In some embodiments cleavage produces a 3 end that may be extended along a polynucleotide template. In some embodiments only a portion of the plurality of first second and or third oligonucleotides comprise a cleavage site e.g. about less than about or more than about 10 20 30 40 50 60 70 80 90 or more . The cleavage site may be cleavable by any suitable means including but not limited to chemical enzymatic and photochemical cleavage. The cleavage groups may be positioned between the first nucleotide and the solid support or at or after any number of nucleotides in the oligonucleotide such as about less than about or more than about 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 20 or more nucleotides from the point of attachment to the solid support.

Processes for chemical enzymatic and photochemical cleavage and cleavage sites cleaved by such processes are known in the art. Examples of cleavage means include but are not limited to restriction enzyme digestion in which case the cleavage site is an appropriate restriction site for the enzyme which directs cleavage of one or both strands of a duplex template RNase digestion or chemical cleavage of a bond between a deoxyribonucleotide and a ribonucleotide in which case the cleavage site may include one or more ribonucleotides chemical reduction of a disulphide linkage with a reducing agent e.g. TCEP in which case the cleavage site should include an appropriate disulphide linkage chemical cleavage of a diol linkage with periodate in which case the cleavage site should include a diol linkage generation of an abasic site and subsequent hydrolysis. Cleavage may be followed by blocking to produce a 3 end that cannot be extended such as by a polymerase a ligase and or other enzymes. An example of a blocking agents include but are not limited to amines e.g. ethanolamine which may be added before during or after the addition of a cleaving agent. Additional non limiting examples of cleavage processes and cleavage sites are described in US20120053074 which is incorporated by reference in its entirety.

In some embodiments a plurality of target polynucleotides are amplified according to a method that comprises exposing a sample comprising a plurality of target polynucleotides to an apparatus of the invention. In some embodiments the amplification process comprises bridge amplification. General methods for conducting standard bridge amplification are known in the art. By way of example WO 1998 044151 and WO 2000 018957 both describe methods of nucleic acid amplification which allow amplification products to be immobilized on a solid support in order to form arrays comprised of clusters or colonies formed from a plurality of identical immobilized polynucleotide strands and a plurality of identical immobilized complementary strands. In some embodiments a plurality of polynucleotides are sequenced according to a method that comprises exposing a sample comprising a plurality of target polynucleotides to an apparatus of the invention. General methods for conducting sequencing using a plurality of oligonucleotides attached to a solid support are known in the art such as methods disclosed in US20120053074 and US20110223601 which are hereby incorporated by reference in their entirety. Non limiting exemplary methods for amplifying and or sequencing target polynucleotides in accordance with the methods and apparatuses of the invention are provided herein. In general amplification of specific target polynucleotides permits the generation of sequencing data that is enriched for target polynucleotides such as target genomic sequences relative to non target polynucleotides. In some embodiments the enrichment of sequencing data for target polynucleotides especially sequencing data for causal genetic variants relative to non target polynucleotides is about or at least about 10 fold 100 fold 500 fold 1000 fold 5000 fold 10000 fold 50000 fold 100000 fold 1000000 fold or more.

Non limiting examples of substrates comprising oligonucleotides methods for their production and systems and methods for their operation are provided in WO 2008 002502 which in hereby incorporated by reference in its entirety.

In one aspect the invention provides a method for sequencing a plurality of target polynucleotides in a sample. In one embodiment the method comprises a fragmenting target polynucleotides to produce fragmented polynucleotides b joining adapter oligonucleotides to the fragmented polynucleotides each of the adapter oligonucleotides comprising sequence D to produce adapted polynucleotides comprising sequence D hybridized to complementary sequence D at both ends of the adapted polynucleotides optionally wherein sequence D is produced by extension of a target polynucleotide 3 end c amplifying the adapted polynucleotides using amplification primers comprising sequence C sequence D and a barcode associated with the sample wherein sequence D is positioned at the 3 end of the amplification primers d hybridizing amplified target polynucleotides to a plurality of different first oligonucleotides that are attached to a solid surface e performing bridge amplification on a solid surface and f sequencing a plurality of polynucleotides from step e . The solid surface may comprise a plurality of oligonucleotides as described herein including an apparatus as described herein and optionally produced according to the methods described herein. In some embodiments the solid surface comprises i a plurality of different first oligonucleotides comprising sequence A and sequence B wherein sequence A is common among all first oligonucleotides and further wherein sequence B is different for each different first oligonucleotide is at the 3 end of each first oligonucleotide and is complementary to a sequence comprising a causal genetic variant or a sequence within 200 nucleotides of a causal genetic variant ii a plurality of second oligonucleotides comprising sequence A at each 3 end and iii a plurality of third oligonucleotides comprising sequence C at each 3 end. In some embodiments one or more of sequences A B C and D are different sequences. In some embodiments one or more of sequences A B C and D are about less than about or more than about 5 10 15 20 25 30 40 50 60 70 80 90 or more different from one or more of the other of sequences A B C and D e.g. have less than about 10 20 30 40 50 60 70 80 90 or more sequence identity . In some embodiments one or more of sequences A B C and D comprise about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 or more nucleotides each.

Samples from which the target polynucleotides are derived can comprise multiple samples from the same individual samples from different individuals or combinations thereof. In some embodiments a sample comprises a plurality of polynucleotides from a single individual. In some embodiments a sample comprises a plurality of polynucleotides from two or more individuals. An individual is any organism or portion thereof from which target polynucleotides can be derived non limiting examples of which include plants animals fungi protists monerans viruses mitochondria and chloroplasts. Sample polynucleotides can be isolated from a subject such as a cell sample tissue sample fluid sample or organ sample derived therefrom or cell cultures derived from any of these including for example cultured cell lines biopsy blood sample cheek swab or fluid sample containing a cell e.g. saliva . The subject may be an animal including but not limited to a cow a pig a mouse a rat a chicken a cat a dog etc. and is usually a mammal such as a human. Samples can also be artificially derived such as by chemical synthesis. In some embodiments samples comprise DNA. In some embodiments samples comprise genomic DNA. In some embodiments samples comprise mitochondrial DNA chloroplast DNA plasmid DNA bacterial artificial chromosomes yeast artificial chromosomes oligonucleotide tags polynucleotides from an organism e.g. bacteria virus or fungus other than the subject from whom the sample is taken or combinations thereof. In some embodiments the samples comprise DNA generated by amplification such as by primer extension reactions using any suitable combination of primers and a DNA polymerase including but not limited to polymerase chain reaction PCR reverse transcription and combinations thereof. Where the template for the primer extension reaction is RNA the product of reverse transcription is referred to as complementary DNA cDNA . Primers useful in primer extension reactions can comprise sequences specific to one or more targets random sequences partially random sequences and combinations thereof. Reaction conditions suitable for primer extension reactions are known in the art. In general sample polynucleotides comprise any polynucleotide present in a sample which may or may not include target polynucleotides. In some embodiments a sample from a single individual is divided into multiple separate samples e.g. 2 3 4 5 6 7 8 9 10 or more separate samples that are subjected to the methods of the invention independently such as analysis in duplicate triplicate quadruplicate or more.

Methods for the extraction and purification of nucleic acids are well known in the art. For example nucleic acids can be purified by organic extraction with phenol phenol chloroform isoamyl alcohol or similar formulations including TRIzol and TriReagent. Other non limiting examples of extraction techniques include 1 organic extraction followed by ethanol precipitation e.g. using a phenol chloroform organic reagent Ausubel et al. 1993 with or without the use of an automated nucleic acid extractor e.g. the Model 341 DNA Extractor available from Applied Biosystems Foster City Calif. 2 stationary phase adsorption methods U.S. Pat. No. 5 234 809 Walsh et al. 1991 and 3 salt induced nucleic acid precipitation methods Miller et al. 1988 such precipitation methods being typically referred to as salting out methods. Another example of nucleic acid isolation and or purification includes the use of magnetic particles to which nucleic acids can specifically or non specifically bind followed by isolation of the beads using a magnet and washing and eluting the nucleic acids from the beads see e.g. U.S. Pat. No. 5 705 628 . In some embodiments the above isolation methods may be preceded by an enzyme digestion step to help eliminate unwanted protein from the sample e.g. digestion with proteinase K or other like proteases. See e.g. U.S. Pat. No. 7 001 724. If desired RNase inhibitors may be added to the lysis buffer. For certain cell or sample types it may be desirable to add a protein denaturation digestion step to the protocol. Purification methods may be directed to isolate DNA RNA or both. When both DNA and RNA are isolated together during or subsequent to an extraction procedure further steps may be employed to purify one or both separately from the other. Sub fractions of extracted nucleic acids can also be generated for example purification by size sequence or other physical or chemical characteristic. In addition to an initial nucleic acid isolation step purification of nucleic acids can be performed after any step in the methods of the invention such as to remove excess or unwanted reagents reactants or products. Methods for determining the amount and or purity of nucleic acids in a sample are known in the art and include absorbance e.g. absorbance of light at 260 nm 280 nm and a ratio of these and detection of a label e.g. fluorescent dyes and intercalating agents such as SYBR green SYBR blue DAPI propidium iodine Hoechst stain SYBR gold ethidium bromide .

In some embodiments target polynucleotides are fragmented into a population of fragmented polynucleotides of one or more specific size range s . In some embodiments the amount of sample polynucleotides subjected to fragmentation is about less than about or more than about 50 ng 100 ng 200 ng 300 ng 400 ng 500 ng 600 ng 700 ng 800 ng 900 ng 1000 ng 1500 ng 2000 ng 2500 ng 5000 ng 10 g or more. In some embodiments fragments are generated from about less than about or more than about 1 10 100 1000 10000 100000 300000 500000 or more genome equivalents of starting DNA. Fragmentation may be accomplished by methods known in the art including chemical enzymatic and mechanical fragmentation. In some embodiments the fragments have an average or median length from about 10 to about 10 000 nucleotides. In some embodiments the fragments have an average or median length from about 50 to about 2 000 nucleotides. In some embodiments the fragments have an average or median length of about less than about more than about or between about 100 2500 200 1000 10 800 10 500 50 500 50 250 or 50 150 nucleotides. In some embodiments the fragments have an average or median length of about less than about or more than about 200 300 500 600 800 1000 1500 or more nucleotides. In some embodiments the fragmentation is accomplished mechanically comprising subjecting sample polynucleotides to acoustic sonication. In some embodiments the fragmentation comprises treating the sample polynucleotides with one or more enzymes under conditions suitable for the one or more enzymes to generate double stranded nucleic acid breaks. Examples of enzymes useful in the generation of polynucleotide fragments include sequence specific and non sequence specific nucleases. Non limiting examples of nucleases include DNase I Fragmentase restriction endonucleases variants thereof and combinations thereof. For example digestion with DNase I can induce random double stranded breaks in DNA in the absence of Mg and in the presence of Mn . In some embodiments fragmentation comprises treating the sample polynucleotides with one or more restriction endonucleases. Fragmentation can produce fragments having 5 overhangs 3 overhangs blunt ends or a combination thereof. In some embodiments such as when fragmentation comprises the use of one or more restriction endonucleases cleavage of sample polynucleotides leaves overhangs having a predictable sequence. In some embodiments the method includes the step of size selecting the fragments via standard methods such as column purification or isolation from an agarose gel. In some embodiments the method comprises determining the average and or median fragment length after fragmentation. In some embodiments samples having an average and or median fragment length above a desired threshold are again subjected to fragmentation. In some embodiments samples having an average and or median fragment length below a desired threshold are discarded.

In some embodiments the 5 and or 3 end nucleotide sequences of fragmented polynucleotides are not modified prior to ligation with one or more adapter oligonucleotides also referred to as adapters . For example fragmentation by a restriction endonuclease can be used to leave a predictable overhang followed by ligation with one or more adapter oligonucleotides comprising an overhang complementary to the predictable overhang on a polynucleotide fragment. In another example cleavage by an enzyme that leaves a predictable blunt end can be followed by ligation of blunt ended polynucleotide fragments to adapter oligonucleotides comprising a blunt end. In some embodiments the fragmented polynucleotides are blunt end polished or end repaired to produce polynucleotide fragments having blunt ends prior to being joined to adapters. The blunt end polishing step may be accomplished by incubation with a suitable enzyme such as a DNA polymerase that has both 3 to 5 exonuclease activity and 5 to 3 polymerase activity for example T4 polymerase. In some embodiments end repair is followed by or concludes with addition of 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 or more nucleotides such as one or more adenine A tailing one or more thymine one or more guanine or one or more cytosine to produce an overhang. Polynucleotide fragments having an overhang can be joined to one or more adapter oligonucleotides having a complementary overhang such as in a ligation reaction. For example a single adenine can be added to the 3 ends of end repaired polynucleotide fragments using a template independent polymerase followed by ligation to one or more adapters each having an overhanging thymine at a 3 end. In some embodiments adapter oligonucleotides can be joined to blunt end double stranded DNA fragment molecules which have been modified by extension of the 3 end with one or more nucleotides followed by 5 phosphorylation. In some cases extension of the 3 end may be performed with a polymerase such as for example Klenow polymerase or any other suitable polymerases known in the art or by use of a terminal deoxynucleotide transferase in the presence of one or more dNTPs in a suitable buffer containing magnesium. In some embodiments target polynucleotides having blunt ends are joined to one or more adapters comprising a blunt end. Phosphorylation of 5 ends of fragmented polynucleotides may be performed for example with T4 polynucleotide kinase in a suitable buffer containing ATP and magnesium. The fragmented polynucleotides may optionally be treated to dephosphorylate 5 ends or 3 ends for example by using enzymes known in the art such as phosphatases.

In some embodiments fragmentation is followed by ligation of adapter oligonucleotides to the fragmented polynucleotides. An adapter oligonucleotide includes any oligonucleotide having a sequence at least a portion of which is known that can be joined to a target polynucleotide. Adapter oligonucleotides can comprise DNA RNA nucleotide analogues non canonical nucleotides labeled nucleotides modified nucleotides or combinations thereof. Adapter oligonucleotides can be single stranded double stranded or partial duplex. In general a partial duplex adapter comprises one or more single stranded regions and one or more double stranded regions. Double stranded adapters can comprise two separate oligonucleotides hybridized to one another also referred to as an oligonucleotide duplex and hybridization may leave one or more blunt ends one or more 3 overhangs one or more 5 overhangs one or more bulges resulting from mismatched and or unpaired nucleotides or any combination of these. In some embodiments a single stranded adapter comprises two or more sequences that are able to hybridize with one another. When two such hybridizable sequences are contained in a single stranded adapter hybridization yields a hairpin structure hairpin adapter . When two hybridized regions of an adapter are separated from one another by a non hybridized region a bubble structure results. Adapters comprising a bubble structure can consist of a single adapter oligonucleotide comprising internal hybridizations or may comprise two or more adapter oligonucleotides hybridized to one another. Internal sequence hybridization such as between two hybridizable sequences in an adapter can produce a double stranded structure in a single stranded adapter oligonucleotide. Adapters of different kinds can be used in combination such as a hairpin adapter and a double stranded adapter or adapters of different sequences. Different adapters can be joined to target polynucleotides in sequential reactions or simultaneously. In some embodiments identical adapters are added to both ends of a target polynucleotide. For example first and second adapters can be added to the same reaction. Adapters can be manipulated prior to combining with target polynucleotides. For example terminal phosphates can be added or removed.

In some embodiments an adapter is a mismatched adapter formed by annealing two partially complementary polynucleotide strands so as to provide when the two strands are annealed at least one double stranded region and at least one unmatched region. The double stranded region of the adapter is a short double stranded region typically comprising 5 or more consecutive base pairs formed by annealing of the two partially complementary polynucleotide strands. This term simply refers to a double stranded region of nucleic acid in which the two strands are annealed and does not imply any particular structural conformation. In some embodiments the double stranded region is about less than about or more than about 5 10 15 20 25 30 or more nucleotides in length. Generally it is advantageous for the double stranded region of a mismatched adapter to be as short as possible without loss of function. By function in this context is meant that the double stranded region form a stable duplex under standard reaction conditions for an enzyme catalyzed nucleic acid ligation reaction which conditions are known to those skilled in the art e.g. incubation at a temperature in the range of from 4 C. to 25 C. in a ligation buffer appropriate for the enzyme such that the two strands forming the adapter remain partially annealed during ligation of the adapter to a target molecule. It is not absolutely necessary for the double stranded region to be stable under the conditions typically used in the annealing steps of primer extension or PCR reactions. Typically the double stranded region is adjacent to the ligatable end of the adapter i.e. the end that is joined to a target polynucleotide in a ligation reaction. The ligatable end of the adapter may be blunt or in other embodiments short 5 or 3 overhangs of one or more nucleotides may be present to facilitate promote ligation. The 5 terminal nucleotide at the ligatable end of the adapter is typically phosphorylated to enable phosphodiester linkage to a 3 hydroxyl group on a sample polynucleotide. The term unmatched region refers to a region of the adapter wherein the sequences of the two polynucleotide strands forming the adapter exhibit a degree of non complementarity such that the two strands are not capable of annealing to each other under standard annealing conditions for a primer extension or PCR reaction. The two strands in the unmatched region may exhibit some degree of annealing under standard reaction conditions for an enzyme catalyzed ligation reaction provided that the two strands revert to single stranded form under annealing conditions.

Adapter oligonucleotides can contain one or more of a variety of sequence elements including but not limited to one or more amplification primer annealing sequences or complements thereof one or more sequencing primer annealing sequences or complements thereof one or more barcode sequences one or more common sequences shared among multiple different adapters or subsets of different adapters one or more restriction enzyme recognition sites one or more overhangs complementary to one or more target polynucleotide overhangs one or more probe binding sites e.g. for attachment to a sequencing platform such as a flow cell for massive parallel sequencing such as an apparatus as described herein or flow cells as developed by Illumina Inc. one or more random or near random sequences e.g. one or more nucleotides selected at random from a set of two or more different nucleotides at one or more positions with each of the different nucleotides selected at one or more positions represented in a pool of adapters comprising the random sequence and combinations thereof. Two or more sequence elements can be non adjacent to one another e.g. separated by one or more nucleotides adjacent to one another partially overlapping or completely overlapping. For example an amplification primer annealing sequence can also serve as a sequencing primer annealing sequence. Sequence elements can be located at or near the 3 end at or near the 5 end or in the interior of the adapter oligonucleotide. When an adapter oligonucleotide is capable of forming secondary structure such as a hairpin sequence elements can be located partially or completely outside the secondary structure partially or completely inside the secondary structure or in between sequences participating in the secondary structure. A sequence element may be of any suitable length such as about less than about or more than about 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 or more nucleotides in length. Adapter oligonucleotides can have any suitable length at least sufficient to accommodate the one or more sequence elements of which they are comprised. In some embodiments adapters are about less than about or more than about 10 15 20 25 30 35 40 45 50 55 60 65 70 75 80 90 100 200 or more nucleotides in length

In some embodiments the adapter oligonucleotides joined to fragmented polynucleotides from one sample comprise one or more sequences common to all adapter oligonucleotides and a barcode that is unique to the adapters joined to polynucleotides of that particular sample such that the barcode sequence can be used to distinguish polynucleotides originating from one sample or adapter joining reaction from polynucleotides originating from another sample or adapter joining reaction. In some embodiments an adapter oligonucleotide comprises a 5 overhang a 3 overhang or both that is complementary to one or more target polynucleotide overhangs. Complementary overhangs can be one or more nucleotides in length including but not limited to 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 or more nucleotides in length. Complementary overhangs may comprise a fixed sequence. Complementary overhangs of an adapter oligonucleotide may comprise a random sequence of one or more nucleotides such that one or more nucleotides are selected at random from a set of two or more different nucleotides at one or more positions with each of the different nucleotides selected at one or more positions represented in a pool of adapters with complementary overhangs comprising the random sequence. In some embodiments an adapter overhang is complementary to a target polynucleotide overhang produced by restriction endonuclease digestion. In some embodiments an adapter overhang consists of an adenine or a thymine.

In some embodiments adapter oligonucleotides comprise one strand comprising the sequence element sequence D. In some embodiments adapter oligonucleotides comprise sequence D hybridized to complementary sequence D where sequence D is on the same or different strand as sequence D. In some embodiments the 3 end of a target polynucleotide is extended along an adapter oligonucleotide to generate complementary sequence D . In a preferred embodiment fragmented polynucleotides and adapter oligonucleotides are combined and treated e.g. by ligation and optionally by fragment extension to produce double stranded adapted polynucleotides comprising fragmented polynucleotide sequence joined to adapter oligonucleotide sequences at both ends where both ends of the adapted polynucleotides comprise sequence D hybridized to sequence D . In some embodiments the amount of fragmented polynucleotides subjected to adapter joining is about less than about or more than about 50 ng 100 ng 200 ng 300 ng 400 ng 500 ng 600 ng 700 ng 800 ng 900 ng 1000 ng 1500 ng 2000 ng 2500 ng 5000 ng 10 g or more e.g. a threshold amount . In some embodiments the amount of fragmented polynucleotides is determined before proceeding with adapter joining where adapter joining is not performed if the amount is below a threshold amount.

The terms joining and ligation as used herein with respect to two polynucleotides such as an adapter oligonucleotide and a sample polynucleotide refers to the covalent attachment of two separate polynucleotides to produce a single larger polynucleotide with a contiguous backbone. Methods for joining two polynucleotides are known in the art and include without limitation enzymatic and non enzymatic e.g. chemical methods. Examples of ligation reactions that are non enzymatic include the non enzymatic ligation techniques described in U.S. Pat. Nos. 5 780 613 and 5 476 930 which are herein incorporated by reference. In some embodiments an adapter oligonucleotide is joined to a fragmented polynucleotide by a ligase for example a DNA ligase or RNA ligase. Multiple ligases each having characterized reaction conditions are known in the art and include without limitation NAD dependent ligases including tRNA ligase Taq DNA ligase DNA ligase DNA ligase Tth DNA ligase DNA ligase I and II thermostable ligase Ampligase thermostable DNA ligase VanC type ligase 9 N DNA Ligase Tsp DNA ligase and novel ligases discovered by bioprospecting ATP dependent ligases including T4 RNA ligase T4 DNA ligase T3 DNA ligase T7 DNA ligase Pfu DNA ligase DNA ligase 1 DNA ligase III DNA ligase IV and novel ligases discovered by bioprospecting and wild type mutant isoforms and genetically engineered variants thereof. Ligation can be between polynucleotides having hybridizable sequences such as complementary overhangs. Ligation can also be between two blunt ends. Generally a 5 phosphate is utilized in a ligation reaction. The 5 phosphate can be provided by the fragmented polynucleotide the adapter oligonucleotide or both. 5 phosphates can be added to or removed from polynucleotides to be joined as needed. Methods for the addition or removal of 5 phosphates are known in the art and include without limitation enzymatic and chemical processes. Enzymes useful in the addition and or removal of 5 phosphates include kinases phosphatases and polymerases. In some embodiments both of the two ends joined in a ligation reaction e.g. an adapter end and a fragmented polynucleotide end provide a 5 phosphate such that two covalent linkages are made in joining the two ends at one or both ends of a fragmented polynucleotide. In some embodiments 3 phosphates are removed prior to ligation. In some embodiments an adapter oligonucleotide is added to both ends of a fragmented polynucleotide wherein one or both strands at each end are joined to one or more adapter oligonucleotides. In some embodiments separate ligation reactions are carried out for different samples using a different adapter oligonucleotide comprising at least one different barcode sequence for each sample such that no barcode sequence is joined to the target polynucleotides of more than one sample to be analyzed in parallel.

Non limiting examples of adapter oligonucleotides include the double stranded adapter formed by hybridizing CACTCAGCAGCACGACGATCACAGATGTGTATAAGAGACAGT SEQ ID NO 17 to GTGAGTCGTCGTGCTGCTAGTGTCTACACATATTCTCTGTC SEQ ID NO 18 . Additional non limiting examples of adapter oligonucleotides are described in US20110319290 and US20070128624 which are incorporated herein by reference.

In some embodiments adapted polynucleotides are subjected to an amplification reaction that amplifies target polynucleotides in the sample. In some embodiments amplification uses primers comprising sequence C sequence D and a barcode associated with the sample wherein sequence D is positioned at the 3 end of the amplification primers. Amplification primers may be of any suitable length such as about less than about or more than about 5 10 15 20 25 30 35 40 45 50 55 60 65 70 75 80 90 100 or more nucleotides any portion or all of which may be complementary to the corresponding target sequence to which the primer hybridizes e.g. about less than about or more than about 5 10 15 20 25 30 35 40 45 50 or more nucleotides . Amplification refers to any process by which the copy number of a target sequence is increased. Methods for primer directed amplification of target polynucleotides are known in the art and include without limitation methods based on the polymerase chain reaction PCR . Conditions favorable to the amplification of target sequences by PCR are known in the art can be optimized at a variety of steps in the process and depend on characteristics of elements in the reaction such as target type target concentration sequence length to be amplified sequence of the target and or one or more primers primer length primer concentration polymerase used reaction volume ratio of one or more elements to one or more other elements and others some or all of which can be altered. In general PCR involves the steps of denaturation of the target to be amplified if double stranded hybridization of one or more primers to the target and extension of the primers by a DNA polymerase with the steps repeated or cycled in order to amplify the target sequence. Steps in this process can be optimized for various outcomes such as to enhance yield decrease the formation of spurious products and or increase or decrease specificity of primer annealing. Methods of optimization are well known in the art and include adjustments to the type or amount of elements in the amplification reaction and or to the conditions of a given step in the process such as temperature at a particular step duration of a particular step and or number of cycles. In some embodiments an amplification reaction comprises at least 5 10 15 20 25 30 35 50 or more cycles. In some embodiments an amplification reaction comprises no more than 5 10 15 20 25 35 50 or more cycles. Cycles can contain any number of steps such as 1 2 3 4 5 6 7 8 9 10 or more steps. Steps can comprise any temperature or gradient of temperatures suitable for achieving the purpose of the given step including but not limited to strand denaturation primer annealing and primer extension. Steps can be of any duration including but not limited to about less than about or more than about 1 5 10 15 20 25 30 35 40 45 50 55 60 70 80 90 100 120 180 240 300 360 420 480 540 600 or more seconds including indefinitely until manually interrupted. Cycles of any number comprising different steps can be combined in any order.

In some embodiments amplification comprises hybridization between sequence D at the 3 end of an amplification primer and sequence D of an adapted polynucleotide extension of the amplification primer along the adapted polynucleotide to produce a primer extension product comprising sequence D derived from the amplification primer and sequence D produced during primer extension. In some embodiments the amplification process is repeated one or more times by denaturing the primer extension product from a template polynucleotide and repeating the process using the primer extension product as template for further primer extension reactions. In some embodiments the first cycle of primer extension is repeated using the same primer as the primer used in the first primer extension reaction such as for about less than about or more than about 5 10 15 20 25 30 35 50 or more cycles. In some embodiments one or more primer extensions by the amplification primer is followed by one or more amplification cycles using a second amplification primer having a 3 end comprising a sequence complementary to a sequence added to the adapted polynucleotides by amplification with the first amplification primer e.g. complementary to the complement of sequence C or a portion thereof . In some embodiments the second amplification primer comprises sequence C or a portion thereof at the 3 end. A non limiting example of a second amplification primer includes CGAGATCTACACGCCTCCCTCGCGCCATCAG SEQ ID NO 19 . In some embodiments amplification by the second amplification primer comprises about less than about or more than about 5 10 15 20 25 30 35 50 or more cycles. In some embodiments the amount of adapted polynucleotides subjected to amplification is about less than about or more than about 50 ng 100 ng 200 ng 300 ng 400 ng 500 ng 600 ng 700 ng 800 ng 900 ng 1000 ng 1500 ng 2000 ng 2500 ng 5000 ng 10 g or more e.g. a threshold amount . In some embodiments the amount of adapted polynucleotides is determined before proceeding with amplification where amplification is not performed if the amount is below a threshold amount.

In some embodiments the amplification primer comprises a barcode. As used herein the term barcode refers to a known nucleic acid sequence that allows some feature of a polynucleotide with which the barcode is associated to be identified. In some embodiments the feature of the polynucleotide to be identified is the sample from which the polynucleotide is derived. In some embodiments barcodes are about or at least about 3 4 5 6 7 8 9 10 11 12 13 14 15 or more nucleotides in length. In some embodiments barcodes are shorter than 10 9 8 7 6 5 or 4 nucleotides in length. In some embodiments barcodes associated with some polynucleotides are of different lengths than barcodes associated with other polynucleotides. In general barcodes are of sufficient length and comprise sequences that are sufficiently different to allow the identification of samples based on barcodes with which they are associated. In some embodiments a barcode and the sample source with which it is associated can be identified accurately after the mutation insertion or deletion of one or more nucleotides in the barcode sequence such as the mutation insertion or deletion of 1 2 3 4 5 6 7 8 9 10 or more nucleotides. In some embodiments each barcode in a plurality of barcodes differ from every other barcode in the plurality at at least three nucleotide positions such as at least 3 4 5 6 7 8 9 10 or more nucleotide positions. A plurality of barcodes may be represented in a pool of samples each sample comprising polynucleotides comprising one or more barcodes that differ from the barcodes contained in the polynucleotides derived from the other samples in the pool. Samples of polynucleotides comprising one or more barcodes can be pooled based on the barcode sequences to which they are joined such that all four of the nucleotide bases A G C and T are approximately evenly represented at one or more positions along each barcode in the pool such as at 1 2 3 4 5 6 7 8 or more positions or all positions of the barcode . In some embodiments the methods of the invention further comprise identifying the sample from which a target polynucleotide is derived based on a barcode sequence to which the target polynucleotide is joined. In general a barcode comprises a nucleic acid sequence that when joined to a target polynucleotide serves as an identifier of the sample from which the target polynucleotide was derived.

In some embodiments separate amplification reactions are carried out for separate samples using amplification primers comprising at least one different barcode sequence for each sample such that no barcode sequence is joined to the target polynucleotides of more than one sample in a pool of two or more samples. In some embodiments amplified polynucleotides derived from different samples and comprising different barcodes are pooled before proceeding with subsequent manipulation of the polynucleotides such as before amplification and or sequencing on a solid support . Pools can comprise any fraction of the total constituent amplification reactions including whole reaction volumes. Samples can be pooled evenly or unevenly. In some embodiments target polynucleotides are pooled based on the barcodes to which they are joined. Pools may comprise polynucleotides derived from about less than about or more than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 20 25 30 40 50 75 100 or more different samples. Samples can be pooled in multiples of four in order to represent all four of the nucleotide bases A G C and T at one or more positions along the barcode evenly for example 4 8 12 16 20 24 28 32 36 40 44 48 52 56 60 64 96 128 192 256 384 and so on. Non limiting examples of barcodes include AGGTCA CAGCAG ACTGCT TAACGG GGATTA AACCTG GCCGTT CGTTGA GTAACC CTTAAC TGCTAA GATCCG CCAGGT TTCAGC ATGATC and TCGGAT. In some embodiments the barcode is positioned between sequence D and sequence C of an amplification primer or after sequence C and sequence D in a 5 to 3 direction downstream . In some embodiments the amplification primer comprises or consists of the sequence CGAGATCTACACGCCTCCCTCGCGCCATCAG CACTCAGCAGCACGACGATCAC SEQ ID NO 21 where each X represents zero one or more nucleotides of a barcode sequence.

In some embodiments target polynucleotides are hybridized to a plurality of oligonucleotides that are attached to a solid support such as any apparatus described herein. Hybridization may be before or after one or more sample processing steps such as adapter joining and amplification. In preferred embodiments target polynucleotides are hybridized to oligonucleotides on a solid support after both adapter joining and one or more amplification reactions. Oligonucleotides on the solid support may hybridize to random polynucleotide sequences specific sequences common to multiple different target polynucleotides e.g. one or more sequences derived from an adapter oligonucleotide such as sequences D D or a portion thereof one or more sequences derived from an amplification primer such as sequences C C or a portion thereof or combinations of these sequences specific to different target polynucleotides such as represented by sequence B as described herein or combinations of these. In some embodiments the solid support comprises a plurality of different first oligonucleotides comprising sequence A and sequence B wherein sequence A is common among all first oligonucleotides and further wherein sequence B is different for each different first oligonucleotide is at the 3 end of each first oligonucleotide. In some embodiments the plurality of first oligonucleotides comprises about less than about or more than about 5 10 25 50 75 100 125 150 175 200 300 400 500 750 1000 2500 5000 7500 10000 20000 50000 or more different oligonucleotides each comprising a different sequence B. In some embodiments sequence B of one or more of the plurality of first oligonucleotides comprises a sequence selected from the group consisting of SEQ ID NOs 22 121 shown in e.g. 1 5 10 25 50 75 or 100 different oligonucleotides each with a different sequence from . In some embodiments sequence B or the target sequence to which it specifically hybridizes comprises a causal genetic variant as described herein. In some embodiments sequence B or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of a causal genetic variant as described herein. Causal genetic variants are typically located downstream of a first oligonucleotide such that at least a portion of the causal genetic variant serves as template for extension of a first oligonucleotide. The solid support may further comprise a plurality of second oligonucleotides comprising sequence A at the 3 end of each second oligonucleotide and a plurality of third oligonucleotides comprising sequence C at the 3 end of each third oligonucleotide as described herein.

In some embodiments sequence B of one or more of the plurality of first oligonucleotides or the target sequence to which it specifically hybridizes comprises a non subject sequence. In some embodiments sequence B or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of a non subject sequence. In general a non subject sequence corresponds to a polynucleotide derived from an organism other than the individual being tested or from whom a sample is taken such as DNA or RNA from bacteria archaea viruses protists fungi or other organism. Non subject sequence can also include nucleic acids from a fetus such as cell free nucleic acid also referred to as extracellular nucleic acid from a fetus. A non subject sequence may be indicative of the identity of an organism or class of organisms and may further be indicative of a disease state such as infection. An example of non subject sequences useful in identifying an organism include without limitation rRNA sequences such as 16s rRNA sequences see e.g. WO2010151842 . In some embodiments non subject sequences are analyzed instead of or separately from causal genetic variants. In some embodiments causal genetic variants and non subject sequences are analyzed in parallel such as in the same sample e.g. using a mixture of first oligonucleotides some with a sequence B that specifically hybridizes to a sequence comprising or near a causal genetic variant and some with a sequence B that specifically hybridizes to a sequence comprising or near a non subject sequence and or in the same report.

In some embodiments sequence B of one or more of the plurality of first oligonucleotides or the target sequence to which it specifically hybridizes comprises an ancestry informative marker AIM . In some embodiments sequence B or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of an AIM. An AIM may be used to classify a person as belonging to or not belonging to one or more populations such as a population that is at increased risk for one of the causal genetic variants. For example an AIM can be diagnostic for a population in which a trait is at increased prevalence. In certain instances the AIM may distinguish between populations with finer granularity for example between sub continental groups or related ethnic groups. In some embodiments AIMs are analyzed instead of or separately from causal genetic variants and or non subject sequences. In some embodiments AIMs causal genetic variants and or non subject sequences are analyzed in parallel such as in the same sample e.g. using a mixture of first oligonucleotides some with a sequence B that specifically hybridizes to a sequence comprising or near a causal genetic variant and some with a sequence B that specifically hybridizes to a sequence comprising or near an AIM and or in the same report.

In some embodiments the method further comprises performing bridge amplification on the solid support. In general bridge amplification uses repeated steps of annealing of primers to templates primer extension and separation of extended primers from templates. These steps can generally be performed using reagents and conditions known to those skilled in PCR or reverse transcriptase plus PCR techniques. Thus a nucleic acid polymerase can be used together with a supply of nucleoside triphosphate molecules or other molecules that function as precursors of nucleotides present in DNA RNA such as modified nucleoside triphosphates to extend primers in the presence of a suitable template. Excess deoxyribonucleoside triphosphates are desirably provided. Preferred deoxyribonucleoside triphosphates are abbreviated dTTP deoxythymidine nucleoside triphosphate dATP deoxyadenosine nucleoside triphosphate dCTP deoxycytosine nucleoside triphosphate and dGTP deoxyguanosine nucleoside triphosphate . Preferred ribonucleoside triphosphates are UTP ATP CTP and GTP. However alternatives are possible. These may be naturally or non naturally occurring. A buffer of the type generally used in PCR reactions may also be provided. A nucleic acid polymerase used to incorporate nucleotides during primer extension is preferably stable under the reaction conditions utilized in order that it can be used several times. Thus where heating is used to separate a newly synthesized nucleic acid strand from its template the nucleic acid polymerase is preferably heat stable at the temperature used. Such heat stable polymerases are known to those skilled in the art. They are obtainable from thermophilic micro organisms and include the DNA dependent DNA polymerase known as Taq polymerase and also thermostable derivatives thereof.

Typically annealing of a primer to its template takes place at a temperature of 25 to 90 C. A temperature in this range will also typically be used during primer extension and may be the same as or different from the temperature used during annealing and or denaturation. Once sufficient time has elapsed to allow annealing and also to allow a desired degree of primer extension to occur the temperature can be increased if desired to allow strand separation. At this stage the temperature will typically be increased to a temperature of 60 to 100 C. High temperatures can also be used to reduce non specific priming problems prior to annealing and or to control the timing of amplification initiation e.g. in order to synchronize amplification initiation for a number of samples. Alternatively the strands maybe separated by treatment with a solution of low salt and high pH 12 or by using a chaotropic salt e.g. guanidinium hydrochloride or by an organic solvent e.g. formamide .

Following strand separation e.g. by heating a washing step may be performed. The washing step may be omitted between initial rounds of annealing primer extension and strand separation such as if it is desired to maintain the same templates in the vicinity of immobilized primers. This allows templates to be used several times to initiate colony formation. The size of colonies produced by amplification on the solid support can be controlled e.g. by controlling the number of cycles of annealing primer extension and strand separation that occur. Other factors which affect the size of colonies can also be controlled. These include the number and arrangement on a surface of immobilized primers the conformation of a support onto which the primers are immobilized the length and stiffness of template and or primer molecules temperature and the ionic strength and viscosity of a fluid in which the above mentioned cycles can be performed.

A non limiting example of an amplification process in accordance with the methods of the invention is illustrated in and described below. First a first oligonucleotide attached to the solid support and comprising sequence B at its 3 end hybridizes to a complementary target sequence B such as a sequence unique to a specific target polynucleotide in a plurality of different target polynucleotides e.g. a particular genomic DNA sequence . In this way sequence B serves as a probe. The target polynucleotide in comprises sequences derived from adapter oligonucleotides e.g. sequences D and D and from amplification primers e.g. C and C . Extension of the first oligonucleotide produces a first extension product attached to the solid support the first extension product comprising from 5 to 3 sequences A B C and D where sequence C is complementary to sequence C and sequence D is complementary to sequence D. The first extension product is then separated from the target polynucleotide template e.g. by heat or chemical denaturation . Sequence C of the first extension product then hybridizes to one of a plurality of third oligonucleotides attached to the solid support the third oligonucleotide comprising sequence C at its 3 end. Extension of the third oligonucleotide produces a second extension product attached to the solid support the second extension product comprising from 5 to 3 sequences C D B and A where sequence B is complementary to sequence B and sequence A is complementary to sequence A. The two extension products form a double stranded polynucleotide bridge with one strand at both ends attached to the solid support. The first and second extension products are then denatured and subsequence hybridizations between the extension products and other oligonucleotides followed by extension replicate the first and second extension products. For example each first extension product may hybridize to a further third oligonucleotide to produce additional copies of the second extension product. In addition a second extension product may hybridize to one of a plurality of second oligonucleotides attached to the solid support the second oligonucleotide comprising sequence A at its 3 end. Extension of the second oligonucleotide produces an extension product comprising the sequence of a first extension product. Successive rounds of extension along extension products radiates outward from an initial first extension product to produce a cluster or colony of first extension products and their complementary second extension products derived from a single target polynucleotide. This process may be modified to accommodate oligonucleotides comprising different sequences or sequence arrangements different target polynucleotides or combinations of target polynucleotides types of solid supports and other considerations depending on a particular bridge amplification reaction. In general this process provides for amplification on a solid support of specific target polynucleotides from sample polynucleotides comprising target polynucleotides and non target polynucleotides. Generally target polynucleotides are selectively amplified while non target polynucleotides in the sample are not amplified or are amplified to a much lower degree such as about or less than about 10 fold 100 fold 500 fold 1000 fold 2500 fold 5000 fold 10000 fold 25000 fold 50000 fold 100000 fold 1000000 fold or more lower than one or more target polynucleotides.

In some embodiments the amount of amplified polynucleotides from a previous amplification step that is subjected to bridge amplification is about less than about or more than about 50 ng 100 ng 500 ng 1 g 2 g 3 g 4 g 5 g 6 g 7 g 8 g 9 g 10 g 12 g 13 g 14 g 15 g 20 g 25 g 26 g 27 g 28 g 29 g 30 g 40 g 50 g or more e.g. a threshold amount . In some embodiments the amount of amplified polynucleotides from a previous amplification step is determined before proceeding with bridge amplification where bridge amplification is not performed if the amount is below a threshold amount.

In some embodiments bridge amplification is followed by sequencing a plurality of oligonucleotides attached to the solid support. General methods for sequencing polynucleotides attached to a solid support including reagents and reaction conditions are known in the art. In some embodiments sequencing comprises or consists of single end sequencing. In some embodiments sequencing comprises or consists of paired end sequencing. Sequencing can be carried out using any suitable sequencing technique wherein nucleotides are added successively to a free 3 hydroxyl group resulting in synthesis of a polynucleotide chain in the 5 to 3 direction. The identity of the nucleotide added is preferably determined after each nucleotide addition. Sequencing techniques using sequencing by ligation wherein not every contiguous base is sequenced and techniques such as massively parallel signature sequencing MPSS where bases are removed from rather than added to the strands on the surface are also within the scope of the invention as are techniques using detection of pyrophosphate release pyrosequencing . Such pyrosequencing based techniques are particularly applicable to sequencing arrays of beads where the beads have been amplified in an emulsion such that a single template from the library molecule is amplified on each bead.

One particular sequencing method which can be used in the methods of the invention relies on the use of modified nucleotides that can act as reversible chain terminators. Such reversible chain terminators comprise removable 3 blocking groups for example as described in WO04018497 and U.S. Pat. No. 7 057 026. Once such a modified nucleotide has been incorporated into the growing polynucleotide chain complementary to the region of the template being sequenced there is no free 3 OH group available to direct further sequence extension and therefore the polymerase cannot add further nucleotides. Once the identity of the base incorporated into the growing chain has been determined the 3 block may be removed to allow addition of the next successive nucleotide. By ordering the products derived using these modified nucleotides it is possible to deduce the DNA sequence of the DNA template. Such reactions can be done in a single experiment if each of the modified nucleotides has attached thereto a different label known to correspond to the particular base to facilitate discrimination between the bases added at each incorporation step. Non limiting examples of suitable labels are described in WO 2007 135368 the contents of which are incorporated herein by reference in their entirety. Alternatively a separate reaction may be carried out containing each of the modified nucleotides added individually.

The modified nucleotides may carry a label to facilitate their detection. In a particular embodiment the label is a fluorescent label. Each nucleotide type may carry a different fluorescent label. However the detectable label need not be a fluorescent label. Any label can be used which allows the detection of the incorporation of the nucleotide into the DNA sequence. One method for detecting fluorescently labeled nucleotides comprises using laser light of a wavelength specific for the labeled nucleotides or the use of other suitable sources of illumination. Fluorescence from the label on an incorporated nucleotide may be detected by a CCD camera or other suitable detection means. Suitable detection means are described in WO 2007 123744 the contents of which are incorporated herein by reference in their entirety.

In some embodiments a first sequencing reaction proceeds from a 3 end created by cleavage at a cleavage site contained in an oligonucleotide attached to the solid support which oligonucleotide was extended during bridge amplification. In some embodiments the cleaved strand is separated from its complementary strand before sequencing by extension of the attached oligonucleotide. In some embodiments the attached oligonucleotide having the newly freed 3 end created by cleavage is extended using a polymerase having strand displacement activity such that the cleaved strand is displaced as the new strand is extended. In some embodiments extension of the attached oligonucleotide proceeds along the full length of the template extension product from the amplification reaction which in some embodiments includes extension beyond a last identified nucleotide. In some embodiments the template extension product is then cleaved at a cleavage site contained in an oligonucleotide attached to the solid support and the oligonucleotide extended during the sequencing reaction is linearized for produce a freed first sequencing extension product. The 5 end of the first sequencing product may then serve as a template for a second sequencing reaction which can proceed by extension of a sequencing primer such as a sequencing primer described herein or by extension from the 3 end created by cleavage at the cleavage site. In some embodiments the average or median number of nucleotides identified along a template polynucleotide being sequenced is about less than about or more than about 10 15 20 25 30 35 40 45 50 60 70 80 90 100 150 200 300 400 500 or more.

In some embodiments sequencing comprises treating bridge amplification products to remove substantially all or remove or displace at least a portion of one of the immobilized strands in the bridge structure in order to generate a template that is at least partially single stranded. The portion of the template which is single stranded will thus be available for hybridization with a sequencing primer. The process of removing all or a portion of one immobilized strand in a bridged double stranded nucleic acid structure may be referred to herein as linearization and is described in further detail in WO07010251 the contents of which are incorporated herein by reference in their entirety.

Bridged template structures may be linearized by cleavage of one or both strands with a restriction endonuclease or by cleavage of one strand with a nicking endonuclease. Other methods of cleavage can be used as an alternative to restriction enzymes or nicking enzymes including but not limited to chemical cleavage e.g. cleavage of a diol linkage with periodate cleavage of abasic sites by cleavage with endonuclease for example USER as supplied by NEB part number M5505S by exposure to heat or alkali cleavage of ribonucleotides incorporated into amplification products otherwise comprised of deoxyribonucleotides photochemical cleavage or cleavage of a peptide linker. In some embodiments a linearization step may be avoided such as when the solid phase amplification reaction is performed with only one amplification oligonucleotide covalently immobilized and another amplification oligonucleotide free in solution. Following the cleavage step regardless of the method used for cleavage the product of the cleavage reaction may be subjected to denaturing conditions in order to remove the portion s of the cleaved strand s that are not attached to the solid support. Suitable denaturing conditions for example sodium hydroxide solution formamide solution or heat are known in the art such as described in standard molecular biology protocols Sambrook et al. 2001 Molecular Cloning A Laboratory Manual 3rd Ed Cold Spring Harbor Laboratory Press Cold Spring Harbor Laboratory Press NY Current Protocols eds Ausubel et al. . Denaturation results in the production of a sequencing template which is partially or substantially single stranded. A sequencing reaction may then be initiated by hybridization of a sequencing primer to the single stranded portion of the template. Thus the invention encompasses methods wherein the nucleic acid sequencing reaction comprises hybridizing a sequencing primer to a single stranded region of a linearized amplification product sequentially incorporating one or more nucleotides into a polynucleotide strand complementary to the region of amplified template strand to be sequenced identifying the base present in one or more of the incorporated nucleotide s and thereby determining the sequence of a region of the template strand.

In some embodiments the sequencing primer comprises a sequence complementary to one or more sequences derived from an adapter oligonucleotide an amplification primer an oligonucleotide attached to the solid support or a combination of these. In some embodiments the sequencing primer comprises sequence D or a portion thereof. In some embodiments a sequencing primer comprises sequence C or a portion thereof. A sequencing primer can be of any suitable length such as about less than about or more than about 5 10 15 20 25 30 35 40 45 50 55 60 65 70 75 80 90 100 or more nucleotides any portion or all of which may be complementary to the corresponding target sequence to which the primer hybridizes e.g. about less than about or more than about 5 10 15 20 25 30 35 40 45 50 or more nucleotides . In some embodiments a sequencing primer comprises the sequence CACTCAGCAGCACGACGATCACAGATGTGTATAAGAGACAG SEQ ID NO 20 .

In general extension of a sequencing primer produces a sequencing extension product. The number of nucleotides added to the sequencing extension product that are identified in the sequencing process may depend on a number of factors including template sequence reaction conditions reagents used and other factors. In some embodiments the average or median number of nucleotides identified along a growing sequencing primer is about less than about or more than about 10 15 20 25 30 35 40 45 50 60 70 80 90 100 150 200 300 400 500 or more. In some embodiments a sequencing primer is extended along the full length of the template primer extension product from the amplification reaction which in some embodiments includes extension beyond a last identified nucleotide.

In some embodiments the sequencing extension product is subjected to denaturing conditions in order to remove the sequencing extension product from the attached template strand to which it is hybridized in order to make the template partially or completely single stranded and available for hybridization with a second sequencing primer. The second sequencing primer may be the same as or different from the first sequencing primer. In some embodiments the second sequencing primer hybridizes to a sequence located closer to the 5 end of the target nucleic acid than the sequence to which the first sequencing primer hybridizes. In some embodiments the second sequencing primer hybridizes to a sequence located closer to the 3 end of the target nucleic acid than the sequence to which the first sequencing primer hybridizes. In some embodiments only one of the first and second sequencing primers is extended along a barcode sequence thereby identifying the nucleotides in the barcode sequence. In some embodiments one sequencing primer e.g. the first sequencing primer hybridizes to a sequence located 5 from the barcode such that extension of this sequencing primer does not generate sequence complementary to the barcode and another sequencing primer e.g. the second sequencing primer hybridizes to a sequence located 3 from the barcode such that extension of this sequencing primer generates sequence complementary to the barcode . In some embodiments the second sequencing primer comprises SEQ ID NO 19.

The invention is not intended to be limited to use of the sequencing methods outlined above as essentially any sequencing methodology which relies on successive incorporation of nucleotides into a polynucleotide chain can be used. Suitable techniques include for example those described in U.S. Pat. No. 6 306 597 US20090233802 US20120053074 and US20110223601 which are incorporated by reference in their entireties. In the cases where strand resynthesis is employed both strands must be immobilized to the surface in a way that allows subsequent release of a portion of the immobilized strand. This can be achieved through a number of mechanisms as described in WO07010251 the contents of which are incorporated herein by reference in their entirety. For example one primer can contain a uracil nucleotide which means that the strand can be cleaved at the uracil base using the enzyme uracil DNA glycosylase UDG which removes the nucleotide base and endonuclease VIII that excises the abasic nucleotide. This enzyme combination is available as USER from New England Biolabs NEB part number M5505 . The second primer may comprise an 8 oxoguanine nucleotide which is then cleavable by the enzyme FPG NEB part number M0240 . This design of primers provides complete control of which primer is cleaved at which point in the process and also where in the cluster the cleavage occurs. The primers may also be chemically modified for example with a disulfide or diol modification that allows chemical cleavage at specific locations.

In some embodiments sequencing data are generated for about less than about or more than about 5 10 25 50 100 150 200 250 300 400 500 750 1000 2500 5000 7500 10000 20000 50000 or more different target polynucleotides from a sample in a single reaction container e.g. a channel in a flow cell . In some embodiments sequencing data are generated for a plurality of samples in parallel such as about less than about or more than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 24 48 96 192 384 768 1000 or more samples. In some embodiments sequencing data are generated for a plurality of samples in a single reaction container e.g. a channel in a flow cell such as about less than about or more than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 24 48 96 192 384 768 1000 or more samples and sequencing data are subsequently grouped according to the sample from which the sequenced polynucleotides originated. In a single reaction sequencing data may be generated for about or at least about 10 10 10 2 10 3 10 4 10 5 10 10 10 or more target polynucleotides or clusters from a bridge amplification reaction which may comprise sequencing data for about less than about or more than about 10 10 10 2 10 3 10 4 10 5 10 10 10 or more target polynucleotides or clusters for each sample in the reaction. In some embodiments the presence absence or genotype of about less than about or more than about 5 10 25 50 75 100 125 150 175 200 300 400 500 750 1000 2500 5000 7500 10000 20000 50000 or more causal genetic variants is determined for a sample based on the sequencing data. The presence absence or genotype of one or more causal genetic variants may be determined with an accuracy of about or more than about 80 85 90 95 97.5 99 99.5 99.9 or higher.

In some embodiments one or more or all of the steps in a method of the invention are automated such as by use of one or more automated devices. In general automated devices are devices that are able to operate without human direction an automated system can perform a function during a period of time after a human has finished taking any action to promote the function e.g. by entering instructions into a computer after which the automated device performs one or more steps without further human operation. Software and programs including code that implements embodiments of the present invention may be stored on some type of data storage media such as a CD ROM DVD ROM tape flash drive or diskette or other appropriate computer readable medium. Various embodiments of the present invention can also be implemented exclusively in hardware or in a combination of software and hardware. For example in one embodiment rather than a conventional personal computer a Programmable Logic Controller PLC is used. As known to those skilled in the art PLCs are frequently used in a variety of process control applications where the expense of a general purpose computer is unnecessary. PLCs may be configured in a known manner to execute one or a variety of control programs and are capable of receiving inputs from a user or another device and or providing outputs to a user or another device in a manner similar to that of a personal computer. Accordingly although embodiments of the present invention are described in terms of a general purpose computer it should be appreciated that the use of a general purpose computer is exemplary only as other configurations may be used.

In some embodiments automation may comprise the use of one or more liquid handlers and associated software. Several commercially available liquid handling systems can be utilized to run the automation of these processes see for example liquid handlers from Perkin Elmer Beckman Coulter Caliper Life Sciences Tecan Eppendorf Apricot Design Velocity 11 as examples . In some embodiments automated steps include one or more of fragmentation end repair A tailing addition of adenine overhang adapter joining PCR amplification sample quantification e.g. amount and or purity of DNA and sequencing. In some embodiments hybridization of amplified polynucleotides to oligonucleotides attached to a solid surface extension along the amplified polynucleotides as templates and or bridge amplification is automated e.g. by use of an Illumina cBot . Non limiting examples of devices for conducting bridge amplification are described in WO2008002502. In some embodiments sequencing is automated. A variety of automated sequencing machines are commercially available and include sequencers manufactured by Life Technologies SOLiD platform and pH based detection Roche 454 platform Illumina e.g. flow cell based systems such as Genome Analyzer HiSeq or MiSeq systems . Transfer between 2 3 4 5 or more automated devices e.g. between one or more of a liquid handler bridge a amplification device and a sequencing device may be manual or automated. In some embodiments one or more steps in a method of the invention e.g. all steps or all automated steps are completed in about or less than about 72 48 24 20 18 16 14 12 10 9 8 7 6 5 4 3 2 1 or fewer hours. In some embodiments the time from sample receipt DNA extraction fragmentation adapter joining amplification or bridge amplification to production of sequencing data is about or less than about 72 48 24 20 18 16 14 12 10 9 8 7 6 5 4 3 2 1 or fewer hours.

In one aspect the invention provides a method of enriching a plurality of different target polynucleotides in a sample. In some embodiments the method comprises a joining an adapter oligonucleotide to each of the target polynucleotides wherein the adapter oligonucleotide comprises sequence Y b hybridizing a plurality of different oligonucleotide primers to the adapted target polynucleotides wherein each oligonucleotide primer comprises sequence Z and sequence W wherein sequence Z is common among all oligonucleotide primers and further wherein sequence W is different for each different oligonucleotide primer is positioned at the 3 end of each oligonucleotide primer and is complementary to a sequence comprising a causal genetic variant or a sequence within 200 nucleotides of a causal genetic variant c in an extension reaction extending the oligonucleotide primers along the adapted target polynucleotides to produce extended primers comprising sequence Z and sequence Y wherein sequence Y is complementary to sequence Y and d exponentially amplifying the purified extension products using a pair of amplification primers comprising i a first amplification primer comprising sequence V and sequence Z wherein sequence Z is positioned at the 3 end of the first amplification primer and ii a second amplification primer comprising sequence X and sequence Y wherein sequence Y and is positioned at the 3 end of the second amplification primer. In some embodiments each oligonucleotide primer comprises a first binding partner. In some embodiments the method further comprises before step d exposing the extended primers to a solid surface comprising a second binding partner that binds to the first binding partner thereby purifying the extended primers away from one or more components of the extension reaction. In some embodiments one or more of sequences V W X Y and Z are different sequences. In some embodiments sequence V and sequence X are the same. In some embodiments sequence V and or sequence X are not included in their respective primers. In some embodiments one or more of sequences V W X Y and Z are about less than about or more than about 5 10 15 20 25 30 40 50 60 70 80 90 or more different from one or more of the other of sequences V W X Y and Z e.g. have less than about 10 20 30 40 50 60 70 80 90 or more sequence identity . In some embodiments one or more of sequences V W X Y and Z comprise about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 or more nucleotides each. In some embodiments sequence V or sequence Z is equivalent to sequence A sequence W is equivalent to sequence B sequence X is equivalent to sequence C and or sequence Y is equivalent to sequence D as described with respect to other aspects of the invention.

In one aspect the invention provides a method of enriching a plurality of different target polynucleotides in a sample. In some embodiments the method comprises a hybridizing a plurality of different oligonucleotide primers to the target polynucleotides wherein each oligonucleotide primer comprises sequence Z and sequence W wherein sequence Z is common among all oligonucleotide primers and further wherein sequence W is different for each different oligonucleotide primer is positioned at the 3 end of each oligonucleotide primer and is complementary to a sequence comprising a causal genetic variant or a sequence within 200 nucleotides of a causal genetic variant b in an extension reaction extending the oligonucleotide primers along the target polynucleotides to produce extended primers c joining an adapter oligonucleotide to each extended primer wherein the adapter oligonucleotide comprises sequence Y and further wherein sequence Y is the complement of a sequence Y and d exponentially amplifying the purified extension products using a pair of amplification primers comprising i a first amplification primer comprising sequence V and sequence Z wherein sequence Z is positioned at the 3 end of the first amplification primer and ii a second amplification primer comprising sequence X and sequence Y wherein sequence Y and is positioned at the 3 end of the second amplification primer. In some embodiments each oligonucleotide primer comprises a first binding partner. In some embodiments the method further comprises before step c exposing the extended primers to a solid surface comprising a second binding partner that binds to the first binding partner thereby purifying the extended primers away from one or more components of the extension reaction. In some embodiments one or more of sequences V W X Y and Z are different sequences. In some embodiments sequence V and sequence X are the same. In some embodiments sequence V and or sequence X are not included in their respective primers. In some embodiments one or more of sequences V W X Y and Z are about less than about or more than about 5 10 15 20 25 30 40 50 60 70 80 90 or more different from one or more of the other of sequences V W X Y and Z e.g. have less than about 10 20 30 40 50 60 70 80 90 or more sequence identity . In some embodiments one or more of sequences V W X Y and Z comprise about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 or more nucleotides each. In some embodiments sequence V or sequence Z is equivalent to sequence A sequence W is equivalent to sequence B sequence X is equivalent to sequence C and or sequence Y is equivalent to sequence D as described with respect to other aspects of the invention.

Samples from which the target polynucleotides are derived can comprise multiple samples from the same individual samples from different individuals or combinations thereof. In some embodiments a sample comprises a plurality of polynucleotides from a single individual. In some embodiments a sample comprises a plurality of polynucleotides from two or more individuals. Examples of sources of sample polynucleotides and methods for their purification are described herein such as with regard to other aspects of the invention.

In some embodiments target polynucleotides are fragmented into a population of fragmented polynucleotides of one or more specific size range s . In some embodiments the amount of sample polynucleotides subjected to fragmentation is about less than about or more than about 50 ng 100 ng 200 ng 300 ng 400 ng 500 ng 600 ng 700 ng 800 ng 900 ng 1000 ng 1500 ng 2000 ng 2500 ng 5000 ng 10 g or more. In some embodiments fragments are generated from about less than about or more than about 1 10 100 1000 10000 100000 300000 500000 or more genome equivalents of starting DNA. Fragmentation may be accomplished by methods known in the art including chemical enzymatic and mechanical fragmentation. In some embodiments the fragments have an average or median length from about 10 to about 10 000 nucleotides. In some embodiments the fragments have an average or median length from about 50 to about 2 000 nucleotides. In some embodiments the fragments have an average or median length of about less than about more than about or between about 100 2500 200 1000 10 800 10 500 50 500 50 250 or 50 150 nucleotides. In some embodiments the fragments have an average or median length of about less than about or more than about 200 300 500 600 800 1000 1500 or more nucleotides. Example methods of fragmentation and optional end repair including optional A tailing are described herein such as with regard to other aspects of the invention. End repair may be performed at any step before joining of adapter oligonucleotides such as before or after extension of oligonucleotide primers.

In some embodiments fragmentation or oligonucleotide primer extension is followed by ligation of adapter oligonucleotides to the fragmented or extended polynucleotides see e.g. . Examples of adapter oligonucleotides and methods for their manipulation and joining to target polynucleotides are described herein such as with regard to other aspects of the invention. In some embodiments adapter oligonucleotides comprise one strand comprising the sequence element sequence Y. In some embodiments adapter oligonucleotides comprise one strand comprising the sequence element sequence Y which is the complement of sequence Y. In some embodiments adapter oligonucleotides comprise sequence Y hybridized to complementary sequence Y where sequence Y is on the same or different strand as sequence Y. In some embodiments the 3 end of a target polynucleotide or extended primer is extended along an adapter oligonucleotide to generate sequence Y or sequence Y . In some embodiments fragmented polynucleotides and adapter oligonucleotides are combined and treated e.g. by ligation and optionally by fragment extension to produce double stranded adapted polynucleotides comprising fragmented polynucleotide sequence joined to adapter oligonucleotide sequences at both ends where both ends of the adapted polynucleotides comprise sequence Y hybridized to sequence Y . In some embodiments extended primers that are hybridized to target polynucleotides are combined and treated e.g. by ligation and optionally by 3 end extension to produce double stranded adapted polynucleotides comprising sequence Y hybridized to sequence Y at one end. In some embodiments the amount of fragmented polynucleotides subjected to further manipulation e.g. adapter joining or oligonucleotide primer extension is about less than about or more than about 50 ng 100 ng 200 ng 300 ng 400 ng 500 ng 600 ng 700 ng 800 ng 900 ng 1000 ng 1500 ng 2000 ng 2500 ng 5000 ng 10 g or more e.g. a threshold amount . In some embodiments the amount of fragmented polynucleotides is determined before proceeding with further manipulation where further manipulation is not performed if the amount is below a threshold amount.

In some embodiments primer extension products comprising sequences complementary to target polynucleotide sequences are produced in an extension reaction. In general an extension reaction comprises extension of an oligonucleotide primer hybridized to a target polynucleotide. Oligonucleotide primers may be of any suitable length such as about less than about or more than about 5 10 15 20 25 30 35 40 45 50 55 60 65 70 75 80 90 100 or more nucleotides any portion or all of which may be complementary to the corresponding target sequence to which the primer hybridizes e.g. about less than about or more than about 5 10 15 20 25 30 35 40 45 50 or more nucleotides . Primer extension may comprise one or more cycles of a PCR reaction such as denaturation primer annealing and primer extension which may be repeated any number of times with or without a reverse primer. For example in the absence of a reverse primer multiple cycles may be used to linearly amplify one or more target polynucleotides by repeated extension of primers along the corresponding targets without using extended primers as templates for further amplification. Examples of oligonucleotides useful as primers and methods for their use in primer extension reactions e.g. amplification are provided herein such as with regard to other aspects of the invention. An illustration of a non limiting example of an amplification method is provided in .

In some embodiments an oligonucleotide primer comprises sequence Z which is common to each of a plurality of different oligonucleotide primers in a reaction and sequence W which is different for each different oligonucleotide primer and is positioned at the 3 end of each oligonucleotide primer. In some embodiments the plurality of oligonucleotide primers comprises about less than about or more than about 5 10 25 50 75 100 125 150 175 200 300 400 500 750 1000 2500 5000 7500 10000 20000 50000 or more different oligonucleotides each comprising a different sequence W. In some embodiments sequence W of one or more of the plurality of oligonucleotide primers comprises a sequence selected from the group consisting of SEQ ID NOs 22 121 shown in e.g. 1 5 10 25 50 75 or 100 different oligonucleotides each with a different sequence from . In some embodiments sequence W or the target sequence to which it specifically hybridizes comprises a causal genetic variant as described herein. In some embodiments sequence W or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of a causal genetic variant as described herein. Causal genetic variants are typically located downstream of an oligonucleotide primer such that at least a portion of the causal genetic variant serves as template for extension of an oligonucleotide primer. Typically extension of an oligonucleotide primer along a target polynucleotide comprising sequence Y derived from an adapter oligonucleotide produces a primer extension product comprising primer derived sequences a the 5 end and sequences complementary to adapter derived sequences near the 3 end e.g. sequence Y the complement of Y .

In some embodiments sequence W of one or more of the plurality of oligonucleotide primers or the target sequence to which it specifically hybridizes comprises a non subject sequence. In some embodiments sequence W or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of a non subject sequence. In general a non subject sequence corresponds to a polynucleotide derived from an organism other than the individual being tested such as DNA or RNA from bacteria archaea viruses protists fungi or other organism. A non subject sequence may be indicative of the identity of an organism or class of organisms and may further be indicative of a disease state such as infection. An example of non subject sequences useful in identifying an organism include without limitation rRNA sequences such as 16s rRNA sequences see e.g. WO2010151842 . In some embodiments non subject sequences are analyzed instead of or separately from causal genetic variants. In some embodiments causal genetic variants and non subject sequences are analyzed in parallel such as in the same sample e.g. using a mixture of oligonucleotide primers some with a sequence W that specifically hybridizes to a sequence comprising or near a causal genetic variant and some with a sequence W that specifically hybridizes to a sequence comprising or near a non subject sequence and or in the same report.

In some embodiments sequence W of one or more of the plurality of oligonucleotide primers or the target sequence to which it specifically hybridizes comprises an ancestry informative marker AIM . In some embodiments sequence W or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of an AIM. An AIM may be used to classify a person as belonging to or not belonging to one or more populations such as a population that is at increased risk for one of the causal genetic variants. For example an AIM can be diagnostic for a population in which a trait is at increased prevalence. In certain instances the AIM may distinguish between populations with finer granularity for example between sub continental groups or related ethnic groups. In some embodiments AIMs are analyzed instead of or separately from causal genetic variants and or non subject sequences. In some embodiments AIMs causal genetic variants and or non subject sequences are analyzed in parallel such as in the same sample e.g. using a mixture of first oligonucleotides some with a sequence B that specifically hybridizes to a sequence comprising or near a causal genetic variant and some with a sequence B that specifically hybridizes to a sequence comprising or near an AIM and or in the same report.

In some embodiments the oligonucleotide primers comprise a first binding partner such as a member of a binding pair. In general binding partner refers to one of a first and a second moiety wherein the first and the second moiety have a specific binding affinity for each other. Suitable binding pairs for use in the invention include but are not limited to antigens antibodies for example digoxigenin anti digoxigenin dinitrophenyl DNP anti DNP dansyl X anti dansyl Fluorescein anti fluorescein lucifer yellow anti lucifer yellow and rhodamine anti rhodamine biotin avidin or biotin streptavidin calmodulin binding protein CBP calmodulin hormone hormone receptor lectin carbohydrate peptide cell membrane receptor protein A antibody hapten antihapten enzyme cofactor and enzyme substrate. Other suitable binding pairs include polypeptides such as the FLAG peptide Hopp et al. BioTechnology 6 1204 1210 1988 the KT3 epitope peptide Martin et al. Science 255 192 194 1992 tubulin epitope peptide Skinner et al. J. Biol. Chem. 266 15163 15166 1991 and the T7 gene 10 protein peptide tag Lutz Freyermuth et al. Proc. Natl. Acad. Sci. USA 87 6393 6397 1990 and the antibodies each thereto. Further non limiting examples of binding partners include agonists and antagonists for cell membrane receptors toxins and venoms viral epitopes hormones such as steroids hormone receptors peptides enzymes and other catalytic polypeptides enzyme substrates cofactors drugs including small organic molecule drugs opiates opiate receptors lectins sugars saccharides including polysaccharides proteins and antibodies including monoclonal antibodies and synthetic antibody fragments cells cell membranes and moieties therein including cell membrane receptors and organelles. In some embodiments the first binding partner is a reactive moiety and the second binding partner is a reactive surface that reacts with the reactive moiety such as described herein with respect to other aspects of the invention. In some embodiments the oligonucleotide primers are attached to the solid surface prior to initiating the extension reaction. Methods for the addition of binding partners to oligonucleotides are known in the art and include addition during such as by using a modified nucleotide comprising the binding partner or after synthesis.

In some embodiments extension of the oligonucleotide primers is followed by purification of extended primers on a solid surface. In some embodiments adapter joining is followed by purification of extended primers on a solid surface. Typically the solid surface comprises a second binding partner which is the second member of a binding pair and binds to the first binding partner. In some embodiments a solid surface may have a wide variety of forms including membranes slides plates micromachined chips microparticles beads and the like. Solid surfaces may comprise a wide variety of materials including but not limited to glass plastic silicon alkanethiolate derivatized gold cellulose low cross linked and high cross linked polystyrene silica gel polyamide and the like and can have various shapes and features e.g. wells indentations channels etc. . The surface can be hydrophilic or capable of being rendered hydrophilic and may comprise inorganic powders such as silica magnesium sulfate and alumina natural polymeric materials particularly cellulosic materials and materials derived from cellulose such as fiber containing papers e.g. filter paper chromatographic paper etc. synthetic or modified naturally occurring polymers such as nitrocellulose cellulose acetate poly vinyl chloride polyacrylamide cross linked dextran agarose polyacrylate polyethylene polypropylene poly 4 methylbutene polystyrene polymethacrylate poly ethylene terephthalate nylon poly vinyl butyrate etc. either used by themselves or in conjunction with other materials glass available as Bioglass ceramics metals and the like. Natural or synthetic assemblies such as liposomes phospholipid vesicles and cells can also be employed. The surface can have any one of a number of shapes such as strip rod particle including bead and the like.

In some embodiments the solid surface comprises a bead or plurality of beads. The beads may be of any convenient size and fabricated from any number of known materials. Example of such materials include inorganics natural polymers and synthetic polymers. Specific examples of these materials include cellulose cellulose derivatives acrylic resins glass silica gels polystyrene gelatin polyvinyl pyrrolidone co polymers of vinyl and acrylamide polystyrene cross linked with divinylbenzene or the like as described e.g in Merrifield Biochemistry 1964 3 1385 1390 polyacrylamides latex gels polystyrene dextran rubber silicon plastics nitrocellulose natural sponges silica gels control pore glass metals cross linked dextrans e.g. Sephadex agarose gel Sepharose and other solid phase supports known to those of skill in the art. The beads are generally about 2 to about 100 m in diameter or about 5 to about 80 pm in diameter in some cases about 10 to about 40 m in diameter. In some embodiments the beads can be magnetic paramagnetic or otherwise responsive to a magnetic field. Having beads responsive to a magnetic field can be useful for isolation and purification of the beads having polynucleotides attached thereto such as by the application of a magnetic field and isolation of the beads e.g. by removal of the beads from solution or removal of solution from the beads . Non limiting examples of beads responsive to a magnetic field include Dynabeads manufactured by Life Technologies Carlsbad Calif. . Other methods to separate beads can also be used. For example the capture beads may be labeled with a fluorescent moiety which would make the nucleic acid bead complex fluorescent. The target capture bead complex may be separated for example by flow cytometry or fluorescence cell sorter. Beads may also be separated by centrifugation. Isolation of polynucleotides by attachment to beads may further comprise the step of washing the beads such as in a suitable wash buffer. Generally purification of primer extension products comprises purification away from one or more components of the primer extension reaction such that the one or more components from which the extension products are purified are reduced in amount such as by 10 fold 5 fold 100 fold 500 fold 1000 fold 10000 fold 100000 fold or more or below detectable levels. In some embodiments purification includes a denaturation step such that primer extension products are purified away from the target polynucleotide templates to which they were hybridized.

Extended primers may be subjected to amplification such as linear or exponential amplification. Methods for amplification are known in art examples of which are described herein such as with respect to other aspects of the invention. Exponential amplification includes PCR amplification and any other amplification methods where primer extension products serve as templates for further rounds of primer extension. Amplification typically utilizes one or more amplification primers examples of which are described herein such as with regard to other aspects of the invention. Amplification primers may be of any suitable length such as about less than about or more than about 5 10 15 20 25 30 35 40 45 50 55 60 65 70 75 80 90 100 or more nucleotides any portion or all of which may be complementary to the corresponding target sequence to which the primer hybridizes e.g. about less than about or more than about 5 10 15 20 25 30 35 40 45 50 or more nucleotides . In general PCR involves the steps of denaturation of the target to be amplified if double stranded hybridization of one or more primers to the target and extension of the primers by a DNA polymerase with the steps repeated or cycled in order to amplify the target sequence. Steps in this process can be optimized for various outcomes such as to enhance yield decrease the formation of spurious products and or increase or decrease specificity of primer annealing. Methods of optimization are well known in the art and include adjustments to the type or amount of elements in the amplification reaction and or to the conditions of a given step in the process such as temperature at a particular step duration of a particular step and or number of cycles. In some embodiments an amplification reaction comprises at least 5 10 15 20 25 30 35 50 or more cycles. In some embodiments an amplification reaction comprises no more than 5 10 15 20 25 35 50 or more cycles. Cycles can contain any number of steps such as 1 2 3 4 5 6 7 8 9 10 or more steps. Steps can comprise any temperature or gradient of temperatures suitable for achieving the purpose of the given step including but not limited to strand denaturation primer annealing and primer extension. Steps can be of any duration including but not limited to about less than about or more than about 1 5 10 15 20 25 30 35 40 45 50 55 60 70 80 90 100 120 180 240 300 360 420 480 540 600 or more seconds including indefinitely until manually interrupted. Cycles of any number comprising different steps can be combined in any order.

In some embodiments amplification comprises generating primer extension products using a pair of amplification primers. Amplification primers may comprise sequences complementary to complete or one or more portions of sequences derived from adapter oligonucleotide sequences sequences derived from oligonucleotide primer sequences sequences that are not complementary to template polynucleotides e.g. 5 non complementary sequences one or more other sequence elements e.g. sequence elements as described herein or combinations of these. In some embodiments a second amplification primer comprises sequence X and sequence Y where sequence Y is positioned at the 3 end of the second amplification primer.

In some embodiments the oligonucleotide primer and or one or more amplification primers comprise a barcode. Examples of barcodes are described herein such as with regard to other aspects of the invention. In some embodiments separate amplification reactions are carried out for separate samples using amplification primers comprising at least one different barcode sequence for each sample such that no barcode sequence is joined to the target polynucleotides of more than one sample to be analyzed in parallel. In some embodiments amplified polynucleotides derived from different samples and comprising different barcodes are pooled before proceeding with subsequent manipulation of the polynucleotides such as before sequencing . Pools may comprise polynucleotides derived from about less than about or more than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 20 25 30 40 50 75 100 or more different samples. Pools may subsequently be subjected to sequencing and the source samples of sequenced target polynucleotides may be identified based on their associated barcodes.

In some embodiments exponentially amplified target polynucleotides are sequenced. Sequencing may be performed according to any method of sequencing known in the art including sequencing processes described herein such as with reference to other aspects of the invention. Sequence analysis using template dependent synthesis can include a number of different processes. For example in the ubiquitously practiced four color Sanger sequencing methods a population of template molecules is used to create a population of complementary fragment sequences. Primer extension is carried out in the presence of the four naturally occurring nucleotides and with a sub population of dye labeled terminator nucleotides e.g. dideoxyribonucleotides where each type of terminator ddATP ddGTP ddTTP ddCTP includes a different detectable label. As a result a nested set of fragments is created where the fragments terminate at each nucleotide in the sequence beyond the primer and are labeled in a manner that permits identification of the terminating nucleotide. The nested fragment population is then subjected to size based separation e.g. using capillary electrophoresis and the labels associated with each different sized fragment is identified to identify the terminating nucleotide. As a result the sequence of labels moving past a detector in the separation system provides a direct readout of the sequence information of the synthesized fragments and by complementarity the underlying template See e.g. U.S. Pat. No. 5 171 534 .

Other examples of template dependent sequencing methods include sequence by synthesis processes where individual nucleotides are identified iteratively as they are added to the growing primer extension product.

Pyrosequencing is an example of a sequence by synthesis process that identifies the incorporation of a nucleotide by assaying the resulting synthesis mixture for the presence of by products of the sequencing reaction namely pyrophosphate. In particular a primer template polymerase complex is contacted with a single type of nucleotide. If that nucleotide is incorporated the polymerization reaction cleaves the nucleoside triphosphate between the and phosphates of the triphosphate chain releasing pyrophosphate. The presence of released pyrophosphate is then identified using a chemiluminescent enzyme reporter system that converts the pyrophosphate with AMP into ATP then measures ATP using a luciferase enzyme to produce measurable light signals. Where light is detected the base is incorporated where no light is detected the base is not incorporated. Following appropriate washing steps the various bases are cyclically contacted with the complex to sequentially identify subsequent bases in the template sequence. See e.g. U.S. Pat. No. 6 210 891.

In related processes the primer template polymerase complex is immobilized upon a substrate and the complex is contacted with labeled nucleotides. The immobilization of the complex may be through the primer sequence the template sequence and or the polymerase enzyme and may be covalent or noncovalent. For example immobilization of the complex can be via a linkage between the polymerase or the primer and the substrate surface. In alternate configurations the nucleotides are provided with and without removable terminator groups. Upon incorporation the label is coupled with the complex and is thus detectable. In the case of terminator bearing nucleotides all four different nucleotides bearing individually identifiable labels are contacted with the complex. Incorporation of the labeled nucleotide arrests extension by virtue of the presence of the terminator and adds the label to the complex allowing identification of the incorporated nucleotide. The label and terminator are then removed from the incorporated nucleotide and following appropriate washing steps the process is repeated. In the case of non terminated nucleotides a single type of labeled nucleotide is added to the complex to determine whether it will be incorporated as with pyrosequencing. Following removal of the label group on the nucleotide and appropriate washing steps the various different nucleotides are cycled through the reaction mixture in the same process. See e.g. U.S. Pat. No. 6 833 246 incorporated herein by reference in its entirety for all purposes. For example the Illumina Genome Analyzer System is based on technology described in WO 98 44151 hereby incorporated by reference wherein DNA molecules are bound to a sequencing platform flow cell via an anchor probe binding site otherwise referred to as a flow cell binding site and amplified in situ on a glass slide. A solid surface on which DNA molecules are amplified typically comprise a plurality of first and second bound oligonucleotides the first complementary to a sequence near or at one end of a target polynucleotide and the second complementary to a sequence near or at the other end of a target polynucleotide. This arrangement permits bridge amplification such as described herein. The DNA molecules are then annealed to a sequencing primer and sequenced in parallel base by base using a reversible terminator approach. Hybridization of a sequencing primer may be preceded by cleavage of one strand of a double stranded bridge polynucleotide at a cleavage site in one of the bound oligonucleotides anchoring the bridge thus leaving one single strand not bound to the solid substrate that may be removed by denaturing and the other strand bound and available for hybridization to a sequencing primer. Typically the Illumina Genome Analyzer System utilizes flow cells with 8 channels generating sequencing reads of 18 to 36 bases in length generating 1.3 Gbp of high quality data per run see www.illumina.com .

In yet a further sequence by synthesis process the incorporation of differently labeled nucleotides is observed in real time as template dependent synthesis is carried out. In particular an individual immobilized primer template polymerase complex is observed as fluorescently labeled nucleotides are incorporated permitting real time identification of each added base as it is added. In this process label groups are attached to a portion of the nucleotide that is cleaved during incorporation. For example by attaching the label group to a portion of the phosphate chain removed during incorporation i.e. a or other terminal phosphate group on a nucleoside polyphosphate the label is not incorporated into the nascent strand and instead natural DNA is produced. Observation of individual molecules typically involves the optical confinement of the complex within a very small illumination volume. By optically confining the complex one creates a monitored region in which randomly diffusing nucleotides are present for a very short period of time while incorporated nucleotides are retained within the observation volume for longer as they are being incorporated. This results in a characteristic signal associated with the incorporation event which is also characterized by a signal profile that is characteristic of the base being added. In related aspects interacting label components such as fluorescent resonant energy transfer FRET dye pairs are provided upon the polymerase or other portion of the complex and the incorporating nucleotide such that the incorporation event puts the labeling components in interactive proximity and a characteristic signal results that is again also characteristic of the base being incorporated See e.g. U.S. Pat. Nos. 6 056 661 6 917 726 7 033 764 7 052 847 7 056 676 7 170 050 7 361 466 and 7 416 844 and US 20070134128 .

In some embodiments the nucleic acids in the sample can be sequenced by ligation. This method uses a DNA ligase enzyme to identify the target sequence for example as used in the polony method and in the SOLiD technology Applied Biosystems now Invitrogen . In general a pool of all possible oligonucleotides of a fixed length is provided labeled according to the sequenced position. Oligonucleotides are annealed and ligated the preferential ligation by DNA ligase for matching sequences results in a signal corresponding to the complementary sequence at that position.

In some embodiments sequencing data are generated for a plurality of samples in parallel such as about less than about or more than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 24 48 96 192 384 768 1000 or more samples. In some embodiments sequencing data are generated for a plurality of samples in a single reaction container e.g. a channel in a flow cell such as about less than about or more than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 24 48 96 192 384 768 1000 or more samples and sequencing data are subsequently grouped according to the sample from which the sequenced polynucleotides originated e.g. based on a barcode sequence .

In some embodiments sequencing data are generated for about less than about or more than about 5 10 25 50 100 150 200 250 300 400 500 750 1000 2500 5000 7500 10000 20000 50000 or more different target polynucleotides from a sample in a single reaction container e.g. a channel in a flow cell . In some embodiments sequencing data are generated for a plurality of samples in parallel such as about less than about or more than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 24 48 96 192 384 768 1000 or more samples. In some embodiments sequencing data are generated for a plurality of samples in a single reaction container e.g. a channel in a flow cell such as about less than about or more than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 24 48 96 192 384 768 1000 or more samples and sequencing data are subsequently grouped according to the sample from which the sequenced polynucleotides originated. In a single reaction sequencing data may be generated for about or at least about 10 10 10 2 10 3 10 4 10 5 10 10 10 or more target polynucleotides or clusters from a bridge amplification reaction which may comprise sequencing data for about less than about or more than about 10 10 10 2 10 3 10 4 10 5 10 10 10target polynucleotides or clusters for each sample in the reaction. In some embodiments the presence or absence of about less than about or more than about 5 10 25 50 75 100 125 150 175 200 300 400 500 750 1000 2500 5000 7500 10000 20000 50000 or more causal genetic variants is determined for a sample based on the sequencing data. The presence or absence of one or more causal genetic variants may be determined with an accuracy of about or more than about 80 85 90 95 97.5 99 99.5 99.9 or higher.

In some embodiments one or more or all of the steps in a method of the invention are automated such as by use of one or more automated devices. In general automated devices are devices that are able to operate without human direction an automated system can perform a function during a period of time after a human has finished taking any action to promote the function e.g. by entering instructions into a computer after which the automated device performs one or more steps without further human operation. Software and programs including code that implements embodiments of the present invention may be stored on some type of data storage media such as a CD ROM DVD ROM tape flash drive or diskette or other appropriate computer readable medium. Various embodiments of the present invention can also be implemented exclusively in hardware or in a combination of software and hardware. For example in one embodiment rather than a conventional personal computer a Programmable Logic Controller PLC is used. As known to those skilled in the art PLCs are frequently used in a variety of process control applications where the expense of a general purpose computer is unnecessary. PLCs may be configured in a known manner to execute one or a variety of control programs and are capable of receiving inputs from a user or another device and or providing outputs to a user or another device in a manner similar to that of a personal computer. Accordingly although embodiments of the present invention are described in terms of a general purpose computer it should be appreciated that the use of a general purpose computer is exemplary only as other configurations may be used.

In some embodiments automation may comprise the use of one or more liquid handlers and associated software. Several commercially available liquid handling systems can be utilized to run the automation of these processes see for example liquid handlers from Perkin Elmer Beckman Coulter Caliper Life Sciences Tecan Eppendorf Apricot Design Velocity 11 as examples . In some embodiments automated steps include one or more of fragmentation end repair A tailing addition of adenine overhang adapter joining PCR amplification sample quantification e.g. amount and or purity of DNA and sequencing. In some embodiments bridge amplification is automated e.g. by use of an Illumina cBot . In some embodiments sequencing is automated. A variety of automated sequencing machines are commercially available and include sequencers manufactured by Life Technologies SOLiD platform and pH based detection Roche 454 platform Illumina e.g. flow cell based systems such as Genome Analyzer devices . Transfer between 2 3 4 5 or more automated devices e.g. between one or more of a liquid handler bridge a amplification device and a sequencing device may be manual or automated. In some embodiments one or more steps in a method of the invention e.g. all steps or all automated steps are completed in about or less than about 72 48 24 20 18 16 14 12 10 9 8 7 6 5 4 3 2 1 or fewer hours. In some embodiments the time from sample receipt DNA extraction fragmentation adapter joining amplification or bridge amplification to production of sequencing data is about or less than about 72 48 24 20 18 16 14 12 10 9 8 7 6 5 4 3 2 1 or fewer hours.

In one aspect the invention provides a method of detecting genetic variation in a subject s genome. In some embodiments the method comprises generating and analyzing sequencing data. In one embodiment the method comprises a providing a plurality of clusters of polynucleotides wherein i each cluster comprises multiple copies of a nucleic acid duplex attached to a support ii each duplex in a cluster comprises a first molecule comprising sequences A B G D C from 5 to 3 and a second molecule comprising sequences C D G B A from 5 to 3 iii sequence A is complementary to sequence A sequence B is complementary to sequence B sequence C is complementary to sequence C sequence D is complementary to sequence D and sequence G is complementary to sequence G iv sequence G is a portion of a target polynucleotide sequence from a subject and is different for each of a plurality of clusters and v sequence B is located 5 with respect to sequence G in the corresponding target polynucleotide sequence b sequencing sequence G by extension of a first primer comprising sequence D to produce an R1 sequence for each cluster c sequencing sequence B by extension of a second primer comprising sequence A to produce R2 sequence for each cluster d performing a first alignment using a first algorithm to align all R1 sequences to one or more first reference sequences e performing a second alignment using a second algorithm to locally align R1 sequences identified in said first alignment as likely to contain an insertion or deletion with respect to said one or more first reference sequences to produce a single consensus alignment for each insertion or deletion f performing an R2 alignment by aligning all R2 sequences to one or more second reference sequences and g transmitting a report identifying sequence variation identified by steps d to f to a receiver. In some embodiments sequence A B C and D correspond to sequence A B C and D respectively as described with regard to other aspects of the invention.

In some embodiments the method comprises a providing sequencing data for a plurality of clusters of polynucleotides wherein i each cluster comprised multiple copies of a nucleic acid duplex attached to a support ii each duplex in a cluster comprised a first molecule comprising sequences A B G D C from 5 to 3 and a second molecule comprising sequences C D G B A from 5 to 3 iii sequence A is complementary to sequence A sequence B is complementary to sequence B sequence C is complementary to sequence C sequence D is complementary to sequence D and sequence G is complementary to sequence G iv sequence G is a portion of a target polynucleotide sequence from a subject and is different for each of a plurality of clusters v sequence B is located 5 with respect to sequence Gin the corresponding target polynucleotide sequence viii the sequencing data comprises R1 sequences generated by extension of a first primer comprising sequence D and vi the sequencing data comprises R2 sequences generated by extension of a second primer comprising sequence A b performing a first alignment using a first algorithm to align all R1 sequences to one or more first reference sequences c performing a second alignment using a second algorithm to locally align R1 sequences identified in said first alignment as likely to contain an insertion or deletion with respect to said one or more first reference sequences to produce a single consensus alignment for each insertion or deletion d performing an R2 alignment by aligning all R2 sequences to one or more second reference sequences and e transmitting a report identifying sequence variation identified by steps b to d to a receiver. In some embodiments sequence A B C and D correspond to sequence A B C and D respectively as described with regard to other aspects of the invention.

In general a cluster of polynucleotides comprises multiple copies of a nucleic acid duplex that co localize to a position on a support. A variety of suitable solid supports and support materials are known in the art non limiting examples of which are provided herein such as with regard to other aspects of the invention. Clusters of polynucleotides may be produced by bridge amplification. Suitable methods and apparatuses for performing bridge amplification are provided herein such as with regard to other aspects of the invention. In some embodiments a solid support comprises a plurality of clusters with each cluster in the plurality formed by amplification of a different target polynucleotide sequence. The portion of a target polynucleotide sequence to be amplified such as a sequence G may be bound to a support in a process comprising extension of a first oligonucleotide immobilized on the support. In some embodiments the solid support comprises a plurality of different first oligonucleotides comprising sequence A and sequence B wherein sequence A is common among all first oligonucleotides and further wherein sequence B is different for each different first oligonucleotide and is at the 3 end of each first oligonucleotide. In some embodiments the plurality of first oligonucleotides comprises about less than about or more than about 5 10 25 50 75 100 125 150 175 200 300 400 500 750 1000 2500 5000 7500 10000 20000 50000 or more different oligonucleotides each comprising a different sequence B. In some embodiments sequence B of one or more of the plurality of first oligonucleotides comprises a sequence selected from the group consisting of SEQ ID NOs 22 121 shown in e.g. 1 5 10 25 50 75 or 100 different oligonucleotides each with a different sequence from . In some embodiments sequence B or the target sequence to which it specifically hybridizes comprises a causal genetic variant as described herein. In some embodiments sequence B or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of a causal genetic variant as described herein. Causal genetic variants are typically located downstream of a first oligonucleotide such that at least a portion of the causal genetic variant serves as template for extension of a first oligonucleotide. The solid support may further comprise a plurality of second oligonucleotides comprising sequence A at the 3 end of each second oligonucleotide and a plurality of third oligonucleotides comprising sequence C at the 3 end of each third oligonucleotide as described herein. An example of bridge amplification of a portion of a target polynucleotide sequence using bound first second and third oligonucleotides to produce clusters of duplexes is illustrated in with sequence G represented by the black line between sequences B and D and sequence G represented by the black line between sequence B and D.

In some embodiments sequence B of one or more of the plurality of first oligonucleotides or the target sequence to which it specifically hybridizes comprises a non subject sequence. In some embodiments sequence B or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of a non subject sequence. In general a non subject sequence corresponds to a polynucleotide derived from an organism other than the individual being tested or from whom a sample is taken such as DNA or RNA from bacteria archaea viruses protists fungi or other organism. Non subject sequence can also include nucleic acids from a fetus such as cell free nucleic acid also referred to as extracellular nucleic acid from a fetus. A non subject sequence may be indicative of the identity of an organism or class of organisms and may further be indicative of a disease state such as infection. An example of non subject sequences useful in identifying an organism include without limitation rRNA sequences such as 16s rRNA sequences see e.g. WO2010151842 . In some embodiments non subject sequences are analyzed instead of or separately from causal genetic variants. In some embodiments causal genetic variants and non subject sequences are analyzed in parallel such as in the same sample e.g. using a mixture of first oligonucleotides some with a sequence B that specifically hybridizes to a sequence comprising or near a causal genetic variant and some with a sequence B that specifically hybridizes to a sequence comprising or near a non subject sequence and or in the same report.

In some embodiments sequence B of one or more of the plurality of first oligonucleotides or the target sequence to which it specifically hybridizes comprises an ancestry informative marker AIM . In some embodiments sequence B or the target sequence to which it specifically hybridizes is within about less than about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 60 70 80 90 100 200 500 or more nucleotides of an AIM. An AIM may be used to classify a person as belonging to or not belonging to one or more populations such as a population that is at increased risk for one of the causal genetic variants. For example an AIM can be diagnostic for a population in which a trait is at increased prevalence. In certain instances the AIM may distinguish between populations with finer granularity for example between sub continental groups or related ethnic groups. In some embodiments AIMs are analyzed instead of or separately from causal genetic variants and or non subject sequences. In some embodiments AIMs causal genetic variants and or non subject sequences are analyzed in parallel such as in the same sample e.g. using a mixture of first oligonucleotides some with a sequence B that specifically hybridizes to a sequence comprising or near a causal genetic variant and some with a sequence B that specifically hybridizes to a sequence comprising or near an AIM and or in the same report.

In some embodiments one or more sequences of a plurality of clusters are sequenced. Example methods of sequencing are described herein such as with regard to other aspects of the invention. Sequencing data may be produced by extension of one or more sequencing primers for each cluster. A sequencing primer can be of any suitable length such as about less than about or more than about 5 10 15 20 25 30 35 40 45 50 55 60 65 70 75 80 90 100 or more nucleotides any portion or all of which may be complementary to the corresponding target sequence to which the primer hybridizes e.g. about less than about or more than about 5 10 15 20 25 30 35 40 45 50 or more nucleotides . A sequencing primer primer D may comprise or consist of sequence D such that it specifically hybridizes to complementary sequence D . In some embodiments the first nucleotide downstream of sequence D is the first nucleotide of sequence G such that the first nucleotide added in extension of primer D corresponds to sequence G. A sequencing primer primer C may comprise or consist of sequence C such that it specifically hybridizes to complementary sequence C . In some embodiments the first nucleotide downstream of sequence C is the first nucleotide of a barcode sequence such that the first nucleotide added in extension of primer C corresponds to a barcode sequence. A sequencing primer primer A may comprise or consist of sequence A such that it specifically hybridizes to complementary sequence A . In some embodiments the first nucleotide downstream of sequence A is the first nucleotide of sequence B such that the first nucleotide added in extension of primer A corresponds to sequence B. In some embodiments a sequencing primer comprises the sequence CACTCAGCAGCACGACGATCACAGATGTGTATAAGAGACAG SEQ ID NO 20 .

Two or more different sequencing primers may be used in successive sequencing reactions to produce multiple sequencing reads for each cluster. For example successive sequencing reactions may be performed for each of primers A C and D in any order e.g. primer D then primer C then primer A . A sequencing reaction may be preceded by one or more of strand cleavage strand denaturation or a wash step to remove one or more components of a previous reaction e.g. a sequencing primer . A sequencing reaction may comprise multiple cycles of individual nucleotide primer extension with each addition followed by an identification step to determine the identity of the added base. The number of cycles of individual nucleotide extension may be about or more than about 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 30 40 50 60 70 80 90 100 150 200 250 500 or more such as for each of a plurality of sequencing primers used in successive sequencing reactions or collectively for all sequencing primers used in successive sequencing reactions. In some embodiments the number of cycles of individual nucleotide extension is selected based on the length of the sequence to be identified such as a barcode or probe sequence and may be less than about 30 25 20 15 10 9 8 7 6 5 or fewer cycles. The number of cycles for each of a plurality of sequencing primers used in successive sequencing reactions may be different. For example 59 cycles of extension of primer D may be followed by 6 cycles of extension of primer C which may then be followed by 15 cycles of extension of primer A for 80 total cycles of extension.

Extension of a first sequencing primer second sequencing primer and a third primer that is an indexing primer may produce an R1 an R2 and a barcode sequence respectively for each cluster. In general multiple sequences are identified as originating from a single cluster based on physical co localization of successive extension reactions such as a position on an array of clusters. In some embodiments sequencing data e.g. R1 and or R2 sequences are generated for about less than about or more than about 5 10 25 50 100 150 200 250 300 400 500 750 1000 2500 5000 7500 10000 20000 50000 or more different target polynucleotides from a sample in a single reaction container e.g. a channel in a flow cell such as by extension of one or more sequencing primers. In some embodiments sequencing data are generated for a plurality of samples in parallel such as about less than about or more than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 24 48 96 192 384 768 1000 or more samples. In some embodiments sequencing data are generated for a plurality of samples in a single reaction container e.g. a channel in a flow cell such as about less than about or more than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 24 48 96 192 384 768 1000 or more samples and sequencing data are subsequently grouped according to the sample from which the sequenced polynucleotides originated e.g. based on a barcode sequence . Grouping of sequencing data based on barcode sequences may be performed before or after performing one or more alignments such as described herein and optionally before removing one or more sequences from analysis. In general once sequencing reads are grouped based on a barcode each group of reads is further processed independently of other groups. In some embodiments each barcode differs from every other barcode in a plurality of different barcodes analyzed in parallel. Typically a barcode sequence is associated with a single sample in a pool of samples sequenced in a single reaction. In some embodiments each of a plurality of barcode sequences is uniquely associated with a single sample in a pool of samples sequenced simultaneously. In some embodiments a barcode sequence is located 5 from sequence D .

In a single reaction sequencing data e.g. R1 and or R2 sequence may be generated for about or at least about 10 10 10 2 10 3 10 4 10 5 10 10 10 or more target polynucleotides or clusters from a bridge amplification reaction which may comprise sequencing data for about less than about or more than about 10 10 10 2 10 3 10 4 10 5 10 10 10target polynucleotides or clusters for each sample in the reaction. A sequencing system may output sequencing data in any of a variety of output data file types or formats including but not limited to .fasta .csfasta seq.txt qseq.txt .fastq .sff prb.txt .sms srs and or .qv. In some embodiments the presence or absence of about less than about or more than about 5 10 25 50 75 100 125 150 175 200 300 400 500 750 1000 2500 5000 7500 10000 20000 50000 or more causal genetic variants is determined for a sample based on the sequencing data. The presence absence or allele ratio of one or more causal genetic variants may be determined with an accuracy of about or more than about 80 85 90 95 97.5 99 99.5 99.9 or higher. In some embodiments the presence absence or quantity of one or more non subject sequence and or one or more AIM is determined with an accuracy of about or more than about 80 85 90 95 97.5 99 99.5 99.9 or higher.

In some embodiments sequences identified in one or more sequencing reactions for a plurality of clusters are aligned to a reference sequence. In general alignment involves placing one sequence along another sequence iteratively introducing gaps along each sequence scoring how well the two sequences match and preferably repeating for various positions along the reference. The best scoring match is deemed to be the alignment and represents an inference about the degree of relationship between the sequences. In some embodiments a reference sequence to which sequencing reads are compared is a reference genome such as the genome of a member of the same species as the subject. A reference genome may be complete or incomplete. In some embodiments a reference genome consists only of regions containing target polynucleotides. In some embodiments a reference sequence comprises or consists of a human genome. In some embodiments a reference sequence comprises or consists of sequences of polynucleotides of one or more organisms other than the individual being tested or from whom a sample is taken such as sequences from one or more bacteria archaea viruses protists fungi or other organism. In some embodiments a reference sequence comprises or consists of a plurality of known sequences such as all probe sequences used to amplify target polynucleotide sequences e.g. every sequence B and or sequence B for every different target polynucleotide . Sequencing data generated from the extension of one primer e.g. R1 sequences from primer D may be aligned to the same or different reference sequence as sequencing data generated from the extension of another primer e.g. R2 sequences from primer A . Sequencing data generated from the extension of one primer may be aligned to a reference sequence two or more times with each alignment using a different alignment algorithm. R1 sequences may be aligned independently of R2 sequences. A first alignment of R1 and R2 sequences may use the same alignment algorithm.

In an alignment a base in the sequencing read alongside a non matching base in the reference indicates that a substitution mutation has occurred at that point. Similarly where one sequence includes a gap alongside a base in the other sequence an insertion or deletion mutation an indel is inferred to have occurred. When it is desired to specify that one sequence is being aligned to one other the alignment is sometimes called a pairwise alignment. Multiple sequence alignment generally refers to the alignment of two or more sequences including for example by a series of pairwise alignments. In some embodiments scoring an alignment involves setting values for the probabilities of substitutions and indels. When individual bases are aligned a match or mismatch contributes to the alignment score by a substitution probability which could be for example 1 for a match and 0.33 for a mismatch. An indel deducts from an alignment score by a gap penalty which could be for example 1. Gap penalties and substitution probabilities can be based on empirical knowledge or a priori assumptions about how sequences mutate. Their values affect the resulting alignment. Examples of algorithms for performing alignments include without limitation the Smith Waterman SW algorithm the Needleman Wunsch NW algorithm algorithms based on the Burrows Wheeler Transform BWT and hash function aligners such as Novoalign Novocraft Technologies available at www.novocraft.com ELAND Illumina San Diego Calif. SOAP available at soap.genomics.org.cn and Maq available at maq.sourceforge.net .

In some embodiments an alignment according to the invention is performed using a computer program. One exemplary alignment program which implements a BWT approach is Burrows Wheeler Aligner BWA available from the SourceForge web site maintained by Geeknet Fairfax Va. . BWT typically occupies 2 bits of memory per nucleotide making it possible to index nucleotide sequences as long as 4G base pairs with a typical desktop or laptop computer. The pre processing includes the construction of BWT i.e. indexing the reference and the supporting auxiliary data structures. BWA includes two different algorithms both based on BWT. Alignment by BWA can proceed using the algorithm bwa short designed for short queries up to about 200 bp with low error rate 

An alignment program that implements a version of the Smith Waterman algorithm is MUMmer available from the SourceForge web site maintained by Geeknet Fairfax Va. . MUMmer is a system for rapidly aligning entire genomes whether in complete or draft form Kurtz S. et al. Genome Biology 5 R12 2004 Delcher A. L. et al. Nucl. Acids Res. 27 11 1999 . For example MUMmer 3.0 can find all 20 basepair or longer exact matches between a pair of 5 megabase genomes in 13.7 seconds using 78 MB of memory on a 2.4 GHz Linux desktop computer. MUMmer can also align incomplete genomes it can easily handle the 100s or 1000s of contigs from a shotgun sequencing project and will align them to another set of contigs or a genome using the NUCmer program included with the system.

Other non limiting examples of alignment programs include BLAT from Kent Informatics Santa Cruz Calif. Kent W. J. Genome Research 4 656 664 2002 SOAP2 from Beijing Genomics Institute Beijing Conn. or BGI Americas Corporation Cambridge Mass. Bowtie Langmead et al. Genome Biology 10 R25 2009 Efficient Large Scale Alignment of Nucleotide Databases ELAND or the ELANDv2 component of the Consensus Assessment of Sequence and Variation CASAVA software Illumina San Diego Calif. RTG Investigator from Real Time Genomics Inc. San Francisco Calif. Novoalign from Novocraft Selangor Malaysia Exonerate European Bioinformatics Institute Hinxton UK Slater G. and Birney E. BMC Bioinformatics 6 31 2005 Clustal Omega from University College Dublin Dublin Ireland Sievers F. et al. Mol Syst Biol 7 article 539 2011 ClustalW or ClustalX from University College Dublin Dublin Ireland Larkin M. A. et al. Bioinformatics 23 2947 2948 2007 and FASTA European Bioinformatics Institute Hinxton UK Pearson W. R. et al. PNAS 85 8 2444 8 1988 Lipman D. J. Science 227 4693 1435 41 1985 .

In some embodiments any or all of the steps of the invention are automated. For example a Perl script or shell script can be written to invoke any of the various programs discussed above see e.g. Tisdall Mastering Perl for Bioinformatics O Reilly Associates Inc. Sebastopol Calif. 2003 Michael R. Mastering Unix Shell Scripting Wiley Publishing Inc. Indianapolis Ind. 2003 . Alternatively methods of the invention may be embodied wholly or partially in one or more dedicated programs for example each optionally written in a compiled language such as C then compiled and distributed as a binary. Methods of the invention may be implemented wholly or in part as modules within or by invoking functionality within existing sequence analysis platforms. In certain embodiments methods of the invention include a number of steps that are all invoked automatically responsive to a single starting queue e.g. one or a combination of triggering events sourced from human activity another computer program or a machine . Thus the invention provides methods in which any or the steps or any combination of the steps can occur automatically responsive to a queue. The output can be provided in the format of a computer file. In certain embodiments the output is a FASTA file VCF file text file or an XML file containing sequence data such as a sequence of the nucleic acid aligned to a sequence of the reference genome. In other embodiments the output contains coordinates or a string describing one or more mutations in the subject nucleic acid relative to the reference genome. Alignment strings known in the art include Simple UnGapped Alignment Report SUGAR Verbose Useful Labeled Gapped Alignment Report VULGAR and Compact Idiosyncratic Gapped Alignment Report CIGAR Ning Z. et al. Genome Research 11 10 1725 9 2001 . In some embodiments the output is a sequence alignment such as for example a sequence alignment map SAM or binary alignment map BAM file comprising a CIGAR string the SAM format is described e.g. in Li et al. The Sequence Alignment Map format and SAMtools Bioinformatics 2009 25 16 2078 9 . In some embodiments CIGAR displays or includes gapped alignments one per line. CIGAR is a compressed pairwise alignment format reported as a CIGAR string.

In some embodiments R1 sequence from a cluster comprises sequence G from a plurality of different target polynucleotides and R2 sequence from a cluster comprises sequence B where sequence B is a probe sequence used to generate a cluster of amplified duplexes. When each sequence B is selected to target a specific target polynucleotide its sequence and location within the reference sequences e.g. reference genome is generally known and R1 sequences from the same cluster may be expected to fall within an anticipated nucleotide distance. An anticipated nucleotide distance may be based on an average or median fragment length for samples comprising fragmented sample polynucleotides or an upper threshold distance representing an unlikely fragment length based on such median or average fragment length. Thus in some embodiments an R1 sequence that aligns to a position further away than the threshold distance from the R2 sequence from the same cluster may be erroneous and is discarded. In some embodiments the upper threshold distance along a reference sequence between aligned R1 and R2 sequences from the same cluster above which sequence reads for a cluster are discarded is about or more than about 1000 2500 5000 7500 10000 12500 15000 20000 or more base pairs. In some embodiments alignments of R1 sequences to non unique regions of a reference sequence e.g. a reference genome are discarded and the sequences re aligned to a smaller subset of unique sequences within the reference sequence.

Typically a base quality score is determined for each nucleotide in a sequencing result and relates to the probability that a particular base call is wrong. An example of a base quality score is a Phred quality score Q where Q 10 logP and where P represents the probability that the corresponding base call is incorrect. In some embodiments base quality scores are used to evaluate alignments of sequencing reads to a reference sequence such as by determining a mapping quality score for each of a plurality of alignments. Methods for calculating mapping quality scores are known in the art. For example alignments having a quality score below a threshold value may be discarded re aligned or replaced with an alternative alignment having a higher score. In some embodiments an alignment with a mapping quality score below a threshold value and having more than one optimal alignment is re aligned to a subset of sequences within the reference sequence such as only regions of a reference genome containing target polynucleotides. In some embodiments a threshold mapping quality score is about or less than about 100 75 50 25 20 10 5 4 3 2 1 or 0.

In some embodiments sequencing reads likely to be duplicative are removed following an initial alignment. When sequencing reads are mapped duplicative reads may be marked as duplicates by the alignment algorithm. For example a mark duplicates subroutine within the alignment algorithm examines all of the records in a file of aligned sequences e.g. a .BAM file and decides which reads are duplicates of other reads. Generally speaking there are two types of duplicates optical duplicates which are typically caused by defects in the primary analysis software and PCR duplicates which are caused by duplicative PCR reactions. However from a computational point of view optical duplicates and PCR duplicates are indistinguishable. One way to determine if two sequence reads are duplicates or not is to compare the base sequences two duplicate reads should have duplicate base sequences. However due to sequencing errors it may be the case that two duplicate reads are sequenced such that a sequencing error for one read will cause its base sequence to differ significantly from the other read. Therefore rather than compare base sequences to determine if two reads are duplicates their alignments can be compared instead. If two reads are duplicates then the entire set of alignments for both reads will generally be the same. In some embodiments duplicates are marked for removal and or discarded using one or more algorithms distinct from the alignment algorithm. In general when barcode sequences are used sequencing reads are only deleted when occurring within the same barcode sequence grouping.

In some embodiments a second alignment using a second algorithm is performed after a first alignment using a first algorithm. The second alignment may be with respect to the same reference sequence as the first alignment a different reference sequence from that used in the first alignment or without use of a reference sequence such as when all sequencing reads overlapping a particular region are aligned with one another . For example sequences identified in a first alignment as likely to contain an insertion and or deletion indel with respect to the first reference sequence may be locally aligned to produce a single consensus sequence for an insertion and or deletion contained in a target polynucleotide. A first alignment may align individual sequences to a reference sequence independently. In some cases a sequencing read with a true indel may be aligned with multiple mismatches rather than an indel when an alignment model with multiple mismatches scores higher than the indel containing alignment. Typically multiple sequences are aligned as overlapping a single nucleotide position such as in a tiled fashion . Overlapping regions containing more than a predicted amount of sequence variation for example more than two alleles for a unique locus in a genome of a human subject may indicate the likely presence of an indel. The location of some indels for a particular reference sequence may be known such that sequences overlapping the location of a known indel identifies the sequence as likely to contain an indel. The likelihood of containing an indel may be expressed numerically based on one or more such factors such as a likelihood of at least about 60 70 80 90 95 99 or more. In some embodiments all sequences overlapping a region of interest such as a causal genetic variant and optionally also containing or likely to contain an indel are locally aligned using a second algorithm in order to produce a single consensus sequence for the region of interest. A region of interest may be of any suitable size such as about less than about or more than about 5 10 15 20 25 50 100 250 500 or more nucleotides in length. A second alignment may be a local multiple sequence alignment of all sequencing reads overlapping one or more nucleotide positions. In some embodiments the second alignment identifies a single consensus sequence by optimizing the alignment of all sequencing reads at a position. In some embodiments the consensus sequence produced by the second alignment contains at least 1 2 3 4 5 6 7 8 9 10 15 20 or 25 fewer nucleotide mismatches with respect to the reference sequence than one or more of the sequences realigned to produce the consensus sequence. In some embodiments the algorithm used to perform the second alignment is capable of identifying an insertion and or a deletion of about or more than about 1 2 3 4 5 6 7 8 9 10 15 20 25 or more nucleotides relative to the reference sequence with an accuracy of about or more than about 80 85 90 95 97 99 or higher.

Typically the second algorithm is different from the first algorithm and the second algorithm may require more resources of the system e.g. a computer system running the algorithm to perform the same number of alignments. For example performing the first alignment with a system using the first algorithm may take less time to align all R1 reads than would be taken if the system used the second algorithm to perform the first alignment of all R1 reads. In some embodiments performing the first alignment with the first algorithm takes about or less than about 90 80 70 60 50 40 30 25 20 15 10 5 or less of the time that would be taken by the same system to perform the first alignment using the second algorithm. As a further example performing the first alignment with a system using the first algorithm may use less system memory to align all R1 reads than would be used if the system used the second algorithm to perform the first alignment of all R1 reads. In some embodiments performing the first alignment with the first algorithm uses about or less than about 90 80 70 60 50 40 30 25 20 15 10 5 or less memory than would be used by the same system to perform the first alignment using the second algorithm. The first algorithm may perform the first alignment using heuristics. In some embodiments the first algorithm is based on the Burrows Wheeler transform such as the Burrows Wheeler Aligner. In some embodiments the second algorithm is based on the Smith Waterman algorithm.

In some embodiments an amplified sequence derived from one or more target polynucleotides from a sample e.g. sequence G from at least 75 80 85 90 95 or 100 of all clusters is from 1 nucleotide in length to about less than about or more than about 10 25 50 100 250 500 1000 2000 5000 or more nucleotides in length. In general if an amplified sequence derived from a target polynucleotide for a cluster e.g. sequence G is shorter than the number of cycles of nucleotide extension performed in sequencing the amplified sequence also referred to as the read length then sequence data returned for that sequencing read will likely contain sequence of a first oligonucleotide used to initially capture the amplified sequence e.g. sequence B or B . When sequence B or B exists in the reference sequence e.g. reference genome the sequence may correctly align but any true mutation in the target polynucleotide may be masked or inferred with lower confidence. To avoid negative effects of first oligonucleotide sequence contained in an R1 sequence base calls likely to correspond to sequence B or B for a cluster may be deleted. Sequence B or B for a cluster may be identified in a separate sequencing reaction such as producing R2 sequence. R1 sequence may then be compared to R2 sequence for the same cluster to determine whether or not one or more nucleotides of R1 correspond to sequence B or B . If no R2 sequence or no R2 sequence comprising any sequence B is obtained for a cluster deleting first oligonucleotide sequence may comprise deleting a portion of an R1 sequence for a cluster when the portion of R1 sequence to be deleted is identical to at least a portion of any sequence B such as any B found in any cluster or corresponding to any sequence B used to amplify a target polynucleotide the portion includes either the 5 or 3 nucleotide of R1 and either i no R2 sequence was produced for the cluster or ii R2 sequence produced is not identical to any sequence B. In general nucleotide sequence comprising sequence B or B found in the interior of an R1 sequence that is not including the 5 or 3 end of the sequencing read indicates that the amplified sequence was captured using a more distant sequence B.

In some embodiments genetic variation detected by a method of the invention is used to calculate a plurality of probabilities. Each probability may be a probability of a subject or a subject s present or future offspring having or developing a disease or trait. In some embodiments each probability is based on the R1 sequences for the subject and one or more such probabilities may be included in a report of analysis results. In general calculation of a probability that the tested subject has or will develop a disease or trait is based on a level of risk associated with one or more tested causal genetic variants non subject sequences and or AIMs. For example if two causal genetic variants contribute to the risk of developing a disease in an additive fashion then the presence of both causal genetic variants in a subject would indicate that the risk of that disease in the subject is increased by the value resulting from adding the risks associated with each. In general calculation of a probability that an offspring of the subject will have a disease or trait is based on a level of risk associated with one or more tested causal genetic variants and or AIMs and the probability that an offspring will inherit the causal genetic variants and or AIMs. Risk calculations may be based on risk correlations maintained in one or more databases which databases may be updated based on external reports and or records of genotyping results and associated phenotypes of tested subjects. In some embodiments the calculations are performed by a computer in accordance with instructions contained in a computer readable medium. In some embodiments the statistical confidence of a probability that the subject or subject s offspring will have or develop a disease or trait is at least about 70 80 85 90 95 97.5 99 or higher. Confidence may be based on a number of factors such as confidence in sequencing accuracy number of associated genetic variants tested and confidence in the risk associate with each genetic variant. Example methods for calculating probabilities are described in US20100022406.

In some embodiments of any aspect of the invention a computer system is used to execute one or more steps of the described methods. illustrates a non limiting example of a computer system useful in the methods of the invention. In some embodiments the computer system is integrated into and is part of an analysis system like a liquid handler bridge amplification system e.g. an Illumina cBot and or a sequencing system e.g. an Illumina Genome Analyzer HiSeq or MiSeq system . In some embodiments the computer system is connected to or ported to an analysis system. In some embodiments the computer system is connected to an analysis system by a network connection. A computer system or digital device may be used to receive and store results analyze the results and or produce a report of the results and analysis. The computer system may be understood as a logical apparatus that can read instructions from media e.g. software and or network port e.g. from the internet which can optionally be connected to a server having fixed media. A computer system may comprise one or more of a CPU disk drives input devices such as keyboard and or mouse and a display e.g. a monitor . Data communication such as transmission of instructions or reports can be achieved through a communication medium to a server at a local or a remote location. The communication medium can include any means of transmitting and or receiving data. For example the communication medium can be a network connection a wireless connection or an internet connection. Such a connection can provide for communication over the World Wide Web. It is envisioned that data relating to the present invention can be transmitted over such networks or connections or any other suitable means for transmitting information including but not limited to mailing a physical report such as a print out for reception and or for review by a receiver. The receiver can be but is not limited to an individual a health care provider a health care manager or electronic system e.g. one or more computers and or one or more servers . In some embodiments a computer readable medium includes a medium suitable for transmission of a result of an analysis of a biological sample. The medium can include a result regarding analysis of an individual s genetic profile wherein such a result is derived using the methods described herein. The data and or results may be displayed at any time on a display such as a monitor and may also be stored or printed in the form of a genetic report.

Causal genetic variants associated with phenotypes may be obtained from scientific literature and sent to a computer system for comparison with sequence results for a sample from a subject. Genotypes of causal genetic variants and results from biological samples may be sent to stored and analyzed by a computer system or other digital device which produces a report of the results and analyses of genomic data. The results and analyses may be accessed online by a receiver such as a health care provider via an online portal or website. The results and analyses may be viewed online saved on a receiver s computer printed or be mailed to a receiver. The results may be used for personalized health management such as at the direction of a physician or other health professional. For example the subject may be referred to or contacted by a genetic counselor to receive genetic counseling.

The database may have one or more of a variety of optional components that for example provide more information about the sequencing results produced by methods of the invention. In some embodiments there is provided a computer readable medium encoded with computer executable software that includes instructions for a computer to execute functions associated with the identified causal genetic variants. Such computer system may include any combination of such codes or computer executable software depending upon the types of evaluations desired to be completed. The computer system may also have code for linking each of the sequences e.g. genotypes for causal genetic variants to at least one phenotype such as a condition for example a medical condition including but not limited to a risk for having or developing the phenotype. Each medical condition in turn can be linked to at least one recommendation by a medical specialist and code for generating a report comprising the recommendation. The system can also have code for generating a report. Different types of reports can be generated for example reports based on the level of detail a receiver may want or have paid for. For example a receiver may have ordered analysis for a single phenotype such as a condition and thus a report may comprise the results for that single phenotype such as a condition. Another receiver may have requested a genetic profile for a panel or an organ system or another individual may have requested a comprehensive genetic profile that includes analysis of all clinically relevant causal genetic variants. Reports may comprise one or more of subject information e.g. name date of birth ethnicity sample type date of sample collection and or date of sample receipt description of analysis method s results for all causal genetic variants tested results for all disease or traits tested results for diseases or traits having a positive score e.g. a risk above a threshold level such as about or more than about 1 50000 1 25000 1 10000 1 5000 1 2500 1 1000 1 500 1 100 1 50 1 10 or higher results for causal genetic variants associated with a disease or trait having a positive score results for two or more individuals such as individuals that are parents or planning to have children risk of having or developing a disease or trait risk of a present or future child having or developing a disease or trait risk of a fetus having or developing a disease or trait methods of risk calculation and recommendations for further action.

The report generated can be reviewed and further analyzed by a genetic counselor and or other medical professional such as a managing doctor or licensed physician or other third party. The genetic counselor or medical professional or both or other third party can meet with the individual to discuss the results analysis and the genetic report. Discussions can include information about the causal genetic variant s such as the causal genetic variant s that is or are tested presence absence and or genotype how the causal genetic variant s can be inherited or transmitted for example using the pedigree generated from a questionnaire the prevalence of the causal genetic variant s prevalence or incidence of associated phenotypes and information about associated phenotypes for example specific conditions or traits such as medically or clinically relevant conditions such as how the phenotype may affect the individual and preventative measures that may be taken. The genetic counselor or medical professional may incorporate other information such as other genetic information or information from questionnaires in their analysis and discussion with the individual. Information about the phenotype such as condition or trait can include recommendations such as follow up suggestions such as further genetic counseling predictive medicine recommendations or preventive medicine recommendations for the individual s personal physician or other healthcare provider. Screening information such as methods of breast cancer screening may be discussed for example if an individual was found to be at a higher risk of breast cancer. Other topics that may be discussed include lifestyle modifications and medications. For example lifestyle modifications may be suggested such as dietary changes and specific diet plans may be recommended or an exercise regimen may be suggested and specific exercise facilities or trainers may be referred to the individual. Common misconceptions may also be included allowing the individual to be aware of preventive measures or other interventions that may be thought of as being helpful or useful but that have been shown in published literature to either not be beneficial or to actually be harmful. Alternative therapies may also be included such as alternative medicines such as dietary supplements or alternative therapies such as acupuncture or yoga. Family planning options may also be included as well as monitoring options such as such as screening exams or laboratory tests that may detect or help monitor for the presence of a phenotype or the progression of a phenotype. Medications that may prevent limit the onset or delay the progression of a phenotype such as a disease to which the individual is predisposed or a medication with high efficacy and low side effects may be suggested for an individual or medications or classes of medications that an individual should avoid due to possibility of adverse reaction s . For example the medical professional may make an assessment of the individual s likely drug response including metabolism efficacy and or safety. The medical professional can also discuss therapeutic treatments such as prophylactic treatments and monitoring such as doctor visits and exams radiologic exams self exams or laboratory tests for potential need of treatment or effects of treatment based on information from the individual s genetic profile either alone or in combination with information about the individual s environmental factors such as lifestyle habits diagnosed medical conditions current medications and others . Additional resources may also be listed such as including information for the individual or the individual s physician or other healthcare professional to acquire additional information about the phenotype the causal genetic variant s or both such as links to websites that contain information on the phenotype such as an internal website from the company that produces the genetic report or external websites such as national organizations for the phenotype. Additional resources may also include reference to telephone numbers books or people that the individual may seek out to acquire more information about the phenotype the causal genetic variant s or both.

In one aspect the invention provides a method comprising offering a first and optionally a second service wherein a the first service comprises predicting the probability that an offspring of the couple will have each of a plurality of traits caused by causal genetic variants wherein the prediction is based on the respective genotypes of the two individuals in the couple and b the second service comprises predicting the probable phenotype of an offspring of the couple for a plurality of traits wherein the probability is determined based on the respective phenotypes and or the family history of the individuals in the couple. In one embodiment at least one prediction is further based on the respective genetically inferred ancestries of the individuals. In another embodiment the first service is offered as a service for a fee and the second service is offered as a free service.

In one aspect this invention provides a system comprising a computer readable medium configured to store family history information from each member of a couple b computer readable medium configured to store data comprising genetic information about each member of the couple c computer readable medium comprising computer code that when executed i predicts each individual s carrier status with respect to traits caused by alleles identified in the genetic information or ii predicts probable traits of offspring of the couple determinable by the family histories and or the genetic information and d a display that displays i carrier status of at least one member of the couple or ii probable traits of the offspring. In some embodiments the system further comprises e a webpage configured to accept an offer to purchase a DNA test kit. In some embodiments the display is electronic for example a webpage. In some embodiments the system further comprises e a display that displays referrals to a genetic counselor and or other medical professional for example medical geneticists or obstetrician gynecologist based on the genetic information.

The internet and the world wide web offer access to and distribution of information. In some embodiments a website can be particularly suited to efficiently providing various functionality for allowing customers to purchase genetic testing and receive the results of genetic testing. The system typically will include a server on which the website resides. Users use an interface connected to the server such as a computer monitor or a telephone screen to interact with the website by clicking or rolling over links that pop up information or direct the user to another webpage. Websites typically are interactive allowing the user to input information or a query and obtain a response on the interface.

In some embodiments of a system and business method a website can allow a customer to purchase manage and view the results of genetic testing as well as to learn more generally about the probability that potential offspring will develop a disease or trait. For example a customer can be a couple of prospective parents who seek to learn whether their offspring will be at risk for developing Mendelian disease. A customer can be presented with the offer to purchase genetic testing to determine one or more of i the carrier status of the customer ii the likelihood that the customer will develop one or more diseases or traits and iii the probability that an offspring of the customer will develop one or more diseases or traits based on causal genetic variants identified in the customer s DNA.

If the customer chooses to purchase genetic testing then the customer may pay a fee for example through an online credit card transaction in exchange for genetic testing direct phone consultation with a genetic counselor on the company s staff and or referrals to genetic counselors and or other relevant medical professionals. The genetic testing and referrals can be paid for by a fee at the point of purchase or can be included in an initial user registration fee. In some embodiments the services are free and revenue is generated by the company by advertising other products in conjunction with a particular product. For example after a customer places an order online the order is sent to a server for processing. Once payment has been verified the order processing server can send an electronic notification to a shipping vendor to mail a DNA collection kit to the customer. In an embodiment the DNA collection kit is separate from the genetic testing service or the user or customer already has or obtains the DNA collection kit from another source. Notifications can also periodically be sent electronically to the customer comprising order confirmation and updates on order and shipping status. In some embodiments of a business method of the invention a customer can deposit a sample into the collection kit. Any sample that would be obvious to one skilled in the art can be deposited into or onto a collection kit. A sample can be any material containing nucleic acid to be analyzed that would be obvious to one skilled in the art such as bodily fluid like saliva or blood. The collection kit can then be returned to the company for sending to a genotyping lab or can be returned directly to a genotyping lab for processing. A genotyping lab either internal within the company contracted to work with the company or external from the company can isolate the customer s DNA from the provided sample. After the DNA has been isolated from the sample a genotyping device such as an apparatus described herein can be used to test the DNA for the presence of one or more of i ancestry informative markers ii causal genetic variants and iii non subject sequences one or more of which are also referred to herein as Raw Genotypic Information . In some embodiments the DNA does not have to be isolated from the sample to test the DNA for the presence of Raw Genotypic Information.

Raw Genotypic Information can be sent electronically to a server for storage and processing. Computer code on the server can execute on the Raw Genotypic Information to infer the ancestry of the customer and or to confirm the presence of causal genetic variants and or non subject sequences if any. The processed genotypic information can then be electronically sent to a server where computer code on the server can execute on the processed genotypic information to predict the probability that an offspring of the customer will have each of a plurality of traits caused by causal genetic variants found to be present in the customer s processed genotypic information. Results can then be electronically transmitted to a server for storage.

In an example a notification can be sent to the customer to alert the customer to the availability of the results. The notification can be electronic non limiting examples of which include a text message an email or other data packet or the notification can be non electronic non limiting examples of which include a phone call from a genetic counselor or printed communication such as a report sent through the mail. The results provided to a customer can inform the customer of the carrier status of the customer for one or more diseases or traits and or the chances that the customer or customer s future offspring will develop one or more diseases or traits. After the customer has received results and referrals the customer s order can be considered fulfilled and results and referrals can remain accessible to the customer through an online website account. The customer can then choose to further pursue a referral offline if the customer so desires but outside of the purview of the website.

In one aspect the invention provides compositions that can be used in the above described methods. Compositions of the invention can comprise any one or more of the elements described herein. For example compositions may include one or more of the following one or more solid supports comprising oligonucleotides attached thereto one or more oligonucleotides for attachment to a solid support one or more adapter oligonucleotides one or more amplification primers one or more oligonucleotide primers comprising a first binding partner one or more solid surfaces e.g. beads comprising a second binding partner one or more sequencing primers reagents for utilizing any of these reaction mixtures comprising any of these and instructions for using any of these.

In one aspect the invention provides kits containing any one or more of the elements disclosed in the above methods and compositions. In some embodiments a kit comprises a composition of the invention in one or more containers. For example kits may include one or more of the following one or more solid supports comprising oligonucleotides attached thereto one or more oligonucleotides for attachment to a solid support one or more adapter oligonucleotides one or more amplification primers one or more oligonucleotide primers comprising a first binding partner one or more solid surfaces e.g. beads comprising a second binding partner one or more sequencing primers reagents for utilizing any of these and instructions for using any of these. In some embodiments the kit further comprises one or more of a a DNA ligase b a DNA dependent DNA polymerase c an RNA dependent DNA polymerase d random primers e primers comprising at least 4 thymidines at the 3 end f a DNA endonuclease g a DNA dependent DNA polymerase having 3 to 5 exonuclease activity h a plurality of primers each primer having one of a plurality of selected sequences i a DNA kinase j a DNA exonuclease k magnetic beads and 1 one or more buffers suitable for one or more of the elements contained in the kit. The adapters primers other oligonucleotides and reagents can be without limitation any of those described herein. Elements of the kit can further be provided without limitation in any amount and or combination such as in the same kit or same container . The kits may further comprise additional agents for use according to the methods of the invention. The kit elements can be provided in any suitable container including but not limited to test tubes vials flasks bottles ampules syringes or the like. The agents can be provided in a form that may be directly used in the methods of the invention or in a form that requires preparation prior to use such as in the reconstitution of lyophilized agents. Agents may be provided in aliquots for single use or as stocks from which multiple uses such as in a number of reaction may be obtained.

The following examples are given for the purpose of illustrating various embodiments of the invention and are not meant to limit the present invention in any fashion. The present examples along with the methods described herein are presently representative of preferred embodiments are exemplary and are not intended as limitations on the scope of the invention. Changes therein and other uses which are encompassed within the spirit of the invention as defined by the scope of the claims will occur to those skilled in the art.

Genomic DNA gDNA is extracted in 96 well format leaving wells A1 G12 and H12 empty which will later contain a no template control the universal negative standard containing Coriell sample NA12878 genomic DNA lacking every causal genetic variant tested and a sample comprising one of a plurality of known causal genetic variants respectively . 50 L from each well are transferred into a corresponding well of an absorbance plate. Absorbance at 260 nm is measured using a Tecan M200 plate reader to calculate DNA quantity. 50 L of gDNA are transferred from the absorbance plate into an Eppendorf twin.tec plate. Control samples are added to their respective position on the twin.tec plate. The gDNA and controls are fragmented in a SonicMan Matrical Spokane Wash. sonicator according to the following protocol at 10 C. Pre chill 180 s cycles 100 sonication 3.0 s power 35 lid chill 1.0 s plate chill 0 post chill 0. A 2 L sample is analyzed for fragmentation size distribution using a Fragment Analyzer Advanced Analytical Technologies Ames Iowa . Samples having a median fragment size of at least 200 base pairs and no more than 1000 bp are subjected to further processing. Samples with a median fragment size below 200 bp are discarded and reprocessed from extracted gDNA. Samples with a median fragment size above 1000 bp are either subjected to further sonication to reach the desired size range or are discarded and reprocessed from extracted gDNA.

Sonicated gDNA is transferred into a round bottomed sample plate for use in conjunction with the Beckman Biomek FXP. The Biomek automates the processes of end repair addition of adenine overhangs and adapter ligation. The Biomek system comprises an Agencourt SPRIPlate Super Magnet Plate a Biomek FXP Dual Arm System with Multichannel Pipettor and Span 8 Pipettor with pump control module computer and monitor peltier controller two waste containers and two water containers and BioMek FXP Control Software. This process utilizes the SPRIworks HT Fragmentation Library Kit which contains end repair buffer and enzyme a tailing buffer and enzyme ligation buffer and enzyme and Agencourt AMPure XP beads. After each reaction processed gDNA is cleaned using magnetic bead separation. Adapter ligation is followed by quantifying DNA in the processed sample using absorbance at 260 nm as measured by the Tecan M200. Samples with less than 900 ng are not processed further but are instead reprocessed from the original extracted sample. After the absorbance reading the sample plate is returned to the Biomek FXP for PCR amplification. The first step is division of each sample into four separate samples on a 384 well plate such that amplification for each sample source is performed in quadruplicate. Amplification primers comprise a barcode sequence to allow identification of the sample source of a sequence. PCR includes the use of an ABI GeneAmp PCR system 9700 with dual 384 well blocks 1.5 mL tube racks 24 channel 200 L multichannel pipettor and 96 well aluminum plate holder. Samples are automatically thermally cycled according to following protocol 95 C for 5 minutes 27 cycles of 98 C for 20 seconds 65 C for 15 seconds 72 C for 1 minute. When amplification is complete the four sub samples from each sample source are recombined into a single well of a 96 well plate.

Amplified polynucleotides are purified by magnetic bead separation. 1.8 sample volumes of magnetic beads are added to each sample which are allowed to sit at room temperature for about 5 minutes. The plate is placed on a magnetic separator for about 2 minutes until the slurry is completely clear and all beads have been collected on the side of each well. Buffer solution is then aspirated and 200 L of 70 ethanol are added. The ethanol is allowed to sit at room temperature for about 30 seconds before being aspirated. The plate is then removed from the magnet and DNA is eluted in about 40 L of elution buffer EB 10 mM Tris HCl pH 8.5 . The plate is returned to the magnet and allowed to sit at room temperature for about 2 minutes until the beads have collected on the sides of the well. The 40 L sample from each well is then transferred to a corresponding well of a new absorbance quantitation plate. DNA quantity in each well is checked by measuring absorbance at 260 nm as above. Samples having a concentration of at least 500 ng L are further processed for sequencing. Wells with lower concentrations are failed and the corresponding samples are re amplified.

Amplified samples are pooled across rows of the 96 well plate to produce pools of 12 samples where amplified polynucleotides of each sample comprise a barcode unique to that sample among the 12 samples in the pool. The volume of each sample added to the pool is calculated such that the total amount of DNA in the sample submitted for sequencing is approximately 11.25 g. Each pool is concentrated by cleanup on magnetic beads as above with elution in 38.5 L EB. 1 L of each pool is used to quantify total DNA on a NanoDrop machine Thermo Scientific Wilmington Del. . Samples below 10 g are failed and pooling and cleanup are repeated. Samples having at least 10 g are further processed for sequencing.

Before polynucleotides in each pool are attached bridge amplified and sequenced a cBot reagent plate is prepared. Reagent plates are prepared ten at a time using commercially supplied Phusion High Fidelity PCR Master Mix with HF Buffer New England Biolabs Detergent free Phusion HF Buffer Pack New England Biolabs 0.1N NaOH HT1 buffer 5 SSC 0.05 Tween 20 and HT2 buffer 0.3 SSC 0.05 Tween 20 . Five Nova Biostorage 8 tube strips are placed into positions 1 2 3 7 and 10 of ten separate Nova Biostorage RoBo Racks. 1.25 mL of Phusion master mix are added to a 15 mL tube followed by addition of 1.25 mL of RNase and DNase free water and vortexing for 10 seconds to generate 1 Phusion master mix. 440 L of 5 Phusion HF buffer are added to another 15 mL tube labeled HF followed by addition of 1760 L of RNase and DNase free water and mixed to generate 1 HF buffer. Reagents are dispensed into rows of the reagent plates as follows Row 1 720 L HT1 buffer Row 2 230 L Phusion master mix Row 3 200 L 1 HF buffer Row 7 300 L HT2 buffer and Row 10 215 L 0.1N NaOH. Each tube strip is then covered with Nova Biostage tube caps and all plates are frozen until needed.

Each sample pool is then prepared for sequencing by attachment to a flow cell. The system for attachment and bridge amplification comprises a cBot system a NanoDrop Absorbance Spectrometer Applied Biosystems Veriti 96 well Thermal Cycler 0.2 mL Veriti Thermocycler Program and cBot attachment and bridge amplification programs. Samples are heated to 95 C. for 5 minutes. 12.5 L of 4 Hybridization buffer 10 SSC 0.2 Tween 20 is added to each sample which are placed on ice until loaded on the Illumina cBot machine. A sipper comb flowcell reagent plate and sample tubes are then loaded on the cBot. For each sample pool polynucleotides are attached to a channel of the flow cell by extension of oligonucleotides attached to the surface of the channel target capture step of . The attached oligonucleotides comprise a collection of different oligonucleotides that specifically hybridize to members of a collection of about 5000 different interrogation positions located upstream of selected causal genetic variants. Clusters of bridge amplified sequences are then generated on the cBot using standard procedures.

Clusters are sequenced using a Genome Analyzer IIx GAIIx Illumina San Diego Calif. . The sequencing system comprises a Genome Analyzer IIx a Paired End Module Sequencing Control Software GAIIx programs sequencing pre wash prime post wash 500 mL capacity plastic beakers a large square ice bucket and a scale with 0.1 g tolerance. Sequencing is performed in two rounds. In a first round sequencing data is generated from a first primer that hybridizes downstream of 3 along the extended strand the barcode and adjacent to the target genomic DNA sequences thereby generating sequencing data for the target gDNA regions comprising causal genetics variants. In the second round sequencing data is generated from a second primer that hybridizes upstream of 5 along the extended strand the barcode sequence such that barcode sequence data is produced for each cluster. The order of these sequencing reactions could be reversed. Barcodes for each cluster are then matched to their corresponding gDNA sequence such that the sample source for each gDNA sequence can be identified. The raw data from the GAIIx is combined into individual reads each with quality scores using standard Illumina software. Reads are aligned to a reference genome using a Burrows Wheeler Aligner and variants are found from this alignment using the genome analysis toolkit GATK. The output file from the GATK listing all found discrepancies between the sequencing reads and the reference assembly is then used to generate a genotype report which is sent securely to the ordering physician for a consultation with the patient that provided the sample.

Example processes for the amplification of a plurality of different target polynucleotides are illustrated in which differ primarily in the inclusion of a solid phase purification step in . also illustrates an example amplification process and differs from the process illustrated in primarily in that oligonucleotide primer extension is performed before adapter joining instead of after adapter joining Amplification may or may not include a solid phase purification step. illustrates an amplification process as in and also example bridge amplification and sequencing processes. The amplification process illustrated in may be used in conjunction with any bridge amplification method and associated sequencing method.

First a partially single stranded adapter is ligated to fragmented polynucleotides. The partially single stranded adapter has a double stranded region at one end sequence U hybridized to complementary sequence U and the single stranded sequence Y that does not hybridize to the target polynucleotide under the hybridization and extension conditions used. Ligation adds sequence Y to both 5 ends of the target polynucleotides. Next a plurality of different oligonucleotide primers each having a different target specific sequence W at the 3 end are hybridized to their respective target polynucleotides and extended producing an extended oligonucleotide with sequence Y complement of Y at the 3 end. Extension may be performed before adapter ligation such as illustrated in . The oligonucleotide primers may lack a first binding partner as in or may comprise a first binding partner as the in the small overhanging circle in . If the extended oligonucleotides do comprise a binding partner they may be purified by selectively binding to a solid surface comprising a second binding partner that binds to the first binding partner as in the bead larger circle in . Bound and extended oligonucleotides may be purified such as by holding in place on a magnetically responsive bead in the presence of a magnetic field while reaction solution is removed beads washed and new reaction solution added e.g. components of a further amplification reaction . Extended oligonucleotides purified or not are then amplified with a pair of amplification primers. One amplification primer comprises sequence X and sequence Y with sequence Y at the 3 end for hybridization to sequence Y . The X Y primer is extended along the extended oligonucleotides to produce a plurality of extended X Y oligonucleotides comprising sequences X Y W and Z 5 to 3 where W is the complement of W and Z is the complement of Z . Another amplification primer comprises sequences V and Z with Z at the 3 end for hybridization to sequence Z of an extended X Y primer. The V Z primer is extended along the extended X Y primer to produce a plurality of sequences comprising V Z Y and X 5 to 3 where X is the complement of X which may then serve as a template for extension of a further X Y primer which may then serve as a template for extension of a further V Z primer and so on for each successive primer extension reaction in the amplification process. The predominant amplified sequences comprise a plurality of different target polynucleotides each contained in a polynucleotide comprising one strand comprising sequences V Z W Y and X from 5 to 3 and another strand comprising sequences X Y W Z and V from 5 to 3 with target polynucleotide sequence located between Z Y and between Z Y. These amplified polynucleotides may then be subjected to sequencing.

Sequencing may follow the process illustrated in the lower half of . A first bound oligonucleotide is hybridized to a sequence near or at the 3 end of an amplified polynucleotide typically by complementarity to a sequence added during the exponential amplification step thereby specifically amplifying and ultimately sequencing exponentially amplified products . Extension of each first bound oligonucleotide provides nucleation points for bridge amplification to produce clusters of double stranded bridge polynucleotides with the same sequence. Extension products of first bound oligonucleotides are denatured to remove the hybridized templates. An extended first bound oligonucleotide then hybridize to a second bound oligonucleotide typically by complementary to a sequence at or near the 3 end and derived from sequence added during the exponential amplification step. Extended second bound oligonucleotides may then serve as templates for extension of further first oligonucleotides which may then serve as templates for extension of further second oligonucleotides and so on. Here some or all first oligonucleotides comprise a cleavage site which is cleaved after completing the bridge amplification process. Bound polynucleotides are then subjected to denaturing conditions such as heating e.g. about 95 C. or chemically denatured to remove one strand of a plurality of bound bridge polynucleotides. The remaining bound strands are then free for hybridization with a sequencing primer illustrated above first read in . Sequencing data is then generated by sequential steps of nucleotide extension and detection extending the sequencing primer. The extended first sequencing primer may then be denatured and removed from the template in order to repeat the sequencing process from a second sequencing primer that is different from the first. Where one sequencing primer is used only to generate enough sequencing data to identify a barcode sequence that sequencing reaction may be significantly shorter than the other sequencing reaction e.g. less than about 3 4 5 6 7 8 9 10 11 12 13 14 15 or more cycles of nucleotide addition . While only illustrates bridge amplification and sequencing of a single target polynucleotide bridge amplification and sequencing typically involves a plurality of different target polynucleotides amplified in a previous amplification step all of which are bridge amplified and sequenced in parallel.

Polynucleotides e.g. DNA and or RNA are extracted from a sample from a subject suspected to contain viral and or bacterial polynucleotides using standard methods known in the art. Sample polynucleotides are fragmented end repaired and A tailed such as in Example 1. Adapter oligonucleotides comprising sequence D are then joined to the sample polynucleotides which are then amplified using amplification primers comprising sequence C sequence D and a barcode. Amplified target polynucleotides are hybridized to a plurality of different first oligonucleotides that are attached to a solid surface. Each first oligonucleotide comprises sequence A and sequence B where sequence B is different for each different first oligonucleotide is at the 3 end of each first oligonucleotide and is complementary to a sequence comprising a non subject sequence or a sequence within 200 nucleotides of a non subject sequence. Specifically the first oligonucleotides are selected to amplify sequences having high depth outside the subject s genome such as viral or bacterial sequences unique to a particular class order family genus species or other taxonomic group of virus or bacteria. Sequences amplified may include 16s rRNA sequences. Polynucleotides from a healthy control are processed simultaneously. Target polynucleotides are then bridge amplified and sequenced according to methods of the invention. Sequencing data produced for the non subject sequences may be used to identify an infectious agent. Sequencing data produced for the non subject sequences may be used to detect relative levels of different taxonomic groups of bacteria e.g. ratios of one or more taxonomic groups to one or more other taxonomic groups or shifts in these. The identities or relative levels of bacteria or infectious agent are then used as the basis for making a medical recommendation or taking medical action.

This example sequence manipulation and alignment procedure pipeline begins with raw data from Genome Analyzer IIx GAIIx or HiSeq sequencers Illumina San Diego Calif. to infer genotypes and compute metrics from patient samples. Sequencing data is generated from runs of barcoded samples in a 12 multiplexed configuration per Flowcell lane according to a method of the invention. The sequencer raw data includes basecalls BCL files and various quality control and calibration metrics. The raw basecalls and metrics are first compiled into QSEQ files and then filtered merged and demultiplexed based on barcode sequences into sample specific FASTQ files. FASTQ reads are aligned to the HG19 genome to create an initial BAM file. This BAM file undergoes several transformations to filter clip and refine alignments and to recalibrate quality metrics. The final BAM file is used to infer genotypes for known variants and to discover novel ones producing a callset. The callset VCF files is then filtered using various call metrics to create a final set of high confidence such as about or more than about 80 85 90 95 99 or higher confidence variant calls per sample. Finally various metrics are computed per sample lane and batch and the calls and metrics are loaded into a laboratory information management system HMS for visualization review and final report generation. The pipeline can be run in whole or in part locally and or using cloud computing such as on the Amazon cloud. Users may interact with the pipeline using any suitable communication mechanism. For example interaction may be via Django management commands Django Software Foundation Lawrence Kans. a shell script for executing each step of the pipeline or an application programming interface written in a suitable programming language e.g. PHP Ruby on Rails Django or an interface like Amazon EC2 . Overviews of the operation of this example pipeline are illustrated in .

Sequencing occurs on a flowcell with 8 lanes. Each lane has 12 or more with HiSeq samples each with a unique 6 7 nucleotide barcode sequence. Each lane is subdivided into some number of tiles 120 for GAIIx 48 for HiSeq . The sequencer outputs 3 reads per flowcell cluster. Read1 R1 is the sequence of one edge of a gDNA fragment 59 bp generated by extension of a first primer. Read2 is the barcode sequence 6 bp generated by extension of a third primer. Read3 R2 is part of the probe sequence 15 bp generated by extension of a second primer.

The raw sequencing data processed in the first step of the pipeline creating FASTQ files is typically large such as about or more than about 100 GB 150 GB 200 GB 250 GB 300 GB 400 GB 500 GB 1000 GB or more . Accordingly it can be advantageous to utilize cloud computing for some or all of the analysis steps. In this example the first step is run locally and the resulting FASTQ files are uploaded to Amazon S3 an online storage web service provided by Amazon Seattle Wash. and processed using Amazon EC2 instances a cloud computing web service provided by Amazon Seattle Wash. . Amazon s Simple Queue Service SQS is used to assign tasks. The final calls and metrics are then downloaded and loaded into a local database. The EC2 instances pull tasks and FAQSTQ files from SQS and S3 respectively process them and upload the results to S3. Instances may be initiated and or terminated manually or may be partially or fully automated.

The FASTQ2BAM process aligns reads to the genome and further processes the alignment. Format changing sorting and indexing are performed as needed. All steps are run on files for individual samples and all samples represented in a batch may be run in parallel on different machines. The steps in FASTQ2BAM include align bwa fix align mark duplicates realign bam recalibrate bam and clip alignment. In the align bwa step reads in a FASTQ file are aligned to the reference genome using the BWA aligner. This step is called twice once to align read1 to a reference genome and once to align read3 to a collection of the probe sequences used to amplify target polynucleotides. The output sequence alignment map SAM file is converted to a binary alignment map BAM file and then sorted and indexed. In order to improve detection of indels the default BWA parameters are modified as follows decrease seed length to 16 increase number of allowed gaps in alignment to 3 decrease the gap open and extension penalties to 6 and 2 respectively increase the number of allowed gap extensions to 20 for larger variants custom contigs may be added to the genome to infer their presence . The fix alignment step then modifies some of the alignments to improve their accuracy and remove alignments likely to be erroneous. Because read3 includes probe derived sequence at variable distances from read1 in this scenario it does not fit some of the statistical assumptions made by aligners such as expected distance between reads and customary paired end mapping would be less efficient. To improve alignment accuracy read1 and read3 are aligned independently which is generally faster than paired end mapping and then the fix alignment step processes the results to discard any reads where read1 and read3 are on different strands or positioned more than 10000 base pairs away from each other read1 not filtered if read3 does not map and remap reads with multiple best scoring positions to a subset of the genome consisting only of regions of interest ROI e.g. regions containing a causal genetic variant non subject sequence or AIM typically near a probe sequence . illustrate an example alignment of reads in a CFTR exon with a non unique region before and after fix align respectively which closes an artificial gap in the illustrated sequence pileup.

The mark duplicates step identifies and tags PCR and optical duplicates using a tool from the Picard toolset java based command line utilities for processing sequencing data in BAM format available from the SourceForge web site maintained by Geeknet Fairfax Va. . Without discarding duplicates non uniform PCR efficiency between the reference and alternate allele could lead to allelic bias where the counts of reference and alternate alleles for a variant may be biased by PCR. Without additional information it is assumed that two reads from a sample that map to the same position are duplicates and thus all except one are marked as duplicates and excluded from subsequent analysis. Depth of coverage may be increased by using primers with different barcodes in the same sample such that an additional read having the same sequence as another would not be discarded if the associated barcodes from the respective clusters were different.

The realign bam step performs multiple sequence Smith Waterman alignment around indels and typically has the effects of better identifying true indels and reducing or eliminating the number of false positive SNPs. The initial aligner BWA or any similar tool aligns each read independently and heuristically. A read with a true INDEL may align as a read with a cluster of SNPs because that alignment model scores better than one with the INDEL given the set of parameters and heuristics used. In multiple sequence alignment of the same reads the aligner tries to optimize the score of an alignment model of all reads to the reference and to each other thus unless the same cluster of SNPs can align all reads the true alignment will typically score higher. The realignment step performs a multiple sequence exact realignment around any INDELs found in the ROI. show the same reads before and after local realignment. A realignment may be performed around any indel in an ROI. Alternatively or additionally a realignment may be performed around known indels such as around indels in one or more reference sets of indels such as sets reported in Mills et al. Genome Res. 2011 June 21 6 830 839 Durbin et al. Nature 2010 October 28 467 7319 1061 1073 and Bhangale et al. Nature Genetics 2006 38 1457 1462 .

The recalibrate bam step recalibrates base qualities using empirical batch data. Illumina software estimates a quality value for each base using various quality control metrics using a simple model of the sequencing chemistry it does not take specific error modes into account. In this step a GATK tool that uses high scoring alignments is used to determine empirical base quality analyze the covariation in empirical quality between many features of a sequenced base reported quality surrounding bases read position etc and recalibrate all qualities using the covariation model. This step provides more accurate base qualities which leads to more accurate calling statistics.

The clip alignment step removes bases from aligned reads that include probe sequence from the corresponding cluster. For amplified sequences from a subject that are shorter than the read length read1 will typically contain sequences from the corresponding probe. When probe sequences are derived from the reference genome these reads will align to the genome but will mask out any true SNPs thus introducing an allelic bias towards the reference sequence. This step identifies when a read overlaps its own probe and selectively removes the overlapping bases from the read. For all reads where both read1 and read3 map and where read1 overlaps the probe the overlapping bases of read1 are removed from the alignment clipped . If read3 of a read doesn t map then read1 is clipped if it overlaps any probe but only if the overlap occurs at either end of the read probe sequence that does not include a terminal base indicates the read was not generated from that probe . In either case clipping is performed by modifying the CIGAR alignment string to include the S operation for the clipped bases updating the start position and setting the base qualities of the clipped bases to 0. Thus the sequence still exists but the alignment is modified to exclude the clipped bases.

The BAM2VCF process uses the final alignments BAM files from the FASTQ2BAM process to determine genotypes using a Bayesian method to compute probability of variants given sequencing data and prior knowledge. All steps are run on files for individual samples and can be run in parallel and on different machines. The strategy for variant identification calling is to create an initial set of identified variants a callset using very lenient thresholds to maximize sensitivity and then to filter it based on call metrics and other criteria. The steps in the BAM2VCF process include genome whitelist genotype novel and hard filter vcf steps. The genome whitelist step infers genotypes for an input BAM file based on a comparison to a given a reference list of known variant positions and alleles at those positions. In this step a computer algorithm programmed to identify variants a caller is configured to output all variants and to skip any confidence based filtering. The output of this step is a variant call format VCF file which is further processed in additional steps.

The genotype novel step identifies variant sites within the ROI that differ from the reference genome and infers the genotypes at those sites. In this step the caller is configured to output only genotypes not included in the reference list of known variants used in the previous step and to skip any confidence based filtering. Calls from the genotype novel step may contain many false positives. The hard filter vcf step filters genotype determinations using several call metrics. These metrics fall into two broad categories 1 those that quantify the confidence of the base calls mapping variant or genotype determination and 2 those that quantify the likelihood of common sources of errors such as strand bias position bias or presence of sequence features such as homopolymer runs that are known to cause Illumina chemistry errors. Modified thresholds may be based on recommendations by the Genome Analysis Tool Kit GATK . Alternatively or additionally a machine learning approach may be used to identify thresholds for a desired sensitivity and specificity.

To aid the evaluation of the processes in this example a record for each sample is made of the number of reads with corresponding barcode reads mapped to the genome reads after fix align step reads after excluding PCR optical duplicates reads where read1 and read3 map more than 10000 bp apart reads in non unique regions that the fix align step attempts to remap reads successfully remapped reads that are clipped and have a corresponding read3 reads that are clipped and do not have a corresponding read3 reads in X and Y chromosomes which may be used to infer sex and SNP calls that match the SNP identity in the reference genome. Thresholds as to any one or combination of these metrics may be set such that results for any sample falling below the threshold are discarded. Any one or combination of these metrics may also be aggregated for an entire sequencing lane in addition to the total number of reads per lane and the number of reads passing an initial filter. Thresholds as to any one or combination of sequencing lane metrics may also be set for the exclusion of data resulting from lanes failing to pass the threshold s . Concordance and discordance between any two callsets may be analyzed for validation studies or for setting thresholds for future sample analyses.

For any genomic variants identified a pileup image may be generated which illustrates an alignment of all reads underlying any variant call. A pileup image may be produced using a genomics data visualizer such as the Integrative Genomics Viewer IGV provided by the Broad Institute Cambridge Mass. . To do this an IGV script is generated that 1 loads the genome and BAM files and 2 iterates through each variant position and outputs a snapshot PNG of the pileups. IGV is run under a virtual framebuffer e.g. xvfb and the resulting PNG files are cropped using command line Imagick tool to remove IGV chrome.

An algorithm is used in a process of selecting optimal probe sequences for initial capture of target sequence for amplification and sequencing a process also referred to as probe design . The probe sequences may then be used in the production of a collection of oligonucleotide primers or first oligonucleotides bound to a solid support. The probe design process may be repeated such as to incorporate additions to the list of variants and corresponding target sequences to be sequenced. Accordingly the algorithm allows the addition of previously designed regions of interest ROIs and probes so that regions that are already covered by a previously designed ROI are not redesigned.

The initial unit of probe design is the Region of Interest ROI which can be a list of the exons of a gene a single genomic base regions or points that are non coding or combinations of these that can possibly overlap. The first step in the process is to load and then reconcile all of the different regions for which probes are to be designed. The design engine class keeps track of all ROIs to be considered and later all of the probes that have been designed for each ROI. Small ROIs such as variants initially entered as point mutations are padded to a length of 100 bp before being processed further. Then all overlapping ROIs are combined into single ROIs so that duplicate probes are not designed.

Two ROIs are merged if and only if they reduce the number of ROI Tiles covering the combined ROIs. The number of tiles covering the two separate areas is calculated along with the number of tiles to cover a hypothetically joined ROI. The case that requires the fewest tiles is used for subsequent steps of probe design. An algorithm is used to determine the number of tiles covering a given genomic region.

Once ROIs have been padded and merged all ROIs are at least 100 bp long and none overlap. The resulting ROIs are long e.g. longer than a specified tile length or short e.g. less than or equal to a specified tile length . Long ROIs are subdivided into ROI tiles which are unit lengths of sequence for which a probe will be designed. Short ROIs being less than or equal to the tile length are not subdivided. Each potential ROI tile is evaluated on how well the probes designed from it perform. The maximum number of tiles is also calculated as the upper bound on this calculation. All tile numbers between the minimum and the maximum number of tiles possible are considered in order from least number of tiles to greatest. These numbers of tiles are equal to the ceiling of the number of bases in the ROI to be split divided by the min or max length of the ROI depending on what number is being calculated. These numbers are ROI TILE 250 bases and MAX ROI TILE where MAX ROI TILE TILE SIZE READ LENGTH RECESS . TILE SIZE is between 300 440 bases long. READ LENGTH is 40 60 bases long. RECESS is set at 10 bases in length. Once a number of tiles whose probe design yields all valid probes is found the iteration ends. This in effect minimizes the number of tiles required to cover a region while at the same time ensuring the best probes are chosen according to the criteria below.

The probe design algorithm works on a given ROI tile in isolation from other ROI tiles so the ROI tile can be considered the fundamental unit of this probe design process. Each ROI tile will have a forward and reverse tile designed for it so that all bases may be evidenced from either strand upon sequencing. The probe design algorithm works by considering the forward and reverse primers for every READ LENGTH tile in a probe design window that is calculated for each ROI tile. Each probe in this window is then scored based on criteria described below to create a set of scores from most important to least important where for all scores a lower score is better. Therefore the best probe is merely the one that appears first in a multiple field ascending sort of the probe score sets. Each ROI tile partition causes the probe design algorithm to be run for each of the possible ROI tiles. The iteration starts with the condition of the fewest ROI tiles and if such a partition doesn t yield valid probes conditions of which are described below the number of ROI tiles is increased and the partition is re done.

The probe window is defined as follows 1 the length of the probe window is defined as TILE SIZE length of the current ROI tile RECESS 2 the start coordinate is then defined as RECESS by away from the end of the ROI itself and the stop coordinate is calculated by adding the length of the probe window above to the RECESS coordinate and 3 all 40 mers in this range are then considered as probes for evaluation. provides an illustration of the positional relationship of sequence regions considered in this step.

Criteria used to evaluate each probe in the order considered include uniqueness of the near 24 mer overlap with any common SNPs in the near 24 mer mappability of the entire 40 mer NtBspQI overlap repeat masking overlap with any common SNPs in entire 40 mer near 24 mer overlap with disease variant 40 mer overlap with disease variant GC and Distance to ROI. While an ideal probe is unique across the genome sometimes finding such a location near an arbitrary site is not possible. To compensate the near 24 mer defined as the 3 most 24 bases of the oligonucleotide comprising the probe sequence or its complement is selected to be as unique as possible. Because the extension of the captured genomic species occurs from 5 to 3 the quality of the basepair binding site nearest the double stranded to single stranded junction has a large effect on the efficiency of the capture a stronger bond making it more likely that the captured sequence will be extended. To measure binding quality the University of California Santa Cruz 24 mer mappability track available through the UCSC genome browser is used which gives for each base in the genome the mapping score of the 24 mer beginning at that base. The score is given as 1 N where N is the number of matches to that 24 mer in the genome. Only two results out of this test are considered whether the score equals 1 i.e. is unique or is less than one i.e. has multiple binding sites . The first case is preferred.

Overlap with common SNPs in the near 24 mer is not desired. Any mismatch in the capture probe binding site reduces binding efficiency. Because nearby SNPs are often in linkage disequilibrium this difference in binding efficiency would introduce a great deal of allelic bias. It is therefore desirable for any allele found to have the greatest chance possible of having the same probe binding site as other alleles in that gene. The UCSC Common SNP Track is used to make this calculation. There are two categories those with no overlaps and those with one or more overlaps. The former is greatly preferred.

The mappability of the entire 40 mer determines the same score with the same categories as the 24 mer mappability but using the UCSC 40 mer Mappability track instead of the 24 mer track. This new track has a similar definition only 40 mer mappability is considered instead of 24 mer mappability.

The enzyme NtBspQI may be useful in oligonucleotide synthesis or manipulation. Accordingly the number of bases of overlap between the probe and the recognition site of the enzyme is scored.

In evaluating repeat masking the UCSC repeat mask track annotated repeats is used to calculate the weighted average of the values for the bases that makeup the each 40 mer. The repeat mask track assigns values 0 or 1 to each base depending on whether it is masked or not. Thus the higher the score the more it is masked. Designing probes for masked bases is not desired so a lower ideally 0 score is better. These scores are divided into quartiles so up to 25 masked comprises a class scored as 0 up to 50 another as does 75 and 100 .

Just as overlap of a probe near 24 mer with any common SNP is evaluated so too is overlap of the entire 40 mer probe sequence with any common SNP evaluated and scored.

Whether or not the near 24 mer overlaps a disease variant is also evaluated. This test is similar to the common SNP overlap test except the near 24 mer is evaluate for overlap with any causal genetic variants to be sequenced. A score of 0 is given for no overlap and a score of 1 is given for the presence of overlap. A similar analysis is then performed for the entire 40 mer.

The GC of a probe sequence received one of two scores 0 for a GC between 20 80 and 1 for outside this range. Finally distance to the ROI is evaluated. All other things being equal the probe closest to the ROI is preferred. The score for ROI distance is equal to the number of base pairs between the end of the probe and the start of the ROI it targets.

A valid probe is a probe that meets all of the following criteria and for which no further iteration of probe design for the ROI is required 1 mappability of the near 24 mer 1 3.5 2 Mappability of the entire 40 mer 1 3.5 3 repeat fraction 0.25 4 no overlaps with common SNPs in the near 24 mer no NtBspQI recognition site in the probe sequence.

Exemplary processes of delivering a probability that a user is a carrier of rare genetic disease are demonstrated in . illustrate pipelines for order fulfillment for web and medical customers respectively. An order can be placed by a physician or a consumer. An order can be placed for a single test or for a couple or family. The order can be accepted through a web site. The ordering system can accept contact information demographic details and billing information. Contact information can include without limitation name address telephone number and email addresses. Demographic information can include without limitation sex date of birth and self reported ethnicity. An order confirmation notification can be sent using the provided contact information. Acceptable orders are added to a database and the states of these orders can be subsequently maintained by a state machine.

A sample collection kit is then sent to the user. A sample is collected that is any human tissue or fluid. The sample can also be isolated DNA from a human. Examples of samples useful for this example include but are not limited to saliva blood urine buccal cells amniotic fluid cell scrapings and cell culture. The sample is then genotyped using a device described herein. Phenotype solicitation for example retrieving self identification of phenotypic traits of a user can be performed in parallel with sample processing.

Sample collection can be performed at home in a physician s office or at a specialized collection site. Sample collection and return can be tracked by advancing the state of the order tracking state machine. Samples received by the accessioning facility can be registered in the database system by advancing their state in the state machine. After acceptance at the accessioning facility samples can be delivered to the genotyping facility. The genotyping facility can return raw genome data to a secure data storage server by secure file transfer protocols. File upload can trigger an advance of the state machine. This advance can trigger a server configured to perform genotype calling to retrieve the raw genome data from the data storage server as well as any phenotype data associated with the order. The genotyping algorithm can produce a fully probabilistic genotype call.

Quality control metrics can be generated from the calling process. An example quality control metric is the percentage of probabilistic genotype calls in which at least one genotype has a posterior probability greater than a threshold value. A batch of samples is processed together. When processed as a batch the individual probabilistic genotype calls can be used to generate batch level quality control statistics. Probabilistic genotype calls can be stored in a database. Successful genotype calling can trigger an advance of the order state. For a couple or family order the state machine can hold for completion of the entire order else single orders can be passed to the next state. If phenotype data is required for risk calculation then the state machine can delay processing until all phenotype data is collected. The state machine can also trigger a notification to the patient that phenotype data is required. If all genotype and phenotype data are ready then the state machine can advance trigger the risk calculation server to perform risk calculation. The results of risk calculation can be serialized and transferred to a results reporting system. This is a machine readable format of the results. The state machine can advance the order when the transfer is completed. The results reporting server can combine the probabilistic risk calculation with appropriate text and formatting to generate a human readable report. This human readable report can be further formatted for display on a website. This human readable report can be formatted for other media such as PDF files for printing. The final results reports can be automatically released using an auto verification system. A human can review the reports for release. The reviewers may be a clinical laboratory scientist and a physician. The results are accessed via a web portal which links to a view of the results and a summary of the quality control metrics. Acceptance of the report by the clinical laboratory scientist releases the results to the physician. The physician can review the results in a similar portal and approve the final release of the results.

While preferred embodiments of the present invention have been shown and described herein it will be obvious to those skilled in the art that such embodiments are provided by way of example only. Numerous variations changes and substitutions will now occur to those skilled in the art without departing from the invention. It should be understood that various alternatives to the embodiments of the invention described herein may be employed in practicing the invention. It is intended that the following claims define the scope of the invention and that methods and structures within the scope of these claims and their equivalents be covered thereby.

