---

title: Transcription factors
abstract: The invention provides polynucleotide sequences isolated from plants encoding transcription factors. Polypeptides encoded by the polynucleotides are also provided. Products and methods of use are disclosed.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08962947&OS=08962947&RS=08962947
owner: Arborgen Inc.
number: 08962947
owner_city: Ridgeville
owner_country: US
publication_date: 20120206
---
This application is a divisional of U.S. application Ser. No. 12 187 745 filed Aug. 7 2008 which is a divisional of U.S. application Ser. No. 10 863 905 filed Jun. 7 2004 which claims priority to U.S. Provisional Application No. 60 476 189 filed Jun. 6 2003.

The instant application contains a lengthy Sequence Listing which has been submitted via CD R in lieu of a printed paper copy and is hereby incorporated by reference in its entirety. Said CD R recorded on Oct. 30 2004 are labeled Copy 1 and Copy 2 respectively and each contains only one identical 6.71 MB file 44463295.APP .

The present invention relates to polynucleotide sequences isolated from plants that encode transcription factors together with polypeptides encoded by such polynucleotides. In particular this invention relates to polynucleotide and polypeptide sequences isolated from and and the use of such polynucleotide and polypeptide sequences for regulating gene transcription and gene expression.

During transcription a single stranded RNA complementary to the DNA sequence to be transcribed is formed by the action of RNA polymerases. Initiation of transcription in eucaryotic cells is regulated by complex interactions between cis acting DNA motifs and trans acting protein factors. Among the cis acting regulatory regions are sequences of DNA termed promoters. A promoter is located close to the transcription initiation site and comprises a nucleotide sequence that associates with an RNA polymerase either directly or indirectly. Promoters usually consist of proximal e.g. TATA box and more distant elements e.g. CCAAT box . Enhancers are cis acting DNA motifs which may be situated 5 prime and or 3 prime from the initiation site.

Both promoters and enhancers are generally composed of several discrete often redundant elements each of which may be recognized by one or more trans acting regulatory proteins known as transcription factors. Regulation of the complex patterns of gene expression observed both spatially and temporally in all developing organisms is thought to arise from the interaction of enhancer and promoter bound general and tissue preferred transcription factors with DNA Izawa T Foster R and Chua N H 1993230 1131 1144 Menkens A E Schindler U and Cashmore A R 199513 506 510 . Developmental decisions in organisms as diverse as and are regulated by transcription factors. These DNA binding regulatory molecules have been shown to control the expression of genes responsible for the differentiation of different cell types for example the differentiation of leaf trichomes and xylem tissue in Kirik V Schnittger A Radchuk V Adler K Hulskamp M and Baumlein H 2001235 2 366 77 Baima S Possenti M Matteucci A Wisman E Altamura M M Ruberti I and Morelli G. 2001 126 2 643 55 formation of endoderm from embryonic cells in and the initiation of gene expression in response to environmental and phytohormonal stress in plants Yanagisawa S and Sheen J 199810 75 89 .

Transcription factors generally bind DNA in a sequence specific manner and either activate or repress transcription initiation. The specific mechanisms of these interactions remain to be fully elucidated. At least three types of separate domains have been identified within transcription factors. One is essential for sequence specific DNA recognition one for the activation repression of transcriptional initiation and one for the formation of protein protein interactions such as dimerization . Studies indicate that many plant transcription factors can be grouped into distinct classes based on their conserved DNA binding domains Katagiri F and Chua N H 1992 8 22 27 Menkens A E Schindler U and Cashmore A R 1995 13 506 510 Martin C and Paz Ares J 1997 13 67 73 . Each member of these families interacts and binds with distinct DNA sequence motifs that are often found in multiple gene promoters controlled by different regulatory signals.

Several transcription factor families have been identified in plants. For example nucleotide sequences encoding the following transcription factors families have been identified Alfin like AP2 APETALA2 and EREBPs ethylene responsive element binding proteins ARF AUX IAA bHLH bZIP C2C2 Zn C2C2 Co like C2C2 Dof C2C2 GATA C2C2 YABBY C2H2 Zn C3H type CCAAT CCAAT HAP3 CCAAT HAP5 CPP Zn DRAP1 E2F DP GARP GRAS HMG BOX HOMED BOX HSF Jumanji LFY LIM MADS Box SEQ ID NO 3668 MYB NAC Polycomb like RAV like SBP TCP TFIID Transfactor Trihelix TUBBY and WRKY SEQ ID NO 3670 .

Because transcription factors regulate transcription and orchestrate gene expression in plants and other organisms control of transcription factor gene expression provides a powerful means for altering plant phenotype. The multigenic control of plant phenotype presents difficulties in determining the genes responsible for phenotypic determination. One major obstacle to identifying genes and gene expression differences that contribute to phenotype in plants is the difficulty with which the expression of more than a handful of genes can be studied concurrently. Another difficulty in identifying and understanding gene expression and the interrelationship of the genes that contribute to plant phenotype is the high degree of sensitivity to environmental factors that plants demonstrate.

There have been recent advances using genome wide expression profiling. In particular the use of DNA microarrays has been useful to examine the expression of a large number of genes in a single experiment. Several studies of plant gene responses to developmental and environmental stimuli have been conducted using expression profiling. For example microarray analysis was employed to study gene expression during fruit ripening in strawberry Aharoni et al. 129 1019 1031 2002 wound response in Cheong et al. 129 661 7 2002 pathogen response in Schenk et al. 97 11655 60 2000 and auxin response in soybean Thibaud Nissen et al. 132 118. Whetten et al. 47 275 91 2001 discloses expression profiling of cell wall biosynthetic genes in L. using cDNA probes. Whetten et al. examined genes which were differentially expressed between differentiating juvenile and mature secondary xylem. Additionally to determine the effect of certain environmental stimuli on gene expression gene expression in compression wood was compared to normal wood. A total of 156 of the 2300 elements examined showed differential expression. Whetten supra at 285. Comparison of juvenile wood to mature wood showed 188 elements as differentially expressed. Id. at 286.

Although expression profiling and in particular DNA microarrays provide a convenient tool for genome wide expression analysis their use has been limited to organisms for which the complete genome sequence or a large cDNA collection is available. See Hertzberg et al. 98 14732 7 2001a Hertzberg et al. 25 585 2001b . For example Whetten supra states A more complete analysis of this interesting question awaits the completion of a larger set of both pine and poplar ESTs. Whetten et al. at 286. Furthermore microarrays comprising cDNA or EST probes may not be able to distinguish genes of the same family because of sequence similarities among the genes. That is cDNAs or ESTs when used as microarray probes may bind to more than one gene of the same family.

Methods of manipulating gene expression to yield a plant with a more desirable phenotype would be facilitated by a better understanding of transcription factor gene expression in various types of plant tissue at different stages of plant development and upon stimulation by different environmental cues. The ability to control plant architecture and agronomically important traits would be improved by a better understanding of how cell cycle gene expression effects formation of plant tissues how cell cycle gene expression causes plant cells to enter or exit cell division and how plant growth and transcription factor gene are connected. Among the large number of transcription factor genes the expression of which can change during development of a plant only a fraction are likely to effect phenotype.

Accordingly there exists a need for transcription factors that can be used for regulating gene expression in plants.

Accordingly there is a need for transcription factor genes and polynucleotides that can be used for regulating gene expression in plants. Additionally there is a need for tools and methods which can correlate changes in transcription factor gene expression to phenotype. There is also a need for polynucleotides useful in such methods. There is a further need for methods. There is a further need for methods of identifying transcription factor genes and gene products that impact plant phenotype and that can be manipulated to obtain a desired phenotype.

In one aspect the invention provides an isolated polynucleotide comprising a nucleic acid sequence that codes for a polypeptide that is capable of at least one of i binding to a nucleic acid molecule or ii regulating expression of a gene in a plant.

In one embodiment the polynucleotide is a transcription factor that functions in a plant cell. In another embodiment the isolated polynucleotide comprises a nucleic acid sequence selected from the group consisting of SEQ ID NO. 1 494 496 820 1641 1972 3588 3592.

In one embodiment the isolated polynucleotide is normally expressed in a species of or . In another embodiment the polynucleotide is normally expressed in . In another embodiment the polynucleotide is normally expressed in

In one embodiment the isolated polynucleotide is normally expressed in a species of conifer. In another embodiment the conifer is selected from the group consisting of Eastern white pine Western white Sugar pine Red pine Pitch pine Jack pine Longleaf pine Shortleaf pine Loblolly pine Slash pine Virginia pine Ponderosa pine Jeffrey pine and Lodgepole pine Radiata pine and hybrid crosses thereof. In a further embodiment the conifer is selected from the group consisting of Albospica Aurea Kashmir Shalimar Silver Mist White Imp ssp. ssp. Nana var. Pendula Crippsii Boulevard Filifera Aurea Blue Sport Sekkan Sugi Vilmoriniana Glauca var. Blue Ice Blue Sapphire Autumn Gold Torulosa Tollesons Remontii Sanders Blue var. Umbraculifera Vanderwolf Pyramid Pendula French Blue Mitsch Weeping Filiformis Golden Splendor Post Sentinal Caslewellan Naylors Blue and hybrid crosses thereof.

In one embodiment the conifer is a Southern Yellow pine tree. In a further embodiment the Southern Yellow pine is selected from the group consisting of and and hybrids.

In another embodiment the isolated polynucleotide is normally expressed in a tree selected from the group consisting of chestnut ash beech basswood birch black cherry black walnut butternut chinkapin cottonwood elm hackberry hickory holly locust magnolia maple oak poplar acacia aspen teak red alder royal paulownia sassafras sweetgum sycamore tupelo willow and yellow poplar and intra and inter species hybrid crosses thereof.

In another embodiment the polynucleotide is normally expressed in a gymnosperm or an angiosperm. In another embodiment the polynucleotide expresses a polypeptide that is capable of at least one of i binding to a nucleic acid molecule or ii regulating expression of a gene in a monocotyledenous plant.

In another embodiment the monocotyledenous plant is selected from the group consisting of turfgrass wheat maize rice oat barley orchid iris lily onion sugarcane and sorghum.

In another embodiment the turfgrass is selected from the group consisting of spp. spp. Kentucky Bluegrass And Perennial Ryegrass Mix and

In one embodiment the polynucleotide expresses a polypeptide that is is capable of at least one of i binding to a nucleic acid molecule or ii regulating expression of a gene in a dicotyledenous plant.

In another embodiment the dicotyledenous plant is selected from the group consisting of cotton tobacco tomato potato aspen Sweetgum acacia poplar willow teak mahogany chestnut elm sugar beet broccoli cassava sweet potato pepper poinsettia legumes alfalfa soybean carrot strawberry lettuce oak maple walnut rose mint squash daisy geranium and cactus.

In another embodiment the polypeptide is capable of upregulating or downregulating the expression of a gene in a plant.

In another embodiment the phenotype of a plant which expresses the isolated polynucleotide in at least one cell is different from the phenotype of a plant of the same species that does not express the isolated polynucleotide in any of its cells.

In another embodiment the phenotype of the plant expressing the isolated polynucleotide comprises a difference in lignin quality compared to a plant of the same species that does not express the isolated polynucleotide.

In another embodiment the difference in lignin quality is characterized by change in the structure of the lignin molecule.

In another embodiment the phenotype of the plant expressing the isolated polynucleotide comprises a difference in wood composition compared to a plant of the same species that does not express the isolated polynucleotide.

In another embodiment the phenotype of the plant expressing the isolated polynucleotide comprises a difference in fiber composition compared to a plant of the same species that does not express the isolated polynucleotide.

In another embodiment the phenotype of the plant expressing the isolated polynucleotide comprises a difference in plant cell division compared to a plant of the same species that does not express the isolated polynucleotide.

In another embodiment the phenotype of the plant expressing the isolated polynucleotide comprises a difference in plant cell development compared to a plant of the same species that does not express the isolated polynucleotide.

In another aspect the invention provides the isolated polynucleotide comprising the sequence of any one of SEQ ID NOs. 1 494 496 820 1641 1972 3588 3592 or variant thereof.

In one embodiment the variant encodes a polypeptide that is capable of at least one of i binding to a nucleic acid molecule or ii regulating expression of a gene in a plant.

In another aspect the invention provides a plant transcription factor comprising the amino acid sequence of any one of SEQ ID NOs. 821 1640 1973 2304 3593 3666 or variant thereof wherein said transcription factor is capable of at least one of i binding to a nucleic acid molecule or ii regulating expression of a gene in a plant.

In one embodiment the variant has a sequence identity that is greater than or equal to 99 98 97 96 95 94 93 92 91 90 89 88 87 86 85 84 83 82 81 80 79 78 77 76 75 74 73 72 71 70 69 68 67 66 65 64 63 62 61 or 60 in sequence to any one of SEQ ID NOs. 1 494 496 820 1641 1972 3588 3592.

In one embodiment the variant has a sequence identity that is greater than or equal to 99 98 97 96 95 94 93 92 91 90 89 88 87 86 85 84 83 82 81 80 79 78 77 76 75 74 73 72 71 70 69 68 67 66 65 64 63 62 61 or 60 in sequence to any one of SEQ ID NOs. 821 1640 1973 2304 3593 3666.

In another aspect the invention provides a DNA construct comprising i at least one polynucleotide that has the sequence of any one of SEQ ID NOs. 1 494 496 820 1641 1972 3588 3592 ii a promoter and iii a desired nucleic acid wherein said polynucleotide encodes a plant transcription factor that regulates the activity of said promoter and wherein said promoter and said desired gene are operably linked.

In another aspect the invention provides a DNA construct comprising i at least one polynucleotide that has the sequence of any one of SEQ ID NOs. 1 494 496 820 1641 1972 3588 3592 ii a first promoter iii a second promoter and iv a desired nucleic acid wherein a said polynucleotide encodes a plant transcription factor that regulates the activity of said second promoter b said second promoter and said desired nucleic acid are operably linked and c said polynucleotide is operably linked to and expressed by said first promoter.

In another aspect the invention provides a DNA construct comprising i at least one polynucleotide that has the sequence of any one of SEQ ID NOs. 1 494 496 820 1641 1972 3588 3592 and ii a promoter wherein a said polynucleotide encodes a plant transcription factor that regulates the activity of a promoter that is endogenous to a plant cell and b said promoter and said polynucleotide are operably linked.

In one embodiment the promoter is selected from the group consisting of a constitutive promoter a strong promoter or an inducible promoter.

In another embodiment the regulatable promoter is regulated by any one of auxin ethylene abscisic acid wounding methyl jasmonate or gibberellic acid.

In another embodiment the promoter is selected from the group consisting of the nucleic acid sequence identified in any one of SEQ ID NO 1642 to 1643.

In another embodiment the RNA transcript has an antisense sequence of a gene that is endogenous to a plant cell.

In another embodiment the RNA transcript induces RNA interference of a gene that is normally expressed in a plant cell.

In another aspect the invention provides a plant cell comprising a DNA construct that comprises i at least one polynucleotide that has the sequence of any one of SEQ ID NOs. 1 494 496 820 1641 1972 3588 3592 ii a promoter and iii a desired nucleic acid wherein said polynucleotide encodes a plant transcription factor that regulates the activity of said promoter and wherein said promoter and said desired gene are operably linked.

In another aspect the invention provides a plant cell comprising a DNA construct comprising i at least one polynucleotide that has the sequence of any one of SEQ ID NOs. 1 494 496 820 1641 1972 3588 3592 ii a first promoter iii a second promoter and iv a desired nucleic acid wherein a said polynucleotide encodes a plant transcription factor that regulates the activity of said second promoter b said second promoter and said desired gene are operably linked and c said polynucleotide is operably linked to and expressed by said first promoter. In one embodiment the invention provides a transgenic plant comprising the plant cell.

In another aspect the invention provides a plant cell comprising a DNA construct comprising i at least one polynucleotide that has the sequence of any one of SEQ ID NOs. 1 494 496 820 1641 1972 3588 3592 and ii a promoter wherein a said polynucleotide encodes a plant transcription factor that regulates the activity of a promoter that is endogenous to a plant cell and b said promoter and said polynucleotide are operably linked. In one embodiment the invention provides a transgenic plant comprising the plant cell.

In another aspect the invention provides an isolated polynucleotide comprising the sequence encoding the catalytic domain of any one of SEQ ID NOs. 821 1640 1973 2304 3593 3666 wherein said polynucleotide codes for a polypeptide that is capable of at least one of i binding to a nucleic acid molecule or ii regulating expression of a gene in a plant.

In another aspect the invention provides a method for producing a transgenic plant comprising a transforming a plant cell with a DNA construct that comprises i at least one polynucleotide that has the sequence of any one of SEQ ID NOs. 1 494 496 820 1641 1972 3588 3592 ii a promoter and iii a desired nucleic acid wherein said polynucleotide encodes a plant transcription factor that regulates the activity of said promoter and wherein said promoter and said desired gene are operably linked b culturing said transformed plant cell under conditions that promote growth of a plant wherein a polypeptide encoded by said polynucleotide and the product of said desired nucleic acid are both expressed in the plant cell and wherein said plant is a transgenic plant that exhibits a phenotype that is different from a plant of the same species that does not contain said DNA construct. In one embodiment the plant cell is located within a plant explant tissue.

In another embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in lignin quality compared to a plant of the same species that does not contain the DNA construct.

In another embodiment the difference in lignin quality is characterized by change in the structure of the lignin molecule.

In another embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in wood composition compared to a plant of the same species that does not contain the DNA construct.

In another embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in fiber yield compared to a plant of the same species that does not contain the DNA construct.

In another embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in plant cell division compared to a plant of the same species that does not contain the DNA construct.

In another embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in plant cell development compared to a plant of the same species that does not contain the DNA construct.

In another embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in any one of flower color petal shape petal size aroma leaf shape leaf size or plant height compared to a plant of the same species that does not contain the DNA construct.

In another aspect the present invention provides a method for producing a transgenic plant comprising a transforming a plant cell with a DNA construct that comprises i at least one polynucleotide that has the sequence of any one of SEQ ID NOs. 1 494 496 820 1641 1972 3588 3592 and ii a promoter wherein said polynucleotide and said promoter are operably linked and b culturing said transformed plant cell under conditions that promote growth of a plant wherein the polynucleotide encodes a polypeptide that is capable of at least one of binding to a part of the genome of the plant cell or regulating expression of a gene in the plant cell genome wherein said plant is a transgenic plant that exhibits a phenotype that is different from a plant of the same species that does not contain said DNA construct.

In one embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in lignin quality compared to a plant of the same species that does not contain the DNA construct.

In another embodiment the difference in lignin quality is characterized by change in the structure of the lignin molecule.

In another embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in wood composition compared to a plant of the same species that does not contain the DNA construct.

In another embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in fiber yield compared to a plant of the same species that does not contain the DNA construct.

In another embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in plant cell division compared to a plant of the same species that does not contain the DNA construct.

In another embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in plant cell development compared to a plant of the same species that does not contain the DNA construct.

In another embodiment the phenotype of the plant expressing the polynucleotide and the desired nucleic acid is characterized by a difference in any one of flower color petal shape petal size aroma leaf shape leaf size or plant height compared to a plant of the same species that does not contain the DNA construct.

In one aspect the invention provides a method for screening for a promoter that can be regulated by a plant transcription factor comprising a expressing in a plant cell a DNA construct that comprises i at least one polynucleotide that has the sequence of any one of SEQ ID NOs. 1 494 496 820 1641 1972 3588 3592 ii a constitutive promoter iii a candidate promoter and iv a reporter gene wherein said polynucleotide encodes a plant transcription factor wherein said candidate promoter and said reporter gene are operably linked and wherein said polynucleotide is operably linked to and expressed by said constitutive promoter b detecting the level of expression of said reporter gene and c comparing the level of expression of said reporter gene with the level of expression of a second reporter gene from a plant cell that contains a DNA construct comprising said candidate promoter operably linked to said second reporter gene.

In another aspect the invention provides a wood pulp obtained from a transgenic tree that expresses a transcription factor comprising the amino acid sequence of any one of SEQ ID NOs. 822 1640 3593 3596.

In another aspect the invention provides a transgenic plant that expresses a transcription factor comprising the amino acid sequence of any one of SEQ ID NOs. 822 1640 3593 3596 and wherein the transcription factor confers a trait to the plant selected from the group consisting of increased drought tolerance reduced or increased height reduced or increased branching enhanced cold and frost tolerance improved vigor enhanced color enhanced health and nutritional characteristics improved storage enhanced yield enhanced salt tolerance enhanced heavy metal tolerance increased disease tolerance increased insect tolerance increased water stress tolerance enhanced sweetness improved taste improved texture decreased phosphate content increased germination increased micronutrient uptake improved starch composition improved flower longevity and production of novel proteins or peptides.

In another aspect the invention provides a transgenic plant expressing a transcription factor comprising the amino acid sequence of any one of SEQ ID NOs. 822 1640 3593 3596 wherein said plant has a reduced or increased period of juvenality compared to a wild type plant of the same species.

In another aspect the invention provides a transgenic plant expressing a transcription factor comprising the amino acid sequence of any one of SEQ ID NOs. 822 1640 3593 3596 wherein said plant has self abscising branches.

In another aspect the invention provides a transgenic plant expressing a transcription factor comprising the amino acid sequence of any one of SEQ ID NOs. 822 1640 3593 3596 wherein said plant has accelerated or delayed reproductive development compared with a wild type plant of the same species.

In another aspect the invention provides an isolated nucleotide sequence having the nucleotide sequence of any of SEQ ID NO. 1 494 496 820 1641 1972 3588 3592 and nucleotide sequences having 60 sequence identity with the nucleotide sequence of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and which bind DNA.

In another aspect the invention provides an isolated nucleotide sequence having the nucleotide sequence of any of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and nucleotide sequences having 65 sequence identity with any of the nucleotide sequences of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and which are involved in transcription.

In another aspect the invention provides an isolated nucleotide sequence having the nucleotide sequence of any of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and nucleotide sequences having 70 sequence identity with any of the nucleotide sequences of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and which regulate expression of a gene in a plant.

In another aspect the invention provides an isolated nucleotide sequence having the nucleotide sequence of any of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and nucleotide sequences having 75 sequence identity with any of the nucleotide sequences of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and which encode a DNA binding protein.

In another aspect the invention provides an isolated nucleotide sequence having the nucleotide sequence of any of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and nucleotide sequences having 80 identity with any of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and which mediate transcription of a gene in a plant.

In another aspect the invention provides an isolated nucleotide sequence having the nucleotide sequence of any of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and nucleotide sequences having 85 identity with any of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and which bind DNA.

In another aspect the invention provides an isolated nucleotide sequence having the nucleotide sequence of any of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and nucleotide sequences having 90 identity with any of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 and which regulate expression of a gene in a plant.

In another aspect the invention provides an isolated nucleotide sequence having the nucleotide sequence of any of SEQ ID NO 181 188 and nucleotide sequences having 79 identity with any of SEQ ID NO 181 188 and which are involved in gene transcription.

In another aspect the invention provides a method of correlating polynucleotide expression in two different samples comprising 

detecting a level of expression of one or more polynucleotides encoding a product encoded by a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592 and conservative variants thereof in a first sample 

comparing the level of expression of the one or more polynucleotides in the first sample to the level of expression of the one or more polynucleotides in the second sample and

correlating a difference in expression level of the one or more polynucleotides between the first and second samples.

In one embodiment the first sample and the second sample are each from a different type of plant tissue.

In another embodiment the first sample and the second sample are from the same tissue and wherein the first sample and the second sample are each harvested during a different season of the year.

In another embodiment the first sample and the second sample are obtained from plants in different stages of development.

In another aspect the invention provides a method of correlating the possession of a plant phenotype to the level of polynucleotide expression in the plant of one or more polynucleotides comprising 

detecting a level of expression of one or more polynucleotides encoding a product encoded by a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592 and conservative variants thereof in a first plant possessing a phenotype 

detecting a level of expression of the one or more polynucleotides in a second plant lacking the phenotype 

comparing the level of expression of the one or more polynucleotides in the first plant to the level of expression of the one or more polynucleotides in the second plant and

correlating a difference in expression level of the one or more polynucleotides between the first and second plants to possession of the phenotype.

In one embodiment the first and second samples are both obtained from a plant tissue selected from the group consisting of vascular tissue apical meristem vascular cambium xylem phloem root flower cone fruit and seed.

In one embodiment the plant tissue of the first sample and second sample are each obtained from a different type of tissue.

In another embodiment the first and second samples are each obtained from a plant tissue in a different stage of development.

In another embodiment both the first and second plants or plant cells are of a same species selected from and species.

In yet another embodiment the first and second plants or plant cells are of a species selected from or

In yet another embodiment the step of detecting is effected using one or more polynucleotides capable of hybridizing to a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592 under standard hybridization conditions.

In yet another embodiment the step of detecting is effected using one or more polynucleotides capable of hybridizing to a polynucleotide expression product encoded by a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592 under standard hybridization conditions.

In yet another embodiment at least one of the one or more polynucleotides hybridizes to a 3 untranslated region of one of the one or more polynucleotides.

In another embodiment one of the one or more polynucleotides hybridizes to the 3 untranslated region of one of the one or more polynucleotides.

In another embodiment one or more polynucleotides comprises a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592.

In another embodiment one or more polynucleotides comprises a nucleic acid sequence selected from the group consisting of SEQ ID NOs 2742 3587.

In another embodiment one or more polynucleotides is selected from the group consisting of DNA and RNA.

In another embodiment one or more polynucleotides is selected from the group consisting of DNA and RNA.

In another embodiment prior to the detecting steps the step of amplifying the one or more polynucleotides in the first and second plant or plant cells.

In another embodiment further comprising prior to the detecting steps the step of labeling the one or more polynucleotides in the first and second plant or plant cells with a detectable label.

In another aspect the invention provides a combination for detecting expression of one or more polynucleotides comprising two or more oligonucleotides wherein each oligonucleotide is capable of hybridizing to a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592.

In another aspect the invention provides a combination for detecting expression of one or more polynucleotides comprising two or more oligonucleotides wherein each oligonucleotide is capable of hybridizing to a polynucleotide expression product encoded by a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592.

In another embodiment the invention provides two or more oligonucleotides hybridizes to a different one of the nucleic acid sequences selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592.

In another embodiment two or more oligonucleotides hybridizes to a nucleotide sequence encoded by a different one of the nucleic acid sequences selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592.

In another embodiment at least one of the two or more oligonucleotides hybridizes to a 3 untranslated region of a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592.

In another embodiment at least one of the two or more oligonucleotides hybridizes to nucleic acid sequence that is complementary to a 3 untranslated region of a nucleic acid sequence selected from the group consisting of SEQ ID Nos 1 494 496 820 1641 1972 3588 3592.

In another embodiment each of the two or more oligonucleotides are comprised of fewer than about 100 nucleotide bases.

In another embodiment at least one of the two or more oligonucleotides comprises a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1973 2304 3593 3666.

In another embodiment at least one of the two or more oligonucleotides comprises a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1973 2304 3593 3666.

In another embodiment each of the two or more oligonucleotides hybridizes to a gene encoding a protein selected from the group consisting of AB13 VP1 Alfin like AP2 EREBP ARF ARID AUX IAA bHLH bZIP C2C2 Zn C2C2 Co like C2C2 Dof C2C2 GATA C2C2 YABBY C2H2 Zn C3H type CCAAT CCAAT DR1 CCAAT HAP2 CCAAT HAP3 CCP Zn E2F DP EIL GARP GRAS HMB BOX HOMED BOX HSF Jumonji LIM MADS Box SEQ ID NO 3668 MYB NAC NIN like RAV like SBP TCP trihelix TUBBY and WRKY SEQ ID NO 3670 .

In another embodiment each of the two or more oligonucleotides hybridizes to a nucleic acid sequence encoded by a gene encoding a protein selected from the group consisting of AB13 VP1 Alfin like AP2 EREBP ARF ARID AUX IAA bHLH bZIP C2C2 Zn C2C2 Co like C2C2 Doff C2C2 GATA C2C2 YABBY C2H2 Zn C3H type CCAAT CCAAT DR1 CCAAT HAP2 CCAAT HAP3 CCP Zn E2F DP EIL GARP GRAS HMB BOX HOMED BOX SEQ ID NO 3668 HSF Jumonji LIM MADS Box MYB NAC RAV like SBP TCP trihelix TUBBY and WRKY SEQ ID NO 3670 .

In another embodiment each of the two or more oligonucleotides hybridizes to a gene encoding a different one of the proteins.

In another embodiment each of the two or more oligonucleotides hybridizes to a nucleic acid sequence encoded by a gene encoding a different one of the proteins.

In another embodiment each of the two or more oligonucleotides hybridizes to a nucleic acid sequence encoded by a different gene.

In another embodiment the combination comprises from about 2 to about 5000 of the two or more oligonucleotides.

In another embodiment the invention provides a microarray comprising the combination of any one of claims provided on a solid support wherein each of said two or more oligonucleotides occupies a unique location on said solid support.

In another aspect the invention proviA method for detecting one or more polynucleotides in a sample comprising 

contacting the sample with two or more oligonucleotides wherein each oligonucleotide is capable of hybridizing to a gene comprising a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592 under standard hybridization conditions and

detecting the one or more polynucleotides of interest which are hybridized to the one or more oligonucleotides.

In another aspect the present invention provides a method for detecting one or more nucleic acid sequences encoded by one or more polynucleotides in a sample comprising 

contacting the sample with two or more oligonucleotides wherein each oligonucleotide is capable of hybridizing to a nucleic acid sequence encoded by a gene comprising a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592 under standard hybridization conditions and

detecting the one or more nucleic acid sequences which are hybridized to the one or more oligonucleotides.

In another embodiment each of the two or more oligonucleotides hybridizes to a gene comprising a different one of the nucleic acid sequences selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592.

In another embodiment each of the two or more oligonucleotides hybridizes to a nucleic acid sequence encoded by a gene comprising a different one of the nucleic acid sequences selected from the group consisting of SEQ ID Nos 1 494 496 820 1641 1972 3588 3592.

In another embodiment at least one of the two or more oligonucleotides hybridizes to a 3 untranslated region of a gene comprising a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592.

In another embodiment at least one of the two or more oligonucleotides hybridizes to a nucleic acid sequence that is complementary to a 3 untranslated region of a gene comprising a nucleic acid sequence selected from the group consisting of SEQ ID Nos 1 494 496 820 1641 1972 3588 3592.

In another embodiment each of the two or more oligonucleotides are comprised of fewer than about 100 nucleotide bases.

In another embodiment at least one of the two or more oligonucleotides comprises a nucleic acid sequence selected from the group consisting of SEQ ID Nos 1973 2304 3593 3666.

In another embodiment at least one of the two or more oligonucleotides comprises a nucleic acid sequence selected from the group consisting of SEQ ID NOs 1973 2304 3593 3666.

In another embodiment each of the two or more oligonucleotides hybridizes to a gene encoding a protein selected from the group consisting of AB13 VP1 Alfin like AP2 EREBP ARF ARID AUX IAA bHLH bZIP C2C2 Zn C2C2 Co like C2C2 Dof C2C2 GATA C2C2 YABBY C2H2 Zn C3H type CCAAT CCAAT DR1 CCAAT HAP2 CCAAT HAP3 CCP Zn E2F DP EIL GARP GRAS HMB BOX HOMED BOX HSF Jumonji LIM MADS Box SEQ ID NO 3668 MYB NAC NIN like RAV like SBP TCP trihelix TUBBY and WRKY SEQ ID NO 3670 .

In another embodiment each of the two or more oligonucleotides hybridizes to a nucleic acid sequence encoded by a gene encoding a protein selected from the group consisting of AB13 VP1 Alfin like AP2 EREBP ARF ARID AUX IAA bHLH bZIP C2C2 Zn C2C2 Co like C2C2 Dof C2C2 GATA C2C2 YABBY C2H2 Zn C3H type CCAAT CCAAT DR1 CCAAT HAP2 CCAAT HAP3 CCP Zn E2F DP EIL GARP GRAS HMB BOX HOMED BOX HSF Jumonji LIM MADS Box SEQ ID NO 3668 MYB NAC RAV like SBP TCP trihelix TUBBY and WRKY SEQ ID NO 3670 .

In another embodiment each of the two or more oligonucleotides hybridizes to a gene encoding a different one of the proteins.

In another embodiment each of the two or more oligonucleotides hybridizes to a nucleic acid sequence encoded by a gene encoding a different one of the proteins.

In another embodiment two or more oligonucleotides are provided on a solid support wherein each of the two of more oligonucleotides occupy a unique location on the solid support.

In another embodiment the solid support comprises from about 2 to about 5000 of the two or more oligonucleotides.

In another embodiment further comprising prior to the contacting step the step of amplifying the one or more polynucleotides or nucleic acid sequences in the sample.

In another embodiment further comprising prior to the contacting step the step of labeling the one or more polynucleotides or nucleic acid sequences in the sample with a detectable label.

In another embodiment the invention provides a kit for detecting gene expression comprising the microarray with one or more buffers or reagents for a nucleotide hybridization reaction.

The above mentioned and additional features of the present invention and the manner of obtaining them will become apparent and the invention will be best understood by reference to the following more detailed description. All references disclosed herein are hereby incorporated by reference in their entirety as if each was incorporated individually.

The present invention provides isolated polynucleotides that encode plant transcription factors together with isolated polypeptides encoded by such polynucleotides.

Transformation of a plant with a polynucleotide sequence encoding a protein involved in the regulation of gene expression can be employed to modify properties such as cellulose synthesis lignin deposition other aspects of wood development flower development root development branching seasonal responses such as light and cold controls on meristem identity and disease resistance. To this end the present invention provides a polynucleotide sequence encoding a polypeptide sequence having the function of a plant transcription factor. The present invention also provides a DNA construct having a promoter operably linked to a polynucleotide sequence wherein said polynucleotide sequence encodes a plant transcription factor. Additionally the invention provides methods for assaying the activity of an inventive transcription factor sequence methods for using a transcription factor for modifying growth wood development and or fiber composition in a plant.

The present invention uses terms and phrases that are well known to those practicing the art. Unless defined otherwise all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Generally the nomenclature used herein and the laboratory procedures in cell culture molecular genetics and nucleic acid chemistry and hybridization described herein are those well known and commonly employed in the art. Standard techniques are used for recombinant nucleic acid methods polynucleotide synthesis microbial culture cell culture tissue culture transformation transfection transduction analytical chemistry organic synthetic chemistry chemical syntheses chemical analysis and pharmaceutical formulation and delivery. Generally enzymatic reactions and purification and or isolation steps are performed according to the manufacturers specifications. The techniques and procedures are generally performed according to conventional methodology Sambrook Russel MC A LM3ed. Cold Spring Harbor Laboratory Press Cold Spring Harbor N.Y. 2001 .

ABI3 VP1 The maize Vp1 gene and abi3 gene of are believed to be orthologs based on similarities of the mutant phenotypes and amino acid sequence conservation. VP1 fully restores abscisic acid ABA sensitivity to abi3 mutants during seed germination and suppresses the early flowering phenotype of abi3. VP1 mediates a novel interaction between ABA and auxin signaling that results in developmental arrest and altered patterns of gene expression. Suzuki M et al. 2001 28 4 409 18. Auxin and abscisic acid are important in many plant developmental processes including leaf and root development Brady S M Sarkar S F Bonetta D and McCourt P 200334 1 67 75 .

AP2 The AP2 APETALA2 and EREBPs ethylene responsive element binding proteins are the prototypic members of a family of transcription factors unique to plants whose distinguishing characteristic is that they contain the so called AP2 DNA binding domain. AP2 EREBP genes form a large multigene family and they play a variety of roles throughout the plant life cycle. AP2 EREBP genes are key regulators of several developmental processes including floral organ identity determination and leaf epidermal cell identity. In the homeotic gene APETALA2 AP2 has been shown to control three salient processes during development 1 the specification of flower organ identity throughout floral organogenesis Jofuku et al. 6 1211 1225 1994 2 establishment of flower meristem identity Irish and Sussex 2 8 741 753 1990 and 3 the temporal and spatial regulation of flower homeotic gene activity Drews et al. 65 6 991 1002 1991 . DNA sequence analysis suggests that AP2 encodes a theoretical polypeptide of 432 aa with a distinct 68 aa repeated motif termed the AP2 domain. This domain has been shown to be essential for AP2 functions and contains within the 68 aa an eighteen amino acid core region that is predicted to form an amphipathic helix Jofuku et al. 6 1211 1225 1994 . Ap2 like domain containing transcription factors have been also been identified in both Okamuro et al. 94 7076 7081 1997 and in tobacco with the identification of the ethylene responsive element binding proteins EREBPs Ohme Takagi and Shinshi 7 2 173 182 1995 . In these RAP2 related to AP2 genes encode two distinct subfamilies of AP2 domain containing proteins designated AP2 like and EREBP like Okamuro et al. 94 7076 7081 1997 . In vitro DNA binding has not been shown to date using the RAP2 proteins. Based upon the presence of two highly conserved motifs YRG and RAYD SEQ ID NO 3672 within the AP2 domain it has been proposed that binding DNA binding occurs in a manner similar to that of AP2 proteins.

Alfin like Alfin1 is a transcription factor that functions in roots. Alfin1 overexpression also improves salt tolerance and root growth of the transgenic plants Winicov I. 2000 210 3 416 22 .

Angiosperm vascular plants having seeds enclosed in an ovary. Angiosperms are seed plants that produce flowers that bear fruits. Angiosperms are divided into dicotyledonous and monocotyledonous plants.

ARF Auxin response factors ARFs are a recently discovered family of transcription factors that bind with specificity to auxin response elements AuxREs in promoters of primary or early auxin responsive genes. ARFs have an amino terminal DNA binding domain related to the carboxyl terminal DNA binding domain in the maize transactivator VIVIPAROUS1. Some ARFs contain transcriptional activation domains while others contain repression domains. ARFs appear to play a pivotal role in auxin regulated gene expression of primary response genes Guilfoyle T J Ulmasov T and Hagen G. 199854 7 619 27 . ARF genes in have been shown to be important in controlling both axis formation in the embryo and auxin dependent cell expansion Hardtke C S Ckurshumova W Vidaurre D P Singh S A Stamatiou G Tiwari S B Hagen G Guilfoyle T J and Berleth T. 2004 131 5 1089 100 . Auxin responses are important in meristem and wood development in plants Uggla C Magel E Moritz T and Sundberg B 2001125 4 2029 39 .

ARID Dead ringer Dri is a founding member of a recently defined ARID family of DNA binding proteins whose members share a conserved DNA binding domain termed the A T rich interaction domain. This family includes the B cell specific factor Bright and the factor Eyelid Osa . dri is developmentally regulated and is expressed in a restricted set of cells including some neural cells and differentiating cells of the gut and salivary gland ducts. It is unlikely that Dri is a general transcription co factor or chromatin modifier as is Eyelid since transcription of only a small number of the genes are disrupted in dri mutant embryos Valentine 1998 and Shandala 1999 .

The ARID domain can be found in many genomes of plants and at least one ARID gene family can be clearly traced from plant to metazoans Rbbp2 family by the conservation of the order of multiple conserved domains.

Dri has been shown to be a sequence specific DNA binding protein. The in vitro sequence specificity of Dri is strikingly similar to that of many homeodomain proteins. Dri preferentially binds the PuATTAA sequence. It is therefore likely that the phenotypes exhibited by dri mutant embryos result from disruption to the expression of regulatory genes. ARID proteins have been implicated in the control of cell growth differentiation and development Wilsker D Patsialou A Dallas P B and Moran E. 200213 3 95 106 .

AUX IAA Indole 3 acetic acid IAA or auxin is indispensable for plant growth and development. The hormone rapidly and specifically activates within minutes transcription of a select set of early genes that are thought to mediate the various auxin effects which include effects on meristem and wood development. The concept of early genes or primary response genes has successfully been used in several biological systems to access and explore upstream and downstream segments of signal transduction pathways. Molecular and genetic studies conducted by a number of groups indicate that Aux IAA proteins play a central role in auxin responses Tiwari S B Hagen G and Guilfoyle T. 200315 2 533 43 Moyle R Schrader J Stenberg A Olsson O Saxena S Sandberg G and Bhalerao R P. 200231 6 675 85 .

bZIP The basic leucine zipper bZIP is a conserved family of transcription factors defined by a basic leucine zipper bZIP motif Landschultz et al. 240 1759 1764 1988 McKnight 264 54 64 1991 Foster et al. 8 2 192 200 1994 . Transcriptional regulation of gene expression is mediated by both the bZIPs and other families of transcription factors through the concerted action of sequence specific transcription factors that interact with regulatory elements residing in the promoter regions of the corresponding gene. The bZIP bipartite DNA binding structure consists of a region enriched in basic amino acids basic region adjacent to a leucine zipper that is characterized by several leucine residues regularly spaced at seven amino acid intervals Vinson et al. 246 911 916 1989 . Whereas the basic region directly contacts the DNA the leucine zipper mediates homodimerisation and heterodimerisation of protein monomers through a parallel interaction of the hydrophobic dimerization interfaces of two helices resulting in a coiled coil structure O Shea et al. 243 538 542 1989 254 539 544 1991 Hu et al. 250 1400 1403 1990 Rasmussen et al. 88 561 564 1991 .

Dof proteins are a relatively new class of transcription factor and are thought to mediate the regulation of some patterns of plant gene expression in part by combinatorial interactions between bZIP proteins and other types of transcription factors binding to closely linked sites. Such an example of this combinatorial interaction has been observed between bZIP and Dof transcription factors Singh 118 1111 1120 1998 . These Dof proteins possess a single zinc finger DNA binding domain that is highly conserved in plants Yanagisawa 1 213 1996 . Specific binding of the Dof protein to bZIP transcription factors has been demonstrated and it has been proposed that this specific interaction results in the stimulation of bZIP binding to DNA target sequences in plant promoters Chen et al. 10 955 966 1996 . Examples of such Dof bZIP interactions have been reported in the literature including for example the glutathionine S transferase 6 gene GST6 promoter which has been shown to contain several Dof binding sites closely linked to the ocs element a recognized bZIP binding site Singh 118 1111 1120 1998 .

The bZIP family of G box binding factors from including GBF1 GBF2 and GBF3 for example interact with the palindromic G box motif CCACGTGG . However it has been demonstrated that the DNA binding specificity of such transcription factors for example GBF1 may be influenced by the nature of the nucleotides flanking the ACGT core Schindler et al. 11 1274 1289 1992a . In vivo transient and transgenic plant expression studies have shown that these ACGT elements are necessary for maximal transcriptional activation and have been identified in a multitude of plant genes regulated by diverse environmental physiological and environmental cues. Classification of these transcription factors based upon their ability to bind to the ACGT core motif yielded a relatively diverse group of proteins including for example the CamV 35S promoter as 1 binding protein which exhibits DNA binding site requirements distinct from those proteins interacting with the G box Tabata et al. 10 1459 1467 1991 . Thus in addition to defining the individual classes of bZIP proteins on the basis of their DNA binding specificity such proteins can also be classified according to their heterodimerisation characteristics Cao et al. 5 1538 1552 1991 Schindler et al. 11 1261 1273 1992b .

Environmentally inducible promoters require the presence of two cis acting elements critical for promoter activity one of which is the moderately conserved G box CCACGTGG deVetten et al. 4 10 1295 1307 1992 . A mutation in one of the two elements abolishes or severely reduces the ability of the promoter to respond to environmental changes. The sequence of the second cis acting element positioned near the G box is not conserved among different environmentally inducible promoters but may be similar among promoters induced by the same signal. The spacing between the G box and the second cis acting element appears to be critical suggesting a direct interaction between the respective binding factors deVetten and Ferl 26 9 1055 1068 1994 Ramachandran et al. 4 5 642 646 1994 .

Basic helix loop helix zipper proteins represent an additional class of bZIP transcription factors described in the literature and includes for example the Myc proteins. These proteins contain two regions characteristic of transcription factors an N terminal transactivation domain consisting of several phosphorylation sites and a C terminal basic helix loop helix bHLH leucine zipper motif known to mediate dimerization and sequence specific DNA binding via three distinct domains the leucine zipper helix loop helix and basic regions Toledo Ortiz G Huq E and Quail P H. 200315 8 1749 70 . It is predicted that this family of TFs has a range of different roles in plant cell and tissue development as well as plant metabolism including specifying epidermal cell fate in roots Bernhardt C Lee M M Gonzalez A Zhang F Lloyd A and Schiefelbein J. 2003130 26 6431 9 fruit development Liljegren S J Roeder A H Kempin S A Gremski K Ostergaard L Guimil S Reyes D K and Yanofsky M F 2004116 6 843 53 the formation of ER bodies Matsushima R Fukao Y Nishimura M and Hara Nishimura I. 2004 Plant Cell. May 21 Epub ahead of print and be involved in anthocyanin biosynthesis Ramsay N A Walker A R Mooney M and Gray J C 200352 3 679 88 .

CCAAT The CCAAT box element identified by Gelinas et al. 313 6000 323 325 1985 has been shown to occur between 80 bp and 300 bp from the transcription start site and may operate in either orientation with possible cooperative interactions with multiple boxes Tasanen et al. 267 16 11513 11519 1992 or other conserved motifs Muro et al. 267 18 12767 12774 1992 Rieping and Schoffl 231 2 226 232 1992 . CCAAT box related motifs have been identified in a number of promoters in a variety of organisms including yeast Hahn et al. 240 4850 317 321 1988 rat Maity et al. 87 14 5378 5382 1990 Vuorio et al. 265 36 22480 22486 1990 and plants Rieping and Schoffl 231 2 226 232 1992 Kehoe et al. 6 8 1123 1134 1994 . In both yeast and vertebrates a protein complex has been shown to bind to the CCAAT motif. In yeast the complex consists of three proteins known as HAP2 HAP3 and HAP5 Pinkham and Guarente 5 12 3410 3416 1985 .

In there exists an analagous DR1 transcription factor. The identification of a Dr1 like protein in strongly argues for the ubiquity of this protein among eukaryotic genera and for a conserved mechanism to regulate transcription initiation that involves Dr1. Kuromori Yamamoto Cloning of cDNAs from that encode putative protein phosphatase 2C and a human Dr1 like protein by transformation of a fission yeast mutant 22 24 5296 301 1994 .

CAATT binding factors have been implicated with plant fertility in Levesque Lemay M Albani D Aldcorn D Hammerlindl J Keller W and Robert L S 200321 8 804 8. Epub 2003 Mar. 4 and embryogenesis Lee H Fischer R L Goldberg R B and Harada J J. 2003100 4 2152 6 .

C2C2 Co like The vegetative and reproductive flowering phases of development are clearly separated. The onset of flowering is promoted by long photoperiods but the constans co mutant flowers later than wild type under these conditions Putterill J Robson F Lee K Simon R and Coupland G 199580 6 847 57 Valverde F Mouradov A Soppe W Ravenscroft D Samach A and Coupland G 2004 303 5660 1003 6 . Some transgenic plants containing extra copies of CO flowered earlier than wild type suggesting that CO activity limits flowering time. Double mutants were constructed containing co and mutations affecting gibberellic acid responses meristem identity or phytochrome function and their phenotypes suggested a model for the role of CO in promoting flowering. CO interaction with phytohormone response and meristem identity means that CO like genes may function to regulate genes in a variety of plant developmental processes.

Despite promoting flowering in response to long days and rice promoting flowering in response to short days the network controlling this response has been found to be highly conserved in these distantly related plants and controlled by Constans Simpson G G. 2003 25 9 829 32 .

C2C2 GATA Many light responsive promoters common in plants contain GATA motifs and a number of nuclear proteins have been defined that interact with these elements. Type IV zinc finger proteins have been extensively characterised in animals and fungi and are referred to as GATA factors by virtue of their affinity for promoter elements containing this sequence Lowry J A and Atchley W R. 200050 2 103 15 .

Proteins containing a domain structure containing the C X2 C X20 C X2 C motif SEQ ID NO 3673 a CCT domain and an uncharacterized conserved domain were found exclusively in plants indicating that they belong to a novel family of plant specific GATA type transcription factors. The overexpression of one such factor ZIM in resulted in the elongation of hypocotyls and petiols Shikata M Matsuda Y Ando K Nishii A Takemura M Yokota A and Kohchi T. 200455 397 631 9 .

C2C2 YABBY The expression of these genes is precisely correlated with abaxial cell fate in mutants in which abaxial cell fates are found ectopically reduced or eliminated. Members of this gene family are responsible for the specification of abaxial cell fate in lateral organs of such as leaves and floral organs Siegfried K R Eshed Y Baum S F Otsuga D Drews G N and Bowman J L 1999 126 18 4117 28 . Yabby also plays a role in other plants for example it regulates midrib formation by promoting cell proliferation in the central region of the rice leaf Yamaguchi T Nagasawa N Kawasaki S Matsuoka M Nagato Y and Hirano H Y. 200416 2 500 9 .

C2H2 Zn C2H2 zinc finger protein genes encode nucleic acid binding proteins involved in the regulation of gene activity. AtZFP1 zinc finger protein 1 is one member of a small family of C2H2 zinc finger encoding sequences previously characterized from . The genomic sequence corresponding to the AtZFP1 cDNA has been determined. Molecular analysis demonstrates that AtZFP1 is a unique intronless gene which encodes a 1100 nucleotides mRNA highly expressed in roots and stems Chrispeels H E Oettinger H Janvier N and Tague B W. 200042 2 279 90 .

Plant C2H2 zinc finger transcription factors have been identified as playing important roles in floral organogenesis Yun J Y Weigel D and Lee I. 200243 1 52 7 flowering time Kozaki A Hake S and Colasanti J. 200432 5 1710 20 leaf initiation lateral shoot initiation gametogenesis and seed development Sagasser M Lu G H Hahlbrock K and Weisshaar B 2002 16 0 138 49 .

C3H type Zn C3H type zinc finger proteins are known to be involved in the regulation of cell division in human tumors and may have similar functions in plants.

CPP ZN A novel type of DNA binding protein CPP1 has been identified interacting with the promoter of the soybean leghemoglobin gene Gmlbc3. The DNA binding domain of CPP1 contains two similar Cys rich domains with 9 and 10 Cys respectively. The cpp1 gene is induced late in nodule development and the expression is confined to the distal part of the central infected tissue of the nodule. A constitutively expressed cpp1 gene reduces the expression of a Gmlbc3 promoter gusA reporter construct in roots. These data therefore suggest that CPP1 might be involved in the regulation of the leghemoglobin genes in the symbiotic root nodule Cvitanich C Pallisgaard N Nielsen K A Hansen A C Larsen K Pihakaski Maunsbach K Marcker K A and Jensen E O 200097 14 8163 8 .

Desired Polynucleotide a desired polynucleotide of the present invention is a genetic element such as a promoter enhancer or terminator or gene or polynucleotide that is to be transcribed and or translated in a transformed cell that comprises the desired polynucleotide in its genome. If the desired polynucleotide comprises a sequence encoding a protein product the coding region may be operably linked to regulatory elements such as to a promoter and a terminator that bring about expression of an associated messenger RNA transcript and or a protein product encoded by the desired polynucleotide. Thus a desired polynucleotide may comprise a gene that is operably linked in the 5 to 3 orientation a promoter a gene that encodes a protein and a terminator. Alternatively the desired polynucleotide may comprise a gene or fragment thereof in a sense or antisense orientation the transcription of which produces nucleic acids that may affect expression of an endogenous gene in the plant cell. A desired polynucleotide may also yield upon transcription a double stranded RNA product upon that initiates RNA interference of a gene to which the desired polynucleotide is associated. A desired polynucleotide of the present invention may be positioned within a T DNA such that the left and right T DNA border sequences flank or are on either side of the desired polynucleotide. The present invention envisions the stable integration of one or more desired polynucleotides into the genome of at least one plant cell. A desired polynucleotide may be mutated or a variant of its wild type sequence. It is understood that all or part of the desired polynucleotide can be integrated into the genome of a plant. It also is understood that the term desired polynucleotide encompasses one or more of such polynucleotides. Thus a T DNA of the present invention may comprise one two three four five six seven eight nine ten or more desired polynucleotides.

Dicotyledonous plant dicot a flowering plant whose embryos have two seed halves or cotyledons branching leaf veins and flower parts in multiples of four or five. Examples of dicots include but are not limited to teak mahogany cotton tobacco tomato potato sugar beet broccoli cassava sweet potato pepper poinsettia bean alfalfa soybean carrot strawberry lettuce oak maple walnut rose mint squash daisy geranium avocado and cactus.

DRAP1 NC2 Dr1 Drap1 is a bifunctional basal transcription factor that differentially regulates gene transcription through DPE or TATA box motifs. Purified recombinant dNC2 activates DPE driven promoters and represses TATA driven promoters. A mutant version of dNC2 can activate DPE promoters but is unable to repress TATA promoters. Thus the activation and repression functions are distinct. Rice Song W Solimeo H Rupert R A Yadav N S and Zhu Q 200214 1 181 95 .

E2F DP E2F DP complexes play a pivotal role in the regulation of the G1 S transition in animals. Recently plant E2F and DP related homologs have been cloned. Plant E2F homologs exhibit an overall domain organization similar to that of their animal counterparts although phylogenetic analysis demonstrated that they form a separate subgroup. They are predominantly produced in actively dividing cells with highest transcript levels in early S phase cells Mariconti L Pellegrini B Cantoni R Stevens R Bergounioux C Cella R and Albani D 2002277 12 9911 9 . In tobacco high expression of E2F promotes endoreduplication by accelerating S phase entry in terminally differentiated cells with limited mitotic activity and enhanced E2F activity modulates cell cycle in a cell type specific manner and affects plant morphology depending on a balance between activities for committing to S phase and M phase Kosugi S and Ohashi Y. 2003132 4 2012 22 . In known promoters E2F binding regions are found in the promoters of cell division related genes Egelkrout E M Mariconti L Settlage S B Cella R Robertson D and Hanley Bowdoin L. 200214 12 3225 36 Stevens R Mariconti L Rossignol P Perennes C Cella R and Bergounioux C. 2002277 36 32978 84 .

EIL Overexpression of EIN3 or EIL1 in wild type plants resulted in a constitutive ethylene phenotype and increased ERF1 expression. These results indicate that EIN3 is a transcription factor that acts as a positive regulator of the ethylene signal transduction pathway Chao Q Rothenberg M Solano R Roman G Terzaghi W and Ecker J R 199789 7 1133 44 . Ethylene is important in many plant processes including maturation and wood formation.

Fiber composition as used herein fiber composition refers to trait that can be modified to change the structure appearance or use of fiber. While not limiting traits that determine fiber composition include fiber length coarseness strength color cross sectional and fiber density. For example it is known that fiber length imparts strength whereas fiber coarseness determines texture and flexibility.

Foreign foreign with respect to a nucleic acid means that that nucleic acid is derived from non plant organisms or derived from a plant that is not the same species as the plant to be transformed or is not derived from a plant that is not interfertile with the plant to be transformed does not belong to the species of the target plant. According to the present invention foreign DNA or RNA represents nucleic acids that are naturally occurring in the genetic makeup of fungi bacteria viruses mammals fish or birds but are not naturally occurring in the plant that is to be transformed. Thus a foreign nucleic acid is one that encodes for instance a polypeptide that is not naturally produced by the transformed plant. A foreign nucleic acid does not have to encode a protein product.

GARP GARP transcription factors are represented by the family of Response Regulator ARR genes that mediate responses to cytokinin and ethylene. The ARR family can be divided into two groups Type A and Type B which differ in their sequence and domain structure. Type A genes are responsive to cytokinin while Type B genes are induced by ethylene and osmotic stress. Both Type A and Type B family genes have a two component signal transduction system. comprising a histidyl aspartyl phosphorelay and a response regulator receiver. Stock et al. 69 183 215 2000 .

Gene A gene is a segment of a DNA molecule that contains all the information required for synthesis of a product polypeptide chain or RNA molecule that includes both coding and non coding sequences.

Genetic element a genetic element is any discreet nucleotide sequence such as but not limited to a promoter gene terminator intron enhancer spacer 5 untranslated region 3 untranslated region or recombinase recognition site.

Genetic modification stable introduction of DNA into the genome of certain organisms by applying methods in molecular and cell biology.

Gymnosperm as used herein refers to a seed plant that bears seed without ovaries. Examples of gymnosperms include conifers cycads ginkgos and ephedras.

GRAS Sequence analysis of the products of the GRAS GAI RGA SCR gene family indicates that they share a variable amino terminus and a highly conserved carboxyl terminus that contains five recognizable motifs. The importance of the GRAS gene family in plant biology has been established by the functional analyses of SCR GAI and RGA. These genes appear to have a function in patterning particularly radial patterning which is important in the development of stems roots and floral organs Pysh et al. 18 111 119 1999 . GRAS proteins exert important roles in very diverse processes such as signal transduction meristem maintenance and development Bolle C. 2004 218 5 683 92 .

Homeotic transcription factors In animals homeotic transcription factors have in animals been implicated in a number of developmental processes including for example the control of pattern formation in insects and vertebrate embryos and the specification of cell differentiation in many tissues Ingham 335 25 34 1988 McGinnis and Krumlauf 68 283 302 1992 . Homeodomain secondary structures are characterized by a distinctive helix turn helix motif initially identified in bacterial DNA binding domains. This helix turn helix sequence structure motif spans approximately 20 amino acids and is characterized by two short helices separated by a sharp 90 degree bend or turn Harrison and Aggarwal 59 933 969 1990 . This helix has been shown to bind in the major groove of the DNA helix.

Plant homeobox genes have been identified in a number of plant species including maize parsley and soybean. Expression pattern analysis of maize homeobox gene family members suggests that these transcription factors may be involved in defining specific regions in the vegetative apical meristem potentially involved in the initiation of leaf structures Jackson et al. 120 405 413 1994 . Such observations imply that the plant homeobox genes as for the animal homeobox genes may be involved in the determination of cell fate.

Homeodomain zipper HD zip represents an additional family of homeodomain proteins. These homeodomain zipper proteins HD zip possess both the characteristic homeodomain linked to an additional leucine zipper dimerization motif. This family includes for example Athb 1 and Athb 2 Sessa et al. 12 3507 3517 1993 and Athb 4 Carabelli et al. 4 469 479 1993 .

HSF Heat shock factors HSF are the transcriptional activators of the heat shock response. The conversion of constitutively expressed HSF to a form that can bind DNA requires the trimerization of the protein involving leucine zipper interactions as shown for yeast chicken and human HSFs. Like other metazoan HSFs the endogenous HSF displays heat shock inducible DNA binding activity in gel retardation assays Hubel A Lee J H Wu C and Schoffl F 1995248 2 136 41 . Overexpression of heat shock protein in plants results in plants exhibiting a thermotolerance Sanmiya K Suzuki K Egawa Y and Shono M. 2004 557 1 3 265 8 Sung D Y and Guy C L. 2003132 2 979 87 .

Introduction as used herein refers to the insertion of a nucleic acid sequence into a cell by methods including infection transfection transformation or transduction.

Jumonji There is an absence of literature regarding jumonji transcriptional regulators in plants. In animals however there is a small amount of literature covering this family. Overexpression has been shown to decrease cell proliferation and suggests a role during regulation of cell proliferation signaling Ohno T Nakajima K Kojima M Toyoda M and Takeuchi T 2004317 3 925 9 Kitajima K Kojima M Kondo S and Takeuchi T 200129 4 507 14 . The jumonji protein contains an ARID domain and a jmjC domain. Frequently jumonji proteins are associated with a small N terminal jmjN domain and or a C terminal ZnC5HC2 domain and or a PHD Zn finger Toyoda M Kojima M Takeuchi T. 2000274 2 332 6 .

Juvenility describes a physiological difference between a young tree and a mature tree. In the present invention juvenility refers to differences in microfibril angle wood density cellulose yield regenerability and reproductive ability between a young tree and a mature tree. For example it has been shown that as a plant tissue matures the tissue loses its ability to regenerate.

Lignin as used herein refers to a polymeric composition composed of phenylpropanoid units including polymerized derivatives of monolignols coniferyl coumaryl and sinapyl alcohol. Lignin quality refers to the ability of a lignin composition to impart strength to cell wall matrices assist in the transport of water and or impede degradation of cell wall polysaccharides. Lignin composition or lignin structure may be changed by altering the relative amounts of each of monolignols or by altering the type of lignin. For example guaiacyl lignins derived from ferulic acid are prominent in softwood species whereas guaiacyl syringyl lignins derived from ferulic acid and sinapic acid are characteristic of hardwood species. The degradation of lignin from softwoods such as pine requires substantially more alkali and longer incubations compared with the removal of lignin from hardwoods. Additionally lignin composition may be regulated by either up regulation or down regulation of enzymes involved lignin biosynthesis. For example key lignin biosynthsesis enzymes include 4 coumaric acid coenzyme A ligase 4CL Cinnamyl Alcohol dehydrogenase CAD and Sinapyl Alcohol Dehydrogenase SAD .

LIM The LIM domain is a specialized double zinc finger motif found in a variety of proteins in association with domains of divergent functions such as the homeodomain see the sunflower pollen specific SF3 transcription factor Baltz et al. 2 713 721 1992 or forming proteins composed primarily of LIM domains Dawid et al. 144 156 162 1998 . LIM domains interact specifically with other LIM domains and with many different protein domains. LIM domains are thought to function as protein interaction modules mediating specific contacts between members of functional complexes and modulating the activity of some of the constituent proteins. Nucleic acid binding by LIM domains while suggested by structural considerations remains an unproven possibility. However it is possible that together with the homeodomain the LIM domain could bind to the regulatory regions of developmentally controlled genes as has been proposed for the paired box a conserved sequence motif first identified in the paired PRD and gooseberry GSB homeodomain proteins from Triesman et al. 5 594 604 1991 . The PRD box is also able to bind DNA in the absence of the homeodomain. LIM domain proteins can be nuclear cytoplasmic or can shuttle between compartments. In the animal systems several important LIM proteins have been shown to be associated with the cytoskeleton having a role in adhesion plaque and actin microfilament organization. Among nuclear LIM proteins the LIM homeodomain proteins form a major subfamily with important functions in cell lineage determination and pattern formation during animal development. In plants a LIM protein has been demonstrated to control a number of genes in the lignin biosynthesis pathway critically important for developing wood Kawaoka A Ebinuma H 2001 Transcriptional control of lignin biosynthesis by tobacco LIM protein. 57 1149 1157 Kawaoka et al. 22 289 301 2000 .

MADS MADS box SEQ ID NO 3668 transcription factors interact with a conserved region of DNA known as the MADS box. All MADS box SEQ ID NO 3668 transcription factors contain a conserved DNA binding dimerization region known as the MADS domain SEQ ID NO 3668 which has been identified throughout the different kingdoms Riechmann and Meyerowitz 378 10 1079 1101 1997 . Many of the MADS box SEQ ID NO 3668 genes isolated from plants are expressed primarily in floral meristems or floral organs and are believed to play a role in either specifying inflorescence and floral meristem identity or in determining floral organ identity. One class of regulatory genes responsible for floral meristem identity and the pattern of meristem development includes the genes APETALA1 AP1 APETALA2 AP2 CAULIFLOWER CAL LEAFY LFY and AGAMOUS AG from . Both LFY and AP1 have been shown to encode putative transcription factors Weigel et al. 69 843 859 1992 with AP1 and AG each encoding putative transcription factors of the MADS box domain family Yanofsky et al. 346 35 39 1990 . Mutations in the Lfy gene have been shown to result in a partial conversion of flowers into inflorescence shoots. MADS box SEQ ID NO 3668 genes are required for anther and pollen maturation Schreiber D N Bantin J and Dresselhaus T. 2004134 3 1069 79 the transition from vegetative to reproductive growth in plants Murai K Miyamae M Kato H Takumi S and Ogihara Y. 200344 12 1255 65 an flowering time Trevaskis B Bagnall D J Ellis M H Peacock W J and Dennis E S. 2003100 22 13099 104 .

Monocotyledonous plant monocot a flowering plant having embryos with one cotyledon or seed leaf parallel leaf veins and flower parts in multiples of three. Examples of monocots include but are not limited to turfgrass maize rice oat wheat barley sorghum orchid iris lily onion and palm. Examples of turfgrass include but are not limited to spp. bentgrass species including colonial bentgrass and creeping bentgrasses kentucky bluegrass spp. ryegrass species including annual ryegrass and perennial ryegrass tall fescue commutata fine fescue common bermudagrass varieties including Tifgreen Tifway II and Santa Ana as well as hybrids thereof kikuyugrass st. augustinegrass zoysiagrass and

Myb The Myb family of transcription factors is a group of functionally diverse transcriptional activators found in both plants and animals that is characterized by a conserved amino terminal DNA binding domain containing either two in plant species or three in animal species imperfect tandem repeats of approximately 50 amino acids Rosinski and Atchley 46 1 74 83 1998 Stober Grasser et al. 7 3 589 596 1992 . Comparisons between the amino acid sequences of representative plant and mammalian MYB proteins indicate that there is a greater conservation between the same repeat from different proteins than between the R2 and R3 repeats from the same protein Martin and Paz Ares 13 2 67 73 1997 . More than 100 MYB genes have been reported from Romero et al. 14 3 273 284 1998 Myb genes such at AtmybL2 have been isolated that include only one of the typical two or three tryptophan repeats found in other myb like proteins Kirik Baumlein 183 1 2 109 13 1996 . A myb like gene has been previously isolated from developing xylem and when ectopically expressed in transgenic plants the plants showed accelerated lignification Patzlaff A McInnis S Courtenay A Surman C Newman L J Smith C Bevan M W Mansfield S Whetten R W Sederoff R R Campbell M M. 200336 6 743 54 . A pine myb gene Pt MYB1 may regulate transcription from cis acting AC elements in pine xylem Patzlaff A Newman L J Dubos C Whetten R W Smith C McInnis S Bevan M W Sederoff R R and Campbell M M. 200353 4 597 608 .

DNA binding studies have demonstrated that there are differences but also frequent overlaps in binding specificity among plant MYB proteins in line with the distinct but often related functions that are beginning to be recognized for these proteins. Studies involving the eight putative base contacting residues in MYB DNA binding domains have revealed that at least six are fully conserved in all plant MYB proteins identified to date and the remaining two are conserved in at least 80 of these proteins Martin and Paz Ares 13 2 67 73 1997 . Mutational analysis involving residues that do not contact bases have indicated that the sequence specific binding capacity of MYBs is affected and this may account for some of the differences in the DNA binding specificity between plant MYB proteins Solano et al. 272 5 2889 2895 1997 . This large sized gene family may contribute to the regulatory flexibility underlying the developmental and metabolic plasticity displayed by plants.

NAC NAC proteins are characterized by their conserved N terminal NAC domains that can bind both DNA and other proteins. The NAC domain consists of a twisted beta sheet surrounded by a few helical elements. NAC proteins are involved in developmental processes including formation of the shoot apical meristem floral organs and lateral shoots as well as in plant hormonal control and defense Ernst H A Olsen A N Larsen S AND Lo Leggio L. 20045 3 297 303 . Auxin plays a key role in lateral root formation but the signaling pathway for this process is poorly understood. NAC1 a new member of the NAC family is induced by auxin and mediates auxin signaling to promote lateral root development. NAC1 is a transcription activator consisting of an N terminal conserved NAC domain that binds to DNA and a C terminal activation domain. This factor activates the expression of two downstream auxin responsive genes DBP and AIR3.

NIN like The NIN protein was discovered via a mutant phenotype conferring arrested nodule development. It was demonstrated that the NIN protein is required for the formation of infection threads and nodule primordia. NIN protein has sequence similarity to transcription factors and a predicted DNA binding dimerization domain similar to other plant proteins involved in nitrogen related processes Schauser L Roussis A Stiller J and Stougaard J. 19991999 402 6758 191 5 . The NIN like family of transcription factors is characterized by the RWP RK domain SEQ ID NO 3669 Borisov A Y Madsen L H Tsyganov V E Umehara Y Voroshilova V A Batagov A O Sandal N Mortensen A Schauser L Ellis N Tikhonovich I A and Stougaard J. 2003131 3 1009 17 . An N terminal Octicosapeptide OPR is found in 11 out of 19 of the plant NIN like proteins.

Operably linked combining two or more molecules in such a fashion that in combination they function properly in a plant cell. For instance a promoter is operably linked to a structural gene when the promoter controls transcription of the structural gene.

Phenotype phenotype is a distinguishing feature or characteristic of a plant which may be altered according to the present invention by integrating one or more desired polynucleotides and or screenable selectable markers into the genome of at least one plant cell of a transformed plant. The desired polynucleotide s and or markers may confer a change in the phenotype of a transformed plant by modifying any one of a number of genetic molecular biochemical physiological morphological or agronomic characteristics or properties of the transformed plant cell or plant as a whole. Thus expression of one or more stably integrated desired polynucleotide s in a plant genome may yield a phenotype selected from the group consisting of but not limited to increased drought tolerance enhanced cold and frost tolerance improved vigor enhanced color enhanced health and nutritional characteristics improved storage enhanced yield enhanced salt tolerance enhanced heavy metal tolerance increased disease tolerance increased insect tolerance increased water stress tolerance enhanced sweetness improved vigor improved taste improved texture decreased phosphate content increased germination increased micronutrient uptake improved starch composition and improved flower longevity.

Plant tissue a plant is any of various photosynthetic eukaryotic multicellular organisms of the kingdom Plantae characteristically producing embryos containing chloroplasts and having cellulose cell walls. A part of a plant i.e. a plant tissue may be treated according to the methods of the present invention to produce a transgenic plant. Many suitable plant tissues can be transformed according to the present invention and include but are not limited to somatic embryos pollen leaves stems calli stolons microtubers and shoots. Thus the present invention envisions the transformation of angiosperm and gymnosperm plants such as turfgrass wheat maize rice barley oat sugar beet potato tomato tobacco alfalfa lettuce carrot strawberry cassava sweet potato geranium soybean oak pine fir acacia walnut and palm. According to the present invention plant tissue also encompasses plant cells. Plant cells include suspension cultures callus embryos meristematic regions callus tissue leaves roots shoots gametophytes sporophytes pollen seeds and microspores. Plant tissues may be at various stages of maturity and may be grown in liquid or solid culture or in soil or suitable media in pots greenhouses or fields. A plant tissue also refers to any clone of such a plant seed progeny propagule whether generated sexually or asexually and descendents of any of these such as cuttings or seed. Of particular interest are conifers such as pine fir and spruce monocots such as Kentucky bluegrass creeping bentgrass maize and wheat and dicots such as cotton tomato lettuce tobacco and geranium.

Plant transformation and cell culture broadly refers to the process by which plant cells are genetically modified and transferred to an appropriate plant culture medium for maintenance further growth and or further development. Such methods are well known to the skilled artisan.

POLYCOMB Polycomb group PcG proteins play an important role in developmental and epigenetic regulation of gene expression in fruit fly and mammals. Recent evidence has shown that homologs of PcG proteins are also important for the regulation of plant development. Recent studies in plants have shown that PcG proteins regulate diverse developmental processes and as in animals they affect both homeotic gene expression and cell proliferation Reyes J C and Grossniklaus U. 200314 1 77 84 . PcG proteins have also been shown to repress expression of introduced and endogenous genes in fruit fly. All examples of polycomb based repression likely operate through formation of a repressive chromatin structure Hsieh T F Hakim O Ohad N and Fischer R L. 2003 8 9 439 45 .

Progeny a progeny of the present invention such as the progeny of a transgenic plant is one that is born of begotten by or derived from a plant or the transgenic plant. Thus a progeny plant i.e. an F1 generation plant is an offspring or a descendant of the transgenic plant produced by the inventive methods. A progeny of a transgenic plant may contain in at least one some or all of its cell genomes the desired polynucleotide that was integrated into a cell of the parent transgenic plant by the methods described herein. Thus the desired polynucleotide is transmitted or inherited by the progeny plant. The desired polynucleotide that is so inherited in the progeny plant may reside within a T DNA construct which also is inherited by the progeny plant from its parent. The term progeny as used herein also may be considered to be the offspring or descendants of a group of plants.

Promoter promoter is intended to mean a nucleic acid preferably DNA that binds RNA polymerase and or other transcription regulatory elements. As with any promoter the promoters of the current invention will facilitate or control the transcription of DNA or RNA to generate an mRNA molecule from a nucleic acid molecule that is operably linked to the promoter. As stated earlier the RNA generated may code for a protein or polypeptide or may code for an RNA interfering or antisense molecule.

A plant promoter is a promoter capable of initiating transcription in plant cells whether or not its origin is a plant cell. Exemplary plant promoters include but are not limited to those that are obtained from plants plant viruses and bacteria such as or which comprise genes expressed in plant cells. Examples of promoters under developmental control include promoters that preferentially initiate transcription in certain tissues such as xylem leaves roots or seeds. Such promoters are referred to as tissue preferred promoters. Promoters which initiate transcription only in certain tissues are referred to as tissue specific promoters. A cell type specific promoter primarily drives expression in certain cell types in one or more organs for example vascular cells in roots or leaves. An inducible or repressible promoter is a promoter which is under environmental control. Examples of environmental conditions that may effect transcription by inducible promoters include anaerobic conditions or the presence of light. Tissue specific tissue preferred cell type specific and inducible promoters constitute the class of non constitutive promoters. A constitutive promoter is a promoter which is active under most environmental conditions and in most plant parts.

Polynucleotide is a nucleotide sequence comprising a gene coding sequence or a fragment thereof comprising at least 15 consecutive nucleotides preferably at least 30 consecutive nucleotides and more preferably at least 50 consecutive nucleotides a promoter an intron an enhancer region a polyadenylation site a translation initiation site 5 or 3 untranslated regions a reporter gene a selectable marker or the like. The polynucleotide may comprise single stranded or double stranded DNA or RNA. The polynucleotide may comprise modified bases or a modified backbone. The polynucleotide may be genomic an RNA transcript such as an mRNA or a processed nucleotide sequence such as a cDNA . The polynucleotide may comprise a sequence in either sense or antisense orientations.

An isolated polynucleotide is a polynucleotide sequence that is not in its native state e.g. the polynucleotide is comprised of a nucleotide sequence not found in nature or the polynucleotide is separated from nucleotide sequences with which it typically is in proximity or is next to nucleotide sequences with which it typically is not in proximity.

RAY like RAV like transcription factors are unique to higher plants. RAV stands for Related to ABI3 VP1 and have been placed in the AP2 EREBP transcription factor family because they contain the AP2 domain. However they also contain a domain homologous to the B3 domain. The AP2 domain binds to 5 CAACA 3 and the B3 domain binds to 5 CACCTG 3 . This dual binding is autonomous and achieves high affinity and specificity of binding Kagaya Y Ohmiya K and Hattori T. 199927 2 470 8 . Interestingly some RAV like proteins such as those found in contain only the B3 domain.

Regenerability as used herein refers to the ability of a plant to redifferentiate from a de differentiated tissue.

SBP The SPL gene family represents a group of structurally diverse genes encoding putative transcription factors found apparently only in plants. The distinguishing characteristic of the SPL gene family is the SBP box encoding a conserved protein domain of 76 amino acids in length the SBP domain which is responsible for the interaction with DNA. SBP genes appear to have a function in differentiation of plant organs both in vegetative and floral organs Unte U S Sorensen A M et al. 200315 4 1009 19 Cardon et al. Gene 237 91 104 1999 Moreno et al. Genes Dev. 11 616 628 1997 Cardon et al. Plant J. 12 367 377 1997 . SBP box genes have been isolated from trees and implicated in the regulation of flower development Lannenpaa M Janonen I Holtta Vuori M Gardemeister M Porali I and Sopanen T. 2004120 3 491 500 .

Seed a seed may be regarded as a ripened plant ovule containing an embryo and a propagative part of a plant as a tuber or spore. Seed may be incubated prior to mediated transformation in the dark for instance to facilitate germination. Seed also may be sterilized prior to incubation such as by brief treatment with bleach. The resultant seedling can then be exposed to a desired strain of

Selectable screenable marker a gene that if expressed in plants or plant tissues makes it possible to distinguish them from other plants or plant tissues that do not express that gene. Screening procedures may require assays for expression of proteins encoded by the screenable marker gene. Examples of such markers include the beta glucuronidase GUS gene and the luciferase LUX gene. Examples of selectable markers include the neomycin phosphotransferase NPTII gene encoding kanamycin and geneticin resistance the hygromycin phosphotransferase HPT or APHIV gene encoding resistance to hygromycin acetolactate synthase als genes encoding resistance to sulfonylurea type herbicides genes BAR and or PAT coding for resistance to herbicides which act to inhibit the action of glutamine synthase such as phosphinothricin Liberty or Basta or other similar genes known in the art.

Sequence identity as used herein sequence identity or identity in the context of two nucleic acid or polypeptide sequences includes reference to the residues in the two sequences which are the same when aligned for maximum correspondence over a specified region. When percentage of sequence identity is used in reference to proteins it is recognized that residue positions which are not identical often differ by conservative amino acid substitutions where amino acid residues are substituted for other amino acid residues with similar chemical properties e.g. charge or hydrophobicity and therefore do not change the functional properties of the molecule. Where sequences differ in conservative substitutions the percent sequence identity may be adjusted upwards to correct for the conservative nature of the substitution. Sequences which differ by such conservative substitutions are said to have sequence similarity or similarity. Means for making this adjustment are well known to those of skill in the art. Typically this involves scoring a conservative substitution as a partial rather than a full mismatch thereby increasing the percentage sequence identity. Thus for example where an identical amino acid is given a score of 1 and a non conservative substitution is given a score of zero a conservative substitution is given a score between zero and 1. The scoring of conservative substitutions is calculated e.g. according to the algorithm of Meyers and Miller 4 11 17 1988 e.g. as implemented in the program PC GENE Intelligenetics Mountain View Calif. USA .

As used herein percentage of sequence identity means the value determined by comparing two optimally aligned sequences over a comparison window wherein the portion of the polynucleotide sequence in the comparison window may comprise additions or deletions i.e. gaps as compared to the reference sequence which does not comprise additions or deletions for optimal alignment of the two sequences. The percentage is calculated by determining the number of positions at which the identical nucleic acid base or amino acid residue occurs in both sequences to yield the number of matched positions dividing the number of matched positions by the total number of positions in the window of comparison and multiplying the result by 100 to yield the percentage of sequence identity.

 Sequence identity has an art recognized meaning and can be calculated using published techniques. See CMB Lesk ed. Oxford University Press 1988 B IAGP Smith ed. Academic Press 1993 CAOSD PI Griffin Griffin eds. Humana Press 1994 SAIMB Von Heinje ed. Academic Press 1987 SAP Gribskov Devereux eds. Macmillan Stockton Press 1991 and Carillo Lipton 48 1073 1988 . Methods commonly employed to determine identity or similarity between two sequences include but are not limited to those disclosed in GTHC Bishop ed. Academic Press 1994 and Carillo Lipton supra. Methods to determine identity and similarity are codified in computer programs. Preferred computer program methods to determine identity and similarity between two sequences include but are not limited to the GCG program package Devereux et al. 12 387 1984 BLASTP BLASTN FASTA Atschul et al. 215 403 1990 and FASTDB Brutlag et al. 6 237 1990 .

TCP The TCP family has been termed after its first characterised members TB1 CYC and PCFs . They are expressed in rapidly growing floral primordia. This together with the proposed involvement of cyc and tb1 in influencing meristem growth suggests that many members of the TCP family may affect cell division Cubas P Lauter N Doebley J and Coen E. 199918 2 215 22 .

Transcription factor Transcription factor refers to a polypeptide sequence that regulates the expression of a gene or genes by either directly binding to one or more nucleotide sequences associated with a gene coding sequence or indirectly affecting the activity of another polypeptide s that binds directly to one or more nucleotide sequences associated with a gene coding sequence. A transcription factor may activate up regulate or repress down regulate expression of a gene or genes. A transcription factor may contain a DNA binding domain an activation domain or a domain for protein protein interactions. In the present invention a transcription factor is capable of at least one of 1 binding to a nucleic acid sequence or 2 regulating expression of a gene in a plant. Additionally the inventive polynucleotide sequences and the corresponding polypeptide sequences function as transcription factors in any plant species including angiosperms and gymnosperms.

Transcription and translation terminators The expression DNA constructs of the present invention typically have a transcriptional termination region at the opposite end from the transcription initiation regulatory region. The transcriptional termination region may be selected for stability of the mRNA to enhance expression and or for the addition of polyadenylation tails added to the gene transcription product. Translation of a nascent polypeptide undergoes termination when any of the three chain termination codons enters the A site on the ribosome. Translation termination codons are UAA UAG and UGA.

Transfer DNA T DNA an T DNA is a genetic element that is well known as an element capable of integrating a nucleotide sequence contained within its borders into another genome. In this respect a T DNA is flanked typically by two border sequences. A desired polynucleotide of the present invention and a selectable marker may be positioned between the left border like sequence and the right border like sequence of a T DNA. The desired polynucleotide and selectable marker contained within the T DNA may be operably linked to a variety of different plant specific i.e. native or foreign nucleic acids like promoter and terminator regulatory elements that facilitate its expression i.e. transcription and or translation of the DNA sequence encoded by the desired polynucleotide or selectable marker.

Transformation of plant cells A process by which a nucleic acid is stably inserted into the genome of a plant cell. Transformation may occur under natural or artificial conditions using various methods well known in the art. Transformation may rely on any known method for the insertion of nucleic acid sequences into a prokaryotic or eukaryotic host cell including mediated transformation protocols viral infection whiskers electroporation microinjection polyethylene glycol treatment heat shock lipofection and particle bombardment.

Transgenic plant a transgenic plant of the present invention is one that comprises at least one cell genome in which an exogenous nucleic acid has been stably integrated. According to the present invention a transgenic plant is a plant that comprises only one genetically modified cell and cell genome or is a plant that comprises some genetically modified cells or is a plant in which all of the cells are genetically modified. A transgenic plant of the present invention may be one that comprises expression of the desired polynucleotide i.e. the exogenous nucleic acid in only certain parts of the plant. Thus a transgenic plant may contain only genetically modified cells in certain parts of its structure.

Trihelix GT factors have either one or two trihelix DNA binding domains distantly related to Myb DNA binding domains. Trihelix domains were discovered in proteins that bind to GT elements found in the promoters of many light responsive genes. To date DNA binding proteins characterized by the trihelix motif have been described only in plants and may therefore be involved in plant specific processes. Smalle et al. 95 3318 3322 1998 . Trihelix genes have been shown to be important for light regulated gene expression Nagano Y Inaba T Furuhashi H and Sasaki Y. 2001276 25 22238 43 Wang R Hong G and Han B 2004 324 105 15 . Light responsiveness is important in many plant developmental processes.

TUB TUB and TUBBY are transcription factors originally characterized in mouse where they are important in nervous system function and development Carroll K Gomez C and Shapiro L 20045 1 55 63 . Though similar sequences have been found in plants their function is unknown. 11 Tubby like sequences have been identified in and one of this has been shown to possibly participate in the ABA signaling pathway Lai C P Lee C L Chen P H Wu S H Yang C C and Shaw J F. 2004134 4 1586 97 .

Variant a variant as used herein is understood to mean a nucleotide or amino acid sequence that deviates from the standard or given nucleotide or amino acid sequence of a particular gene or protein. The terms isoform isotype and analog also refer to variant forms of a nucleotide or an amino acid sequence. An amino acid sequence that is altered by the addition removal or substitution of one or more amino acids or a change in nucleotide sequence may be considered a variant sequence. The variant may have conservative changes wherein a substituted amino acid has similar structural or chemical properties e.g. replacement of leucine with isoleucine. A variant may have nonconservative changes e.g. replacement of a glycine with a tryptophan. Analogous minor variations may also include amino acid deletions or insertions or both. Guidance in determining which amino acid residues may be substituted inserted or deleted may be found using computer programs well known in the art such as Vector NTI Suite InforMax MD software. Variant may also refer to a shuffled gene such as those described in Maxygen assigned patents. For instance a variant of the present invention may include variants of sequences and desired polynucleotides that are modified according to the methods and rationale disclosed in U.S. Pat. No. 6 132 970 which is incorporated herein by reference.

Wood composition as used herein refers to trait that can be modified to change the structure appearance or use of wood. While not limiting traits that determine wood composition include cell wall thickness cell length cell density microfibril angle tensile strength tear strength wood color and length and frequency of cell division.

Wood pulp refers to fiber generated from wood having varying degrees of purification. Wood pulp can be used for producing paper paper board and chemical products.

The invention provides methods of obtaining wood wood pulp paper and oil from a plant transformed with a construct of the present invention. Methods for transforming and selecting a transgenic plant are are known in the art. For example pine can be cultured and grown as described in U.S. Patent Application Publication No. 2002 0100083. can be cultured and grown as in for example Rydelius et al. Growing for Pulp and Energy presented at the Mechanization in Short Rotation Intensive Culture Forestry Conference Mobile Ala. 1994. Wood wood pulp paper and oil can be obtained from the plant by any means known in the art.

As noted above the wood and wood pulp obtained in accordance with this invention may demonstrate improved characteristics including but not limited to any one or more of lignin composition lignin structure wood composition cellulose polymerization fiber dimensions ratio of fibers to other plant components plant cell division plant cell development number of cells per unit area cell size cell shape cell wall composition rate of wood formation aesthetic appearance of wood formation of stem defects rate of growth rate of root formation ratio of root to branch vegetative development leaf area index and leaf shape include increased or decreased lignin content increased accessibility of lignin to chemical treatments improved reactivity of lignin increased or decreased cellulose content increased dimensional stability increased tensile strength increased shear strength increased compression strength increased shock resistance increased stiffness increased or decreased hardness decreased spirality decreased shrinkage and differences in weight density and specific gravity.

Phenotype can be assessed by any suitable means. The plants can be evaluated based on their general morphology. Transgenic plants can be observed with the naked eye can be weighed and their height measured. The plant can be examined by isolating individual layers of plant tissue namely phloem and cambium which is further sectioned into meristematic cells early expansion late expansion secondary wall formation and late cell maturation. See e.g. Hertzberg supra. The plants also can be assessed using microscopic analysis or chemical analysis.

Microscopic analysis includes examining cell types stage of development and stain uptake by tissues and cells. Fiber morphology such as fiber wall thickness and microfibril angle of wood pulp fibers can be observed using for example microscopic transmission ellipsometry. See Ye and Sundstr m 80 181 1997 . Wood strength density and grain slope in wet wood and standing trees can be determined by measuring the visible and near infrared spectral data in conjunction with multivariate analysis. See U.S. Patent Application Publication Nos. 2002 0107644 and 2002 0113212. Lumen size can be measured using scanning electron microscopy. Lignin structure and chemical properties can be observed using nuclear magnetic resonance spectroscopy as described in Marita et al. 12939 2001 .

The biochemical characteristic of lignin cellulose carbohydrates and other plant extracts can be evaluated by any standard analytical method known including spectrophotometry fluorescence spectroscopy HPLC mass spectroscopy and tissue staining methods.

WRKY Zn The WRKY SEQ ID NO 3670 proteins are a superfamily of transcription factors with up to 100 representatives in . Family members appear to be involved in the regulation of various physiological programs that are unique to plants including GA signaling pathogen defense senescence and trichome development Zhang Z L Xie Z Zou X Casaretto J Ho T H Shen Q J. 2004134 4 1500 13 Kim C Y and Zhang S. 200438 1 142 51 Robatzek S and Somssich I E. 2002 16 9 1139 49 Johnson C S Kolevski B and Smyth D R. 200214 6 1359 75 . In spite of the strong conservation of their DNA binding domain the overall structures of WRKY SEQ ID NO 3670 proteins are highly divergent and can be categorized into distinct groups which might reflect their different functions.

Zinc finger Zinc finger domains of the type CysHisappear to represent the most abundant DNA binding motif in eukaryotic transcription factors with several thousand being identified to date Berg and Shi 271 5252 1081 1085 1996 . A structural role for zinc in transcription factors was initially proposed in 1983 for the transcription factor IIIA TFIIIA Hanas et al. 258 23 14120 14125 1983 . The CysHisZinc finger domains are characterized by tandem arrays of sequences of C x 2 4 C x 3 LIVMFYWC x 8 H x 3 5 H SEQ ID NO 3674 where X represents a variable amino acid . Structurally the zinc finger consists of two antiparallel a strands followed by an helix Lee et al. 245 4918 635 637 1989 . This structural arrangement allows for the cysteine and histidine side chains to coordinate the zinc with the three other conserved residues forming the hydrophobic core adjacent to the metal coordination unit Berg and Shi 271 5252 1081 1085 1996 . Many proteins possessing a CysHisdomain have been shown to interact with DNA in a sequence specific manner. Crystal structure analysis of the mouse transcription factor Zif268 bound to a specific DNA target indicates that the zinc fingers in the protein DNA complex reside in the major groove of the double helix and interacts with the DNA bases through amino acid side chains referred to as the contact residues Pavletich and Pabo 252 5007 809 817 1991 . The orientations of the zinc finger domains with respect to the DNA are usually identical with each domain contacting a contiguous 3 base pair subsite the majority of which are directed to one strand. There are few interdomain interactions and the DNA recognition by each zinc finger appears to be largely independent of the other domains Berg and Shi 271 5252 1081 1085 1996 .

Plant C2H2 zinc finger transcription factors have been identified as playing important roles in floral organogenesis leaf initiation lateral shoot initiation lateral organ development gametogenesis and seed development. In some cases the same gene can be involved in several different developmental processes such as AtZFP1 Chrispeels H E Oettinger H Janvier N and Tague B W. 200042 2 279 90 Dinneny J R Yadegari R Fischer R L Yanofsky M F and Weigel D. 2004 131 5 1101 10 Weissig H Narisawa S Sikstrom C Olsson P G McCarrey J R Tsonis P A Del Rio Tsonis K and Millan J L. 2003547 1 3 61 8 He Y Gan S. 200454 1 1 9 .

It is understood that the present invention is not limited to the particular methodology protocols vectors and reagents etc. described herein as these may vary. It is also to be understood that the terminology used herein is used for the purpose of describing particular embodiments only and is not intended to limit the scope of the present invention. It must be noted that as used herein and in the appended claims the singular forms a an and the include plural reference unless the context clearly dictates otherwise. Thus for example a reference to a gene is a reference to one or more genes and includes equivalents thereof known to those skilled in the art and so forth. Indeed one skilled in the art can use the methods described herein to express any native gene known presently or subsequently in plant host systems.

The present invention relates to an isolated nucleic molecule comprising a polynucleotide having a sequence selected from the group consisting of any of the polynucleotide sequences of SEQ ID NO 1 494 496 820 1641 1972 3588 3592. The invention also provides functional fragments of the polynucleotide sequences of SEQ ID NO 1 494 496 820 1641 1972 3588 3592. The invention further provides complementary nucleic acids or fragments thereof to any of the polynucleotide sequences of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 as well as a nucleic acid comprising at least 15 contiguous bases which hybridizes to any of the polynucleotide sequences of SEQ ID NO 1 494 496 820 1641 1972 3588 3592.

The present invention relates to an isolated nucleic molecule comprising a polynucleotide having a sequence identity to a sequence selected from the group consisting of any of the polynucleotide sequences set forth in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592. The invention also provides functional fragments of the polynucleotide sequences set forth in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592. The invention further provides complementary nucleic acids or fragments thereof to any of the polynucleotide sequences set forth in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 as well as a nucleic acid comprising at least 15 contiguous bases which hybridizes to any of the polynucleotide sequences recited in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592.

The present invention also relates to an isolated polypeptide sequence comprising a polypeptide having a sequence selected from the group consisting of any of the polypeptide sequences of SEQ ID NO 821 1640 3593 3596. The invention also provides functional fragments of the polypeptide sequences of SEQ ID NO 821 1640 3593 3596.

By isolated nucleic acid molecule s is intended a nucleic acid molecule DNA or RNA which has been removed from its native environment. For example recombinant DNA molecules contained in a DNA construct are considered isolated for the purposes of the present invention. Further examples of isolated DNA molecules include recombinant DNA molecules maintained in heterologous host cells or purified partially or substantially DNA molecules in solution. Isolated RNA molecules include in vitro RNA transcripts of the DNA molecules of the present invention. Isolated nucleic acid molecules according to the present invention further include such molecules produced synthetically.

Nucleic acid molecules of the present invention may be in the form of RNA such as mRNA or in the form of DNA including for instance cDNA and genomic DNA obtained by cloning or produced synthetically. The DNA or RNA may be double stranded or single stranded. Single stranded DNA may be the coding strand also known as the sense strand or it may be the non coding strand also referred to as the anti sense strand.

Unless otherwise indicated all nucleotide sequences determined by sequencing a DNA molecule herein were determined using an automated DNA sequencer such as the Model 373 from Applied Biosystems Inc. and all amino acid sequences of polypeptides encoded by DNA molecules determined herein were predicted by translation of a DNA sequence determined as above. Therefore as is known in the art for any DNA sequence determined by this automated approach any nucleotide sequence determined herein may contain some errors. Nucleotide sequences determined by automation are typically at least about 95 identical more typically at least about 96 to at least about 99.9 identical to the actual nucleotide sequence of the sequenced DNA molecule. The actual sequence can be more precisely determined by other approaches including manual DNA sequencing methods well known in the art. As is also known in the art a single insertion or deletion in a determined nucleotide sequence compared to the actual sequence will cause a frame shift in translation of the nucleotide sequence such that the predicted amino acid sequence encoded by a determined nucleotide sequence may be completely different from the amino acid sequence actually encoded by the sequenced DNA molecule beginning at the point of such an insertion or deletion.

Each nucleotide sequence set forth herein is presented as a sequence of deoxyribonucleotides abbreviated A G C and T . However by nucleotide sequence of a nucleic acid molecule or polynucleotide is intended for a DNA molecule or polynucleotide a sequence of deoxyribonucleotides and for an RNA molecule or polynucleotide the corresponding sequence of ribonucleotides A G C and U where each thymidine deoxynucleotide T in the specified deoxynucleotide sequence in is replaced by the ribonucleotide uridine U .

The present invention is also directed to fragments of the isolated nucleic acid molecules described herein. By a fragment of an isolated DNA molecule having the polynucleotide sequences shown in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 is intended DNA fragments at least 15 nucleotides and more preferably at least 20 nucleotides still more preferably at least 30 nucleotides in length which are useful as diagnostic probes and primers is discussed in more detail below. Of course larger nucleic acid fragments of up to the entire length of the nucleic acid molecules of the present invention are also useful diagnostically as probes according to conventional hybridization techniques or as primers for amplification of a target sequence by the polymerase chain reaction PCR as described for instance in Molecular Cloning A Laboratory Manual 3rd. edition edited by Sambrook Russel. 2001 Cold Spring Harbor Laboratory Press the entire disclosure of which is hereby incorporated herein by reference. By a fragment at least 20 nucleotides in length for example is intended fragments which include 20 or more contiguous bases from the nucleotide sequence of the as shown in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592. The nucleic acids containing the nucleotide sequences listed in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 can be generated using conventional methods of DNA synthesis which will be routine to the skilled artisan. For example restriction endonuclease cleavage or shearing by sonication could easily be used to generate fragments of various sizes. Alternatively the DNA fragments of the present invention could be generated synthetically according to known techniques.

In another aspect the invention provides an isolated nucleic acid molecule comprising a polynucleotide which hybridizes under stringent hybridization conditions to a portion of the polynucleotide in a nucleic acid molecule of the invention described above. By a polynucleotide which hybridizes to a portion of a polynucleotide is intended a polynucleotide either DNA or RNA hybridizing to at least about 15 nucleotides and more preferably at least about 20 nucleotides and still more preferably at least about 30 nucleotides and even more preferably more than 30 nucleotides of the reference polynucleotide. These fragments that hybridize to the reference fragments are useful as diagnostic probes and primers. A probe as used herein is defined as at least about 100 contiguous bases of one of the nucleic acid sequences set forth in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592. For the purpose of the invention two sequences hybridize when they form a double stranded complex in a hybridization solution of 6 SSC 0.5 SDS 5 Denhardt s solution and 100 g of non specific carrier DNA. See Ausubel et al. section 2.9 supplement 27 1994 . Sequences may hybridize at moderate stringency which is defined as a temperature of 60 C. in a hybridization solution of 6 SSC 0.5 SDS 5 Denhardt s solution and 100 g of non specific carrier DNA. For high stringency hybridization the temperature is increased to 68 C. Following the moderate stringency hybridization reaction the nucleotides are washed in a solution of 2 SSC plus 0.05 SDS for five times at room temperature with subsequent washes with 0.1 SSC plus 0.1 SDS at 60 C. for 1 h. For high stringency the wash temperature is increased to 68 C. For the purpose of the invention hybridized nucleotides are those that are detected using 1 ng of a radiolabeled probe having a specific radioactivity of 10 000 cpm ng where the hybridized nucleotides are clearly visible following exposure to X ray film at 70 C. for no more than 72 hours.

The present application is directed to such nucleic acid molecules which are at least 60 65 70 75 80 85 90 95 96 97 98 99 or 100 identical to a nucleic acid sequence described in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592. Preferred however are nucleic acid molecules which are at least 95 96 97 98 99 or 100 identical to the nucleic acid sequence shown in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592. Differences between two nucleic acid sequences may occur at the 5 or 3 terminal positions of the reference nucleotide sequence or anywhere between those terminal positions interspersed either individually among nucleotides in the reference sequence or in one or more contiguous groups within the reference sequence.

As a practical matter whether any particular nucleic acid molecule is at least 95 96 97 98 or 99 identical to a reference nucleotide sequence refers to a comparison made between two molecules using standard algorithms well known in the art and can be determined conventionally using publicly available computer programs such as the BLASTN algorithm. See Altschul et al. 25 3389 3402 1997 .

Polynucleotides may be analyzed using the BLASTX algorithm which compares the six frame conceptual translation products of a nucleotide query sequence both strands against a protein sequence database. The similarity of polypeptide sequences may be examined using the BLASTP algorithm. The BLASTN BLASTX and BLASTP programs are available from the National Center for Biotechnology Information NCBI National Library of Medicine Building 38A Room 8N805 Bethesda Md. 20894 USA. The BLASTN algorithm Version 2.0.4 Feb. 24 1998 and Version 2.0.6 Sep. 16 1998 set to the default parameters described in the documentation and distributed with the algorithm are preferred for use in the determination of polynucleotide variants according to the present invention. The BLASTP algorithm is preferred for use in the determination of polypeptide variants according to the present invention. The computer algorithm FASTA is available from the University of Virginia by contacting David Hudson Assistance Provost for Research University of Virginia PO Box 9025 Charlottesville Va. Version 2.0u4 February 1996 set to the default parameters described in the documentation and distributed with the algorithm may be used in the determination of variants according to the present invention. The use of the FASTA algorithm is described in Pearson and Lipman 85 2444 2448 1988 and Pearson 183 63 98 1990.

The following running parameters are preferred for determination of alignments and similarities using BLASTN that contribute to the E values and percentage identity for polynucleotide sequences Unix running command blastall p blastn d embldb e 10 G0 E0 r 1 v 30 b 30 i queryseq o results the parameters are p Program Name String d Database String e Expectation value E Real G Cost to open a gap zero invokes default behavior Integer E Cost to extend a gap zero invokes default behavior Integer r Reward for a nucleotide match blastn only Integer v Number of one line descriptions V Integer b Number of alignments to show B Integer i Query File File In and o BLAST report Output File File Out Optional.

The following running parameters are preferred for determination of alignments and similarities using BLASTP that contribute to the E values and percentage identity of polypeptide sequences blastall p blastp d swissprotdb e 10 G 0 E 0 v 30 b 30 i queryseq o results wherein the parameters are p Program Name String d Database String e Expectation value E Real G Cost to open a gap zero invokes default behavior Integer E Cost to extend a gap zero invokes default behavior Integer v Number of one line descriptions v Integer b Number of alignments to show b Integer I Query File File In o BLAST report Output File File Out Optional.

The hits to one or more database sequences by a queried sequence produced by BLASTN FASTA BLASTP or a similar algorithm align and identify similar portions of sequences. The hits are arranged in order of the degree of similarity and the length of sequence overlap. Hits to a database sequence generally represent an overlap over only a fraction of the sequence length of the queried sequence.

The BLASTN FASTA and BLASTP algorithms also produce Expect values for alignments. The Expect value E indicates the number of hits one can expect to see over a certain number of contiguous sequences by chance when searching a database of a certain size. The Expect value is used as a significance threshold for determining whether the hit to a database such as the preferred EMBL database indicates true similarity. For example an E value of 0.1 assigned to a polynucleotide hit is interpreted as meaning that in a database of the size of the EMBL database one might expect to see 0.1 matches over the aligned portion of the sequence with a similar score simply by chance. By this criterion the aligned and matched portions of the polynucleotide sequences then have a probability of 90 of being the same. For sequences having an E value of 0.01 or less over aligned and matched portions the probability of finding a match by chance in the EMBL database is 1 or less using the BLASTN or FASTA algorithm.

According to one embodiment variant polynucleotides and polypeptides with reference to each of the polynucleotides and polypeptides of the present invention preferably comprise sequences having the same number or fewer nucleic or amino acids than each of the polynucleotides or polypeptides of the present invention and producing an E value of 0.01 or less when compared to the polynucleotide or polypeptide of the present invention. That is a variant polynucleotide or polypeptide is any sequence that has at least a 99 probability of being the same as the polynucleotide or polypeptide of the present invention measured as having an E value of 0.01 or less using the BLASTN FASTA or BLASTP algorithms set at parameters described above.

Alternatively variant polynucleotides of the present invention hybridize to the polynucleotide sequences recited in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 or complements reverse sequences or reverse complements of those sequences under stringent conditions.

The present invention also encompasses polynucleotides that differ from the disclosed sequences but that as a consequence of the degeneracy of the genetic code encode a polypeptide which is the same as that encoded by a polynucleotide of the present invention. Thus polynucleotides comprising sequences that differ from the polynucleotide sequences recited in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 or complements reverse sequences or reverse complements thereof as a result of conservative substitutions are contemplated by and encompassed within the present invention. Additionally polynucleotides comprising sequences that differ from the polynucleotide sequences recited in of SEQ ID NO 1 494 496 820 1641 1972 3588 3592 or complements reverse complements or reverse sequences thereof as a result of deletions and or insertions totaling less than 10 of the total sequence length are also contemplated by and encompassed within the present invention. Similarly polypeptides comprising sequences that differ from the polypeptide sequences recited in of SEQ ID NO 821 1640 3593 3596 as a result of amino acid substitutions insertions and or deletions totaling less than 10 of the total sequence length are contemplated by and encompassed within the present invention. In certain embodiments variants of the inventive polypeptides possess biological activities that are the same or similar to those of the inventive polypeptides. Such variant polypeptides function as transcription factors and are thus capable of modifying gene expression in a plant. Similarly variant polynucleotides may encode polypeptides that function as transcription factors.

In addition to having a specified percentage identity to an inventive polynucleotide or polypeptide sequence variant polynucleotides and polypeptides preferably have additional structure and or functional features in common with the inventive polynucleotide or polypeptide. Polypeptides having a specified degree of identity to a polypeptide of the present invention share a high degree of similarity in their primary structure and have substantially similar functional properties. In addition to sharing a high degree of similarity in their primary structure to polynucleotides of the present invention polynucleotides having a specified degree of identity to or capable of hybridizing to an inventive polynucleotide preferably have at least one of the following features i they contain an open reading frame or partial open reading frame encoding a polypeptide having substantially the same functional properties as the polypeptide encoded by the inventive polynucleotide or ii they have domains in common.

The polynucleotides of the present invention can be used for specifically directing the expression of polypeptides or proteins in the tissues of plants. The nucleic acids of the present invention can also be used for specifically directing the expression of antisense RNA or RNA involved in RNA interference RNAi such as small interfering RNA siRNA in the tissues of plants which can be useful for inhibiting or completely blocking the expression of targeted genes. As used herein vascular plant tissue refers to xylem phloem or vascular cambium tissue. Preferably the promoters of the invention are either xylem preferred cambium preferred or phloem preferred and direct expression of an operably linked nucleic acid segment in the xylem cambium or phloem respectively. As used herein coding product is intended to mean the ultimate product of the nucleic acid that is operably linked to the promoters. For example a protein or polypeptide is a coding product as well as antisense RNA or siRNA which is the ultimate product of the nucleic acid coding for the antisense RNA. The coding product may also be non translated mRNA. The terms polypeptide and protein are used interchangeably herein. Xylem preferred for example is intended to mean that the nucleic acid molecules of the current invention are more active in the xylem than in any other plant tissue. Most preferably the nucleic acids of the current invention are promoters that are active specifically in the xylem cambium or phloem meaning that the promoters are only active in the xylem cambium or phloem tissue of plants respectively. In other words a xylem specific promoter for example drives the expression of a coding product such that detectable levels of the coding product are expressed only in xylem tissue of a plant. However because of solute transport in plants the coding product that is specifically expressed in the xylem phloem or cambium may be found anywhere in the plant and thus its presence is not necessarily confined to xylem tissue. A vascular preferred promoter on the other hand can be preferentially active is any of the xylem phloem or cambium tissues or in at least two of the three tissue types. A vascular specific promoter is specifically active in any of the xylem phloem or cambium or in at least two of the three.

As used herein promoter is intended to mean a nucleic acid preferably DNA that binds RNA polymerase and or other transcription regulatory elements. As with any promoter the promoters of the current invention will facilitate or control the transcription of DNA or RNA to generate an mRNA molecule from a nucleic acid molecule that is operably linked to the promoter. The RNA may code for a protein or polypeptide or may code for an RNA interfering or antisense molecule. As used herein operably linked is meant to refer to the chemical fusion ligation or synthesis of DNA such that a promoter nucleic acid sequence combination is formed in a proper orientation for the nucleic acid sequence to be transcribed into an RNA segment. The promoters of the current invention may also contain some or all of the 5 untranslated region 5 UTR of the resulting mRNA transcript. On the other hand the promoters of the current invention do not necessarily need to possess any of the 5 UTR.

A promoter as used herein may also include regulatory elements. Conversely a regulatory element may also be separate from a promoter. Regulatory elements confer a number of important characteristics upon a promoter region. Some elements bind transcription factors that enhance the rate of transcription of the operably linked nucleic acid. Other elements bind repressors that inhibit transcription activity. The effect of transcription factors on promoter activity may determine whether the promoter activity is high or low i.e. whether the promoter is strong or weak. 

In a preferred embodiment the promoters described herein are selected from the group consisting of CAD Cinnamyl alcohol Dehydrogenase 4CL 4 coumaric acid coenzyme A ligase SAD Sinapyl Alcohol Dehydrogenase LIM and Pine cellulose synthase.

In another embodiment a constitutive promoter may be used for expressing the inventive polynucleotide sequences. Examples of constitutive plant promoters which may be useful for expressing the TF sequence include the cauliflower mosaic virus CaMV 35S promoter which confers constitutive high level expression in most plant tissues Odel et al. 313 810 1985 the nopaline synthase promoter An et al. 88 547 1988 and the octopine synthase promoter Fromm et al. 1 977 1989 .

In another embodiment a variety of inducible plant gene promoters can be used for expressing the inventive polynucleotide sequences. Inducible promoters regulate gene expression in response to environmental hormonal or chemical signals. Examples of hormone inducible promoters include auxin inducible promoters Baumann et al. 11 323 334 1999 cytokinin inducible promoter Guevara Garcia 38 743 753 1998 and gibberellin responsive promoters Shi et al. 38 1053 1060 1998 . Additionally promoters responsive to heat light wounding pathogen resistance and chemicals such as methyl jasmonate or salicylic acid may be used for expressing the inventive polynucleotide sequences.

The present invention provides DNA constructs comprising the isolated nucleic acid molecules and polypeptide sequences of the present invention. In one embodiment the DNA constructs of the present invention are Ti plasmids derived from

In developing the nucleic acid constructs of this invention the various components of the construct or fragments thereof will normally be inserted into a convenient cloning vector e.g. a plasmid that is capable of replication in a bacterial host e.g. . Numerous vectors exist that have been described in the literature many of which are commercially available. After each cloning the cloning vector with the desired insert may be isolated and subjected to further manipulation such as restriction digestion insertion of new fragments or nucleotides ligation deletion mutation resection etc. to tailor the components of the desired sequence. Once the construct has been completed it may then be transferred to an appropriate vector for further manipulation in accordance with the manner of transformation of the host cell.

A recombinant DNA molecule of the invention typically includes a selectable marker so that transformed cells can be easily identified and selected from non transformed cells. Examples of such markers include but are not limited to a neomycin phosphotransferase nptII gene Potrykus et al. 199 183 188 1985 which confers kanamycin resistance. Cells expressing the nptII gene can be selected using an appropriate antibiotic such as kanamycin or G418. Other commonly used selectable markers include the bar gene which confers bialaphos resistance a mutant EPSP synthase gene Hinchee et al. 6 915 922 1988 which confers glyphosate resistance and a mutant acetolactate synthase gene ALS which confers imidazolinone or sulphonylurea resistance European Patent Application 154 204 1985 .

Additionally vectors may include an origin of replication replicons for a particular host cell. Various prokaryotic replicons are known to those skilled in the art and function to direct autonomous replication and maintenance of a recombinant molecule in a prokaryotic host cell.

In another embodiment pART27 is suitable for use in the present invention. See Gleave A. P. 20 1203 1027 1992 .

The vectors will preferably contain selectable markers for selection in plant cells. Numerous selectable markers for use in selecting transfected plant cells including but not limited to kanamycin glyphosate resistance genes and tetracycline or ampicillin resistance for culturing in and other bacteria.

For secretion of the translated protein into the lumen of the endoplasmic reticulum the periplasmic space or into the extracellular environment appropriate secretion signals may be incorporated into the expressed polypeptide. The signals may be endogenous to the polypeptide or they may be heterologous signals.

In one embodiment a DNA construct of the current invention is designed in a manner such that a polynucleotide sequence described herein is operably linked to a tissue specific promoter. Preferably the polynucleotide encodes a polypeptide involved in cellulose or lignin biosynthesis in plants. Polynucleotides encoding many of the enzymes involved in lignin biosynthesis include but are not limited to cinnamyl alcohol dehydrogenase CAD cinnamate 4 hydroxylase C4H coumarate 3 hydroxylase C3H phenolase PNL O methyl transferase OMT cinnamoyl CoA reductase CCR phenylalanine ammonia lyase PAL 4 coumarate CoA ligase 4CL and peroxidase POX from pine. U.S. Pat. No. 6 204 434. Other enzymes include coniferin glucosidase CBG and caffeic acid 3 O methyltransferase COMT . U.S. Pat. No. 5 451 514 WO 94 23044 and Dharmawardhana et al. 40 365 72 1999 .

In another embodiment the coding sequence operably linked to the promoter may code for a gene product that inhibits the expression or activity of enzymes involved in lignin biosynthesis. For example of particular interest for control of lignin biosynthesis is an antisense gene encoding a 4CL CAD Lim TED2 or a COMT.

In a further embodiment the DNA constructs of the current invention are designed such that the polynucleotide sequences of the current invention are operably linked to DNA or RNA that encodes antisense RNA or interfering RNA which corresponds to genes that code for polypeptides of interest resulting in a decreased expression of targeted gene products. Preferably the gene products targeted for suppression are enzymes involved in lignin biosynthesis. The use of RNAi inhibition of gene expression is described in U.S. Pat. No. 6 506 559 and the use of RNAi to inhibit gene expression in plants is specifically described in WO 99 61631 both of which are herein incorporated by reference.

The use of antisense technology to reduce or inhibit the expression of specific plant genes has been described for example in European Patent Publication No. 271988. Reduction of gene expression led to a change in the phenotype of the plant either at the level of gross visible phenotypic difference for example a lack of lycopene synthesis in the fruit of tomato leading to the production of yellow rather than red fruit or at a more subtle biochemical level for example a change in the amount of polygalacturonase and reduction in depolymerisation of pectins during tomato fruit ripening Smith et. al. 334 724 726 1988 Smith et. al. 14 369 379 1990 . Thus antisense RNA has been demonstrated to be useful in achieving reduction of gene expression in plants.

In one embodiment an inventive polynucleotide sequence is capable of being transcribed inside a plant to yield an antisense RNA transcript is introduced into the plant e.g. into a plant cell. The inventive polynucleotide can be prepared for example by reversing the orientation of a gene sequence with respect to its promoter. Transcription of the exogenous DNA in the plant cell generates an intracellular RNA transcript that is antisense with respect to that gene.

The invention also provides host cells which comprise the DNA constructs of the current invention. As used herein a host cell refers to the cell in which the coding product is ultimately expressed. Accordingly a host cell can be an individual cell a cell culture or cells as part of an organism. The host cell can also be a portion of an embryo endosperm sperm or egg cell or a fertilized egg.

Accordingly the present invention also provides plants or plant cells comprising the DNA constructs of the current invention. Preferably the plants are angiosperms or gymnosperms. The expression construct of the present invention may be used to transform a variety of plants both monocotyledonous e.g. grasses corn grains oat wheat and barley dicotyledonous e.g. tobacco legumes alfalfa oaks maple and Gymnosperms e.g. Scots pine see Aronen 595 1996 white spruce Ellis et al. 11 84 89 1993 and larch Huang et al. 27 201 207 1991 .

In a preferred embodiment the inventive expression vectors are employed to transform woody plants herein defined as a tree or shrub whose stem lives for a number of years and increases in diameter each year by the addition of woody tissue. Preferably the target plant is selected from the group consisting of and pine species most preferably from the group consisting of and its hybrids and . Also preferred the target plant is selected from the group consisting of 

In particular the transgenic plant may be of the species grandis or its hybrids L loblolly pine or or . Beyond the ordinary meaning of plant the term plants is also intended to mean the fruit seeds flower strobilus etc. of the plant. The plant of the current invention may be a direct transfectant meaning that the DNA construct was introduced directly into the plant such as through or the plant may the progeny of a transfected plant. The second or subsequent generation plant may or may not be produced by sexual reproduction i.e. fertilization. Furthermore the plant can be a gametophyte haploid stage or a sporophyte diploid stage .

In one embodiment the present invention provides isolated polynucleotides encoding or partially encoding plant transcription factors that are involved in the regulation of gene expression. The polynucleotides of the present invention were isolated from and but may be isolated from any plant species or synthesized using conventional synthesis techniques.

In specific embodiments isolated polynucleotides of the present invention comprise a sequence selected from the group consisting of sequences identified as SEQ ID NOS 1 494 496 820 1641 1972 3588 3592 complements of the sequences identified as SEQ ID NOS 1 494 496 820 1641 1972 3588 3592 reverse complements of the sequences identified as SEQ ID NOS 1 494 496 820 1641 1972 3588 3592 reverse sequences of the sequences identified as SEQ ID NOS 1 494 496 820 1641 1972 3588 3592 sequences comprising at least a specified number of contiguous residues x mers of any of the above mentioned polynucleotides extended sequences corresponding to any of the above polynucleotides antisense sequences corresponding to any of the above polynucleotides and variants of any of the above polynucleotides as that term is described in this specification.

In another aspect the present invention provides isolated polypeptides encoded by the polynucleotides of SEQ ID NOS 821 1640 3593 3596.

Based on similarity to known sequences from other plant species the isolated polynucleotide sequences were identified as encoding transcription factors as detailed in Tables 1 and 2. The polypeptide sequences were analyzed with publicly available annotation software. EMBL s publicly available InterPro Scan was used for identifying motifs and domains in the present polypeptide sequences. InterPro is a database of protein families domains and functional sites in which identifiable features found in known proteins can be applied to unknown protein sequences. Mulder N. J. et al. 200331 315 318.

As shown in Tables 1 and 2 the polynucleotides of the invention encode transcription factors. These transcription factors can up regulate or down regulate gene expression.

Polypeptides encoded by the polynucleotides of the present invention may be expressed and used in various assays to determine their biological activity. Such polypeptides may be used to raise antibodies to isolate corresponding interacting proteins or other compounds and to quantitatively determine levels of interacting proteins or other compounds.

The present polynucleotides and polypeptides may be introduced into a host plant cell by standard procedures known in the art for introducing recombinant sequences into a target host cell. Such procedures include but are not limited to transfection infection transformation natural uptake electroporation biolistics and . Methods for introducing foreign genes into plants are known in the art and can be used to insert a construct of the invention into a plant host including biological and physical plant transformation protocols. See for example Miki et al. 1993 Procedure for Introducing Foreign DNA Into Plants In Methods in Plant Molecular Biology and Biotechnology Glick and Thompson eds. CRC Press Inc. Boca Raton pages 67 88. The methods chosen vary with the host plant and include chemical transfection methods such as calcium phosphate microorganism mediated gene transfer such as Horsch et al. Science 227 1229 31 1985 electroporation micro injection and biolistic bombardment.

Accordingly the present invention also provides plants or plant cells comprising the polynucleotides or polypeptides of the current invention. In one embodiment the plants are angiosperms or gymnosperms. In another embodiment the plants are selected from and species. In particular the transgenic plant may be of the species and hybrids L loblolly pine or . Beyond the ordinary meaning of plant the term plants is also intended to mean the fruit seeds flower strobilus etc. of the plant. The plant of the current invention may be a direct transfectant meaning that the vector was introduced directly into the plant such as through or the plant may be the progeny of a transfected plant. The progeny may also be obtained by asexual reproduction of a transfected plant. The second or subsequent generation plant may or may not be produced by sexual reproduction i.e. fertilization. Furthermore the plant can be a gametophyte haploid stage or a sporophyte diploid stage .

Methods for transforming tree species are well known in the art. By no means limiting explant refers to plant tissue that is a target for transformation and may include leaf petiole floral and internodal tissues harvested from plants grown in vivo and or in vitro. For example a tree can be transformed as follows. For increased transformation efficiency a tree explant can be harvested and cultured on a pre culture medium before transformation. A pre culture medium as shown in Table 3 is a nutrient medium upon which plant explants are cultured before transformation with and is needed for increasing transformation efficiency and plant regeneration. The pre culture medium comprises an inducer such as acetosyringone. The pre culture medium may optionally comprise plant growth regulators including auxin and cytokinin. Alternatively other pre culture media and time periods of culture may be used.

In the present invention plant explants were pre cultured for four days in the dark on the pre culture medium displayed in Table 3. Woody Plant Medium WPM salts Loyd and McCown 1980 were used in the present pre culture medium however other salt media such as MS medium Murashige and Skoog 1962 or Lepoivre medium may be used. While the present pre culture medium comprises acetosyringone other inducers may be used. Optionally the instant pre culture medium contained both auxin and cytokinin. Other pre culture media and other culture time periods may be used.

Induced culture was prepared by methods known in the art. The induced culture was dripped onto each explant by pipette. Sufficient culture was dripped to ensure that all edges were covered with bacterial solution. Alternatively the explants may be transformed by vacuum infiltration floral dip and other methods of mediated transformation. Following transformation explants covered with culture were placed in the dark for four days of co cultivation. Alternatively the explants may be co cultivated with under light conditions. Additionally the explants may be co cultivated with under light or dark conditions for 2 10 days preferably 4 days. Following co cultivation the explants were transferred to regeneration medium Table 4 with 400 mg l timentin. There is no need to wash explants. Explants were cultured on this medium for four days before transfer to a selection medium. In the present example the selection medium is the regeneration medium supplemented with both timentin and an herbicide selection agent.

Shoot clumps that survive selection are maintained on regeneration medium containing herbicide and timentin and they are transferred every 3 weeks until shoots proliferate and initially elongate. For transformation experiments with a reporter gene such as GUS leaf and stem tissues from the regenerated shoots are stained for GUS expression as soon as the shoots are developed. While any reporter gene may be used such as GFP or luciferase GUS expression was assayed in the present invention by methods known in the art.

GUS staining was performed to monitor the frequency of infection and to ensure that the selected shoots are not escapes or chimeras. Leaf and stem tissues from the regenerated shoots were stained for GUS expression immediately upon shoot development. To determine GUS activity the explants were incubated in a substrate comprising 100 mM phosphate buffer pH 7.0 0.05 dimethyl sulphoxide 0.05 Triton X 100 10 mM EDTA 0.5 mM potassium ferrocyanide and 1.5 mg ml 5 bromo 4 chloro 3 indolyl D glucuronide X gluc . The explants were subjected to 10 minutes of vacuum before an overnight incubation at 37 C. Following overnight incubation GUS foci were counted.

The present invention also provides methods and tools for performing expression profiling of transcription factor polynucleotides. Expression profiling is useful in determining whether polynucleotides are transcribed or translated comparing transcript levels for particular polynucleotide in different tissues genotyping estimating DNA copy number determining identity of descent measuring mRNA decay rates identifying protein binding sites determining subcellular localization of gene products correlating polynucleotide expression to a phenotype or other phenomenon and determining the effect on other polynucleotides of the manipulation of a particular gene. Expression profiling is particularly useful for identifying polynucleotide expression in complex multigenic events. For this reason expression profiling is useful in correlating polynucleotide expression to plant phenotype and formation of plant.

Only a small fraction of the genes of a plant s genome are expressed at a given time in a given tissue sample and all of the expressed genes may not affect the plant phenotype. To identify polynucleotides capable of affecting a phenotype of interest the present invention provides methods and tools for determining for example a polynucleotide expression profile at a given point in plant development and a gene expression profile a given tissue sample. The invention also provides methods and tools for identifying transcription factor polynucleotides whose expression can be manipulated to alter plant phenotype or to alter the biological activity of transcription factor transcription factor polynucleotides transcription and translation products. In support of these methods the invention also provides methods and tools that distinguish expression of different polynucleotides of the same family.

As used herein polynucleotide expression refers to the process of transcription of a DNA sequence into an RNA sequence followed by translation of the RNA into a protein which may or may not undergo post translational processing. Thus the relationship between phenotype and or developmental stage and polynucleotide expression can be observed by detecting quantitatively or qualitatively changes in the level of an RNA or a protein. As used herein the term biological activity includes but is not limited to the activity of a protein gene product including enzyme activity.

The present invention provides oligonucleotides that are useful in these expression profiling methods. Each oligonucleotide is capable of hybridizing under a given set of conditions to a transcription factor polynucleotide or polynucleotide product. In one aspect of the invention a plurality of oligonucleotides is provided wherein each oligonucleotide hybridizes under a given set of conditions to a different cell cycle gene product. Examples of oligonucleotides of the present invention include SEQ ID NOs 2742 3587. Each of the oligos of SEQ ID Nos 2742 3587 hybridizes under standard conditions to a different gene product of one of SEQ ID NOs 1 494 496 820 and 1641 1972 3588 3592. The oligonucleotides of the invention are useful in determining the expression of one or more cell cycle genes in any of the above described methods.

Samples for use in methods of the present invention may be derived from plant tissue. Suitable plant tissues include but are not limited to somatic embryos pollen leaves stems calli stolons microtubers shoots xylem male strolbili pollen cones vascular tissue apical meristem vascular cambium xylem root flower and seed.

According to the present invention plant tissue is used as described previously herein. Plant tissue can be obtained from any of the plants types or species described supra.

In accordance with one aspect of the invention samples can be obtained from plant tissue at different developmental stages from plant tissue at various times of the year e.g. spring versus summer from plant tissues subject to different environmental conditions e.g. variations in light and temperature and or from different types of plant tissue and cells. In accordance with one embodiment plant tissue is obtained during various stages of maturity and during different seasons of the year. For example plant tissue can be collected from stem dividing cells differentiating xylem early developing wood cells differentiated early wood cells and differentiated late wood cells. As another example polynucleotide expression in a sample obtained from a plant with developing wood can be compared to gene expression in a sample obtained from a plant which does not have developing wood.

Differentiating xylem includes samples obtained from compression wood side wood and normal vertical xylem. Methods of obtaining samples for expression profiling from pine and are known. See e.g. Allona et al. 95 9693 8 1998 and Whetton et al. 47 275 91 and Kirst et al. IUFROBC S6.8 June 2003 Umea Sweden .

In one embodiment of the invention polynucleotide expression in one type of tissue is compared to polynucleotide expression in a different type of tissue or to polynucleotide expression in the same type of tissue in a difference stage of development. Polynucleotide expression can also be compared in one type of tissue which is sampled at various times during the year different seasons . For example polynucleotide expression in juvenile secondary xylem can be compared to polynucleotide expression in mature secondary xylem. Similarly polynucleotide expression in cambium can be compared to polynucleotide expression in xylem. Furthermore gene expression in apical meristems can be compared to gene expression in cambium.

In another embodiment of the invention a sample is obtained from a plant having a specific phenotype and polynucleotide expression in that sample is compared to a sample obtained from a plant of the same species that does not have that phenotype. For example a sample can be obtained from a plant exhibiting a fast rate of growth and gene expression can be compared with that of a sample obtained from a plant exhibiting a normal or slow rate of growth. Differentially expressed polynucleotides identified from such a comparison can be correlated with growth rate and therefore useful for manipulating growth rate.

In a further embodiment a sample is obtained from clonally propagated plants. In one embodiment the clonally propagated plants are of the species or . Individual ramets from the same genotype can be sacrificed at different times of year. Thus for any genotype there can be at least two genetically identical trees sacrificed early in the season and late in the season. Each of these trees can be divided into juvenile top to mature bottom samples. Further tissue samples can be divided into for example phloem to xylem in at least 5 layers of peeling. Each of these samples can be evaluated for phenotype and polynucleotide expression.

Where cellular components may interfere with an analytical technique such as a hybridization assay enzyme assay a ligand binding assay or a biological activity assay it may be desirable to isolate the polynucleotide expression products from such cellular components. Polynucleotide expression products including nucleic acid and amino acid gene products can be isolated from cell fragments or lysates by any method known in the art.

Nucleic acids used in accordance with the invention can be prepared by any available method or process or by other processes as they become known in the art. Conventional techniques for isolating nucleic acids are detailed for example in Tijssen LTBMB HWNAP chapter 3 Elsevier Press 1993 Berger and Kimmel 152 1 1987 and GBRL LTTRNA IP Form No. 3786 2000 . Techniques for preparing nucleic acid samples and sequencing polynucleotides from pine and are known. See e.g. Allona et al. supra and Whetton et al. supra and U.S. Application No. 60 476 222.

A suitable nucleic acid sample can contain any type of nucleic acid derived from the transcript of a transcription factor gene or polypeptide i.e. RNA or a subsequence thereof or a nucleic acid for which an mRNA transcribed from a transcription factor gene served as a template. Suitable nucleic acids include cDNA reverse transcribed from a transcript RNA transcribed from that cDNA DNA amplified from the cDNA and RNA transcribed from the amplified DNA. Detection of such products or derived products is indicative of the presence and or abundance of the transcript in the sample. Thus suitable samples include but are not limited to transcripts of a gene or a polynucleotide cDNA reverse transcribed from the transcript cRNA transcribed from the cDNA DNA amplified from the genes and RNA transcribed from amplified DNA. As used herein the category of transcripts includes but is not limited to pre mRNA nascent transcripts transcript processing intermediates and mature mRNAs and degradation products thereof.

It is not necessary to monitor all types of transcripts to practice the invention. For example the expression profiling methods of the invention can be conducted by detecting only one type of transcript such as mature mRNA levels only.

In one aspect of the invention a chromosomal DNA or cDNA library comprising for example fluorescently labeled cDNA synthesized from total cell mRNA is prepared for use in hybridization methods according to recognized methods in the art. See Sambrook et al. supra.

In another aspect of the invention mRNA is amplified using e.g. the MessageAmp kit Ambion . In a further aspect the mRNA is labeled with a detectable label. For example mRNA can be labeled with a fluorescent chromophore such as CyDye Amersham Biosciences .

In some applications it may be desirable to inhibit or destroy RNase that often is present in homogenates or lysates before use in hybridization techniques. Methods of inhibiting or destroying nucleases are well known. In one embodiment of the invention cells or tissues are homogenized in the presence of chaotropic agents to inhibit nuclease. In another embodiment RNase is inhibited or destroyed by heat treatment followed by proteinase treatment.

Protein samples can be obtained by any means known in the art. Protein samples useful in the methods of the invention include crude cell lysates and crude tissue homogenates. Alternatively protein samples can be purified. Various methods of protein purification well known in the art can be found in Marshak et al. SPPC A LCM Cold Spring Harbor Laboratory Press 1996 .

For methods of the invention that comprise detecting a level of polynucleotide expression any method for observing polynucleotide expression can be used without limitation. Such methods include traditional nucleic acid hybridization techniques polymerase chain reaction PCR based methods and protein determination. The invention includes detection methods that use solid support based assay formats as well as those that use solution based assay formats.

Absolute measurements of the expression levels need not be made although they can be made. The invention includes methods comprising comparisons of differences in expression levels between samples. Comparison of expression levels can be done visually or manually or can be automated and done by a machine using for example optical detection means. Subrahmanyam et al. 97 2457 2001 Prashar et al. 303 258 1999 . Hardware and software for analyzing differential expression of genes are available and can be used in practicing the present invention. See e.g. GenStat Software and GeneExpress GX Explorer Training Manual supra Baxevanis Francis Ouellette supra.

In accordance with one embodiment of the invention nucleic acid hybridization techniques are used to observe polynucleotide expression. Exemplary hybridization techniques include Northern blotting Southern blotting solution hybridization and S1 nuclease protection assays.

Nucleic acid hybridization typically involves contacting an oligonucleotide probe and a sample comprising nucleic acids under conditions where the probe can form stable hybrid duplexes with its complementary nucleic acid through complementary base pairing. For example see PCT application WO 99 32660 Berger Kimmel 152 1 1987 . The nucleic acids that do not form hybrid duplexes are then washed away leaving the hybridized nucleic acids to be detected typically through detection of an attached detectable label. The detectable label can be present on the probe or on the nucleic acid sample. In one embodiment the nucleic acids of the sample are detectably labeled polynucleotides representing the mRNA transcripts present in a plant tissue e.g. a cDNA library . Detectable labels are commonly radioactive or fluorescent labels but any label capable of detection can be used. Labels can be incorporated by several approached described for instance in WO 99 32660 supra. In one aspect RNA can be amplified using the MessageAmp kit Ambion with the addition of aminoallyl UTP as well as free UTP. The aminoallyl groups incorporated into the amplified RNA can be reacted with a fluorescent chromophore such as CyDye Amersham Biosciences 

Duplexes of nucleic acids are destabilized by increasing the temperature or decreasing the salt concentration of the buffer containing the nucleic acids. Under low stringency conditions e.g. low temperature and or high salt hybrid duplexes e.g. DNA DNA RNA RNA or RNA DNA will form even where the annealed sequences are not perfectly complementary. Thus specificity of hybridization is reduced at lower stringency. Conversely at higher stringency e.g. higher temperature and or lower salt and or in the presence of destabilizing reagents hybridization tolerates fewer mismatches.

Typically stringent conditions for short probes e.g. 10 to 50 nucleotide bases will be those in which the salt concentration is at least about 0.01 to 1.0 M at pH 7.0 to 8.3 and the temperature is at least about 30 C. Stringent conditions can also be achieved with the addition of destabilizing agents such as formamide.

Under some circumstances it can be desirable to perform hybridization at conditions of low stringency e.g. 6 SSPE T 0.9 M NaCl 60 mM NaHPO pH 7.6 6 mM EDTA 0.005 Triton at 37 C. to ensure hybridization. Subsequent washes can then be performed at higher stringency e.g. 1 SSPE T at 37 C. to eliminate mismatched hybrid duplexes. Successive washes can be performed at increasingly higher stringency e.g. down to as low as 0.25 SSPE T at 37 C. to 50 C. until a desired level of hybridization specificity is obtained.

In general standard conditions for hybridization is a compromise between stringency hybridization specificity and signal intensity. Thus in one embodiment of the invention the hybridized nucleic acids are washed at successively higher stringency conditions and read between each wash. Analysis of the data sets produced in this manner will reveal a wash stringency above which the hybridization pattern is not appreciably altered and which provides adequate signal for the particular oligonucleotide probes of interest. For example the final wash may be selected as that of the highest stringency that produces consistent results and that provides a signal intensity greater than approximately 10 of the background intensity.

Oligonucleotide probes useful in nucleic acid hybridization techniques employed in the present invention are capable of binding to a nucleic acid of complementary sequence through one or more types of chemical bonds usually through complementary base pairing via hydrogen bond formation. A probe can include natural bases i.e. A G U C or T or modified bases 7 deazaguanosine inosine etc. . In addition the nucleotide bases in the probes can be joined by a linkage other than a phosphodiester bond so long as it does not interfere with hybridization. Thus probes can be peptide nucleic acids in which the constituent bases are joined by peptide bonds rather than phosphodiester linkages.

Oligonucleotide probes can be prepared by any means known in the art. Probes useful in the present invention are capable of hybridizing to a nucleotide product of cell cycle genes such as one of SEQ ID NOs 1 235 and 698 717. Probes useful in the invention can be generated using the nucleotide sequences disclosed in SEQ ID NOs 1 235 and 698 717. The invention includes oligonucleotide probes having at least a 2 10 15 20 30 35 40 45 50 55 60 65 70 75 80 85 or 100 nucleotide fragment of a corresponding contiguous sequence of any one of SEQ ID NOs 1 235 and 698 717. The invention includes oligonucleotides of less than 2 1 0.5 0.1 or 0.05 kb in length. In one embodiment the oligonucleotide is 60 nucleotides in length.

Oligonucleotide probes can be designed by any means known in the art. See e.g. Li and Stormo 17 1067 76 2001 . Oligonucleotide probe design can be effected using software. Exemplary software includes ArrayDesigner GeneScan and ProbeSelect. Probes complementary to a defined nucleic acid sequence can be synthesized chemically generated from longer nucleotides using restriction enzymes or can be obtained using techniques such as polymerase chain reaction PCR . PCR methods are well known and are described for example in Innis et al. eds. PCR P A GMA Academic Press Inc. San Diego Calif. 1990 . The probes can be labeled for example with a radioactive biotinylated or fluorescent tag. Optimally the nucleic acids in the sample are labeled and the probes are not labeled. Oligonucleotide probes generated by the above methods can be used in solution or solid support based methods.

The invention includes oligonucleotide probes that hybridize to a product of the coding region or a 3 untranslated region 3 UTR of a transcription factor polynucleotide. In one embodiment the oligonucleotide probe hybridizes to the 3 UTR of any one of SEQ ID Nos 1 494 496 820 1641 1972 3588 3592. The 3 UTR is generally a unique region of the gene even among members of the same family. Therefore the probes capable of hybridizing to a product of the 3 UTR can be useful for differentiating the expression of individual genes within a family where the coding region of the genes likely are highly homologous. This allows for the design of oligonucleotide probes to be used as members of a plurality of oligonucleotides each capable of uniquely binding to a single gene. In another embodiment the oligonucleotide probe comprises any one of SEQ ID NOs 2742 3587. In another embodiment the oligonucleotide probe consists of any one of SEQ ID NOs 2742 3587.

One embodiment of the invention employs two or more oligonucleotide probes in combination to detect a level of expression of one or more transcription factor polynucleotides such as the genes of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592. In one aspect of this embodiment the level of expression of two or more different polynucleotide is detected. The two or more polynucleotide may be from the same or different transcription factor gene families discussed above. Each of the two or more oligonucleotides may hybridize to a different one of the polynucleotides.

One embodiment of the invention employs two or more oligonucleotide probes each of which specifically hybridize to a polynucleotide derived from the transcript of a polynucleotide provided by SEQ ID NOs 1 494 496 820 1641 1972 3588 3592. Another embodiment employs two or more oligonucleotide probes at least one of which comprises a nucleic acid sequence of SEQ ID NOs 1973 2304 3593 3666. Another embodiment employs two or more oligonucleotide probes at least one of which consists of SEQ ID Nos 1973 2304 3593 3666.

The oligonucleotide probes may comprise from about 5 to about 60 or from about 5 to about 500 nucleotide bases such as from about 60 to about 100 nucleotide bases including from about 15 to about 60 nucleotide bases.

One embodiment of the invention uses solid support based oligonucleotide hybridization methods to detect gene expression. Solid support based methods suitable for practicing the present invention are widely known and are described for example in PCT application WO 95 11755 Huber et al. 299 24 2001 Meiyanto et al. 31 406 2001 Relogio et al. 30 e51 2002 . Any solid surface to which oligonucleotides can be bound covalently or non covalently can be used. Such solid supports include filters polyvinyl chloride dishes silicon or glass based chips etc.

One embodiment uses oligonucleotide arrays i.e. microarrays which can be used to simultaneously observe the expression of a number of polynucleotides genes or gene products. Oligonucleotide arrays comprise two or more oligonucleotide probes provided on a solid support wherein each probe occupies a unique location on the support. The location of each probe may be predetermined such that detection of a detectable signal at a given location is indicative of hybridization to an oligonucleotide probe of a known identity. Each predetermined location can contain more than one molecule of a probe but each molecule within the predetermined location has an identical sequence. Such predetermined locations are termed features. There can be for example from 2 10 100 1 000 2 000 or 5 000 or more of such features on a single solid support. In one embodiment each oligonucleotide is located at a unique position on an array at least 2 at least 3 at least 4 at least 5 at least 6 or at least 10 times.

Oligonucleotide probe arrays for detecting gene expression can be made and used according to conventional techniques described for example in Lockhart et al. 14 1675 1996 McGall et al. 93 13555 1996 and Hughes et al. 19 342 2001 . A variety of oligonucleotide array designs is suitable for the practice of this invention.

In one embodiment the one or more oligonucleotides include a plurality of oligonucleotides that each hybridize to a different polynucleotide expressed in a particular tissue type. For example the tissue can be developing wood.

In one embodiment a nucleic acid sample obtained from a plant can be amplified and optionally labeled with a detectable label. Any method of nucleic acid amplification and any detectable label suitable for such purpose can be used. For example amplification reactions can be performed using e.g. Ambion s MessageAmp which creates antisense RNA or aRNA complementary in nucleic acid sequence to the RNA extracted from the sample tissue . The RNA can optionally be labeled using CyDye fluorescent labels. During the amplification step aaUTP is incorporated into the resulting aRNA. The CyDye fluorescent labels are coupled to the aaUTPs in a non enzymatic reaction. Subsequent to the amplification and labeling steps labeled amplified antisense RNAs are precipitated and washed with appropriate buffer and then assayed for purity. For example purity can be assay using a Nanoprop spectrophotometer. The nucleic acid sample is then contacted with an oligonucleotide array having attached to a solid substrate a microarray slide oligonucleotide sample probes capable of hybridizing to nucleic acids of interest which may be present in the sample. The step of contacting is performed under conditions where hybridization can occur between the nucleic acids of interest and the oligonucleotide probes present on the array. The array is then washed to remove non specifically bound nucleic acids and the signals from the labeled molecules that remain hybridized to oligonucleotide probes on the solid substrate are detected. The step of detection can be accomplished using any method appropriate to the type of label used. For example the step of detecting can accomplished using a laser scanner and detector. For example on can use and Axon scanner which optionally uses GenePix Pro software to analyze the position of the signal on the microarray slide.

Oligonucleotide probes used in the methods of the present invention including microarray techniques can be generated using PCR. PCR primers used in generating the probes are chosen for example based on the sequences of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592 to result in amplification of unique fragments of the transcription factor polynucleotides i.e. fragments that hybridize to only one polynucleotide of any one of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592 under standard hybridization conditions . Computer programs are useful in the design of primers with the required specificity and optimal hybridization properties. For example Li and Stormo supra at 1075 discuss a method of probe selection using ProbeSelect which selects an optimum oligonucleotide probe based on the entire gene sequence as well as other gene sequences to be probed at the same time.

In one embodiment oligonucleotide control probes also are used. Exemplary control probes can fall into at least one of three categories referred to herein as 1 normalization controls 2 expression level controls and 3 negative controls. In microarray methods one or more of these control probes may be provided on the array with the inventive transcription factor related oligonucleotides.

Normalization controls correct for dye biases tissue biases dust slide irregularities malformed slide spots etc. Normalization controls are oligonucleotide or other nucleic acid probes that are complementary to labeled reference oligonucleotides or other nucleic acid sequences that are added to the nucleic acid sample to be screened. The signals obtained from the normalization controls after hybridization provide a control for variations in hybridization conditions label intensity reading efficiency and other factors that can cause the signal of a perfect hybridization to vary between arrays. In one embodiment signals e.g. fluorescence intensity or radioactivity read from all other probes used in the method are divided by the signal from the control probes thereby normalizing the measurements.

Virtually any probe can serve as a normalization control. Hybridization efficiency varies however with base composition and probe length. Preferred normalization probes are selected to reflect the average length of the other probes being used but they also can be selected to cover a range of lengths. Further the normalization control s can be selected to reflect the average base composition of the other probes being used. In one embodiment only one or a few normalization probes are used and they are selected such that they hybridize well i.e. without forming secondary structures and do not match any test probes. In one embodiment the normalization controls are mammalian genes.

Expression level controls probes hybridize specifically with constitutively expressed genes present in the biological sample. Virtually any constitutively expressed gene provides a suitable target for expression level control probes. Typically expression level control probes have sequences complementary to subsequences of constitutively expressed housekeeping genes including but not limited to certain photosynthesis genes.

 Negative control probes are not complementary to any of the test oligonucleotides i.e. the inventive transcription factor related oligonucleotides normalization controls or expression controls. In one embodiment the negative control is a mammalian gene which is not complementary to any other sequence in the sample.

The terms background and background signal intensity refer to hybridization signals resulting from non specific binding or other interactions between the labeled target nucleic acids i.e. mRNA present in the biological sample and components of the oligonucleotide array. Background signals also can be produced by intrinsic fluorescence of the array components themselves.

A single background signal can be calculated for the entire array or a different background signal can be calculated for each target nucleic acid. In a one embodiment background is calculated as the average hybridization signal intensity for the lowest 5 to 10 percent of the oligonucleotide probes being used or where a different background signal is calculated for each target gene for the lowest 5 to 10 percent of the probes for each gene. Where the oligonucleotide probes corresponding to a particular cell cycle gene hybridize well and hence appear to bind specifically to a target sequence they should not be used in a background signal calculation. Alternatively background can be calculated as the average hybridization signal intensity produced by hybridization to probes that are not complementary to any sequence found in the sample e.g. probes directed to nucleic acids of the opposite sense or to genes not found in the sample . In microarray methods background can be calculated as the average signal intensity produced by regions of the array that lack any oligonucleotides probes at all.

In another embodiment PCR based methods are used to detect polynucleotide expression. These methods include reverse transcriptase mediated polymerase chain reaction RT PCR including real time and endpoint quantitative reverse transcriptase mediated polymerase chain reaction Q RTPCR . These methods are well known in the art. For example methods of quantitative PCR can be carried out using kits and methods that are commercially available from for example Applied BioSystems and Stratagene . See also Kochanowski QPCR P Humana Press 1999 Innis et al. supra. Vandesompele et al. 3 RESEARCH0034 2002 Stein 59 1235 2002 .

Polynucleotide expression can also be observed in solution using Q RTPCR. Q RTPCR relies on detection of a fluorescent signal produced proportionally during amplification of a PCR product. See Innis et al. supra. Like the traditional PCR method this technique employs PCR oligonucleotide primers typically 15 30 bases long that hybridize to opposite strands and regions flanking the DNA region of interest. Additionally a probe e.g. TaqMan Applied Biosystems is designed to hybridize to the target sequence between the forward and reverse primers traditionally used in the PCR technique. The probe is labeled at the 5 end with a reporter fluorophore such as 6 carboxyfluorescein 6 FAM and a quencher fluorophore like 6 carboxy tetramethyl rhodamine TAMRA . As long as the probe is intact fluorescent energy transfer occurs which results in the absorbance of the fluorescence emission of the reporter fluorophore by the quenching fluorophore. As Taq polymerase extends the primer however the intrinsic 5 to 3 nuclease activity of Taq degrades the probe releasing the reporter fluorophore. The increase in the fluorescence signal detected during the amplification cycle is proportional to the amount of product generated in each cycle.

The forward and reverse amplification primers and internal hybridization probe is designed to hybridize specifically and uniquely with one nucleotide derived from the transcript of a target gene. In one embodiment the selection criteria for primer and probe sequences incorporates constraints regarding nucleotide content and size to accommodate TagMan requirements.

SYBR Green can be used as a probe less Q RTPCR alternative to the Taqman type assay discussed above. ABI P7900 SDSUGAB chap. 1 8 App. A F. 2002 .

A device measures changes in fluorescence emission intensity during PCR amplification. The measurement is done in real time that is as the amplification product accumulates in the reaction. Other methods can be used to measure changes in fluorescence resulting from probe digestion. For example fluorescence polarization can distinguish between large and small molecules based on molecular tumbling see U.S. Pat. No. 5 593 867 .

Proteins can be observed by any means known in the art including immunological methods enzyme assays and protein array proteomics techniques.

Measurement of the translational state can be performed according to several protein methods. For example whole genome monitoring of protein the proteome can be carried out by constructing a microarray in which binding sites comprise immobilized preferably monoclonal antibodies specific to a plurality of proteins having an amino acid sequence of any of SEQ ID Nos 821 1640 1973 2304 3593 3666 or proteins encoded by the polynucleotides of SEQ ID NOs 1 494 496 820 1641 1972 3588 3592 or conservative variants thereof. See Wildt et al. 18 989 2000 . Methods for making polyclonal and monoclonal antibodies are well known as described for instance in Harlow Lane A A LM Cold Spring Harbor Laboratory Press 1988 .

Alternatively proteins can be separated by two dimensional gel electrophoresis systems. Two dimensional gel electrophoresis is well known in the art and typically involves isoelectric focusing along a first dimension followed by SDS PAGE electrophoresis along a second dimension. See e.g. Hames et al. GEP A PA IRL Press 1990 . The resulting electropherograms can be analyzed by numerous techniques including mass spectrometric techniques western blotting and immunoblot analysis using polyclonal and monoclonal antibodies and internal and N terminal micro sequencing.

As discussed above the invention provides methods and tools to correlate transcription factor polynucleotide expression to plant phenotype. Transcription factor polynucleotide expression may be be examined in a plant having a phenotype of interest and compared to a plant that does not have the phenotype or has a different phenotype. Such a phenotype includes but is not limited to increased drought tolerance herbicide resistance reduced or increased height reduced or increased branching enhanced cold and frost tolerance improved vigor enhanced color enhanced health and nutritional characteristics improved storage enhanced yield enhanced salt tolerance enhanced resistance of the wood to decay enhanced resistance to fungal diseases altered attractiveness to insect pests enhanced heavy metal tolerance increased disease tolerance increased insect tolerance increased water stress tolerance enhanced sweetness improved texture decreased phosphate content increased germination increased micronutrient uptake improved starch composition improved flower longevity production of novel resins and production of novel proteins or peptides.

In another embodiment the phenotype includes one or more of the following traits propensity to form reaction wood a reduced period of juvenility an increased period of juvenility self abscising branches accelerated reproductive development or delayed reproductive development.

In a further embodiment the phenotype that is differs in the plants compares includes one or more of the following lignin quality lignin structure wood composition wood appearance wood density wood strength wood stiffness cellulose polymerization fiber dimensions lumen size other plant components plant cell division plant cell development number of cells per unit area cell size cell shape cell wall composition rate of wood formation aesthetic appearance of wood formation of stem defects average microfibril angle width of the S2 cell wall layer rate of growth rate of root formation ratio of root to branch vegetative development leaf area index and leaf shape.

In a further embodiment polynucleotide expression can be correlated to a given point in the cell cycle a given point in plant development and in a given tissue sample. Plant tissue can be examined at different stages of the cell cycle from plant tissue at different developmental stages from plant tissue at various times of the year e.g. spring versus summer from plant tissues subject to different environmental conditions e.g. variations in light and temperature and or from different types of plant tissue and cells. In accordance with one embodiment plant tissue is obtained during various stages of maturity and during different seasons of the year. For example plant tissue can be collected from stem dividing cells differentiating xylem early developing wood cells differentiated spring wood cells differentiated summer wood cells.

It will be apparent to those skilled in the art that various modifications and variations can be made in the methods and compositions of the present invention without departing from the spirit or scope of the invention. Thus it is intended that the present invention cover the modifications and variations of this invention provided they come within the scope of the appended claims and their equivalents.

The following examples are given to illustrate the present invention. It should be understood however that the invention is not to be limited to the specific conditions or details described in these examples. Throughout the specification any and all references to a publicly available document including a U.S. patent are specifically incorporated by reference in their entirety.

Total RNA was extracted from the plant tissue using the protocol of Chang et al. 11 113 116 1993 . mRNA was isolated from the total RNA preparation using either a Poly A Quik mRNA Isolation Kit Stratagene La Jolla Calif. or Dynal Beads Oligo dT SEQ ID NO 3675 Dynal Skogen Norway . A cDNA expression library was constructed from the purified mRNA by reverse transcriptase synthesis followed by insertion of the resulting cDNA clones in Lambda ZAP using a ZAP Express cDNA Synthesis Kit Stratagene according to the manufacturer s protocol. The resulting cDNAs were packaged using a Gigapack II Packaging Extract Stratagene using an aliquot 1 5 l from the 5 l ligation reaction dependent upon the library. Mass excision of the library was done using XL1 Blue MRF cells and XLOLR cells Stratagene with ExAssist helper phage Stratagene . The excised phagemids were diluted with NZY broth Gibco BRL Gaithersburg Md. and plated out onto LB kanamycin agar plates containing X gal and isopropylthio beta galactoside IPTG .

Of the colonies plated and selected for DNA miniprep 99 contained an insert suitable for sequencing. Positive colonies were cultured in NZY broth with kanamycin and cDNA was purified by means of alkaline lysis and polyethylene glycol PEG precipitation. Agarose gel at 1 was used to screen sequencing templates for chromosomal contamination. Dye primer sequences were prepared using a Turbo Catalyst 800 machine Perkin Elmer Applied Biosystems Division Foster City Calif. according to the manufacturer s protocol.

DNA sequence for positive clones was obtained using a Perkin Elmer Applied Biosystems Division Prism 377 sequencer. cDNA clones were sequenced first from the 5 end and in some cases also from the 3 end. For some clones internal sequence was obtained using either Exonuclease III deletion analysis yielding a library of differentially sized subclones in pBK CMV or by direct sequencing using gene specific primers designed to identified regions of the gene of interest.

The determined cDNA sequences were compared with known sequences in the EMBL database using the computer algorithms FASTA and or BLASTN. Multiple alignments of redundant sequences were used to build reliable consensus sequences. The determined cDNA sequences are provided in SEQ ID NOS 1 494 496 820 1641 1972 3588 3592. Based on similarity to known sequences from other plant species the isolated polynucleotide sequences were identified as encoding transcription factors as detailed in Tables 1 and 2. The predicted polypeptide sequences corresponding to the polynucleotide sequences of SEQ ID NOS 1 820 are provided in SEQ ID NOS 821 1640 3593 3596.

DNA sequence for positive clones was obtained using forward and reverse primers on a Perkin Elmer Applied Biosystems Division Prism 377 sequencer and the determined sequences were compared to known sequences in the database as described above.

Based on similarity to known sequences from other plant species the isolated polynucleotide sequences were identified as encoding transcription factors as displayed above in Table 1. The predicted polypeptide sequences corresponding to the polynucleotide sequences of SEQ ID NOS 1 494 496 820 1641 1972 3588 3592 are provided in SEQ ID NOS 821 1640 3593 3596.

To identify additional sequence 5 or 3 of a partial cDNA sequence in a cDNA library 5 and 3 rapid amplification of cDNA ends RACE was performed. using the SMART RACE cDNA amplification kit Clontech Laboratories Palo Alto Calif. . Generally the method entailed first isolating poly A mRNA performing first and second strand cDNA synthesis to generate double stranded cDNA blunting cDNA ends and then ligating of the SMART RACE. Adaptor to the cDNA to form a library of adaptor ligated ds cDNA. Gene specific primers were designed to be used along with adaptor specific primers for both 5 and 3 RACE reactions. Using 5 and 3 RACE reactions 5 and 3 RACE fragments were obtained sequenced and cloned. The process may be repeated until 5 and 3 ends of the full length gene were identified. A full length cDNA may generated by PCR using primers specific to 5 and 3 ends of the gene by end to end PCR.

For example to amplify the missing 5 region of a gene from first strand cDNA a primer was designed 5 3 from the opposite strand of the template sequence and from the region between 100 200 bp of the template sequence. A successful amplification should give an overlap of 100 bp of DNA sequence between the 5 end of the template and PCR product.

RNA was extracted from four pine tissues namely seedling xylem phloem and structural root using the Concert Reagent Protocol Invitrogen Carlsbad Calif. and standard isolation and extraction procedures. The resulting RNA was then treated with DNase using 10 U ul DNase I Roche Diagnostics Basel Switzerland . For 100 g of RNA 9 l 10 DNase buffer Invitrogen Carlsbad Calif. 10 l of Roche DNase 1 and 90 l of Rnase free water was used. The RNA was then incubated at room temperature for 15 minutes and 1 10 volume 25 mM EDTA is added. A RNeasy mini kit Qiagen Venlo The Netherlands was used for RNA clean up according to manufacturer s protocol.

To synthesize cDNA the extracted RNA from xylem phloem seedling and root was used and the SMART RACE cDNA amplification kit Clontech Laboratories Inc Palo Alto Calif. was followed according to manufacturer s protocol. For the RACE PCR the cDNA from the four tissue types was combined. The master mix for PCR was created by combining equal volumes of cDNA from xylem phloem root and seedling tissues. PCR reactions were performed in 96 well PCR plates with 1 l of primer from primer dilution plate 10 mM to corresponding well positions. 49 l of master mix is aliquoted into the PCR plate with primers. Thermal cycling commenced on a GeneAmp 9700 Applied Biosystems Foster City Calif. at the following parameters 

cDNA was separated on an agarose gel following standard procedures. Gel fragments were excised and eluted from the gel by using the Qiagen 96 well Gel Elution kit following the manufacturer s instructions.

PCR products were ligated into pGEMT easy Promega Madison Wis. in a 96 well plate overnight according to the following specifications 60 80 ng of DNA 5 l 2 rapid ligation buffer 0.5 l pGEMT easy vector 0.1 l DNA ligase filled to 10 l with water and incubated overnight.

Each clone was transformed into following standard procedures and DNA was extracted from 12 clones picked by following standard protocols. DNA extraction and the DNA quality was verified on an 1 agarose gel. The presence of the correct size insert in each of the clones was determined by restriction digests using the restriction endonuclease EcoRI and gel electrophoresis following standard laboratory procedures.

Following the identification and cloning of a promoter by the procedure outlined above the promoter is operably linked with a reporter gene to determine those tissue types in which the promoter is active. To this end a construct containing the promoter first is transformed into by electroporation. Briefly 40 l of diluted AgL 1 competent cells are placed on ice and are contacted with about 10 ng of pART27 vector containing the promoter sequence. Electroporation is conducted at the following parameters 

Following electroporation 400 l of YEP liquid media is added and the cells are allowed to recover for one hour at room temperature. Cells then are centrifuged at 6000 rpm for 3 min and are resuspended in 50 l YEP. Cell samples are spread over the surface of a YEP Kan50 Rif50 plate sealed with parafilm and incubated at 29 C. for 2 days for colony growth.

Wild type cv. Columbia 0 plants are then transformed with containing constructs of interest by floral dip infiltration. Briefly cultures are centrifuged at 8600 rcf for 10 min at 20 C. and are resuspended to an optical density of 0.7 0.8. Plants are dipped into an infiltration solution containing the for 5 sec. Plants are drained of excess solution and placed under grow lights in ambient conditions. After 24 hrs the plants are misted and maintained for seed production. Tseeds are surface sterilized in 5 commercial bleach solution and plated on MS media containing Kanamycin 50 mg l and Timentin 250 mg l to select for putative transformants.

Successfully transformed plants are then assayed for the expression of the operably linked reporter gene. Leaf stem root and floral regions are immersed in a staining solution 50 mM NaPO pH 7.2 0.5 Triton X 100 1 mM X Glucuronide cycloheximide salt Ducheffa . A vacuum is applied twice for 5 min to infiltrate the tissue with the staining solution. The tissue is then left shaking overnight at 37 C. for color development. Tissues are checked at three or four time points to check stain development and if samples show early development a piece of tissue is destained in 70 ethanol. This tissue is then examined for GUS expression using a light microscope and photographed.

Primary and secondary pair leaves from the seedlings were harvested from 8 punnets. Leaves were sterilized in 500 ml of 0.175 sodium hypochlorite solution for 10 minutes. Leaves were then rinsed twice in 500 ml of sterile water. Using 20 30 leaves at a time leaves were ground in mortar and pestle and 25 30 ml of FK medium. Cells were filtered through the 40 m nylon mesh. A total of 90 ml of mesophyll cells were obtained in this fashion. Cells were pelleted by centrifuging at 200 g for 2 minutes at 20 C. The pellet was washed once more using equal volume of FK medium. Then the pellet was split in to two equal halves and one half was washed in 45 ml of FK medium and the other in 45 ml of FKH medium. The pellets were re suspended in 60 ml of FK medium and 60 ml of FKH medium respectively. They were cultured in the dark in two 6 well plates on the rotary shaker set at 120 rpm.

Sterile primary leaves 6 8 in number were cut in slivers of 1 mm and placed in 15 ml of cell wall digesting enzyme mix 1 Cellulase Onozuka R 10 and 0.2 pectolyase Y23 in Protoplast isolation buffer . Mesophyll cells cultured in FK medium 40 ml or FKH medium 40 ml were pelleted by centrifuging at 200 g for 2 minutes at 20 C. Each pellet was re suspended in 20 ml of sterile Protoplast isolation buffer containing 200 mg Cellulase Onozuka R 10 and 40 mg Pectolyase Y23. The protoplasts were isolated by incubating the cell suspensions in CellStar culture plates for 2 4 hours on a rotary shaker set at 70 rpm at 23 C. Protoplasts were pelleted by centrifuging the contents of the plates at 200 g for 2 minutes. Each of the pellets was re suspended in 20 ml of 24 sucrose solution.

Transfected protoplast suspensions prepared as described above were individually harvested by adding 9.5 ml of W5 solution mixing the contents of each tube and centrifuging at 70 g for 10 minutes at 20 C. The bulk of the supernatant was removed by decanting and the protoplasts volume was brought up to 900 l. From this 300 L of protoplasts were aliquoted into 5 ml polystyrene round bottom tubes re suspended in a volume of 500 l W5 medium and set aside for analysis of fluorescent reporter gene expression and cell viability. The protoplasts and the remaining solution were transferred to individual microtubes and pelleted by centrifugation at 420 g for 2 minutes at 20 C. The protoplast pellet was assayed for GUS reporter gene expression as described by Jefferson R. A. 19875 387. GUS MUG assays were performed using a Wallac Turku Finland Victor1420 Multilabel Counter. Umbelliferone was detected using a 355 nm excitation filter and a 460 nm emission filter for 1 second.

Cell based assays are used for screening the function of promoters and transcription factors from the Pine and databases. The assays are used to identify transcription factors that are active during tracheary differentiation and lignification by determining whether a promoter responds to trans acting factors in plant cells that are either induced in tracheary element TE forming cells endogenous factors and or introduced by transformation transient assay after introduction of plasmid DNA into the cells . The assay comprises the isolation of mesophyll cells and their culture either in TE inducing or maintenance medium. See Examples 6 9. Control promoterless constructs or constructs comprising promoters that are active during TE formation linked to reporter genes are introduced into the cells or protoplasts prepared from the cells. As described above in Example 8 the transfected protoplasts are harvested by centrifugation and assayed for viability and transgene expression. To correct for experimental variation that may arise from differences in transfection the protoplasts are co transfected with a transfection marker which is also detected by flow cytometry. This system uses fluorescence analysis technologies to capture the data and informatics software to analyze the results. In this way the impact of an introduced gene or gene product can be monitored. Transcriptional repression or activation of a vascular preferred Pine or promoter can be attributed to the candidate transcription factor gene and may be used to support sequence data.

Four color flow cytometry can also be used in the TE assay. The pine ubiquitin promoter is constitutively expressed at a high level in plants therefore pine ubquitin expressing DsRedExpress can be used as the co transfection marker in the cell based assay system. In protoplasts high level of expression of the pine ubiquitin promoter is also found. Pine ubquitin DsRedExpress can be used as a co transfection marker for transfections that involve the two color green and red TE assay.

To correlate a transcription factor with transcriptional regulation of a wood quality trait a cell based assay is performed in two steps. First the transcription factor is tested for activity in combination with promoters individually fused to a fluorescent reporter gene. The promoters used include COMT 306 bp Homeobox 8 691 bp Pine Ubiquitin 2 kb Intron 4CL CAD The COMT and Homeobox 8 promoters are vascular specific whereas the Pine Ubiquitin promoter described in U.S. Pat. No. 6 380 459 B1 is a constitutive promoter. A transcription factor that generates a hit e.g. upregulated transcription or downregulated transcription against one of these two promoters will be screened further.

A transcription factor that either activates or represses transcription from one of the above mentioned promoters will be used for screening vascular specific activity of other candidate vascular specific or vascular preferred promoters. Table 5 lists some candidate vascular specific promoters that can be used with the inventive transcription factors. those skilled in the art will recognize that any vascular preferred promoters may be suitable for use in this assay .

The pFOR293 vector contains a gene encoding a protein similar to the Ethylene Response Element AP2 class of proteins SEQ ID NO 474 which was isolated from a cDNA library made from developing xylem fibers. As described in Example 10 above transcription factor pFOR293 was assayed for the ability to either activate or repress transcription from the Pine Ubiquitin 2 kb Intron promoter.

Following the protocols described above the transcription factor construct pFOR293 was tested for its ability to activate the Pine Ubiquitin promoter. Specifically protoplasts were co transfected with two of three disparate constructs. Test protoplasts were transfected with the effector construct pFOR293 a positive control pFOR263 or pFOR147 or a negative control pART9. Constructs of the pFOR series are based on the primary cloning vector pART7 which has an expression cartridge comprised of the CaMV 35S promoter a multiple cloning site and the transcriptional termination region of the octopine synthase gene Gleave 20 1203 1207 1992 . The vector pFOR293 contains the Ethylene Response Element AP2 transcription factor in its multiple cloning site while the vectors pFOR147 and pFOR263 contain a positive control transcription factor. The protoplasts were also transfected with a second plasmid containing the gene encoding green fluorescence protein EGFP driven by the Ubiquitin promoter or deletion fragments of the promoter.

Control protoplasts were transfected with a plasmid vector pART9 a modified version of pART7 containing the EGFP gene in its multiple cloning site but with the CaMV 35S promoter removed from the expression cartridge. Accordingly pART9 is a promoterless construct which does not express any gene and is used as a control because of its similarity in length and composition to pFOR vectors.

Table 6 below shows the mean fluorescence intensity MFI of EGFP from protoplasts transfected with constructs harboring i the Pine Ubiquitin promoter fused to EGFP Clontech and ii a selection of tree Transcription Factors. In this screen a positive control for transcriptional activation was used pFOR147 and a negative control construct was used pART9 referred to as No Transcription Factor .

Following the protocols described above the transcription factor pFOR293 was tested for its ability to activate the COMT promoter a vascular preferred promoter. Table 7 shows the mean fluorescence intensity MFI of EGFP from protoplasts transfected with constructs harbouring i the COMT promoter fused to EGFP Clontech and ii a selection of tree TFs.

Due to the low level of COMT promoter activity repression is more clearly visualised by determining the percentage cells express a co transfection marker and a reporter gene. Table 8 below presents the results of protoplasts that were transfected with constructs harboring i the COMT promoter fused to EGFP Clontech and ii a selection of tree TFs.

As described in the above examples the transcription factor pFOR293 was assayed for its ability to activate the Homeobox8 promoter. Table 9 below shows the mean fluorescence intensity MFI of EGFP from protoplasts transfected with constructs harbouring i the Homeobox 8 promoter fused to EGFP Clontech and ii a selection of tree TFs. In this screen a positive control for transcriptional activation was used pFOR263 and a negative control construct was used No Transcription Factor .

Due to the low level of Homeobox 8 promoter activity repression is more clearly visualised by determining the percentage cells expressing a co transfection marker and a reporter gene. Table 10 below shows protoplasts transfected with constructs harbouring i the Homeobox 8 promoter fused to EGFP Clontech and ii a selection of tree TFs.

As described in Examples 1 and 2 transcription factors are isolated and identified from and cDNA libraries. Following isolation a transcription factor is cloned in a DNA construct having a promoter operably linked to a reporter gene wherein the transcription factor regulates the activity of the promoter reporter gene fusion. While any promoter can be used this example uses vascular preferred promoters. Based on the expression level of a reporter gene a transcription factor can be identified as a transcriptional activator or repressor relative to a wild type construct that does not contain a transcription factor sequence. A transcriptional activator causes an increase in reporter gene expression relative to a wild type construct. A transcriptional repressor causes a decrease in reporter gene expression relative to a wild type construct. Tables 11 12 displays transcription factors having transcriptional activity with a specific promoter. Transcriptional activity is quantified as a value between one and five wherein a value of five represents an upward maximum of transcriptional activity. Repression is quantified as a value between negative one and negative five wherein a value of negative five represents an upward maximum of transcriptional activity.

The pFOR113 vector contains a gene SEQ ID NO 137 that encodes a protein similar to the DOF class of zinc finger proteins and that was isolated from a cDNA library made from xylem fibres.

As described in Example 11 above transcription factor construct pFOR113 was assayed for the ability to either activate or repress transcription from the Pine Ubiquitin 2 kb Intron promoter.

As shown in the mean fluorescence intensity MFI of EGFP from protoplasts transfected with constructs harbouring i the Pine Ubiquitin promoter fused to EGFP Clontech and ii a selection of tree TFs.

It should be noted that the effects of pFOR113 were more subtle than that observed for pFOR293 so the following experiment was next performed. As described in Example 12 above the gene contained in pFOR113 also contained in the multiple cloning site of pFOR369 was tested with the promoter construct of COMT. As shown in protoplasts from transfected with constructs harbouring i the COMT promoter fused to EGFP Clontech and ii a selection of tree TFs. were assayed for mean fluorescence intensity MFI of EGFP

The inventive polynucleotide sequences can be used to regulate gene expression in any plant including both angiosperms and gymnosperms. The overexpression of a key gene in the lignin biosynthesis pathway may be desirable under circumstances where increased mechanical strength of wood or resistance to pathogens and pests is desired. For example the construct pFOR 434 comprises the Homeobox 8 promoter which is strongly activated by a MYB transcription factor SEQ ID NO 315 . Accordingly the Homeobox 8 promoter can be operably linked to a gene in the lignin biosynthesis pathway. In the presence of the MYB transcription factor expression of the resulting gene product derived from Homeobox 8 promoter lignin biosynthesis gene construct should be higher than the expression product of the same construct in the absence of the MYB transcription factor.

For example ferulate 5 hydroxylase F5H is a key enzyme in the biosynthesis of syringyl lignin monomers. Franke et al. 22 3 223 224 2000 . A DNA vector can be constructed having a MYB transcription factor sequence SEQ ID NO X that binds to the Homeobox 8 promoter operably linked to a sense nucleotide sequence encoding 5FH. As described in Example 4 any plant can be transformed with this DNA construct.

5FH activity can be assayed in a transformed plant according to Franke et al. and references cited therein. Lignin content and composition may be assayed by the methods of Baucher et al. 112 1479 90 1996 .

Under some circumstances it may be desirable to reduce expression of a lignin biosynthesis gene in a plant. For example cinnamyl alcohol dehydrogenase CAD catalyzes the last step of lignin monomer synthesis and has provided a target for successful antisense mediated down regulation of lignin in transgenic plants using other promoters. See Yahiaoui et al. 49 295 306 1998 and references cited therein. Expression of an RNAi molecule corresponding to a portion of CAD results in a decrease in enzyme activity and a corresponding increase in the proportion of cinnamyl aldehydes in the lignin of a transgenic plant.

By use of the inventive polynucleotides of the present invention a DNA vector can be constructed having a transcription factor sequence that binds to a vascular specific promoter operably linked to a gene encoding an RNA interference RNAi molecule corresponding to a portion of the coding region of CAD. For example a DNA vector may have a WRKY SEQ ID NO 3670 transcription factor SEQ ID NO 446 that binds to an COMT promoter operably linked to a nucleotide sequence encoding a CAD RNAi molecule. Any plant may be transformed with the DNA vector as described in Example 4. Transgenic plants may be assayed for CAD activity using the method of Wyrambik et al. 59 9 15 1975 as adapted by Baucher et al. 112 1479 90 1996 . Lignin content and composition can be measured as set forth by Baucher 1996 .

Plant growth and the growth of particular organs such as the roots can be regulated using an inventive polynucleotide sequence. In this example was transformed with a construct comprising a gene encoding an HMG box transcription factor SEQ ID NO 229 driven by the cauliflower mosaic virus promoter. This DNA construct was inserted into a strain of capable of transforming and transformation was carried out using the floral dip method as described above. Seeds were collected and germinated under aseptic conditions in gelled nutrient media. The morphology of the seedlings was compared with that of wild type seedlings and seedlings that had arisen from transformation with pART9. Compared to these control seedlings an unusual growth phenotype was noticed in 16 out of 20 seedlings arising from the transformation with the construct comprising SEQ ID NO 229. In particular 15 of the 20 seedlings examined showed more branching of the primary root and 5 of the 20 seedlings examined showed unusually vigorous growth which may be associated with greater root surface area and nutrient absorption. Such a phenotype is potentially valuable in transgenic plants including forest tree species and plants grown in low nutrient or arid conditions.

Based on the data from the above examples plant gene expression can be regulated using an inventive polynucleotide sequence e.g. a DNA construct having one of the inventive polynucleotide sequences in a sense or antisense orientation. For example can be transformed with a gene encoding a SBP transcription factor. As shown in Table 12 a SBP transcription factor can be used to activate gene expression.

DNA constructs comprising a nucleic acid sequence encoding a SBP transcription factor including the coding region of the SBP transcription factor of SEQ ID NO 781 inserted into the multiple cloning site of pART7 to create pFOR462 are inserted into a strain of capable of transforming a plant. Additionally the pFOR462 construct comprises the Euc COMT promoter operably linked to a desired gene. A desired gene includes any gene involved in wood development. Genes involved in wood development include genes that generate denser cells and or longer cells control microfibril angle and extend cell division. Plants may be transformed as described above in Example 5.

As shown in the above examples plant gene expression can be regulated using an inventive polynucleotide sequence. Vectors can be constructed with one of the inventive polynucleotide sequences in a sense or antisense orientation. For example can be transformed with a gene encoding a C2C2 GATA transcription factor. As shown in Example 14 the construct comprising a transcription factor can be used to repress gene expression.

DNA constructs comprising a nucleic acid sequence encoding a transcription factor including the coding region of the transcription factor of SEQ ID NO 142 are inserted into a strain of capable of transforming a plant. Additionally the construct comprises the Euc COMT promoter operably linked to a desired gene. A desired gene includes any gene involved in wood development. Genes involved in wood development include genes that generate denser cells and or longer cells control microfibril angle.

In silico gene expression can be used to determine the membership of the consensi EST libraries. For each library a consensus is determined from the number of ESTs in any tissue class divided by the total number of ESTs in a class multiplied by 1000. These values provide a normalized value that is not biased by the extent of sequencing from a library. Several libraries were sampled for a consensus value including reproductive bud reproductive bud vegetative fruit leaf phloem cambium xylem root stem sap vegetative whole plant libraries.

A number of the inventive transcription factor sequences exhibit vascular preferred expression more than 50 of the hits by these sequences if the databases were searched at random would be in libraries made from developing vascular tissue and thus are likely to be involved in wood related developmental processes. Many of the remaining transcription factors exhibit vegetative preferred expression suggesting expression in leaf developmental processes and photosynthesis related processes or root preferred expression suggesting expression in root developmental processes and water and nutrient uptake.

As described in Example 1 transcription factors were isolated from cDNA libraries. Following isolation and identification a polynucleotide sequence encoding a transcription factor can be cloned in a DNA construct and transformed into a recipient host cell. Any plant including angiosperms and gymnosperms may be transformed with one of the inventive polynucleotides. As outlined in Example 5 wild type cv. Columbia 0 plants are transformed with containing a DNA construct having a promoter operably linked to a polynucleotide sequence encoding a transcription factor. Shown below in Table 14 expression of a transcription factor in a host plant cell can modify a plant phenotype.

As described in Example 1 transcription factors were isolated from cDNA libraries. Following isolation and identification a polynucleotide sequence encoding a transcription factor can be cloned in a DNA construct and transformed into a recipient host cell. Any plant including angiosperms and gymnosperms may be transformed with one of the inventive polynucleotides. As outlined in Example 5 wild type cv. Columbia 0 plants are transformed with containing a DNA construct having a promoter operably linked to a polynucleotide sequence encoding a transcription factor. Shown below in Table 15 expression of a transcription factor in a host plant cell can modify a plant phenotype.

During the production of cDNA libraries the original transcripts or their DNA counterparts may have features that prevent them from coding for functional proteins. There may be insertions deletions base substitutions or unspliced or improperly spliced introns. If such features exist it is often possible to identify them so that they can be changed. The consensus sequence pinusRadiata001720 equivalent to EST number 011005PRAA002374HT will be used as an example although similar curation can be performed on any other sequences that have homology to sequences in the public databases.

After determination of the DNA sequence BLAST analysis showed that it was related to the gene SHORT VEGETATIVE PHASE or SVP gene At2g22540 on the publicly available genome sequence . However instead of coding for an approximately 240 amino acid polypeptide pinusRadiata001720 was predicted to code for a product of only 157 amino acid residues. This suggested an error in the DNA sequence. To identify where the genuine coding region might be the DNA sequence from position 600 to the end of the EST was translated in each of the three reading frames and the predicted sequences were aligned with the SVP amino acid sequence. It was found that the DNA segment from position 924 to 1170 coded for a sequence with similarity to the carboxyl terminus of SVP. Therefore it appears that an unspliced intron is present in the EST.

Unspliced introns are a relatively minor issue with regard to use of a cloned sequence for overexpression of the gene of interest. The RNA resulting from transcription of the cDNA can be expected to undergo normal processing to remove the intron. Antisense and RNAi constructs are also expected to function to suppress the gene of interest. On other occasions it may be desirable to identify the precise limits of the intron so that it can be removed. When the sequence in question has a published sequence that is highly similar it may be possible to find the intron by aligning the two sequences and identifying the locations where the sequence identity falls off aided by the knowledge that introns start with the sequence GT and end with the sequence AG.

For pinusRadiata001720 there is plausible similarity to SVP up to position 552 where there is a possible EXON intron junction CAAAA gtggg SEQ ID NO 3677 . A second candidate junction is at position 582 where the sequence is TACCA gtacc SEQ ID NO 3678 . In both these cases the putative intron junction falls between the second and third nucleotides of a codon. The likely site of the 3 end of the intron is position 925 where the predicted intron EXON junction is acaag TGGAA SEQ ID NO 3679 and again falls between the second and third bases of a codon. When there is some doubt about the site of the intron because highly similar sequences are not available as is the case for pinusRadiata001720 the intron location can be verified experimentally. For example DNA oligomers can be synthesized flanking the region where the suspected intron is located. For pinusRadiata001720 a sense primer could be synthesized based on sequence in the region from position 400 to 500 and an antisense primer could be synthesized based on sequence in the region from position 1000 to 1100. RNA from radiata pine is isolated and used as a template to make cDNA using reverse transcriptase. The selected primers are then used in a PCR reaction to amplify the correctly spliced DNA segment predicted size of approximately 350 bp smaller than the corresponding segment of the original consensus from the population of cDNAs. The amplified segment is then subjected to sequence analysis and compared to the pinusRadiata001720 sequence to identify the differences.

The same procedure can be used when an alternate splicing event partial intron remaining or partial loss of an exon is suspected. When an EST has a small change such as insertion or deletion of a small number of bases computer analysis of the EST sequence can still indicate its location when a translation product of the wrong size is predicted or if there is an obvious frameshift. Verification of the true sequence is done by synthesis of primers production of new cDNA and PCR amplification as described above.

Example 25 illustrates how transcription factor polynucleotides important for wood development in can be determined and how oligonucleotides which uniquely bind to those genes can be designed and synthesized for use on a microarray.

Open pollinated trees of approximately 16 years of age are selected from plantation grown sites in the United States for loblolly pine and in New Zealand for radiata pine. Trees are felled during the spring and summer seasons to compare the expression of genes associated with these different developmental stages of wood formation. Trees are felled individually and trunk sections are removed from the bottom area approximately one to two meters from the base and within one to two meters below the live crown. The section removed from the basal end of the trunk contains mature wood. The section removed from below the live crown contains juvenile wood. Samples collected during the spring season are termed earlywood or springwood while samples collected during the summer season are considered latewood or summerwood Larson et al. . FPL GTR 129. Madison Wis. U.S. Department of Agriculture Forest Service Forest Products Laboratory. 42p. .

Tissues are isolated from the trunk sections such that phloem cambium developing xylem and maturing xylem are removed. These tissues are collected only from the current year s growth ring. Upon tissue removal in each case the material is immediately plunged into liquid nitrogen to preserve the nucleic acids and other components. The bark is peeled from the section and phloem tissue removed from the inner face of the bark by scraping with a razor blade. Cambium tissue is isolated from the outer face of the peeled section by gentle scraping of the surface. Developing xylem and lignifying xylem are isolated by sequentially performing more vigorous scraping of the remaining tissue. Tissues are transferred from liquid nitrogen into containers for long term storage at 70 C. until RNA extraction and subsequent analysis is performed.

cDNA clones containing sequences that hybridize to the genes showing wood preferred expression are selected from cDNA libraries using techniques well known in the art of molecular biology. Using the sequence information oligonucleotides are designed such that each oligonucleotide is specific for only one cDNA sequence in the library. The oligonucleotide sequences are provided in TABLE 19. 60 mer oligonucleotide probes are designed using the method of Li and Stormo supra or using software such as ArrayDesigner GeneScan and ProbeSelect.

Oligonucleotides are then synthesized in situ described in Hughes et al. 19 324 2002 or as described in Kane et al. 28 4552 2000 . The oligonucleotides can also be synthesized by Sigma Aldrich Saint Louis Mo. USA . Oligonucleotides are volume normalized to a final concentration of 100 M redissolved in 100 l DNAse RNAse free water. All oligonucleotides are desalted and cartridge purified by HPLC in accordance with the quality control specifications of the vendor.

Synthesized 60 mer oligonucleotides are spotted in duplicate onto Corning UltraGAPS gamma amino propyl silane aminosilane coated glass microscope slides Corning N.Y. using Amersham s Lucidea Array spotter Amersham Biosciences NY USA . The position of each oligonucleotide on the slide is known.

All pre and post arraying steps are performed according to specifications described in the US Provisional Patent Application for Methods and Kits for Labeling and Hybridizing cDNA for Microarray Analysis 60 390 142 filed Jun. 20 2002 .

Example 26 illustrates how cell cycle genes important for wood development in can be determined and how oligonucleotides which uniquely bind to those genes can be designed and synthesized for use on a microarray.

RNA is isolated and ESTs generated as described in Sterky et al. supra. The nucleic acid sequences of ESTs derived from samples containing developing wood are compared with nucleic acid sequences of genes known to be involved in the plant cell cycle. ESTs from samples that do not contain developing wood are also compared with sequences of genes known to be involved in the plant cell cycle. An in silico hybridization analysis is performed as described in for example Audic and Claverie 7 986 1997 . Sequences from among the known cell cycle genes that show hybridization in silico to ESTs made from samples containing developing wood but do not hybridize to ESTs from samples not containing developing wood are selected for further examination.

cDNA clones containing sequences that hybridize to the genes showing wood preferred expression are selected from cDNA libraries using techniques well known in the art of molecular biology. Using the sequence information oligonucleotides are designed such that each oligonucleotide is specific for only one cDNA sequence in the library. The oligonucleotide sequences are provided in TABLE 20. 60 mer oligonucleotide probes are designed using the method of Li and Stormo supra or using software such as ArrayDesigner GeneScan and ProbeSelect.

The oligonucleotides are then synthesized in situ described in Hughes et al. 19 324 2002 or as described in Kane et al. 28 4552 2000 and affixed to an activated glass slide Sigma Genosus The Woodlands Tex. using a 5 amino linker. The position of each oligonucleotide on the slide is known.

Example 27 illustrates how to detect expression of transcription factor genes which are important in wood formation using an oligonucleotide microarray prepared as in Example 28. This is an example of a balanced incomplete block designed experiment carried out using aRNA samples prepared from mature phase phloem P cambium C expanding xylem found in a layer below the cambium X1 and differentiating lignifying xylem cells found deeper in the same growth ring X2 . In this example cell cycle gene expression is compared among the four samples namely P C X1 and X2.

RNA is isolated according to the protocol of Chang et al. 11 113 1993 . DNA is removed using DNase I Invitrogen Carlsbad Calif. according to the manufacturer s recommendations. The integrity of the RNA samples is determined using the Agilent 2100 Bioanalyzer Agilent Technologies USA .

10 g of total RNA from each tissue is reverse transcribed into cDNA. All laboratory steps associated with cDNA synthesis and labelling were performed according to specifications described in the U.S. patent application for Methods and Kits for Labeling and Hybridizing cDNA for Microarray Analysis supra .

In the case of phloem tissue it can be difficult to extract sufficient amounts of total RNA for normal labelling procedures. Total RNA is extracted and treated as previously described and 100 ng of total RNA is amplified using the Ovation Nanosample RNA Amplification system from NuGEN CA USA . Similar amplification kits such as those manufactured by Ambion may alternatively be used. The amplified RNA is reverse transcribed into cDNA and labelled as described above.

Hybridization and stringency washes are performed using the protocol as described in the U.S. patent application for Methods and Kits for Labeling and Hybridizing cDNA for Microarray Analysis supra at 42 C. The arrays slides are scanned using a ScanArray 4000 Microarray Analysis System GSI Lumonics Ottawa ON Canada . Raw non normalized intensity values are generated using QUANTARRAY software GSI Lumonics Ottawa ON Canada .

A fully balanced incomplete block experimental design Kerr M. K. and Churchill G. A. 2001 Statistical design and the analysis of gene expression microarray data. Gen. Res. 123 123 128 is used in order to design an array experiment that would allow maximum statistical inferences from analyzed data.

Gene expression data is analyzed using the SAS Microarray Solution software package The SAS Institute Cary N.C. USA . Resulting data was then visualized using JMP The SAS Institute Cary N.C. USA .

Analysis done for this experiment is an ANOVA approach with mixed model specification. Wolfinger et al. 8 625 2001 . Assessing gene significance from cDNA microarray expression data via mixed models. Two steps of linear mixed models are applied. The first one normalization model is applied for global normalization at slide level. The second one gene model is applied for doing rigorous statistical inference on each gene. Both models are stated in Models 1 and 2 . log 1 2 

Yrepresents the intensity of the sspot in the lslide with the kdye applying the jtreatment for the icell line. D S and DSrepresent the mean effect of the jtreatment in the icell line the kdye effect the lslide random effect and the random interaction effect of the kdye in the lslide. is the stochastic error term. Rrepresents the residual of the ggene from model 1 . D S and Drepresent the similar roles as D S and DSexcept they are specific for the ggene. SSrepresent the spot by slide random effect for the ggene. represent the stochastic error term. All random terms are assumed to be normal distributed and mutually independent within each model.

According to the analysis described above certain cDNAs some of which were shown in Table 16 below are found to be differentially expressed.

The involvement of these specific genes in wood development is inferred through the association of the up regulation or down regulation of genes to the particular stages of wood development. Both the spatial continuum of wood development across a section phloem cambium developing xylem maturing xylem at a particular season and tree trunk position and the relationships of season and tree trunk position are considered when making associations of gene expression to the relevance in wood development.

Example 28 demonstrates how one can correlate transcription factor gene expression with agronomically important wood phenotypes such as density stiffness strength distance between branches and spiral grain.

Mature clonally propagated pine trees are selected from among the progeny of known parent trees for superior growth characteristics and resistance to important fungal diseases. The bark is removed from a tangential section and the trees are examined for average wood density in the fifth annual ring at breast height stiffness and strength of the wood and spiral grain. The trees are also characterized by their height mean distance between major branches crown size and forking.

To obtain seedling families that are segregating for major genes that affect density stiffness strength distance between branches spiral grain and other characteristics that may be linked to any of the genes affecting these characteristics trees lacking common parents are chosen for specific crosses on the criterion that they exhibit the widest variation from each other with respect to the density stiffness strength distance between branches and spiral grain criteria. Thus pollen from a plus tree exhibiting high density low mean distance between major branches and high spiral grain is used to pollinate cones from the unrelated plus tree among the selections exhibiting the lowest density highest mean distance between major branches and lowest spiral grain. It is useful to note that plus trees are crossed such that pollen from a plus tree exhibiting high density are used to pollinate developing cones from another plus tree exhibiting high density for example and pollen from a tree exhibiting low mean distance between major branches would be used to pollinate developing cones from another plus tree exhibiting low mean distance between major branches.

Seeds are collected from these controlled pollinations and grown such that the parental identity is maintained for each seed and used for vegetative propagation such that each genotype is represented by multiple ramets. Vegetative propagation is accomplished using micropropagation hedging or fascicle cuttings. Some ramets of each genotype are stored while vegetative propagules of each genotype are grown to sufficient size for establishment of a field planting. The genotypes are arrayed in a replicated design and grown under field conditions where the daily temperature and rainfall are measured and recorded.

The trees are measured at various ages to determine the expression and segregation of density stiffness strength distance between branches spiral grain and any other observable characteristics that may be linked to any of the genes affecting these characteristics. Samples are harvested for characterization of cellulose content lignin content cellulose microfibril angle density strength stiffness tracheid morphology ring width and the like. Samples are also examined for gene expression as described in Example 4. Ramets of each genotype are compared to ramets of the same genotype at different ages to establish age age correlations for these characteristics.

Example 29 demonstrates how the stage of plant development and responses to environmental conditions such as light and season can be correlated to transcription factor gene expression using microarrays prepared as in Example 25. In particular the changes in gene expression associated with wood density are examined.

Trees of three different clonally propagated hybrid genotypes are grown on a site with a weather station that measures daily temperatures and rainfall. During the spring and subsequent summer genetically identical ramets of the three different genotypes are first photographed with north south orientation marks using photography at sufficient resolution to show bark characteristics of juvenile and mature portions of the plant and then felled as in Example 35. The age of the trees is determined by planting records and confirmed by a count of the annual rings. In each of these trees mature wood is defined as the outermost rings of the tree below breast height and juvenile wood as the innermost rings of the tree above breast height. Each tree is accordingly sectored as follows 

Tissue is harvested from the plant trunk as well as from juvenile and mature form leaves. Samples are prepared simultaneously for phenotype analysis including plant morphology and biochemical characteristics and gene expression analysis. The height and diameter of the tree at the point from which each sector was taken is recorded and a soil sample from the base of the tree is taken for chemical assay. Samples prepared for gene expression analysis are weighed and placed into liquid nitrogen for subsequent preparation of RNA samples for use in the microarray experiment. The tissues are denoted as follows 

Thin slices in tangential and radial sections from each of the sectors of the trunk are fixed as described in Ruzin Plant Microtechnique and Microscopy Oxford University Press Inc. New York N.Y. 1999 for anatomical examination and confirmation of wood developmental stage. Microfibril angle is examined at the different developmental stages of the wood for example juvenile transition and mature phases of wood. Other characteristics examined are the ratio of fibers to vessel elements and ray tissue in each sector. Additionally the samples are examined for characteristics that change between juvenile and mature wood and between spring wood and summer wood such as fiber morphology lumen size and width of the S2 thickest cell wall layer. Samples are further examined for measurements of density in the fifth ring and determination of modulus of elasticity using techniques well known to those skilled in the art of wood assays. See e.g. Wang et al. Non destructive Evaluations of Trees Experimental Techniques pp. 28 30 2000 .

For biochemical analysis 50 grams from each of the harvest samples are freeze dried and analyzed using biochemical assays well known to those skilled in the art of plant biochemistry for quantities of simple sugars amino acids lipids other extractives lignin and cellulose. See e.g. Pettersen Schwandt J. Wood Chem. Technol. 11 495 1991 .

In the present example the phenotypes chosen for comparison are high density wood average density wood and low density wood. Nucleic acid samples are prepared as described in Example 3 from trees harvested in the spring and summer. Gene expression profiling by hybridization and data analysis is performed as described in Examples 3 and 4.

Using similar techniques and clonally propagated individuals one can examine cell cycle gene expression as it is related to other complex wood characteristics such as strength stiffness and spirality.

Example 30 demonstrates the ability of the oligonucleotide probes of the invention to distinguish between highly homologous members of a family of transcription factor genes. Hybridization to a particular oligonucleotide on the array identifies a unique HMG box gene that is expressed more strongly in a genotype having a higher density wood than in observed in other genotypes examined. The HMG box gene is also expressed more strongly in mature wood than in juvenile wood and more strongly in summer wood than in spring wood. This gene is not found to be expressed at high levels either in leaves or buds.

The gene expression pattern is confirmed by RT PCR. This gene the putative density related gene is used for in situ hybridization of fixed radial sections. The density related HMG box gene hybridizes most strongly to the vascular cambium in regions of the stem where the xylem is comprised primarily of fibers with few vessel elements and few xylem ray cells.

These results suggest that the HMG box gene product functions in radial cell division which occurs in the cambium and results in diameter growth rather than in axial cell division such as may be important in the apex or leaves. Such a gene would be difficult to identify by cDNA microarrays or other traditional hybridization means because the highly conserved regions present in the gene would result in confusing it with genes encoding enzymes having similar catalytic functions but acting in axial or radial divisions. Furthermore from the sequence similarity based annotation suggesting a function of this gene product in cell division and the observation of this microarray hybridization pattern confirmed by RT PCR and in silico hybridization this gene product functions specifically in developing secondary xylem to guide the cell division patterns of fibers such that higher expression of this gene results in greater fiber production relative to vessel element or ray production. The fiber content is correlated with a principal components analysis PCA variable that accounts for at least 10 of the variation in basic density.

Example 31 describes microarrays for identifying gene expression differences that contribute to the phenotypic characteristics that are important in commercial wood namely wood appearance stiffness strength density fiber dimensions coarseness cellulose and lignin content extractives content and the like.

As in Examples 25 26 woody trees of genera that produce commercially important wood products in this case and are felled from various sites and at various times of year for the collection and isolation of RNA from developing xylem cambium phloem leaves buds roots and other tissues. RNA is also isolated from seedlings of the same genera.

All contigs are compared to both the ESTs made from RNA isolated from samples containing developing wood and the sequences of the ESTs made from RNA of various tissues that do not contain developing wood. Contigs containing primarily ESTs that show more hybridization in silico to ESTs made from RNA isolated from samples containing developing wood than to ESTs made from RNA isolated from samples not containing developing wood are determined to correspond to possible novel genes particularly expressed in developing wood. These contigs are then used for BLAST searches against public domain sequences. Those contigs that hybridize with high stringency to no known genes or genes annotated as having only a hypothetical protein are selected for the next step. These contigs are considered putative novel genes showing wood preferred expression.

The longest cDNA clones containing sequences hybridizing to the putative novel genes showing wood preferred expression are selected from cDNA libraries using techniques well known to those skilled in the art of molecular biology. The cDNAs are sequenced and full length gene coding sequences together with untranslated flanking sequences are obtained where possible. Stretches of 45 80 nucleotides or oligonucleotides are selected from each of the sequences of putative novel genes showing wood preferred expression such that each oligonucleotide probe hybridizes at high stringency to only one sequence represented in the ESTs made from RNA isolated from trees or seedlings of the same genus.

Oligomers are then chemically synthesized and placed onto a microarray slide as described in Example 34. Each oligomer corresponds to a particular sequence of a putative novel gene showing wood preferred expression and to no other gene whose sequence is represented among the ESTs made from RNA isolated from trees or seedlings of the same genus.

Sample preparation and hybridization are carried out as in Example 35. The technique used in this example is more effective than use of a microarray using cDNA probes because the presence of a signal represents significant evidence of the expression of a particular gene rather than of any of a number of genes that may contain similarities to the cDNA due to conserved functional domains or common evolutionary history. Thus it is possible to differentiate homologous genes such as those in the same family but which may have different functions in phenotype determination.

Thus hybridization data gained using the method of Example 30 enable the user to identify which of the putative novel genes actually has a pattern of coordinate expression with known genes a pattern of expression consistent with a particular developmental role and or a pattern of expression that suggests that the gene has a promoter that drives expression in a valuable way.

The hybridization data thus using this method can be used for example to identify a putative novel gene that shows an expression pattern particular to the tracheids with the lowest cellulose microfibril angle in developing spring wood early wood .

Example 32 is directed to generation of a transgenic high throughput cottonwood plant . Transgenic plants are transformed with the following plasmids 35S I GUS pFOR090 pFOR126 pFOR188 pFOR200 pFOR238 and pFOR292. Control plants were not transformed. Plants are transformed using as described in Horsch et al. Science 227 1229 31 1985 . Seedlings are grown until of suitable size to transfer into soil. Height and diameters are measured on all plants and from these data a mean seedling volume index is calculated. This volume index is usually more closely correlated with seedling biomass than height or volume measures alone.

Plants containing the pFor238 plasmid shows reduced early growth of the transformed cottonwood Table 1 . In 5 of the 6 lines growth is severely reduced compared to the Gus controls or the non transformed controls. In the remaining line growth is no better than the controls. The mean growth rates of all lines in the pFOR090 pFOR188 pFOR126 and pFOR292 are similar to the controls. However some individual lines exhibit increased growth rates as compared to the controls. See Table 17.

For SEQ ID NO 269 5 of the 26 lines exhibit early volume production of at least 40 greater than the GUS controls. For SEQ ID NO 270 1 line of the 19 lines exhibit volume growth rates of at least 40 greater than the GUS controls. For SEQ ID NO 538 3 out of 21 lines exhibit growth rates of at least 40 great than the controls. For SEQ ID NO 127 1 of 3 lines exhibit growth rates greater than the control. In total 10 of the lines exhibit growth at least 40 greater than the GUS controls.

These preliminary results also suggest that the different lines are affecting total plant production in different ways. Some lines show a disproportionate increases in height growth. Other lines demonstrate volume growth increases over the controls due primarily to increases in stem diameter growth. In still other lines stem volume increases are due to increases in both height and diameter growth. The magnitude of the growth increases are from these early measurements is encouraging. For example line 1942 of SEQ ID NO 188 has a seedling volume 76 greater than the GUS controls. Measurement of height and diameter of trees grown in fields is determined. These measurements are used for developing age age correlations for growth in these studies. The results identify optimal early selection strategies for greenhouse production.

