---

title: Peptide microarray and method of use
abstract: A peptide microarray comprising a plurality of predicted unique binding peptides being selected by computational prediction of interaction with a protein of interest or a domain thereof. The selected unique binding peptides are pre-synthesized and then printed and/or immobilized onto a solid support surface via N-terminus with a linker. Methods of using the invention peptide microarray for quantitative determination of protein-peptide interaction, epitope mapping, and drug screening are also provided.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08728980&OS=08728980&RS=08728980
owner: The Regents of the University of California
number: 08728980
owner_city: Oakland
owner_country: US
publication_date: 20121101
---
This application claims priority from U.S. Provisional Application Ser. No. 61 555 778 filed on Nov. 4 2011 which is incorporated herein by reference in its entirety.

This invention was made with government support under grant No. GM085188 awarded by National Institutes of Health NIH . The government has certain rights in the invention.

The instant application contains a Sequence Listing which has been submitted in ASCII format via EFS Web and is hereby incorporated by reference in its entirety. Said ASCII copy created on Oct. 22 2012 is named 24978019.txt and is 23 255 bytes in size.

This invention relates to peptide microarrays and methods of use thereof for identifying protein protein interactions.

A peptide microarray also commonly known as peptide chip or peptide epitope microarray is a collection of peptides displayed on a solid surface usually a glass or plastic chip. Peptide chips are used by scientists in biology medicine and pharmacology to study binding properties and functionality and kinetics of protein protein interactions in general. In basic research peptide microarrays are often used to profile an enzyme like kinase phosphatase protease acetyltransferase histone deacetylase etc. to map an antibody epitope or to find key residues for protein binding. Practical applications include seromarker discovery profiling of changing humoral immune responses of individual patients during disease progression monitoring of therapeutic interventions patient stratification and development of diagnostic tools and vaccines.

The assay principle of peptide microarrays is similar to an ELISA protocol. The peptides thousands in several copies are linked to the surface of a glass chip. This peptide chip can directly be incubated with a variety of different biological samples such as purified enzymes or antibodies patient or animal sera cell lysates etc. After several washing steps a secondary antibody with the needed specificity e.g. anti IgG human mouse or anti phosphotyrosine or anti myc is applied. Typically the secondary antibody is tagged by a fluorescence label that can be detected by a fluorescence scanner. Other detection methods include chemiluminescence or autoradiography.

Therefore a peptide microarray is a planar slide with peptides spotted onto it or assembled directly on the surface. Whereas spotting peptides with current techniques can undergo quality control prior to spotting and result from a single synthetic batch peptides synthesized directly on the surface may suffer from batch to batch variation and limited quality control options. However spotting peptides onto a support surface still has current significant limitations. One limitation is the large size of the peptide array required which restricts its capability to be scaled up for proteomic measurements. A second limitation is that the peptide spotting output is often qualitative rather than quantitative. A third limitation is the high cost of peptide synthesis and limited access to the necessary instruments which has prohibited its implementation as a routine technology. Other limitations exist in the current technology.

Therefore there is a need to develop a peptide microarray technology that improves upon the prior art.

The invention provides a peptide microarray comprising a solid support surface and a plurality of predicted unique binding peptides being pre synthesized and immobilized onto the solid support surface by a linker wherein the predicted unique binding peptides have been selected for the microarray by computational prediction of interaction with a protein of interest or a domain thereof. In certain embodiments the peptide microarray of the present invention contains between 10 10 000 50 5 000 100 1 000 200 900 300 800 or 400 700 including any number in between of predicted unique binding peptides or domains thereof immobilized on the solid support surface. In certain embodiments the protein of interest or a domain thereof or the predicted binding peptides are modified including but not limited to methylation or acetylation. In certain embodiments multiple copies of each predicted unique binding peptide are immobilized on the solid support.

In certain embodiments the linker attached to the predicted binding peptides for peptide immobilization on the solid support surface of the microarray includes but is not limited to aminohexanoic acid Ahx or polyethylene glycol PEG . In certain embodiments the predicted binding peptides are capped with Alanine at each terminus. Any suitable linkers or capping molecules now known or later developed in the art that are used in any type of microarray technologies are encompassed in the present invention.

The present invention provides that the predicted unique binding peptides are selected for microarray by computational prediction. In certain embodiments the computational prediction comprises a combination of at least three analytical approaches selected from the group consisting of structural information energetic pattern of said peptide protein interactions e.g. van der Waals electrostatic and desolvation energies molecular dynamics simulation conservation during evolution mutation subcellular location functional filtering processes proteome scan and mass spectra evidence. In certain embodiments the computational prediction comprises a combination of at least four five six seven eight nine ten or more structural and or functional computational approaches now known or later developed in the art for selecting predicted binding peptides.

The present invention further provides a method of identifying one or more binding peptides interacting with a protein of interest or a domain thereof comprising the steps of a providing computational prediction for binding peptides to the protein of interest or a domain thereof b selecting a plurality of the predicted unique binding peptides based on the computational prediction c generating a peptide microarray comprising the plurality of predicted unique binding peptides being pre synthesized and immobilized on a solid support surface by a linker d exposing the microarray to the protein of interest or domain thereof and e detecting interactions of one or more predicted unique binding peptide with the protein of interest or domain thereof on the microarray thereby identifying one or more binding peptides interacting with a protein of interest or a domain thereof.

In certain embodiments the inventive method is capable of identifying a binding peptide protein interaction when the binding peptide and or the protein of interest or a domain thereof are either unmodified or modified including but not limited to methylation or acetylation. The present invention thus provides a use of the peptide microarray in combination with computational prediction for identifying unmodified and or modified binding peptides proteins of interest or a domain or motif thereof. In certain embodiments the modified binding peptides proteins of interest or a domain or motif thereof include post translational modification PTM proteins that regulate lysine methylation.

In other embodiments the inventive method is also capable of identifying and or detecting and quantitating transient interaction between a binding peptide and the protein of interest or a domain thereof. In certain embodiments the interaction between the binding peptides and the protein of interest or a domain thereof is detected via a fluorescent signal that correlates with a binding affinity and provides a quantitative measurement of interaction between the binding peptides and protein of interest or a domain thereof. Any other detection methods now known or later developed in the art that are used in any type of microarray technologies are also encompassed in the present invention. Use of the peptide microarray of the present invention for epitope mapping and drug screening or vaccine development as non limiting examples is also provided.

The invention provides a peptide microarray comprising a solid support surface onto which a plurality of predicted unique binding peptides is immobilized by a linker. The predicted unique binding peptides have been previously selected for the microarray by computational prediction of interaction with a protein of interest or a domain thereof. After the computational prediction selection the predicted unique binding peptides are synthesized before being immobilized onto the solid support surface of the microarray. Any suitable solid support surface now known or later developed that can be used in any types of microarray technologies are encompassed in the present invention. In certain embodiments the solid support surface of the peptide microarray is a glass surface. As used herein the terms microarray and or peptide microarray or peptide chip are well known in the art and refer to a collection of peptide displayed on a solid surface usually a glass or plastic chip and being used to study binding properties and functionality and kinetics of protein protein interactions in general.

The peptide microarray of the present invention with the predicted unique binding peptides pre synthesized and then printed on a glass surface significantly reduces the size of the array. The peptide microarray of the present invention requires much fewer peptides than a traditional peptide microarray due to the computational prediction analysis. Using the peptide microarray of the present invention very little protein is also required for analysis which is particularly useful in disease diagnosis where the amount of patient sample is usually limited. The inventive peptide microarray and method of use thereof significantly reduces the cost of generating and applying the peptide microarray.

In certain embodiments the peptide microarray of the present invention contains 10 10 000 50 5 000 100 1 000 200 900 300 800 or 400 700 including any number in between of predicted unique binding peptides or domains thereof immobilized on the solid support surface. In certain embodiments the microarray contains less than 1 000 unique predicted binding peptides. In certain embodiments the predicted unique binding peptides or domains thereof are modified including but not limited to methylation or acetylation. As used herein the terms peptide and or protein are used interchangeable usually refer to a chain of short polymers of amino acid monomers linked by peptide bonds which are the covalent chemical bonds formed between two molecules when the carboxyl group of one molecule reacts with the amino group of the other molecule. The term peptide usually refers to a shorter chain typically containing fewer than 50 monomer units whereas protein usually refers to a longer chain typically containing over hundreds of amino acid monomers.

As used herein a peptide or polypeptide includes a compound of two or more subunit amino acids amino acid analogs or peptidomimetics. The subunits maybe linked by peptide bonds. In another embodiment the subunit may be linked by other bonds e.g. ester ether etc. As used herein the term amino acid includes either natural and or unnatural or synthetic amino acids including both the D or L optical isomers and amino acid analogs and peptidomimetics. A protein referred to herein is generally 50 amino acids in length or greater. The peptides of the present invention can vary in length from more than 2 and less than 50 amino acids or between 3 30 4 20 5 15 or 6 7 8 9 10 11 or 12 amino acids.

As used herein the proteins and or binding peptides may comprise modified amino acids including but not limited to methylated or acetylated amino acids and or amino acid analogs. If present modifications to the amino acid structure may be imparted before or after assembly of the polymer. The sequence of amino acids may be interrupted by non amino acid components. A peptide chain may be further modified after polymerization such as by conjugation with a labeling component a linker and or a capping molecule on either or both C and N terminus. Proteins and or binding peptides of the present invention may be naturally occurring synthetic recombinant chimeric or any combination thereof. The present invention encompass proteins and or binding peptides from any species or resources including but not limited to human mammal animal plant bacteria fungus virus etc. Methods of isolating and purification of a protein or peptide as well as methods of synthesizing or making a recombinant and or chimeric protein are well known in the art.

As used herein the terms domain or motif can be used interchangeably and refer to a structural or functional fragment or part of a given protein sequence and structure that may evolve function and exist independently of the rest of the protein chain. The length of the domain or motif or fragment can vary and is usually between about 25 amino acids up to 500 amino acids in length.

Before immobilized onto a solid support surface in the present microarray the predicted unique binding peptides are synthesized in situ and then attached to a linker for stabilizing the immobilization. In certain embodiments the linker attached to the predicted binding peptides includes but is not limited to aminohexanoic acid Ahx or polyethylene glycol PEG . In certain embodiments each terminus of the predicted binding peptides is also capped with Alanine. The present invention encompass any suitable linkers or capping molecules now known or later developed in the art that can be used in any type of microarray technologies.

Various publications including patents published applications technical articles and scholarly journals are cited throughout the specification. Each of these cited publications is incorporated by reference herein in its entirety.

The present invention provides that the predicted unique binding peptides are selected for microarray by computational prediction as discussed for instance by Hou et al. 0046 005 2006 8 639 649 2009 the entire contents of each is incorporated by reference herewith. As used herein the term computational prediction refers to any methods and or analytical approaches utilizing a central processing unit of a computer capable of determining primary secondary and tertiary protein structures and functions and protein protein interactions. In certain embodiments the computational prediction comprises a combination of at least three analytical approaches selected from the group consisting of structural information energetic pattern of said peptide protein interactions e.g. van der Waals electrostatic and desolvation energies molecular dynamics simulation conservation during evolution mutation subcellular location functional filtering processes proteome scan and mass spectra evidence. In certain embodiments the computational prediction comprises a combination of at least four or more structural and or functional approaches now known or later developed in the art for selecting predicted binding peptides. For example mass spectra data can be downloaded from databases of PRIDE Vizcaino et al. A guide to the Proteomics Identifications Database proteomics data repository. 2009 9 4276 4283 and peptideAtla Deutsch et al. PeptideAtlas a resource for target selection for emerging targeted proteomics workflows. 2008 9 429 434 and analysis can be performed using software X TANDEM Craig and Beavis TANDEM matching proteins with tandem mass spectra. 2004 20 1466 1467 and INSPECT Tanner et al. InsPecT identification of posttranslationally modified peptides from tandem mass spectra. 2005 77 4626 4639 secondary structures can be predicted by PSI PRED3.0 Jones Protein secondary structure prediction based on position specific scoring matrices. 1999 292 195 202 and SSpro4.0 Chenget al. SCRATCH a protein structure and structural feature prediction server. 2005 33 W72 76 solvent accessible surface area SASA can be calculated using DSSP Kabsch Sander Dictionary of protein secondary structure pattern recognition of hydrogen bonded and geometrical features. 1983 22 2577 2637 multiple sequence alignment and conservation can be evaluated by BLAST Altschul et al. Basic local alignment search tool. 1990 215 403 410 and CLUSTALW2 Larkin et al. Clustal W and Clustal X version 2.02007 23 2947 2948 .

The present invention further provides a method of identifying one or more binding peptides interacting with a protein of interest or a domain thereof comprising the steps of a providing computational prediction for binding peptides to the protein of interest or a domain thereof b selecting a plurality of the predicted unique binding peptides based on the computational prediction c generating a peptide microarray comprising the plurality of predicted unique binding peptides being pre synthesized and immobilized on a solid support surface by a linker d exposing the microarray to the protein of interest or domain thereof and e detecting interactions of one or more predicted unique binding peptide with the protein of interest or domain thereof on the microarray thereby identifying one or more binding peptides interacting with a protein of interest or a domain thereof.

In one aspect the present invention provides a peptide microarray technology that involves chemically synthesizing peptides and printing peptides on a DNA microarray sized glass slide. The slide is incubated with the protein of interest followed by a primary antibody recognizing the protein and then a fluorescently labeled secondary antibody. The slide is scanned with a standard DNA microarray scanner and the fluorescent intensity of each peptide spot is measured. Binding of the protein to a peptide generates a fluorescent signal and the signal intensity correlates with their binding affinity which provides a quantitative measurement of protein peptide interaction. In other embodiments the inventive method is also capable of identifying and or detecting transient interaction between a binding peptide and the protein of interest or a domain thereof. Any other detection methods now known or later developed in the art that are used in any type of microarray technologies are also encompassed in the present invention.

In one embodiment the present invention provides a recombinant protein such as the SH3 domain of c Abl fused to GST and a set of testing peptides. The known peptide binder of this SH3 domain displays a fluorescent signal whereas random peptides as well as peptide binders of other SH3 domains had no signal. The optimization of the peptide microarray printing technology and assay conditions as well as a determination of whether the intensity of the fluorescent signal and the protein peptide binding affinity shows a linear correlation are further provided in Example 1.

In further embodiments the present invention provides a systematic approach that integrates computer modeling and a peptide microarray technology to identify binding peptides of the SH3 domain of the tyrosine kinase Abl1 in the human proteome. The invention provides a comprehensive list of candidate interacting partners for the Abl1 protein among which the presence of numerous methyltransferases and RNA slicing proteins may suggest a novel function of Abl1 in chromatin remodeling and RNA processing. This embodiment of the invention illustrates a powerful approach of integrating computational and experimental methods of using the peptide microarray technology to detect protein interactions mediated by domain peptide recognition.

The present invention further provides that in combination with the computational prediction the peptide microarray and the inventive method can be used for identifying a binding peptide protein interaction when the binding peptide and or the protein of interest or a domain or motif thereof are either unmodified or modified including but not limited to methylation or acetylation. In certain embodiments the modified binding peptides protein of interest or a domain or motif thereof are methylated or acetylated peptides or proteins including but not limited to post translational modification PTM proteins that regulate lysine methylation.

In one aspect the present invention provides a systematic approach that integrates computational prediction including bioinformatics pipeline evidence such as conservation sub cellular localization and mass spectrometry data and a peptide microarray to recognize methylated peptides by Polycomb Chromodomain protein and to identify methyllysine containing peptides from the fly proteome. More detailed discussions of these embodiments are provided below in Example 2.

Thus the present invention provides a peptide microarray technology that can quantitatively determine the binding affinities of peptides on the microarray to the protein of interest and thus provides a quantitative high throughput and cost effective way to conduct proteomics measurements. By printing different sets of peptides on a microarray the peptide microarray technology of the present invention can be used for defining protein binding specificity illustrating protein protein interactions determining epitopes recognized by antibodies and identifying substrates of various enzymes. The commercial applications for the invention peptide microarray technology also include but are not limited to 1 research use to simultaneously study the interaction of protein s of interest with thousands of peptides in a quantitative manner and 2 epitope mapping for disease diagnosis e.g. autoimmune diseases and allergies and drug screening e.g. vaccine development .

The invention is further illustrated by the following examples which are not to be construed in any way as imposing limitations upon the scope thereof. It is apparent for skill artisans that various modifications and changes are possible and are contemplated within the scope of the current invention.

The human tyrosine kinase Abl1 plays important roles in signal transduction and interacts with a broad variety of cellular proteins 1 2 . Fusion of the break point cluster region BCR and ABL genes form the oncogenic BCR ABL which is causally linked to certain human leukemias 3 . The Abl1 protein consists of several modular domains including one SH3 and one SH2 domain that form intra and inter molecular interactions to regulate its kinase activity 1 2 . Although many interacting partners of Abl1 such as Abi1 Abi2 and Rin1 have been reported 2 a proteome wide identification of those mediated by the weak and transient SH3 peptide interaction is still incomplete and remains a challenging problem.

High throughput technologies such as yeast two hybrid and protein complex purification in conjunction with mass spectrometry have greatly facilitated the identification of protein protein interactions. However because they are not optimally designed to detect the weak and transient domain peptide interactions such interactions are often under represented in proteomic screenings 4 5 . On the other hand in vitro binding assays such as phage display peptide array and protein array are often employed to determine the binding specificity of modular domains. Each of these technologies has advantages and limitations. Phage display and peptide array are complementary techniques the former can test binding of a huge number of random peptides against a given domain while the later can determine binding of a set of specific peptides of interest. Compared with these two techniques protein array often presents a relatively smaller number of proteins on a surface to test binding of synthesized peptides or purified proteins. Computational analysis is critical to these in vivo or in vitro experimental measurements as it 1 removes false positives and recovering false negatives for the in vivo data 2 predicts proteome wide interacting partners based on binding motifs determined by in vitro experiments to guide further experimental investigation.

An integrative approach was presented herewith to identify interacting partners of Abl1 in the human proteome computational predictions were made by combining multiple sources of data including structural information energetic pattern of SH3 peptide interactions and conservation during evolution the predicted interactions were then tested using a peptide microarray technology. Different from the popular SPOT technology that synthesizes peptides on a cellulose membrane 6 pre synthesized and quality controlled peptides were printed and immobilized on a glass surface to hybridize with the purified Abl1 SH3 domain which confirmed 237 predicted interactions. Proteome wide identification of putative interacting partners of Abl1 mediated by the SH3 peptide interactions reveals novel functions of Abl1 in chromatin remodeling and RNA processing. It also greatly enhances an understanding of the regulatory mechanism of Abl1.

1. Database Screening Using a Position Specific Scoring Matrix PSSM Derived from Free Energy Calculations

All ten residue long peptides in the UniProt database 7 were scored using the PSSM we developed in a previous study 8 . The PSSM was a 10 20 matrix which represented the difference in binding free energies between the mutated peptides and the template peptide see details in 8 . Briefly the binding free energy of the template peptide was calculated based on the crystal structure PDB entry 1bbz using MM PBSA. Each residue in the peptide to every amino acid of the other 19 was then mutated and the change in binding free energy was calculated. This change reflected the preference of an amino acid at each peptide residue and was encoded in a 10 20 matrix to use as a PSSM to score peptide sequences. The peptide score was computed as 

The conservation analysis was conducted on seven species human chimpanzee rhesus macaque mouse rat dog and cow . The protein sequences of the seven species were taken from the non redundant protein sequence database at the NCBI BLAST server. For each protein containing the putative binding peptide the best match found by PSI BLAST 33 in each species was considered as a homologue if its E value 

Considering the noise in free energy calculation that might affect the accuracy of the PSSM the MIEC SVM model developed in the previous study 12 was used to classify all conserved peptides into binder and non binder groups.

 1 Modeling the complexes for all conserved peptides. The crystal structure of the peptide APSYSPPPPP SEQ ID NO 1 in complex with the Abl SH3 domain PDB entry 1bbz was used as the initial template to model complexes containing the other peptides. The peptide in the template was systematically mutated to another peptide using the scap program 35 . Because of the large number of peptides under consideration only each modeled structure was minimized using the sander program in AMBER9.0 36 and the AMBER03 force field 37 . The solvent effect was taken into account using the generalized Born GB model igb 2 implemented in sander 38 . The maximum number of minimization steps was set to 4 000 and the convergence criterion for the root mean square rms of the Cartesian elements of the energy gradient was 0.05 kcal mol . The first 500 steps were performed with the steepest descent algorithm and the rest of the steps with the conjugate gradient algorithm.

 2 Calculating the molecular interaction fields MIECs for each peptide 11 12 . The MIECs for each residue residue pair were computed using the MM GB protocol. The MIECs included a electrostatic Columbic interaction G b van der Waals interaction AE and c polar contribution to desolvation free energy AGg. The cutoff for calculating Eand Ewas set to 18.0 . A distance independent interior dielectric constant of 1 was used to calculate E. The charges used in the GB calculation were taken from the AMBER03 force field 37 and other GB parameters were taken from 39 . The values of interior and exterior dielectric constants in the GB calculation were set to 1 and 80 respectively. In addition the MIECs for the 9 residue pairs between the adjacent residues in ten residue long peptides were also calculated to consider the conformational preference of the peptide. The molecular interaction component calculations including read in of the SH3 peptide complexes definition of atom types in the GB calculation and assignment of the force field parameters were automatically carried out using the gleap program in AMBER10 40 .

 3 Prediction of binding peptides using the MIEC SVM model. Finally the MIEC SVM model was used to classify each peptide into binder and non binder categories. The LIBSVM program 41 was used in the predictions.

The GST tagged Abl SH3 domain fusion protein was purified as previously described 12 . Protein concentration was determined using the Bradford Assay Bio Rad . The purity of the fusion protein was checked by electrophoresis on SDS PAGE gel and coomassie blue staining. The fusion protein was also subjected to electrophoresis on SDS PAGE gel followed by Western blot using a horseradish peroxidase conjugated anti GST antibody Santa Cruz Biotechnology and the SuperSignal West chemiluminescent substrate Pierce .

All peptides were synthesized by Sigma Aldrich. The peptides were printed onto glass slides in triplicates by ArrayIt Corporation. A buffer blank and a Cy3 marker were also printed in triplicates. The peptide arrays were blocked with the blocking buffer 5 non fat dry milk TBS pH8.0 0.05 Tween 20 for an hour at room temperature. Next the peptide arrays were incubated overnight at 4 C. with purified GST Abl SH3 in the blocking buffer at a final concentration of 5 M. After washing three times for ten minutes with the TBST buffer TBS pH8.0 0.05 Tween 20 an anti GST antibody Santa Cruz Biotechnology was added to a final concentration of 0.2 g ml in the blocking buffer for one hour at room temperature. The arrays were then washed three times for ten minutes with the TBST buffer. Finally the arrays were incubated with the secondary antibody Cy3 conjugated goat anti mouse IgG H L Jackson ImmunoResearch for one hour at room temperature followed by washing for three times for ten minutes with the TBST buffer. As a control the peptide array was incubated with the anti GST and the secondary antibodies alone.

The peptide microarray was processed by the InnoScan 710 laser scanner ArrayIt Corporation . Cy 3 fluorescence was detected using the 532 nm laser at 3 um resolution. The resulted microarray image was analyzed by the microarray image process software Mapix2.8.2 ArrayIt Corporation . The fluorescent intensity of a microarray spot was defined as its own intensity minus the background intensity around it as determined from the scanned image.

To comprehensively identify interacting partners of the Abl1 protein a systematic searching strategy was exploited . All 69 404 peptides in the human proteome that contain the PxxP motif in the UniProt database 7 were first scored using a scoring matrix. This scoring matrix was generated using the Virtual Mutagenesis method previously developed see details in 8 . Briefly each amino acid of the peptide was mutated to the other 19 amino acids and the mutated complex structure was optimized using 2 ns molecular dynamics simulations. The difference in SH3 peptide binding free energy between the wild type and mutated peptides was calculated using the MM PBSA method 9 10 to represent how much an amino acid was preferred at each peptide position.

Next the top 10 000 peptides ranked by the scoring matrix were analyzed and the conservation information was used to remove false positives. Because of functional constraints the binding peptides of the Abl1 SH3 domain are more conserved across species than non binding peptides. For each protein that contains at least one putative binder multiple sequence alignment was generated across seven proteomes including human chimpanzee rhesus macaque mouse rat dog and cow . Because the human Abl1 SH3 domain is completely conserved across these seven species it is reasonable to assume that its interacting partners are also highly conserved. Therefore those non conserved peptides were removed from the candidate list. After applying the conservation filter about half of the peptides were removed and 4981 peptides were left for further analysis.

The goal was to find the most confident interacting partners of Abl1 SH3 domain and the following procedure was used to further remove false positives. As illustrated in the previous studies 11 12 each of these 4981 peptides in complex with the Abl1 SH3 domain was modeled and its molecular interaction energy components MIECs that reflect the energetic characteristics of the binding was calculated. Using the MIEC SVM previously trained on 18 SH3 domains 12 each peptide was classified into binder or non binder category. After this round of filtering 1394 peptides in 714 proteins were predicted as binders of the Abl1 SH3 domain.

Among the top 10 predicted peptides shown in Table 1 five were from proteins known to interact with the Abl1 SH3 domain in bold including EVL that binds to the SH3 domain of Abl1. Considering how difficult such proteome wide prediction is a 50 5 10 rate of identifying known interacting partners demonstrated a satisfactory performance of our approach. On the other hand the remaining peptides may be new binders that have not been reported. To further refine the predictions peptide microarray experiments were conducted on the predicted binding peptides to find the most reliable binders of the Abl1 SH3 domain.

To print peptide microarrays method of immobilizing the peptides on the glass surface as well as the amount of peptide printed needed to decide. Therefore 10 peptides were first selected to test out the conditions. These peptides included 4 known binders of the Abl1 SH3 domain 13 14 and 6 non binders including 4 peptides that bind to other SH3 domains but not Abl1 13 14 and 2 random peptides without the PxxP motif . Same as in the previous study 12 each peptide was 10 amino acid long and two alanines were added at each end. A linker either aminohexanoic acid Ahx or polyethylene glycol PEG was added at the N terminus. The peptides were immobilized to the glass surface via the N terminus. Two spot sizes 150 um and 600 um in diameter which corresponded to 1 nl and 5 nl of 0.3 mg ml peptide solution dispensed to the array were tested respectively. Each printing amount and linker was tested in triplicates. To better quantitatively measure the signal intensity fluorescence and laser scanner were used to detect the binding of the SH3 domain to the peptides instead of chemiluminescence and film in the previous study 12 . Briefly GST Abl SH3 fusion protein was purified and characterized as described before 12 data not shown . Purified protein was overlaid onto the array followed by incubation with an anti GST antibody and then a Cy3 conjugated secondary antibody. Binding of GST Abl SH3 to peptides generated a Cy3 fluorescent signal that was recorded on a laser scanner. The fluorescence intensity for each peptide was an average of triplicates.

In the production run limited by the synthesis cost the top 700 predicted binding peptides were synthesized and printed. 50 binders including 38 with previously measured Kd s 13 and 50 non binders were also included on the array as positive and negative controls respectively. The non binders were randomly selected from the proteome which are unlikely to interact with the Abl1 SH3 domain. Each peptide was printed in triplicates at two concentrations 1 mg ml and 0.5 mg ml. The production array was probed the same way as the test run . Similarly a control experiment was done with just the antibodies and only basal levels of fluorescence were detected data not shown .

To analyze the microarray data the spots that were flagged as noisy were first removed. For the high 1 mg ml and low 0.5 mg ml printing concentrations 724 and 767 out of 800 spots remained for later analyses respectively. Next the statistical cut off was determined for the significant fluorescence signal by modeling the background intensity distribution using a Gamma distribution . The spots were binned based on the fluorescence intensity. To have a better curve fitting the long tail of the histogram that represented strong binding peptides was removed and only the bins with intensity less than 900 and 1500 for the high and low concentrations respectively were kept. The model fitting and p value calculation were conducted using R and P value of 0.05 was used as the cutoff.

After establishing the parameters the 100 control peptides were first examined and it was found that 40 from the 50 positive controls exhibited a strong signal and 48 from the 50 negative controls displayed only basal signal . Therefore the production array showed a satisfactory performance with a sensitivity of 0.80 40 50 and a specificity of 0.96 48 50 . It is worth noting that the 10 false negative peptides were synthesized twice and their binding to the Abl1 SH3 domain were tested using dot blot only one of these 10 peptides showed signals data not shown . Although these 10 peptides were reported to be binders the experiments could not confirm this conclusion and the true sensitivity of the peptide microarray could be 0.98 40 41 .

The 700 predicted binders were then checked and 237 non redundant peptides from 158 proteins were identified as putative binders including 220 and 175 from high and low concentrations respectively. When examining the PSSM scores of the 237 binding peptides and the remaining non binding peptides it was found that the binders have more favorable smaller PSSM scores than the non binders although there is significant overlap between the two distributions . This figure illustrates that the PSSM provides separation between binders and non binders to some extent.

Among the identified binding peptides 13 known binding partners of the Abl1 SH3 domain as well as 2 paralogs of the known binding partners 2 were found Table 2 . There are 52 proteins listed in 2 that interact with the Abl1 through SH3 peptide interaction and 25 13 52 or 29 15 52 of these manually curated interacting partners of the Abl1 SH3 domain were retrieved. Interestingly six of them are also known substrates of the Abl1 kinase ENAH INPPL1 RAPH1 RIN1 WASF2 and WASL. In addition the identified binding proteins also included another previously determined interacting partner of the SH3 domain that were not included in 2 SHIP2 15 and a known substrate and interacting partner of Abl1 WASF3 16 . The interaction between Abl1 and its substrates is often mediated by SH3 domain as in the case of the six substrates mentioned above . It is also known the interactions between WASF family proteins and Abl1 is mediated by SH3 domain 16 . Therefore WASF3 is likely to be a true interacting partner of the Abl1 SH3 domain. It is worth noting that there are numerous databases that document many partners of the Abl1 SH3 domain but the curated list in 2 was only used as the gold standard for known interacting partners which might miss quite a few true positives like SHIP2 . Given the challenge of proteome wide identification of domain peptide interactions an 8.2 13 158 rate of retrieving the curated interacting partners in this study was quite satisfactory while the other confirmed peptides may well be unknown binders.

The instant peptide microarray approach was also compared with another peptide array based SPOT array method for detecting binding peptides of the Abl1 SH3 domain only two known interacting partners M4K1 DYN2 three paralogs of known interacting partners SOS2 ABR and EFS and one known substrate SYNJ2 listed in 2 were discovered in Wu et al. s study 17 2.5 2 81 recovering rate . The superior performance of the instant peptide microarray method demonstrates the power of integrating computational predictions and peptide microarray in identifying domain peptide interactions in the human proteome.

The identified Abl1 binding peptides came from 158 proteins. To illustrate their functions enrichment analysis of their gene ontology GO 18 annotations were performed using the DAVID software package 19 with the default parameters Table 3 . It is unsurprising that many proteins are related to actin cytoskeleton function in which Abl1 is well known to be involved. Unexpectedly putative binding partners that are involved in chromatin modification RNA processing transcriptional regulation and apoptosis were also found Table 3 . Interactions with these proteins suggested possible new functions of Abl1. Several interesting groups of putative interacting partners of Abl1 were highlighted in Table 4.

Proteins are categorized based on their functions chromatin modification bold binding to methylated DNA highlighted in cyan apoptosis highlighted in green and transcription and splicing highlighted in yellow . Colors are shown in greyscales.

Several methyltransferases are critical for transcription for example SET1A SET1B and WBP7 methylate Lys4 of histone protein H3. Mono di and tri methylation of H3K4 represented as H3K4me1 H3K4me2 and H3K4me3 respectively are known to mark active promoters and enhancers 20 . Besides the study WBP7 was also found to interact with Abl1 SH3 domain in 17 although the function of this interaction is unclear.

The analysis revealed the binding of Abl1 to SETD2 a histone methyltransferase that methylates H3K36 which is a mark for transcription elongation and splicing 20 . The SETD2 peptides amino acid 185 194 and 187 196 are located in a proline rich region. Because this region does not contain any identified domain it is likely the peptide is accessible for SH3 binding. SETD2 is known to interact with hyper phosphorylated POLR2A the RNA polymerase II Pol II large subunit that is also a substrate of Abl1. The observation implies possible recruitment of Abl1 by SETD2 to the proximity of the Pol II complex and thereby regulates transcription.

Another identified interacting partner of the Abl1 SH3 domain DOT1 is a histone methyltransferase that methylates H3K79. H3K79me2 is a histone modification mark for transcribed regions 20 . DOT1 also interacts with MLLT10 a myeloid lymphoid or mixed lineage leukemia protein. Confirmation of the interaction between Abl1 and MLLT10 and illustration of its functional importance are needed.

The analysis also revealed that Abl1 recognized peptides in JMJD3 a demethylase that demethylates repressive histone mark of H3K27me3 20 as well as in EP300 an acetyltransferase that is a co factor often located in enhancers 20 . The other identified interacting partners involved in chromatin structure remodeling include a putative polycomb group protein ASXL3 21 and a chromatin modifier CHD8 22 .

Additionally it was found that Abl1 interacted with two methyl CpG binding domain containing proteins MBD5 and MBD6. The functions of these two proteins are still largely unknown. Given Abl1 s potential functions in modification of chromatin structure it could play roles in DNA methylation since there exists interplay between histone modifications and DNA methylation 23 24 .

In addition to these putative interactions between Abl1 and chromatin modification enzymes the work showed that Abl1 SH3 domain also directly interacted with the transcription machinery including TAF1 the largest component of the TFIID basal transcription factor complex 25 and MED19 a component of the Mediator complex and a co activator involved in the regulated transcription of nearly all Pol II dependent genes 26 . Another putative Abl1 binder is ACINU a component of the splicing dependent multiprotein exon junction complex EJC deposited at splice junctions on mRNAs 27 28 . Many proteins involved in RNA splicing and mRNA processing were also identified Table 3 which suggested novel functions of Abl1 in transcription and slicing regulation.

Two potential interacting proteins of Abl1 are known to be involved in apoptosis PDCD7 promotes apoptosis when overexpressed 29 ASPP2 regulates p53 by enhancing the DNA binding and transactivation function of p53 on the promoters of pro apoptotic genes in vivo 30 . In addition CHD8 a chromatin modifier and a putative Abl binder mentioned above can suppress p53 mediated apoptosis by recruiting histone H1 and preventing p53 transactivation activity 31 .

The tyrosine kinase Abl1 is an important therapeutic target in hematopoietic malignancies. A systematic approach of combining computational predictions and peptide microarray was presented herewith to identify proteins bound to Abl1 via the SH3 peptide recognition in the human proteome. A comparison with the documented Abl1 interacting proteins showed a satisfactory performance of this approach. Given the complexity of protein interactions in human and the importance of determining such interactions for therapeutic development the present approach holds great promise for proteome wide search of binding partners of any disease related protein. More excitingly this kind of proteome wide search reveals unknown functions of these proteins as demonstrated. Putative interactions between Abl1 and numerous chromatin remodeling enzymes and RNA processing proteins were found suggesting novel functions of Abl1 in chromatin modification and RNA regulation.

The instant computational approach provides a research tool that integrates computer modeling and bioinformatics analysis that provide complementary information to reduce false positives. Virtual Mutagenesis and MIEC SVM calculations take into account conformational flexibility and energetic characteristics of the SH3 peptide interactions while conservation analysis uses evolution information to remove noise in computer modeling. Thus the instant approach better captures the structural and energetic features of protein recognition than pure bioinformatics analysis based on sequences only.

Because the transient and weak domain peptide interactions are hard for immunoprecipitation in conjunction with mass spectrometry analysis in vitro binding assay such as peptide micro array is often the first step to identify candidate peptides. Since the cost of peptide synthesis is still quite high it is too expensive to include all possible peptides in the micro array. The more accurate the computational methods are the more domain peptide interactions could be identified in the follow up experiments as reflected by the much higher retrieving rate of known interacting partners of the Abl1 SH3 domain listed in 2 by the study than approaches without a computational component.

In the instant study a peptide microarray platform that prints peptides onto the glass surface was employed. This technology uses much less peptides in the order of pico mole than the popular SPOT array in the order of nano mole . Consequently it also needs less protein antibodies and other reagents than the SPOT array. In addition peptide spots on the microarray are much smaller in size 150 um in diameter than those on the SPOT array around 3 mm in diameter which allows high density printing on a very small surface. Hence screenings can be carried out in multiple replicates on a comprehensive scale and in a systemic way. The use of DNA RNA oligo microarray slides in the assay also allows quantitative measurement by a microarray imager.

Signals on peptide microarrays can be detected using fluorescence chemiluminescence or radioisotopes whereas fluorescent dyes are somewhat problematic in SPOT arrays as the synthetic arrays show some background fluorescence 32 . The quality controlled synthesis of peptides reduces the number of false negatives due to lack of or poor quality peptide on the spot which may occur in the SPOT array. The same batch of pre synthesized peptides can be used in thousands of microarray experiments as well as in various in solution assays should there be a need. This not only saves cost but also takes out one variable in data interpretation. The power of our peptide microarray in detecting the weak SH3 peptide binding was first demonstrated in the validation experiment with 10 peptides in the test run and 100 control peptides in the production run. It was further demonstrated by a satisfactory correlation of the identified binders of the Abl SH3 domain to the documented known binders. Furthermore measuring fluorescence intensity provides a quantitative signal that can be analyzed using statistical test to determine binding peptides.

Lysine methylation is one of the important post translational modifications PTMs that regulate protein functions. Up to date proteomic identification of this PTM remains a challenge due to the lack of effective enrichment methods in mass spectrometry experiments. To address this challenge a systematic approach was presented here to predicting peptides in which lysine residues may be methylated to mediate protein protein interactions. The chromodomain of the Polycomb protein in was used as a model system to illustrate the success of this approach. The molecular dynamics simulations and free energy analyses were started on the histone peptides complexed with the Polycomb chromodomain to understand how the binding specificity is achieved. Next virtual mutagenesis was conducted to quantify each domain and peptide residue s contribution to the domain peptide recognition based on which scoring scheme was developed to evaluate the possibility of any lysine containing peptides to be methylated and recognized by the chromodomain. A peptide microarray experiment on a panel of conserved histone peptides showed a satisfactory prediction accuracy of the scoring scheme. Further a bioinformatics pipeline that integrates multiple lines of evidences including conservation sub cellular localization and mass spectrometry data were implemented to scan the fly proteome for a systematic identification of possible methyllysine containing peptides. These putative chromodomain binding peptides suggest unknown functions of the important regulator protein Polycomb and provide a list of candidate methylation events for follow up investigations.

Proteins can exhibit specificity in their binding to methylated lysines the protein Polycomb a member of the chromodomain group of proteins that regulate chromatin structure and that are involved with genetic repression binds strongly to trimethylated Lys27 of histone protein H3 H3K27me3 but only weakly to trimethylated H3K9. Post translational methylation has has also been found on proteins other than histone peptides. The expression of the transient protein p53 for instance which is involved in tumor suppression is regulated by lysine methylation effected by the writer protein Set9. In addition proteins that interact with histones can themselves be marked by lysine methylation which can affect their interactions with histone peptides. While very little is known about the ability of chromodomain proteins to recognize methylated lysines on non histone peptides the Polycomb group protein PcG Pc2 has been observed to interact with lysine residues modified by attachment of a small ubiquitin like modifier SUMO . This phenomenon suggests that PcGs could have functions of interest that are currently unknown besides genetic regulation by modified histone recognition.

Mass spectrometry experiments can detect many types of post translation modifications PTMs and the proteins that manipulate them but its use in the specific case of methylation is limited due to the narrow peak differential in the spectrum. In theory binding assay experiments can determine the binding affinity of a protein to a wide variety of methylated lysine peptides but the experimental data has been scarce. By combining a computational study with a scan of the fruit fly s proteome both the total class of proteins that can interact with methylated lysine and the methylated lysine peptides that can be recognized by PcG proteins are identified and a more complete understanding of the functions of this group of proteins is acquires.

Molecular dynamics MD simulations were performed on Polycomb protein complexed with a peptide from the histone protein H3. First the binding affinities of wild type Polycomb to unmodified H3 and H3K27me1 H3K27me2 and H3K27me3 were determined by simulations and free energy calculation. The models were derived from crystal structures the structures of the Polycomb H3 complexes are shown in . The contributions of each individual residue in both Polycomb and H3 to the binding interaction were derived by energy decomposition analysis see Materials and Methods leading to the discovery of the most critical Polycomb residues. Next mutation matrices were formed both for Polycomb and for H3 to estimate the effect of point mutation in each residue critical for binding. Subsequently peptide microarray binding data were used to verify the prediction of binding to Polycomb on fly histone peptides which demonstrated the satisfactory prediction ability for the H3 mutation matrix. Finally a scan of the fruit fly proteome was conducted to find additional peptides with methylated lysine and estimate their binding affinity relative to the wild type H3 peptide by using the mutation matrix. A further refinement was conducted through a combined analysis of data from conservation sub cellular location and mass spectrometry evidence. The final list of possible Polycomb binders provided insight into the quantity and character of proteins that can interact with Polycomb.

Having determined that 4.0 is the optimal dielectric constant for deriving accurate binding energies with this model see Materials and Methods subsection D residue decompositionwas performed on the Polycomb H3K27me3 complex for this value. The results of this calculation are given in Table I.

The two most important residues in Polycomb for the binding interaction are Tyr26 4.82 kcal mol and Trp47 5.08 kcal mol which are two of the three bulky hydrophobic residues that form a hydrophobic cage for van der Waals attraction of H3K27me3. The third residue of the cage Trp50 is also high on the critical residue list 2.36 kcal mol . Other key hydrophobic residues in Polycomb for the interaction are Leu24 2.57 kcal mol Val25 3.29 kcal mol Leu64 2.84 kcal mol and Leu68 2.26 kcal mol while the most important charged Polycomb residues are Asp23 1.82 kcal mol Glu29 1.22 kcal mol Glu58 1.18 kcal mol Asp65 2.59 kcal mol and Arg67 1.95 kcal mol . A crucial neutral polar residue is Asn62 3.15 kcal mol . Apart from H3K27me3 13.69 kcal mol the most important residues in H3 for the binding interaction are H3R26 8.41 kcal mol H3K23 6.66 kcal mol H3A25 5.41 kcal mol H3A24 4.27 kcal mol H3S28 2.09 kcal mol and H3T22 1.78 kcal mol . The most important electrostatic interactions in the Polycomb H3 complex from visualization of the trajectory are Asp23 with H3R26 Glu29 with H3K23 Glu58 with H3S28 and Asp65 with H3K23.

The mutation matrix for the Polycomb protein in complex with H3K27me3 is provided. The following observations can be made about trends in the mutation matrix. First mutations of the 8 or 9 residues in Polycomb Tyr26 Trp47 Val25 Asn62 Leu64 Asp65 Leu24 Trp50 and possibly Arg67 that are most important to the binding interaction generally weaken the binding affinity especially in the case of Trp47. Second mutations in the lower half of the matrix residues less critical for binding tend to make the interaction with H3K27me3 more favorable with the exception of Glu58. Third mutations to positively charged amino acids lysine and arginine as well as to glycine proline and to a lesser extent alanine tend to make the binding interaction more unfavorable. Finally mutations to negatively charged residues aspartic and glutamic acid tend to make the binding interaction more favorable.

Three aspects of this problem are particularly worthy of investigation first to find polypeptides other than histones that contain methylated lysine second to understand the mechanisms by which a chromodomain protein recognizes a peptide containing a methylated lysine residue and finally to obtain as much functional information as possible about the polypeptides that bind to chromodomain protein. Therefore in another set of simulations point mutations were performed on residues of the H3 peptide in turn positions 20 to 28 except for H3K27me3 which was kept conserved while the Polycomb protein was held to the wild type. In general mutations of peptide residues tend to have a more favorable effect on the binding interaction than mutations of Polycomb residues especially those which are foremost in importance for binding to H3. Several trends are evident from the mutation matrix data first mutations of H3R26 tend to have a weakening effect on the binding affinity second that mutations of H3A25 tend to make the binding interaction more favorable third mutations of H3 residues to Gly Asp Asn and Pro make the binding energy more unfavorable especially in the near vicinity of H3K27me3 and fourth mutations of H3 residues to tyrosine and tryptophan have a strengthening effect on the binding interaction.

There are a total of 46 histone peptide sequences 9 a.a. long with a single lysine trimethylation site which are conserved between the human and fruit fly H2 H3 and H4 proteins Table II . These were screened for Polycomb protein chromodomain binding by peptide microarray. Each peptide sequence was screened as triplets and the fluorescence intensity was quantified with background subtraction individually for each spot from the microarray images by ImageJand the whole data set was fit to a mixed Gaussian distribution model Table III .

A sequence was identified as a positive binder when at least two out of the three microarray spots exhibited a fluorescence intensity that made a p value cutoff of 0.001 from the background Gaussian distribution in the mixed model. Twelve out of the 46 sequences were identified as positive binders under these criteria. Ten of these microarray identified binders also yielded high mutation matrix score in an independent prediction Table IV .

For a thorough and critical assessment of the prediction performance of the mutation matrix microarray was used to test the binding between Polycomb and the 28 histone peptides that it was predicted to bind to as well as 18 peptides that were not predicted to bind. By doing this both true positive and true negative can be identified and a comprehensive analysis of prediction performance is feasible.

The microarray result identified 12 peptides as array positive and 34 as array negative with a p value cutoff of 0.001. A peptide with a mutation matrix score equal to or more favorable than the wild type H3 peptide was predicted to be Polycomb binding. Among 12 true binders 10 were predicted as binders by the mutation matrix score among 34 true non binders the mutation matrix predicted 16 as non binders resulting in a sensitivity and accuracy of 10 12 0.833 and 10 16 46 0.565 respectively and an area under the receiver operating curve ROC of 0.66. The result is satisfactory considering that no any complicated feature structure was conducted and one single score was used to discriminate binders against non binders . To further justify the prediction ability of the mutation score its average for both binders and non binders were also calculated. For 12 binders the average score is 5.48 with standard deviation of 8.31 for 34 non binders the average score is 0.16 with standard deviation of 10.72. The p value of the t test between the two distributions is 0.037 using the R software package which demonstrates a strong trend of discrimination between binder and non binder by mutation matrix score.

Based on the encouraging agreement between the predictions of the mutation matrix and the experimental data the simulations can be used as a tool in scanning the proteome for Polycomb binding peptides. Since there is no systematic way to identify methylated lysines in a high throughput manner the instant computational predictions provide a candidate list of putative methylations to serve as a starting point for experimental study.

The goal is to find peptides in the fly proteome that contain a methylated lysine residue that can be recognized by the Polycomb chromodomain. These peptides can be called as Polycomb binding peptides PBPs . The template peptide H3K27me3 is 9 amino acids long and has the sequence of LATKAARKS SEQ ID NO 88 the second lysine of which is H3K27. Thus peptides of 9 amino acid length with alysine on the eighth residue that show similar or better binding than that of H3 H3K27me3 were searched for. For this purpose a three stage strategy was implemented to narrow down the searching space in a stepwise manner. In addition to removing peptides with a mutation score worse than the H3K27me3 peptide filters of sequence conservation sub cellular location and mass spectrometry evidence were implemented to further reduce false positives see below for details .

Candidate PBPs in the proteome were searched for. The overall scheme is shown in . 103 524 candidate peptides were first identified each of which is a sequence of 9 amino acids for which the eighth residue is lysine. Mutation scores were calculated for all the peptides and 58 131 peptides with favorable binding energy mutation score no higher than zero were kept.

If a peptide binds to the Polycomb chromodomain it is expected to be conserved through evolution since mutations may disrupt the functional interaction. It is thus reasonable to remove peptides that are not conserved at all the 9 amino acids or at the methylated lysine the eighth position. By considering the conservation score for the entire peptide and the lysine site 9 919 peptides were kept for further analysis.

Since the Polycomb protein is localized in the nucleus and functions as a reader of chromatin modifications identifying its interacting partners that are also compartmented to the nucleus was focused on. For the proteins without annotation of their sub cellular location they were kept as their predicted interactions with Polycomb may shed lights on their functional roles. As a result 5 792 peptides from proteins with a sub cellular location annotated to be either nucleus or unknown in Uniprot s general annotationpassed this filter.

Because recognition by Polycomb protein requires the lysine residue to be methylated the mass spectrometry evidence of lysine methylated peptides can provide supporting information for PBP identification. all the publicly available mass spectra were collected and a scan of the proteome was performed to identify lysine methylation. Even though the mass spectrometry experiments were not designed to detect methyllysine such additional evidence can help to identify the most possible binding peptides of the Polycomb chromodomain. It was found that 539 among the 5 792 peptides showed possible addition of methyl group to lysine suggested by the mass spectrometric signals PBPs

To discover unknown functions of the Polycomb protein GO clustering analysis was conducted using DAVIDon the 357 proteins containing the 539 filtered peptides. It was found that GO terms of regulation of transcription and histone modification are most enriched accounting for five out of six top enriched GO clusters. Moreover several important proteins were included in the list such as mediator subunits 23 and 31 which involves recruitment of RNA polymerase II to promoter region that leads to start of transcription and sex combs extra Sce subunit of PRC1 complex regulating gene repression in the context of methylated H3K9 and H3K27. Also several histone related methyltransferase demethyltransferase and acetyltransferase were found in the list such as Art4 Jhd1 Kdm4B and Tip60.

Polycomb s known function is related to H3K9me3 H3K27me3 mediated gene repression by PRC1. From this analysis it was found that Polycomb protein could also involve in gene regulation through chromatin modification with several other proteins including histone methyltransferase acetyltransferase demethyltransferase and several nucleosome remodeling proteins.

ASH1 CARM1 MOF and KDM4B are related to gene regulation by either adding histone marks or remove marks. ASH1 and CARM1 are two methyltransferases. ASH1 specifically trimethylates H3K4 an activation mark and prevents inappropriate Polycomb silencing of important genes during development. CARM1 methylates H3R17 and activate transcription through chromatin remodeling. MOF is a probable histone acetyltransferase to H4K16 on male X chromosome to balance the gene expression between male and female. KDM4B is a probable demethyltransferase that demethylates methylated H3K9 and H3K36. All these proteins are related to either adding activation marks H3K4me3 ASH1 H3R17me2 CARM1 and H4K16ac MOF removing repression mark H3K9me3 KDM4B or removing activation mark H3K36me3 KDM4B . All these findings indicate that Polycomb may be involved in a much more complicated gene regulation network related to chromatin modification than what we have known so far.

RING1 and CAF1 two other putative binding targets of Polycomb are components of PRC1and PRC2 respectively. Intriguingly Polycomb itself is also a component of PRC1. The binding to RING1 is related to recruit Polycomb to form PRC1 complex and the binding to CAF1 is related to recruit PRC1 to PRC2.

CAF1 is also a component of chromatin assembly factor 1 CAF 1 the nucleosome remodeling and deacetylase complex NuRD and the nucleosome remodeling factor NuRF . IPYR another component of NuRF is also included in the putative binder list. These interactions suggest that Polycomb may have further function in chromatin modification related gene regulation.

It was surprising to find that the potential Polycomb binders include several transcription regulation proteins. Most of these proteins are involved in the development or gene transcription. Putative Polycomb binders that are functional in development include hemeotic gene TSH eye development SRC42 and BARH1 neural development SOS and PNT1 and other development related genes VAV DORS EAF CIC and APTE . Putative Polycomb binders involving transcription include TAF the largest component of the TFIID basal transcription factor complex MED23 and MED31 components of Mediator complex and co activators of nearly all Pol II dependent genes RPA2 the second largest component of RNA polymerase I RPB2 and RPB9 components of RNA polymerase II and RPC6 component of RNA polymerase III. Until now little has been known about the function of the interaction of Polycomb in transcription regulation. Although Polycomb may regulate gene transcription process by binding to these important protein factors exploration of the unrevealed realm for Polycomb and its regulatory functions are further needed.

The Polycomb histone H3 complex was investigated with the use of molecular dynamics MD . By specifying the dielectric coefficient in the calculation of the binding free energy the relative binding affinities of Polycomb to H3K27me3 H3K27me2 H3K27me1 and unmodified H3 were able to be replicated including the counterintuitive phenomenon of the chromodomain binding more strongly to H3K27 when monomethylated than dimethylated. Residue decomposition was performed to identify the residues in Polycomb that are most critical for binding to H3K27me3 as well as the relative contributions of the H3 residues to the binding interaction. These residues formed the basis of a mutation matrix in which the effect of mutating any single one of these residues to other amino acids was analyzed.

With an estimate of the effect of mutating a given residue in the Polycomb H3K27me3 complex the proteome was scanned to find peptides containing methylated lysine that could potentially be recognized by Polycomb chromodomain apart from histones. Coupled with additional criteria including conservation sub cellular localization and mass spectrometry data a set of non histone peptides that likely interact with the Polycomb protein mediated by chromodomain methylysine recognition were identified. Considering the lack of proteomic approaches to determine methylated lysine the instant approach not only provides a promising alternative to suggest candidates for follow up investigation of the biological significance of methylation in the Polycomb partner proteins but also reveal unknown functions of the important transcriptional regulator.

As discussed herewith computational methods were developed to identify un modified peptides recognized by such as SH3 domain Xu et al. Proteome wide detection of Abl1 SH3 binding peptides by integrating computational prediction and peptide microarray. 2012 11 O111 010389 Hou et al. Characterization of Domain Peptide Interaction Interface Prediction of SH3 Domain Mediated Protein Protein Interaction Network in Yeast by Generic Structure Based Models. 2012 11 2982 2995 or regulatory subunit of protein kinase A PKA . This instant study made a significant expansion to search for methyl lysine mediated protein protein interactions in the proteome. More importantly the instant approach can be readily applied to identify any PTMs and suggest their possible interacting partners which opens a new avenue of illustrating the mechanisms of how a specific PTM regulates cellular functions.

The Polycomb H3K27me3 complex structure was obtained from the Protein Data Bank PDB . Its PDB entry code is 1PDQ. The other methylation states were formed by manually removing methyl carbons in turn from the structure data in the PDB. The module of LEAP in the AMBER 9.0software package was used to prepare the models for molecular dynamics simulation. The methylated lysine residues were not recognized by the standard residue library of LEAP and were treated as nonstandard residues. The charges for all nonstandard residues were calculated by Gaussian03 using the HF 6 31G basis set and subsequently fitted with RESP. The partial charges on all nonstandard residue atoms as calculated by Gaussian03 and RESP are listed in Table V. The nonstandard residues were all handled by Gaussian03 individually and cappedby ACE and NME residues so that they could be treated as complete molecules. The FF03 force fieldwas used for all atoms in the model except for the methylated zeta nitrogens of H3K27 for which the parameters of the GAFF force fieldwere used. Each Polycomb histone complex was placed in a rectangular periodic box of TIP3P water molecules. The dimensions of the box extended 10 past the solute system on all sides. When necessary Clcounterions were added to neutralize the system for each model.

Each system was minimized in two stages in the first stage the water molecules and counter ions were relaxed while organic molecules were restrained by a harmonic force of 100 kcal mol . In the second stage the restraints were released and the entire system was minimized. Each stage consisted of 400 steps of steepest descent followed by 1600 steps of conjugate gradient minimization. After the minimization stages the systems were heated from 0 to 300 K over 30 ps. The temperature of 300 K and pressure of 1 atm were maintained by a Berendsen thermo and barostat with a coupling timeof 0.2 ps. The SHAKE algorithm was used to restrain all bonds involving hydrogen.Initial temperatures were assigned randomly according to the Maxwell Boltzmann distribution at the start of the heating run. The systems were then equilibrated for 200 ps followed by production runs lasting 4 ns. During the equilibration and production runs the heavy atoms in the Polycomb protein were restrained by a harmonic force of 10 kcal mol in order to ensure complex stability.

In order to determine the optimal interval within the trajectory for calculating binding free energies the root mean square deviation RMSD were calculated for the complex the H3 peptide and the Polycomb protein over the equilibration and production runs. The coordinates of the carbon atoms of the residues were the data used to calculate the RMSD values. For the protein the RMSD remained below 0.20 over the entire 4.2 ns trajectory as expected because of the harmonic restraints . Therefore virtually all fluctuation in the RMSD for the complex is due to the movement of the H3 peptide which was not restrained. Since the RMSD appears to be most stable in the interval between 1.0 and 2.0 ns in the production run this interval was chosen for binding free energy calculation analysis. In order to show that the stability of the peptide conformation is not drastically affected by a point mutation in the Polycomb protein the RMSD of the complex was also provided for the protein mutation Y26G.

The calculated binding free energies for the four methylation states of H3K27 complexed with Polycomb are given in Table VI.

Three different internal dielectric constants 1.0 2.0 and 4.0 were supplied in the calculations. When the simulation binding free energies are graphed against the logarithms of the experimental dissociation constants ln K the correlation coefficients for 1.0 2.0 and 4.0 are respectively 0.8009 0.9375 and 0.9851. Therefore for this system the binding free energies agree most closely with experiment for a dielectric constant of 4.0 including the reversal phenomenon of H3K27me2 binding more weakly to Polycomb than H3K27me1. Surprisingly the van der Waals component of the interaction is 1.21 kcal mol weaker for H3K27me2 than H3K27me1. This phenomenon may be due to the dimethylated cation being more energetically stable than the monomethylated one and thus having less of a tendency to bind to the hydrophobic pocket of the chromodomain. A complete listing of the interaction components of the binding free energy for all H3K27 methylation states is given in Table VII.

The mutation matrix was formed by identifying the 21 most critical binding residues in Polycomb for binding to H3K27me3 according to Table I. Each of these residues was mutated to all of the other 19 naturally occurring amino acids. This produced 21 19 399 separate models each a point mutation of the wild type complex between Polycomb and histone H3. In a separate operation the sequence of the protein was preserved and each of the H3 residues featured in the model except for H3K27 was mutated to each of the other 19 amino acids. This produced another 8 19 152 separate models. The software program SCAPwas used to perform the point mutations. The standard procedure of minimization heating and equilibration see subsection B was performed on each of the models followed by 2 ns of production run. Binding free energies were calculated from the second half the 1.0 2.0 ns interval of the production run for each model.

The RMSD for each system during the equilibration and production run phases of the MD were calculated by the ptraj module in the AMBER package. As a representative sample the RMSD for the Polycomb H3K27me3 wild type is shown in . Based on the stabilization of the RMSD after about 1000 ps the interval for the binding free energy calculation was chosen for the latter half of each production run between 1.0 and 2.0 ns. MM GBSAwas used to calculate the binding free energies for each Polycomb histone complex using the mm pbsa module in AMBER 9.0 according to the general procedure given in 8 and 53 . The binding free energies G were calculated from 100 snapshots taken at equal intervals between the 1.0 and 2.0 ns marks of the production run trajectory. Gwas calculated according to 

Here and are the respective individual free energies of the complex protein and peptide and each term is calculated by summing the contributions from electrostatic potential E van der Waals potential E and solvation free energy G. In turn Gis the sum of the polar contribution Gand nonpolar contribution Gto the solvation free energy 

Gwas calculated by setting the value of IGB to 2 activating the generalized Born parameters of Onufriev et al.. Gwas estimated to be 0.0072 times the solvent accessible surface area SASA as measured by the LCPO method. A grid size of 0.5 was used to solve the Poisson Boltzmann equation and the probe radiuswas set to 1.4 . Three different values 1.0 2.0 and 4.0 were used for the interior dielectric constant c in separate free energy calculations while the external dielectric constant was given throughout as 80.0.

To investigate the molecular basis of recognition specificity component decomposition of the binding free energy was conducted. The contributions to the binding free energy were also broken down for individual residues and by component van der Waals electrostatic and solvation for each of the complexes. Decompositions were performed by invoking the DECOMP module in the MM GBSA program within AMBER. Each H3 or Polycomb residue was evaluated according to its overall contribution to the binding interaction.

The proteome data were downloaded from Uniprot Swissprot 2009 06 16 The Universal Protein Resource UniProt in 20102010 38 D142 148 Jain et al. Infrastructure for the life sciences design and implementation of the UniProt website. 2009 10 136 . In total there are 3047 protein sequences from which all 9 amino acid long peptides containing a lysine at the eighth position from the N terminus were extracted.

The conservation score considered the conservation of both the entire peptide and the methyllysine site. All the protein sequences in 12 fly proteomes from flybasewere downloaded.

For each protein that contains a candidate peptide its homologous proteins in the other 11 fly proteomes were found using NCBI BLAST. Multiple sequence alignment was then generated for the 12 homologous proteins using CLUSTALW2and the overall conservation score was calculated as 

817 332 mass spectra from the PRIDE databasewere collected. All the proteins in Uniprot Swissprot were used as the reference database when scanning possible methyllysines by any of the three methods X TANDEM OMSSA and INSPECT. For the scanning parameter the parent mass error was set to 1 dalton fragment mass error was set to 0.2 dalton and modifications under consideration included mono di and trimethylation. If a lysine was identified as methylated by any of the three programs the peptide was considered as potentially methylated to keep it in the analysis. In total 10 838 possible methyllysines in the proteome were found.

The chromodomain a.a. 1 90 of Polycomb protein was expressed as GST tagged fusion protein and purified as described in 65 . The purity of the protein was examined by SDS PAGE electrophoresis followed by both Coomassie staining and western blot using an Anti GST HRP conjugate Santa Cruz Biotechnology . The concentration of purified protein was determined by BCA assay Amresco . A total of 46 trimethylated histone peptides were synthesized by Sigma Aldrich desalted mass spectrometry checked . The peptides were then printed as triplets onto glass slides ArrayIt together with a Cy3 marker and an anti GST mouse monoclonal antibody Thermo as references.

The peptide microarray was rinsed with TBST buffer 25 mM Tris 125 mM NaCl 0.05 Tween 20 pH 8 followed by blocking with 5 non fat milk in TBST room temperature 1 hour or 4 C. overnight . The slide was then incubated at 4 C. with Polycomb GST fusion protein at a final concentration of 5 mM in 5 non fat milk TBST for 6 hours. After washing three times for 10 minutes each with TBST the slide was incubated with an anti GST mouse monoclonal IgG antibody Thermo at a final concentration of 1 g mL in 5 non fat milk TBST and shaken gently for 1 hour at room temperature. A secondary anti mouse IgG Dylight 488 conjugated antibody Thermo was added to a final concentration of 0.1 g mL after three more cycles of 10 minutes washing with TBST. The slide was shaken for 1 hr at room temperature and washed three times with TBST.

The dried microarray slides were scanned using a Hamamatsu NanoZoomer 2.0HT Slide Scanning System Neuroscience Light Microscopy Facility UCSD . Data quantification were processed using the microarray processing software ImageJ where the fluorescent intensity of a microarray spot was defined as its own intensity minus the background intensity around it on the scanned image.

